<!doctype html>
<html lang="en-US">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1">
<title>Literature &#8211; Smart Microscopy</title>
<script type="text/javascript">
  WebFontConfig = {"google":{"families":["Raleway:r:latin,latin-ext","Merriweather+Sans:r,i,b,bi:latin,latin-ext"]},"api_url":"https:\/\/fonts-api.wp.com\/css"};
  (function() {
    var wf = document.createElement('script');
    wf.src = '/wp-content/mu-plugins/wpcomsh/vendor/automattic/custom-fonts/js/webfont.js';
    wf.type = 'text/javascript';
    wf.async = 'true';
    var s = document.getElementsByTagName('script')[0];
    s.parentNode.insertBefore(wf, s);
	})();
</script><style id="jetpack-custom-fonts-css"></style>
<meta name='robots' content='max-image-preview:large' />
<link rel="alternate" href="/nl/literatuur/" hreflang="nl" />
<link rel="alternate" href="/literature/" hreflang="en" />
<link rel='dns-prefetch' href='//secure.gravatar.com' />
<link rel='dns-prefetch' href='//stats.wp.com' />
<link rel='dns-prefetch' href='//widgets.wp.com' />
<link rel='dns-prefetch' href='//jetpack.wordpress.com' />
<link rel='dns-prefetch' href='//s0.wp.com' />
<link rel='dns-prefetch' href='//public-api.wordpress.com' />
<link rel='dns-prefetch' href='//0.gravatar.com' />
<link rel='dns-prefetch' href='//1.gravatar.com' />
<link rel='dns-prefetch' href='//2.gravatar.com' />
<link rel='preconnect' href='//i0.wp.com' />
<link rel='preconnect' href='//c0.wp.com' />
<link rel="alternate" type="application/rss+xml" title="Smart Microscopy &raquo; Feed" href="/feed/index.rss" />
<link rel="alternate" type="application/rss+xml" title="Smart Microscopy &raquo; Comments Feed" href="/comments/feed/index.rss" />
<link rel="alternate" title="oEmbed (JSON)" type="application/json+oembed" href="/wp-json/oembed/1.0/embed?url=https%3A%2F%2Fsmartmicroscopy.org%2Fliterature%2F&#038;lang=en" />
<link rel="alternate" title="oEmbed (XML)" type="text/xml+oembed" href="/wp-json/oembed/1.0/embed?url=https%3A%2F%2Fsmartmicroscopy.org%2Fliterature%2F&#038;format=xml&#038;lang=en" />
<link rel='stylesheet' id='all-css-12b0842b64f69a581401b133ac8ece26' href='/_static/??-eJx9j1sOgjAQRTckVEhQ+TCupbYD1D6m6QxpdPWiBiUk+DeT3HMfORYKA0NgwQN4IGFN0EISAZNQROLqUFkqp3OXf+Loxt4EEsSSjSooyLiEuvTW6TW2zCC+O9jyvQFHqazwqEc3Ec5YoP9Ijh0mnCPqfV2LbHQ/VVoTY3Qo9ZeYe6PDtDn0gRwT0Gfe/JQ0YGKFGkpvwgu9+HN1PDSntm2q9gmpkoR9' type='text/css' media='all' />
<style id='kind-inline-css'>
:root {
	--kind-color-main:         #d30a6a;
	--kind-color-main-active:    #ab0957;

	--kind-page-bg-color:        #ffffff;
	--kind-page-bg-color-dark:   #cccccc;

	--kind-page-text-color-light:  #e7ffff;

	--kind-color-base:       #4d606a;
	--kind-font-family-base: "Raleway", Arial, sans-serif;
	--kind-font-weight-base: 500;
	--kind-font-size-base:   18px;
	--kind-font-style-base:  normal;

	--kind-color-headings:       #183343;
	--kind-font-family-headings: "Raleway", Arial, sans-serif;
	--kind-font-weight-headings: 700;
	--kind-font-style-headings:  normal;

	--kind-font-size-menu: 18px;

	--kind-completed-project-color: #4d606a;
	--kind-completed-project-opacity: 0.8;
}
:root {
	--kind-header-height: 124px;
	--kind-header-background: #ffffff;
}
.kind-header-logo {
	--kind-color-logo: #183343;
	--kind-color-logo-desc: #4d606a;
	
}
.kind-header-nav {
	--kind-color-menu: #4d606a;
	--kind-color-menu-hover: #dd1400;
}


:root {
	--kind-footer-background: #f7f8f8;
	--kind-footer-heading-color: #183343;
	--kind-footer-color: #4d606a;
	--kind-footer-link-color: #d30a6a;
	--kind-footer-link-color-hover: #ab0957;
}
.kind-footer {
	--kind-footersocial-color: #183343;
	--kind-footersocial-color-hover: #4d606a;
}
.kind-footer-logo {
	--kind-color-logo: #183343;
	--kind-color-logo-desc: #4d606a;
	
}

/*# sourceURL=kind-inline-css */
</style>
<style id='wp-img-auto-sizes-contain-inline-css'>
img:is([sizes=auto i],[sizes^="auto," i]){contain-intrinsic-size:3000px 1500px}
/*# sourceURL=wp-img-auto-sizes-contain-inline-css */
</style>
<style id='wp-emoji-styles-inline-css'>

	img.wp-smiley, img.emoji {
		display: inline !important;
		border: none !important;
		box-shadow: none !important;
		height: 1em !important;
		width: 1em !important;
		margin: 0 0.07em !important;
		vertical-align: -0.1em !important;
		background: none !important;
		padding: 0 !important;
	}
/*# sourceURL=wp-emoji-styles-inline-css */
</style>
<style id='wp-block-library-inline-css'>
:root{--wp-block-synced-color:#7a00df;--wp-block-synced-color--rgb:122,0,223;--wp-bound-block-color:var(--wp-block-synced-color);--wp-editor-canvas-background:#ddd;--wp-admin-theme-color:#007cba;--wp-admin-theme-color--rgb:0,124,186;--wp-admin-theme-color-darker-10:#006ba1;--wp-admin-theme-color-darker-10--rgb:0,107,160.5;--wp-admin-theme-color-darker-20:#005a87;--wp-admin-theme-color-darker-20--rgb:0,90,135;--wp-admin-border-width-focus:2px}@media (min-resolution:192dpi){:root{--wp-admin-border-width-focus:1.5px}}.wp-element-button{cursor:pointer}:root .has-very-light-gray-background-color{background-color:#eee}:root .has-very-dark-gray-background-color{background-color:#313131}:root .has-very-light-gray-color{color:#eee}:root .has-very-dark-gray-color{color:#313131}:root .has-vivid-green-cyan-to-vivid-cyan-blue-gradient-background{background:linear-gradient(135deg,#00d084,#0693e3)}:root .has-purple-crush-gradient-background{background:linear-gradient(135deg,#34e2e4,#4721fb 50%,#ab1dfe)}:root .has-hazy-dawn-gradient-background{background:linear-gradient(135deg,#faaca8,#dad0ec)}:root .has-subdued-olive-gradient-background{background:linear-gradient(135deg,#fafae1,#67a671)}:root .has-atomic-cream-gradient-background{background:linear-gradient(135deg,#fdd79a,#004a59)}:root .has-nightshade-gradient-background{background:linear-gradient(135deg,#330968,#31cdcf)}:root .has-midnight-gradient-background{background:linear-gradient(135deg,#020381,#2874fc)}:root{--wp--preset--font-size--normal:16px;--wp--preset--font-size--huge:42px}.has-regular-font-size{font-size:1em}.has-larger-font-size{font-size:2.625em}.has-normal-font-size{font-size:var(--wp--preset--font-size--normal)}.has-huge-font-size{font-size:var(--wp--preset--font-size--huge)}:root .has-text-align-center{text-align:center}:root .has-text-align-left{text-align:left}:root .has-text-align-right{text-align:right}.has-fit-text{white-space:nowrap!important}#end-resizable-editor-section{display:none}.aligncenter{clear:both}.items-justified-left{justify-content:flex-start}.items-justified-center{justify-content:center}.items-justified-right{justify-content:flex-end}.items-justified-space-between{justify-content:space-between}.screen-reader-text{word-wrap:normal!important;border:0;clip-path:inset(50%);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}.screen-reader-text:focus{background-color:#ddd;clip-path:none;color:#444;display:block;font-size:1em;height:auto;left:5px;line-height:normal;padding:15px 23px 14px;text-decoration:none;top:5px;width:auto;z-index:100000}html :where(.has-border-color){border-style:solid}html :where([style*=border-top-color]){border-top-style:solid}html :where([style*=border-right-color]){border-right-style:solid}html :where([style*=border-bottom-color]){border-bottom-style:solid}html :where([style*=border-left-color]){border-left-style:solid}html :where([style*=border-width]){border-style:solid}html :where([style*=border-top-width]){border-top-style:solid}html :where([style*=border-right-width]){border-right-style:solid}html :where([style*=border-bottom-width]){border-bottom-style:solid}html :where([style*=border-left-width]){border-left-style:solid}html :where(img[class*=wp-image-]){height:auto;max-width:100%}:where(figure){margin:0 0 1em}html :where(.is-position-sticky){--wp-admin--admin-bar--position-offset:var(--wp-admin--admin-bar--height,0px)}@media screen and (max-width:600px){html :where(.is-position-sticky){--wp-admin--admin-bar--position-offset:0px}}
.has-text-align-justify{text-align:justify;}

/*# sourceURL=wp-block-library-inline-css */
</style><style id='wp-block-buttons-inline-css'>
.wp-block-buttons{box-sizing:border-box}.wp-block-buttons.is-vertical{flex-direction:column}.wp-block-buttons.is-vertical>.wp-block-button:last-child{margin-bottom:0}.wp-block-buttons>.wp-block-button{display:inline-block;margin:0}.wp-block-buttons.is-content-justification-left{justify-content:flex-start}.wp-block-buttons.is-content-justification-left.is-vertical{align-items:flex-start}.wp-block-buttons.is-content-justification-center{justify-content:center}.wp-block-buttons.is-content-justification-center.is-vertical{align-items:center}.wp-block-buttons.is-content-justification-right{justify-content:flex-end}.wp-block-buttons.is-content-justification-right.is-vertical{align-items:flex-end}.wp-block-buttons.is-content-justification-space-between{justify-content:space-between}.wp-block-buttons.aligncenter{text-align:center}.wp-block-buttons:not(.is-content-justification-space-between,.is-content-justification-right,.is-content-justification-left,.is-content-justification-center) .wp-block-button.aligncenter{margin-left:auto;margin-right:auto;width:100%}.wp-block-buttons[style*=text-decoration] .wp-block-button,.wp-block-buttons[style*=text-decoration] .wp-block-button__link{text-decoration:inherit}.wp-block-buttons.has-custom-font-size .wp-block-button__link{font-size:inherit}.wp-block-buttons .wp-block-button__link{width:100%}.wp-block-button.aligncenter{text-align:center}
/*# sourceURL=/wp-content/plugins/gutenberg/build/styles/block-library/buttons/style.css */
</style>
<style id='wp-block-columns-inline-css'>
.wp-block-columns{box-sizing:border-box;display:flex;flex-wrap:wrap!important}@media (min-width:782px){.wp-block-columns{flex-wrap:nowrap!important}}.wp-block-columns{align-items:normal!important}.wp-block-columns.are-vertically-aligned-top{align-items:flex-start}.wp-block-columns.are-vertically-aligned-center{align-items:center}.wp-block-columns.are-vertically-aligned-bottom{align-items:flex-end}@media (max-width:781px){.wp-block-columns:not(.is-not-stacked-on-mobile)>.wp-block-column{flex-basis:100%!important}}@media (min-width:782px){.wp-block-columns:not(.is-not-stacked-on-mobile)>.wp-block-column{flex-basis:0;flex-grow:1}.wp-block-columns:not(.is-not-stacked-on-mobile)>.wp-block-column[style*=flex-basis]{flex-grow:0}}.wp-block-columns.is-not-stacked-on-mobile{flex-wrap:nowrap!important}.wp-block-columns.is-not-stacked-on-mobile>.wp-block-column{flex-basis:0;flex-grow:1}.wp-block-columns.is-not-stacked-on-mobile>.wp-block-column[style*=flex-basis]{flex-grow:0}:where(.wp-block-columns){margin-bottom:1.75em}:where(.wp-block-columns.has-background){padding:1.25em 2.375em}.wp-block-column{flex-grow:1;min-width:0;overflow-wrap:break-word;word-break:break-word}.wp-block-column.is-vertically-aligned-top{align-self:flex-start}.wp-block-column.is-vertically-aligned-center{align-self:center}.wp-block-column.is-vertically-aligned-bottom{align-self:flex-end}.wp-block-column.is-vertically-aligned-stretch{align-self:stretch}.wp-block-column.is-vertically-aligned-bottom,.wp-block-column.is-vertically-aligned-center,.wp-block-column.is-vertically-aligned-top{width:100%}
/*# sourceURL=/wp-content/plugins/gutenberg/build/styles/block-library/columns/style.css */
</style>
<style id='wp-block-paragraph-inline-css'>
.is-small-text{font-size:.875em}.is-regular-text{font-size:1em}.is-large-text{font-size:2.25em}.is-larger-text{font-size:3em}.has-drop-cap:not(:focus):first-letter{float:left;font-size:8.4em;font-style:normal;font-weight:100;line-height:.68;margin:.05em .1em 0 0;text-transform:uppercase}body.rtl .has-drop-cap:not(:focus):first-letter{float:none;margin-left:.1em}p.has-drop-cap.has-background{overflow:hidden}:root :where(p.has-background){padding:1.25em 2.375em}:where(p.has-text-color:not(.has-link-color)) a{color:inherit}p.has-text-align-left[style*="writing-mode:vertical-lr"],p.has-text-align-right[style*="writing-mode:vertical-rl"]{rotate:180deg}
/*# sourceURL=/wp-content/plugins/gutenberg/build/styles/block-library/paragraph/style.css */
</style>
<style id='wp-block-spacer-inline-css'>
.wp-block-spacer{clear:both}
/*# sourceURL=/wp-content/plugins/gutenberg/build/styles/block-library/spacer/style.css */
</style>
<style id='wp-block-button-inline-css'>
.wp-block-button__link{align-content:center;box-sizing:border-box;cursor:pointer;display:inline-block;height:100%;text-align:center;word-break:break-word}.wp-block-button__link.aligncenter{text-align:center}.wp-block-button__link.alignright{text-align:right}:where(.wp-block-button__link){border-radius:9999px;box-shadow:none;padding:calc(.667em + 2px) calc(1.333em + 2px);text-decoration:none}.wp-block-button[style*=text-decoration] .wp-block-button__link{text-decoration:inherit}.wp-block-buttons>.wp-block-button.has-custom-width{max-width:none}.wp-block-buttons>.wp-block-button.has-custom-width .wp-block-button__link{width:100%}.wp-block-buttons>.wp-block-button.has-custom-font-size .wp-block-button__link{font-size:inherit}.wp-block-buttons>.wp-block-button.wp-block-button__width-25{width:calc(25% - var(--wp--style--block-gap, .5em)*.75)}.wp-block-buttons>.wp-block-button.wp-block-button__width-50{width:calc(50% - var(--wp--style--block-gap, .5em)*.5)}.wp-block-buttons>.wp-block-button.wp-block-button__width-75{width:calc(75% - var(--wp--style--block-gap, .5em)*.25)}.wp-block-buttons>.wp-block-button.wp-block-button__width-100{flex-basis:100%;width:100%}.wp-block-buttons.is-vertical>.wp-block-button.wp-block-button__width-25{width:25%}.wp-block-buttons.is-vertical>.wp-block-button.wp-block-button__width-50{width:50%}.wp-block-buttons.is-vertical>.wp-block-button.wp-block-button__width-75{width:75%}.wp-block-button.is-style-squared,.wp-block-button__link.wp-block-button.is-style-squared{border-radius:0}.wp-block-button.no-border-radius,.wp-block-button__link.no-border-radius{border-radius:0!important}:root :where(.wp-block-button .wp-block-button__link.is-style-outline),:root :where(.wp-block-button.is-style-outline>.wp-block-button__link){border:2px solid;padding:.667em 1.333em}:root :where(.wp-block-button .wp-block-button__link.is-style-outline:not(.has-text-color)),:root :where(.wp-block-button.is-style-outline>.wp-block-button__link:not(.has-text-color)){color:currentColor}:root :where(.wp-block-button .wp-block-button__link.is-style-outline:not(.has-background)),:root :where(.wp-block-button.is-style-outline>.wp-block-button__link:not(.has-background)){background-color:initial;background-image:none}
/*# sourceURL=/wp-content/plugins/gutenberg/build/styles/block-library/button/style.css */
</style>
<style id='wp-block-heading-inline-css'>
h1:where(.wp-block-heading).has-background,h2:where(.wp-block-heading).has-background,h3:where(.wp-block-heading).has-background,h4:where(.wp-block-heading).has-background,h5:where(.wp-block-heading).has-background,h6:where(.wp-block-heading).has-background{padding:1.25em 2.375em}h1.has-text-align-left[style*=writing-mode]:where([style*=vertical-lr]),h1.has-text-align-right[style*=writing-mode]:where([style*=vertical-rl]),h2.has-text-align-left[style*=writing-mode]:where([style*=vertical-lr]),h2.has-text-align-right[style*=writing-mode]:where([style*=vertical-rl]),h3.has-text-align-left[style*=writing-mode]:where([style*=vertical-lr]),h3.has-text-align-right[style*=writing-mode]:where([style*=vertical-rl]),h4.has-text-align-left[style*=writing-mode]:where([style*=vertical-lr]),h4.has-text-align-right[style*=writing-mode]:where([style*=vertical-rl]),h5.has-text-align-left[style*=writing-mode]:where([style*=vertical-lr]),h5.has-text-align-right[style*=writing-mode]:where([style*=vertical-rl]),h6.has-text-align-left[style*=writing-mode]:where([style*=vertical-lr]),h6.has-text-align-right[style*=writing-mode]:where([style*=vertical-rl]){rotate:180deg}
/*# sourceURL=/wp-content/plugins/gutenberg/build/styles/block-library/heading/style.css */
</style>
<style id='global-styles-inline-css'>
:root{--wp--preset--aspect-ratio--square: 1;--wp--preset--aspect-ratio--4-3: 4/3;--wp--preset--aspect-ratio--3-4: 3/4;--wp--preset--aspect-ratio--3-2: 3/2;--wp--preset--aspect-ratio--2-3: 2/3;--wp--preset--aspect-ratio--16-9: 16/9;--wp--preset--aspect-ratio--9-16: 9/16;--wp--preset--color--black: #000000;--wp--preset--color--cyan-bluish-gray: #abb8c3;--wp--preset--color--white: #ffffff;--wp--preset--color--pale-pink: #f78da7;--wp--preset--color--vivid-red: #cf2e2e;--wp--preset--color--luminous-vivid-orange: #ff6900;--wp--preset--color--luminous-vivid-amber: #fcb900;--wp--preset--color--light-green-cyan: #7bdcb5;--wp--preset--color--vivid-green-cyan: #00d084;--wp--preset--color--pale-cyan-blue: #8ed1fc;--wp--preset--color--vivid-cyan-blue: #0693e3;--wp--preset--color--vivid-purple: #9b51e0;--wp--preset--color--light-grey: #f7f8f8;--wp--preset--color--light-blue: #f5fafe;--wp--preset--color--main: #d30a6a;--wp--preset--color--base: ;--wp--preset--gradient--vivid-cyan-blue-to-vivid-purple: linear-gradient(135deg,rgb(6,147,227) 0%,rgb(155,81,224) 100%);--wp--preset--gradient--light-green-cyan-to-vivid-green-cyan: linear-gradient(135deg,rgb(122,220,180) 0%,rgb(0,208,130) 100%);--wp--preset--gradient--luminous-vivid-amber-to-luminous-vivid-orange: linear-gradient(135deg,rgb(252,185,0) 0%,rgb(255,105,0) 100%);--wp--preset--gradient--luminous-vivid-orange-to-vivid-red: linear-gradient(135deg,rgb(255,105,0) 0%,rgb(207,46,46) 100%);--wp--preset--gradient--very-light-gray-to-cyan-bluish-gray: linear-gradient(135deg,rgb(238,238,238) 0%,rgb(169,184,195) 100%);--wp--preset--gradient--cool-to-warm-spectrum: linear-gradient(135deg,rgb(74,234,220) 0%,rgb(151,120,209) 20%,rgb(207,42,186) 40%,rgb(238,44,130) 60%,rgb(251,105,98) 80%,rgb(254,248,76) 100%);--wp--preset--gradient--blush-light-purple: linear-gradient(135deg,rgb(255,206,236) 0%,rgb(152,150,240) 100%);--wp--preset--gradient--blush-bordeaux: linear-gradient(135deg,rgb(254,205,165) 0%,rgb(254,45,45) 50%,rgb(107,0,62) 100%);--wp--preset--gradient--luminous-dusk: linear-gradient(135deg,rgb(255,203,112) 0%,rgb(199,81,192) 50%,rgb(65,88,208) 100%);--wp--preset--gradient--pale-ocean: linear-gradient(135deg,rgb(255,245,203) 0%,rgb(182,227,212) 50%,rgb(51,167,181) 100%);--wp--preset--gradient--electric-grass: linear-gradient(135deg,rgb(202,248,128) 0%,rgb(113,206,126) 100%);--wp--preset--gradient--midnight: linear-gradient(135deg,rgb(2,3,129) 0%,rgb(40,116,252) 100%);--wp--preset--font-size--small: 13px;--wp--preset--font-size--medium: 20px;--wp--preset--font-size--large: 36px;--wp--preset--font-size--x-large: 42px;--wp--preset--font-family--albert-sans: 'Albert Sans', sans-serif;--wp--preset--font-family--alegreya: Alegreya, serif;--wp--preset--font-family--arvo: Arvo, serif;--wp--preset--font-family--bodoni-moda: 'Bodoni Moda', serif;--wp--preset--font-family--bricolage-grotesque: 'Bricolage Grotesque', sans-serif;--wp--preset--font-family--cabin: Cabin, sans-serif;--wp--preset--font-family--chivo: Chivo, sans-serif;--wp--preset--font-family--commissioner: Commissioner, sans-serif;--wp--preset--font-family--cormorant: Cormorant, serif;--wp--preset--font-family--courier-prime: 'Courier Prime', monospace;--wp--preset--font-family--crimson-pro: 'Crimson Pro', serif;--wp--preset--font-family--dm-mono: 'DM Mono', monospace;--wp--preset--font-family--dm-sans: 'DM Sans', sans-serif;--wp--preset--font-family--dm-serif-display: 'DM Serif Display', serif;--wp--preset--font-family--domine: Domine, serif;--wp--preset--font-family--eb-garamond: 'EB Garamond', serif;--wp--preset--font-family--epilogue: Epilogue, sans-serif;--wp--preset--font-family--fahkwang: Fahkwang, sans-serif;--wp--preset--font-family--figtree: Figtree, sans-serif;--wp--preset--font-family--fira-sans: 'Fira Sans', sans-serif;--wp--preset--font-family--fjalla-one: 'Fjalla One', sans-serif;--wp--preset--font-family--fraunces: Fraunces, serif;--wp--preset--font-family--gabarito: Gabarito, system-ui;--wp--preset--font-family--ibm-plex-mono: 'IBM Plex Mono', monospace;--wp--preset--font-family--ibm-plex-sans: 'IBM Plex Sans', sans-serif;--wp--preset--font-family--ibarra-real-nova: 'Ibarra Real Nova', serif;--wp--preset--font-family--instrument-serif: 'Instrument Serif', serif;--wp--preset--font-family--inter: Inter, sans-serif;--wp--preset--font-family--josefin-sans: 'Josefin Sans', sans-serif;--wp--preset--font-family--jost: Jost, sans-serif;--wp--preset--font-family--libre-baskerville: 'Libre Baskerville', serif;--wp--preset--font-family--libre-franklin: 'Libre Franklin', sans-serif;--wp--preset--font-family--literata: Literata, serif;--wp--preset--font-family--lora: Lora, serif;--wp--preset--font-family--merriweather: Merriweather, serif;--wp--preset--font-family--montserrat: Montserrat, sans-serif;--wp--preset--font-family--newsreader: Newsreader, serif;--wp--preset--font-family--noto-sans-mono: 'Noto Sans Mono', sans-serif;--wp--preset--font-family--nunito: Nunito, sans-serif;--wp--preset--font-family--open-sans: 'Open Sans', sans-serif;--wp--preset--font-family--overpass: Overpass, sans-serif;--wp--preset--font-family--pt-serif: 'PT Serif', serif;--wp--preset--font-family--petrona: Petrona, serif;--wp--preset--font-family--piazzolla: Piazzolla, serif;--wp--preset--font-family--playfair-display: 'Playfair Display', serif;--wp--preset--font-family--plus-jakarta-sans: 'Plus Jakarta Sans', sans-serif;--wp--preset--font-family--poppins: Poppins, sans-serif;--wp--preset--font-family--raleway: Raleway, sans-serif;--wp--preset--font-family--roboto: Roboto, sans-serif;--wp--preset--font-family--roboto-slab: 'Roboto Slab', serif;--wp--preset--font-family--rubik: Rubik, sans-serif;--wp--preset--font-family--rufina: Rufina, serif;--wp--preset--font-family--sora: Sora, sans-serif;--wp--preset--font-family--source-sans-3: 'Source Sans 3', sans-serif;--wp--preset--font-family--source-serif-4: 'Source Serif 4', serif;--wp--preset--font-family--space-mono: 'Space Mono', monospace;--wp--preset--font-family--syne: Syne, sans-serif;--wp--preset--font-family--texturina: Texturina, serif;--wp--preset--font-family--urbanist: Urbanist, sans-serif;--wp--preset--font-family--work-sans: 'Work Sans', sans-serif;--wp--preset--spacing--20: 0.44rem;--wp--preset--spacing--30: 0.67rem;--wp--preset--spacing--40: 1rem;--wp--preset--spacing--50: 1.5rem;--wp--preset--spacing--60: 2.25rem;--wp--preset--spacing--70: 3.38rem;--wp--preset--spacing--80: 5.06rem;--wp--preset--shadow--natural: 6px 6px 9px rgba(0, 0, 0, 0.2);--wp--preset--shadow--deep: 12px 12px 50px rgba(0, 0, 0, 0.4);--wp--preset--shadow--sharp: 6px 6px 0px rgba(0, 0, 0, 0.2);--wp--preset--shadow--outlined: 6px 6px 0px -3px rgb(255, 255, 255), 6px 6px rgb(0, 0, 0);--wp--preset--shadow--crisp: 6px 6px 0px rgb(0, 0, 0);}:where(.is-layout-flex){gap: 0.5em;}:where(.is-layout-grid){gap: 0.5em;}body .is-layout-flex{display: flex;}.is-layout-flex{flex-wrap: wrap;align-items: center;}.is-layout-flex > :is(*, div){margin: 0;}body .is-layout-grid{display: grid;}.is-layout-grid > :is(*, div){margin: 0;}:where(.wp-block-columns.is-layout-flex){gap: 2em;}:where(.wp-block-columns.is-layout-grid){gap: 2em;}:where(.wp-block-post-template.is-layout-flex){gap: 1.25em;}:where(.wp-block-post-template.is-layout-grid){gap: 1.25em;}.has-black-color{color: var(--wp--preset--color--black) !important;}.has-cyan-bluish-gray-color{color: var(--wp--preset--color--cyan-bluish-gray) !important;}.has-white-color{color: var(--wp--preset--color--white) !important;}.has-pale-pink-color{color: var(--wp--preset--color--pale-pink) !important;}.has-vivid-red-color{color: var(--wp--preset--color--vivid-red) !important;}.has-luminous-vivid-orange-color{color: var(--wp--preset--color--luminous-vivid-orange) !important;}.has-luminous-vivid-amber-color{color: var(--wp--preset--color--luminous-vivid-amber) !important;}.has-light-green-cyan-color{color: var(--wp--preset--color--light-green-cyan) !important;}.has-vivid-green-cyan-color{color: var(--wp--preset--color--vivid-green-cyan) !important;}.has-pale-cyan-blue-color{color: var(--wp--preset--color--pale-cyan-blue) !important;}.has-vivid-cyan-blue-color{color: var(--wp--preset--color--vivid-cyan-blue) !important;}.has-vivid-purple-color{color: var(--wp--preset--color--vivid-purple) !important;}.has-black-background-color{background-color: var(--wp--preset--color--black) !important;}.has-cyan-bluish-gray-background-color{background-color: var(--wp--preset--color--cyan-bluish-gray) !important;}.has-white-background-color{background-color: var(--wp--preset--color--white) !important;}.has-pale-pink-background-color{background-color: var(--wp--preset--color--pale-pink) !important;}.has-vivid-red-background-color{background-color: var(--wp--preset--color--vivid-red) !important;}.has-luminous-vivid-orange-background-color{background-color: var(--wp--preset--color--luminous-vivid-orange) !important;}.has-luminous-vivid-amber-background-color{background-color: var(--wp--preset--color--luminous-vivid-amber) !important;}.has-light-green-cyan-background-color{background-color: var(--wp--preset--color--light-green-cyan) !important;}.has-vivid-green-cyan-background-color{background-color: var(--wp--preset--color--vivid-green-cyan) !important;}.has-pale-cyan-blue-background-color{background-color: var(--wp--preset--color--pale-cyan-blue) !important;}.has-vivid-cyan-blue-background-color{background-color: var(--wp--preset--color--vivid-cyan-blue) !important;}.has-vivid-purple-background-color{background-color: var(--wp--preset--color--vivid-purple) !important;}.has-black-border-color{border-color: var(--wp--preset--color--black) !important;}.has-cyan-bluish-gray-border-color{border-color: var(--wp--preset--color--cyan-bluish-gray) !important;}.has-white-border-color{border-color: var(--wp--preset--color--white) !important;}.has-pale-pink-border-color{border-color: var(--wp--preset--color--pale-pink) !important;}.has-vivid-red-border-color{border-color: var(--wp--preset--color--vivid-red) !important;}.has-luminous-vivid-orange-border-color{border-color: var(--wp--preset--color--luminous-vivid-orange) !important;}.has-luminous-vivid-amber-border-color{border-color: var(--wp--preset--color--luminous-vivid-amber) !important;}.has-light-green-cyan-border-color{border-color: var(--wp--preset--color--light-green-cyan) !important;}.has-vivid-green-cyan-border-color{border-color: var(--wp--preset--color--vivid-green-cyan) !important;}.has-pale-cyan-blue-border-color{border-color: var(--wp--preset--color--pale-cyan-blue) !important;}.has-vivid-cyan-blue-border-color{border-color: var(--wp--preset--color--vivid-cyan-blue) !important;}.has-vivid-purple-border-color{border-color: var(--wp--preset--color--vivid-purple) !important;}.has-vivid-cyan-blue-to-vivid-purple-gradient-background{background: var(--wp--preset--gradient--vivid-cyan-blue-to-vivid-purple) !important;}.has-light-green-cyan-to-vivid-green-cyan-gradient-background{background: var(--wp--preset--gradient--light-green-cyan-to-vivid-green-cyan) !important;}.has-luminous-vivid-amber-to-luminous-vivid-orange-gradient-background{background: var(--wp--preset--gradient--luminous-vivid-amber-to-luminous-vivid-orange) !important;}.has-luminous-vivid-orange-to-vivid-red-gradient-background{background: var(--wp--preset--gradient--luminous-vivid-orange-to-vivid-red) !important;}.has-very-light-gray-to-cyan-bluish-gray-gradient-background{background: var(--wp--preset--gradient--very-light-gray-to-cyan-bluish-gray) !important;}.has-cool-to-warm-spectrum-gradient-background{background: var(--wp--preset--gradient--cool-to-warm-spectrum) !important;}.has-blush-light-purple-gradient-background{background: var(--wp--preset--gradient--blush-light-purple) !important;}.has-blush-bordeaux-gradient-background{background: var(--wp--preset--gradient--blush-bordeaux) !important;}.has-luminous-dusk-gradient-background{background: var(--wp--preset--gradient--luminous-dusk) !important;}.has-pale-ocean-gradient-background{background: var(--wp--preset--gradient--pale-ocean) !important;}.has-electric-grass-gradient-background{background: var(--wp--preset--gradient--electric-grass) !important;}.has-midnight-gradient-background{background: var(--wp--preset--gradient--midnight) !important;}.has-small-font-size{font-size: var(--wp--preset--font-size--small) !important;}.has-medium-font-size{font-size: var(--wp--preset--font-size--medium) !important;}.has-large-font-size{font-size: var(--wp--preset--font-size--large) !important;}.has-x-large-font-size{font-size: var(--wp--preset--font-size--x-large) !important;}.has-albert-sans-font-family{font-family: var(--wp--preset--font-family--albert-sans) !important;}.has-alegreya-font-family{font-family: var(--wp--preset--font-family--alegreya) !important;}.has-arvo-font-family{font-family: var(--wp--preset--font-family--arvo) !important;}.has-bodoni-moda-font-family{font-family: var(--wp--preset--font-family--bodoni-moda) !important;}.has-bricolage-grotesque-font-family{font-family: var(--wp--preset--font-family--bricolage-grotesque) !important;}.has-cabin-font-family{font-family: var(--wp--preset--font-family--cabin) !important;}.has-chivo-font-family{font-family: var(--wp--preset--font-family--chivo) !important;}.has-commissioner-font-family{font-family: var(--wp--preset--font-family--commissioner) !important;}.has-cormorant-font-family{font-family: var(--wp--preset--font-family--cormorant) !important;}.has-courier-prime-font-family{font-family: var(--wp--preset--font-family--courier-prime) !important;}.has-crimson-pro-font-family{font-family: var(--wp--preset--font-family--crimson-pro) !important;}.has-dm-mono-font-family{font-family: var(--wp--preset--font-family--dm-mono) !important;}.has-dm-sans-font-family{font-family: var(--wp--preset--font-family--dm-sans) !important;}.has-dm-serif-display-font-family{font-family: var(--wp--preset--font-family--dm-serif-display) !important;}.has-domine-font-family{font-family: var(--wp--preset--font-family--domine) !important;}.has-eb-garamond-font-family{font-family: var(--wp--preset--font-family--eb-garamond) !important;}.has-epilogue-font-family{font-family: var(--wp--preset--font-family--epilogue) !important;}.has-fahkwang-font-family{font-family: var(--wp--preset--font-family--fahkwang) !important;}.has-figtree-font-family{font-family: var(--wp--preset--font-family--figtree) !important;}.has-fira-sans-font-family{font-family: var(--wp--preset--font-family--fira-sans) !important;}.has-fjalla-one-font-family{font-family: var(--wp--preset--font-family--fjalla-one) !important;}.has-fraunces-font-family{font-family: var(--wp--preset--font-family--fraunces) !important;}.has-gabarito-font-family{font-family: var(--wp--preset--font-family--gabarito) !important;}.has-ibm-plex-mono-font-family{font-family: var(--wp--preset--font-family--ibm-plex-mono) !important;}.has-ibm-plex-sans-font-family{font-family: var(--wp--preset--font-family--ibm-plex-sans) !important;}.has-ibarra-real-nova-font-family{font-family: var(--wp--preset--font-family--ibarra-real-nova) !important;}.has-instrument-serif-font-family{font-family: var(--wp--preset--font-family--instrument-serif) !important;}.has-inter-font-family{font-family: var(--wp--preset--font-family--inter) !important;}.has-josefin-sans-font-family{font-family: var(--wp--preset--font-family--josefin-sans) !important;}.has-jost-font-family{font-family: var(--wp--preset--font-family--jost) !important;}.has-libre-baskerville-font-family{font-family: var(--wp--preset--font-family--libre-baskerville) !important;}.has-libre-franklin-font-family{font-family: var(--wp--preset--font-family--libre-franklin) !important;}.has-literata-font-family{font-family: var(--wp--preset--font-family--literata) !important;}.has-lora-font-family{font-family: var(--wp--preset--font-family--lora) !important;}.has-merriweather-font-family{font-family: var(--wp--preset--font-family--merriweather) !important;}.has-montserrat-font-family{font-family: var(--wp--preset--font-family--montserrat) !important;}.has-newsreader-font-family{font-family: var(--wp--preset--font-family--newsreader) !important;}.has-noto-sans-mono-font-family{font-family: var(--wp--preset--font-family--noto-sans-mono) !important;}.has-nunito-font-family{font-family: var(--wp--preset--font-family--nunito) !important;}.has-open-sans-font-family{font-family: var(--wp--preset--font-family--open-sans) !important;}.has-overpass-font-family{font-family: var(--wp--preset--font-family--overpass) !important;}.has-pt-serif-font-family{font-family: var(--wp--preset--font-family--pt-serif) !important;}.has-petrona-font-family{font-family: var(--wp--preset--font-family--petrona) !important;}.has-piazzolla-font-family{font-family: var(--wp--preset--font-family--piazzolla) !important;}.has-playfair-display-font-family{font-family: var(--wp--preset--font-family--playfair-display) !important;}.has-plus-jakarta-sans-font-family{font-family: var(--wp--preset--font-family--plus-jakarta-sans) !important;}.has-poppins-font-family{font-family: var(--wp--preset--font-family--poppins) !important;}.has-raleway-font-family{font-family: var(--wp--preset--font-family--raleway) !important;}.has-roboto-font-family{font-family: var(--wp--preset--font-family--roboto) !important;}.has-roboto-slab-font-family{font-family: var(--wp--preset--font-family--roboto-slab) !important;}.has-rubik-font-family{font-family: var(--wp--preset--font-family--rubik) !important;}.has-rufina-font-family{font-family: var(--wp--preset--font-family--rufina) !important;}.has-sora-font-family{font-family: var(--wp--preset--font-family--sora) !important;}.has-source-sans-3-font-family{font-family: var(--wp--preset--font-family--source-sans-3) !important;}.has-source-serif-4-font-family{font-family: var(--wp--preset--font-family--source-serif-4) !important;}.has-space-mono-font-family{font-family: var(--wp--preset--font-family--space-mono) !important;}.has-syne-font-family{font-family: var(--wp--preset--font-family--syne) !important;}.has-texturina-font-family{font-family: var(--wp--preset--font-family--texturina) !important;}.has-urbanist-font-family{font-family: var(--wp--preset--font-family--urbanist) !important;}.has-work-sans-font-family{font-family: var(--wp--preset--font-family--work-sans) !important;}
:where(.wp-block-columns.is-layout-flex){gap: 2em;}:where(.wp-block-columns.is-layout-grid){gap: 2em;}
/*# sourceURL=global-styles-inline-css */
</style>
<style id='core-block-supports-inline-css'>
.wp-container-core-columns-is-layout-62db62f3{flex-wrap:nowrap;}.wp-container-core-columns-is-layout-bcb7d8a1{flex-wrap:nowrap;}
/*# sourceURL=core-block-supports-inline-css */
</style>

<style id='classic-theme-styles-inline-css'>
/*! This file is auto-generated */
.wp-block-button__link{color:#fff;background-color:#32373c;border-radius:9999px;box-shadow:none;text-decoration:none;padding:calc(.667em + 2px) calc(1.333em + 2px);font-size:1.125em}.wp-block-file__button{background:#32373c;color:#fff;text-decoration:none}
/*# sourceURL=/wp-includes/css/classic-themes.min.css */
</style>
<style id='jetpack-global-styles-frontend-style-inline-css'>
:root { --font-headings: unset; --font-base: unset; --font-headings-default: -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif; --font-base-default: -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif;}
/*# sourceURL=jetpack-global-styles-frontend-style-inline-css */
</style>
<script id="jetpack-mu-wpcom-settings-js-before">
var JETPACK_MU_WPCOM_SETTINGS = {"assetsUrl":"/wp-content/mu-plugins/wpcomsh/jetpack_vendor/automattic/jetpack-mu-wpcom/src/build/"};
//# sourceURL=jetpack-mu-wpcom-settings-js-before
</script>
<script src="/wp-includes/js/jquery/jquery.min.js?ver=3.7.1" id="jquery-core-js"></script>
<script type='text/javascript'  src='/wp-includes/js/jquery/jquery-migrate.min.js?m=1686289764'></script>
<link rel="https://api.w.org/" href="/wp-json/" /><link rel="alternate" title="JSON" type="application/json" href="/wp-json/wp/v2/pages/27" />
<link rel="canonical" href="/literature/" />
	<style>img#wpstats{display:none}</style>
		<meta name="description" content="LITERATURE This page synchronizes an open Zotero library, which you can access here. Anything missing? Let us know in the imagesc forum or contact us! The formatting for this page is a work in progress. Any feedback? Let us know! Filter by type: All types Articles Books Comments Preprints Reviews" />

<!-- Jetpack Open Graph Tags -->
<meta property="og:type" content="article" />
<meta property="og:title" content="Literature" />
<meta property="og:url" content="/literature/" />
<meta property="og:description" content="LITERATURE This page synchronizes an open Zotero library, which you can access here. Anything missing? Let us know in the imagesc forum or contact us! The formatting for this page is a work in prog…" />
<meta property="article:published_time" content="2024-01-23T12:28:01+00:00" />
<meta property="article:modified_time" content="2025-10-02T12:44:41+00:00" />
<meta property="og:site_name" content="Smart Microscopy" />
<meta property="og:image" content="https://s0.wp.com/_si/?t=eyJpbWciOiJodHRwczpcL1wvaTAud3AuY29tXC9zbWFydG1pY3Jvc2NvcHkub3JnXC93cC1jb250ZW50XC91cGxvYWRzXC8yMDI0XC8wMlwvY3JvcHBlZC1zbWFydG1pY3Jvc2NvcHlfbG9nby0xLnBuZz9maXQ9NjMwJTJDNjMwJnNzbD0xIiwidHh0IjoiU21hcnQgTWljcm9zY29weSIsInRlbXBsYXRlIjoiZWRnZSIsImZvbnQiOiIiLCJibG9nX2lkIjoyMjgyOTgzNTB9.Fx-tzTfJ0BgFaZHyEtjV6Jfne4IH-Ff8icbhIDpZDhsMQ" />
<meta property="og:image:width" content="1200" />
<meta property="og:image:height" content="630" />
<meta property="og:image:alt" content="" />
<meta property="og:locale" content="en_US" />
<meta property="og:locale:alternate" content="nl_NL" />
<meta name="twitter:text:title" content="Literature" />
<meta name="twitter:image" content="https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_icon-1.png?fit=240%2C240&amp;ssl=1" />
<meta name="twitter:card" content="summary" />

<!-- End Jetpack Open Graph Tags -->
<link rel="icon" href="https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_icon-1.png?fit=32%2C32&#038;ssl=1" sizes="32x32" />
<link rel="icon" href="https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_icon-1.png?fit=192%2C192&#038;ssl=1" sizes="192x192" />
<link rel="apple-touch-icon" href="https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_icon-1.png?fit=180%2C180&#038;ssl=1" />
<meta name="msapplication-TileImage" content="https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_icon-1.png?fit=270%2C270&#038;ssl=1" />
<style type="text/css" id="wp-custom-css">.kind-search-toggle {
    display: none !important;
}

/* Apply only to forms with the custom class */
.custom-form-gap .contact-form__inset-label-wrap {
    margin-bottom: -10px !important;
}

.custom-form-gap .wp-block-jetpack-button {
    margin-top: -10px !important;
}</style><style id='jetpack-block-button-inline-css'>
.amp-wp-article .wp-block-jetpack-button{color:#fff}.wp-block-jetpack-button{height:fit-content;margin:0;max-width:100%;width:fit-content}.wp-block-jetpack-button.aligncenter,.wp-block-jetpack-button.alignleft,.wp-block-jetpack-button.alignright{display:block}.wp-block-jetpack-button.aligncenter{margin-left:auto;margin-right:auto}.wp-block-jetpack-button.alignleft{margin-right:auto}.wp-block-jetpack-button.alignright{margin-left:auto}.wp-block-jetpack-button.is-style-outline>.wp-block-button__link{background-color:#0000;border:1px solid;color:currentColor}:where(.wp-block-jetpack-button:not(.is-style-outline) button){border:none}.wp-block-jetpack-button .spinner{display:none;fill:currentColor}.wp-block-jetpack-button .spinner svg{display:block}.wp-block-jetpack-button .is-submitting .spinner{display:inline}.wp-block-jetpack-button .is-visually-hidden{clip:rect(0 0 0 0);clip-path:inset(50%);height:1px;overflow:hidden;position:absolute;white-space:nowrap;width:1px}.wp-block-jetpack-button .disable-spinner .spinner{display:none}
/*# sourceURL=/wp-content/plugins/jetpack/_inc/blocks/button/view.css?minify=false */
</style>
<link rel='stylesheet' id='grunion.css-css' href='/wp-content/plugins/jetpack/jetpack_vendor/automattic/jetpack-forms/src/../dist/contact-form/css/grunion.css?ver=15.4-a.3' media='all' />
</head>

<body id="top" class="wp-singular page-template page-template-templates page-template-template-noheader page-template-templatestemplate-noheader-php page page-id-27 wp-custom-logo wp-embed-responsive wp-theme-kind wpf-grey wpft- wpf-guest wpfu-group-4 wpf-theme-2022 wpf-is_standalone-0 wpf-boardid-0 is_wpforo_page-0 is_wpforo_url-0 is_wpforo_shortcode_page-0">
	<a class="skip-link screen-reader-text" href="#content">
		Skip to content	</a>

	
<header class="kind-header">
	<div class="kind-container-fluid">
		<div class="kind-header__inner kind-header__inner-desktop">
			<div class="kind-header__col kind-col-left">
						<a href="/" rel="home" class="kind-header-logo">
			<div class="kind-header-logo__inner">
									<div class="logo" aria-hidden="true">
						<img width="630" height="630" src="https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_logo-1.png?fit=630%2C630&amp;ssl=1" class="attachment-full size-full" alt="Smart Microscopy" decoding="async" fetchpriority="high" srcset="https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_logo-1.png?w=630&amp;ssl=1 630w, https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_logo-1.png?resize=300%2C300&amp;ssl=1 300w, https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_logo-1.png?resize=150%2C150&amp;ssl=1 150w, https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_logo-1.png?resize=600%2C600&amp;ssl=1 600w, https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_logo-1.png?resize=400%2C400&amp;ssl=1 400w, https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_logo-1.png?resize=200%2C200&amp;ssl=1 200w" sizes="(max-width: 630px) 100vw, 630px" data-attachment-id="193" data-permalink="/smartmicroscopy_logo-5/" data-orig-file="https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_logo-1.png?fit=630%2C630&amp;ssl=1" data-orig-size="630,630" data-comments-opened="1" data-image-meta="{&quot;aperture&quot;:&quot;0&quot;,&quot;credit&quot;:&quot;&quot;,&quot;camera&quot;:&quot;&quot;,&quot;caption&quot;:&quot;&quot;,&quot;created_timestamp&quot;:&quot;0&quot;,&quot;copyright&quot;:&quot;&quot;,&quot;focal_length&quot;:&quot;0&quot;,&quot;iso&quot;:&quot;0&quot;,&quot;shutter_speed&quot;:&quot;0&quot;,&quot;title&quot;:&quot;&quot;,&quot;orientation&quot;:&quot;0&quot;}" data-image-title="smartmicroscopy_logo" data-image-description="&lt;p&gt;©Alfredo Rates&lt;/p&gt;
" data-image-caption="" data-medium-file="https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_logo-1.png?fit=300%2C300&amp;ssl=1" data-large-file="https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_logo-1.png?fit=630%2C630&amp;ssl=1" />					</div>
				
									<div class="text">
													<span class="logo-name">Smart Microscopy</span>
																			<span class="logo-desc">All about imaging automation</span>
											</div>
				
			</div>
		</a>
					</div>
			<div class="kind-header__col kind-col-right">
				<nav class="kind-header-nav" aria-label="Primary menu"><ul id="menu-navigate" class="menu kind-nav-menu"><li id="menu-item-45" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-home menu-item-45"><a href="/">What is Smart Microscopy?</a></li>
<li id="menu-item-46" class="menu-item menu-item-type-post_type menu-item-object-page current-menu-item page_item page-item-27 current_page_item menu-item-46"><a href="/literature/" aria-current="page">Literature</a></li>
<li id="menu-item-304" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-304"><a href="/resources/">Resources</a></li>
<li id="menu-item-169" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-169"><a href="/news-about-smart-microscopy/">News</a></li>
<li id="menu-item-355" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-355"><a href="/contact/">Contact</a></li>
</ul></nav>		<button class="kind-search-toggle" type="button" aria-label="Open search">
			<svg width="18" height="19" viewBox="0 0 18 19" fill="none" xmlns="http://www.w3.org/2000/svg">
				<path fill-rule="evenodd" clip-rule="evenodd" d="M2 8.5C2 5.191 4.691 2.5 8 2.5C11.309 2.5 14 5.191 14 8.5C14 11.809 11.309 14.5 8 14.5C4.691 14.5 2 11.809 2 8.5ZM17.707 16.793L14.312 13.397C15.365 12.043 16 10.346 16 8.5C16 4.089 12.411 0.5 8 0.5C3.589 0.5 0 4.089 0 8.5C0 12.911 3.589 16.5 8 16.5C9.846 16.5 11.543 15.865 12.897 14.812L16.293 18.207C16.488 18.402 16.744 18.5 17 18.5C17.256 18.5 17.512 18.402 17.707 18.207C18.098 17.816 18.098 17.184 17.707 16.793Z" fill="currentColor"/>
			</svg>
		</button>
					</div>
		</div>
		
<div class="kind-header__inner kind-header__inner-mobile">
	<div class="kind-header__col kind-col-left">
				<a href="/" rel="home" class="kind-header-mobile-logo">
		<img width="630" height="630" src="https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_logo-1.png?fit=630%2C630&amp;ssl=1" class="attachment-full size-full" alt="Smart Microscopy" decoding="async" srcset="https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_logo-1.png?w=630&amp;ssl=1 630w, https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_logo-1.png?resize=300%2C300&amp;ssl=1 300w, https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_logo-1.png?resize=150%2C150&amp;ssl=1 150w, https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_logo-1.png?resize=600%2C600&amp;ssl=1 600w, https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_logo-1.png?resize=400%2C400&amp;ssl=1 400w, https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_logo-1.png?resize=200%2C200&amp;ssl=1 200w" sizes="(max-width: 630px) 100vw, 630px" data-attachment-id="193" data-permalink="/smartmicroscopy_logo-5/" data-orig-file="https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_logo-1.png?fit=630%2C630&amp;ssl=1" data-orig-size="630,630" data-comments-opened="1" data-image-meta="{&quot;aperture&quot;:&quot;0&quot;,&quot;credit&quot;:&quot;&quot;,&quot;camera&quot;:&quot;&quot;,&quot;caption&quot;:&quot;&quot;,&quot;created_timestamp&quot;:&quot;0&quot;,&quot;copyright&quot;:&quot;&quot;,&quot;focal_length&quot;:&quot;0&quot;,&quot;iso&quot;:&quot;0&quot;,&quot;shutter_speed&quot;:&quot;0&quot;,&quot;title&quot;:&quot;&quot;,&quot;orientation&quot;:&quot;0&quot;}" data-image-title="smartmicroscopy_logo" data-image-description="&lt;p&gt;©Alfredo Rates&lt;/p&gt;
" data-image-caption="" data-medium-file="https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_logo-1.png?fit=300%2C300&amp;ssl=1" data-large-file="https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_logo-1.png?fit=630%2C630&amp;ssl=1" />		</a>
			</div>
	<div class="kind-header__col kind-col-right">
					<button class="kind-offcanvas-toggle" aria-label="Open Off-Canvas">
		<span></span>
		<span></span>
		<span></span>
	</button>
		</div>
</div>
	</div>
		<div class="kind-search" tabindex="-1">
		<div class="kind-search__inner">
			<div class="kind-container">
							</div>
		</div>
		<button class="kind-search-close" aria-label="Close search"></button>
	</div>
		
<div class="nav-overlay"></div>

<div id="site_nav" class="site-nav" tabindex="-1">

	<div class="site-nav-title">
		
		<a href="/" rel="home" class="snt-cell" aria-hidden="true" tabindex="-1">
			<span class="logo-name"><img width="630" height="630" src="https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_logo-1.png?fit=630%2C630&amp;ssl=1" class="attachment-full size-full" alt="Smart Microscopy" data-attachment-id="193" data-permalink="/smartmicroscopy_logo-5/" data-orig-file="https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_logo-1.png?fit=630%2C630&amp;ssl=1" data-orig-size="630,630" data-comments-opened="1" data-image-meta="{&quot;aperture&quot;:&quot;0&quot;,&quot;credit&quot;:&quot;&quot;,&quot;camera&quot;:&quot;&quot;,&quot;caption&quot;:&quot;&quot;,&quot;created_timestamp&quot;:&quot;0&quot;,&quot;copyright&quot;:&quot;&quot;,&quot;focal_length&quot;:&quot;0&quot;,&quot;iso&quot;:&quot;0&quot;,&quot;shutter_speed&quot;:&quot;0&quot;,&quot;title&quot;:&quot;&quot;,&quot;orientation&quot;:&quot;0&quot;}" data-image-title="smartmicroscopy_logo" data-image-description="&lt;p&gt;©Alfredo Rates&lt;/p&gt;
" data-image-caption="" data-medium-file="https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_logo-1.png?fit=300%2C300&amp;ssl=1" data-large-file="https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_logo-1.png?fit=630%2C630&amp;ssl=1" /></span>
		</a>

		
			<button class="trigger-button close kind-offcanvas-close" aria-label="Close Off-Canvas">
		<svg width="24" height="24" viewBox="0 0 14 14" version="1.1" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" focusable="false">
			<path d="M14 1.41L12.59 0 7 5.59 1.41 0 0 1.41 5.59 7 0 12.59 1.41 14 7 8.41 12.59 14 14 12.59 8.41 7 14 1.41z" />
		</svg>	</button>
		</div>

	<nav class="nav-main-menu" aria-label="Primary menu"><ul id="menu-navigate-1" class="main-menu"><li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-home menu-item-45"><a href="/">What is Smart Microscopy?</a></li>
<li class="menu-item menu-item-type-post_type menu-item-object-page current-menu-item page_item page-item-27 current_page_item menu-item-46"><a href="/literature/" aria-current="page">Literature</a></li>
<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-304"><a href="/resources/">Resources</a></li>
<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-169"><a href="/news-about-smart-microscopy/">News</a></li>
<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-355"><a href="/contact/">Contact</a></li>
</ul></nav>
			<div class="search-holder"></div>
		
</div>
</header>
	<main id="content" class="kind-main">

<div class="page-content container">
	<div class="entry-content the-content text-column">
		
<div class="wp-block-columns is-layout-flex wp-container-core-columns-is-layout-62db62f3 wp-block-columns-is-layout-flex" style="padding-top:0;padding-bottom:0">
<div class="wp-block-column is-layout-flow wp-block-column-is-layout-flow" style="flex-basis:80%">
<div style="height:60px" aria-hidden="true" class="wp-block-spacer"></div>



<h1 class="wp-block-heading alignwide has-text-align-center" style="font-size:60px">LITERATURE</h1>
</div>



<div class="wp-block-column is-layout-flow wp-block-column-is-layout-flow" style="padding-top:0;padding-right:0;padding-bottom:0;padding-left:0;font-size:15px;flex-basis:33.33%">
<div style="height:50px" aria-hidden="true" class="wp-block-spacer"></div>


<div data-test='contact-form'
			id='contact-form-27'
			class='wp-block-jetpack-contact-form-container is-single-input-form '
			data-wp-interactive='jetpack/form' data-wp-context='{"formId":"27","formHash":"56695bc6ebad5920b7c3fac7281084932a3334a3","showErrors":false,"errors":[],"fields":[],"isMultiStep":false,"useAjax":true,"submissionData":null,"formattedSubmissionData":[],"submissionSuccess":false,"submissionError":null,"elementId":"jp-form-56695bc6ebad5920b7c3fac7281084932a3334a3","isSingleInputForm":true}'
			data-wp-watch--scroll-to-wrapper="callbacks.scrollToWrapper"
		>
<div class="contact-form-submission contact-form-ajax-submission" data-wp-class--submission-success="context.submissionSuccess"><p class="go-back-message"><a class="link" role="button" tabindex="0" data-wp-on--click="actions.goBack" href="/literature/">Go back</a></p><h4 id="contact-form-success-header"></h4>

<div class="jetpack_forms_contact-form-custom-success-message">Thanks!</div></div><form action='/literature/'
				id='jp-form-56695bc6ebad5920b7c3fac7281084932a3334a3'
				method='post'
				class='contact-form commentsblock wp-block-jetpack-contact-form' aria-label="Literature"
				data-wp-on--submit="actions.onFormSubmit"
				data-wp-on--reset="actions.onFormReset"
				data-wp-class--submission-success="context.submissionSuccess"
				data-wp-class--is-first-step="state.isFirstStep"
				data-wp-class--is-last-step="state.isLastStep"
				data-wp-class--is-ajax-form="context.useAjax"
				novalidate >
<input type='hidden' name='jetpack_contact_form_jwt' value='eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJlbmNyeXB0ZWRfYXR0cmlidXRlcyI6IlczVTlPb0x2QTBjOWE0S3h1NXl4cHRIQ2xPZnQ3ajdNbnZTOHVYQzdJc1FPR2N6K042bTJyaFBuVjJLWHNSdi9aWkU5S1lDNEczRjhDRGliRDJncmMvSFBDSUlxN3FmcStMNjgvaC9ZLzl3SHhiZFFFV2ZOOVd2MFh5Z05mOWhMaVRZTzNrUnl5UVN2NTVKMEpTQ3FqaVViZ3J6V0RWK3IwYjBkT2Y5UGpxWk9IanExV1UyZHNzOWRxQnBkaitpTmNJdms5ZzFXU3RrWHNwaG9ZVHEwb1hHNkVVZWJPKzh6dXoySjhOalkrOHJOWEoyamhGNng1K2NYU3k5eEpCRlBneTRWeWFxV1oyZ3M4M0NXbDVpbnVhWUI5L1ZoRFhNWlc4anVoa3VoSUZxZG04VFNocDBqeWJzY3BkL3VkeGFvVVc5dy9ybERSR2lYZWRlMDRxSTNjcTBXOTZmaU5uNUk5R0VoM3d4RFdNY2pNazBaOHo2WHZaQ1ltWmdPOEJOTWMxS05wVTQ3OWM0QkpYaU05SmJUcDJ2b3M4WVRsRUpzTWpySXRNamhRUkJXVHhUZFJacnl4cUZEbUp4K2NMMW1vak5JbklSLzY2MG02c1NUSlFKUS80NHVGdTUzNzE3am5kZzVXdmtrK2lzSkoxbEVjNmN6eThGNTdsZHFqanREaGtQVGpqS0ZuUGkvSVFGbmwycE91VVorK01kNkpWaEVaQm85TzVTUGs3blhLdmtxaE9sVWVBZklMdlJNU3dSTWZ4ZXk5TjRJby9lUzVqcENCSDRFc1duR2JKNTBTaWg1VkVlckNPeDRHdzlJa2cwMElUOEcwUmVHWUl5dXdRL0xNZGdIbmxjN3QrVVBqWGtGakhBNUFHRDBHQUNjbGVieXovWms4OXhFN0lYbktGSS9jZk45d214Mk9XWTBlb05rYklvU3hSdVdhWVZTeUJsdHY1dXdCaTFncGczT0wwQXVWc2dndHRaVTVUemhWOEpEUmloSWM1TVE2U1pJeG5NRDlGcjI0UVRIQy9yV2psWThwOG51anN1ZkpFc0RlYUs3K1lSK1MxWEdSUkcva2pRMjFlVGlIQWJvZHNIT29sbGFtc0g4MjVIcytwMnRNYW90YVpBcTlFZEZWaEExSHIzeTQrZWV1ZmNGTFJOWWorRzA4andEc0ZkK0hGYzBHYlhQRG42V3R2VHZGR0lGVFhPQ1NsNU1Dd3FSSVkrbDdwRk1sR0sxallNSit6dkJZdW1ORjd0NHV3b1BINkZGNkFEUVFlNnJNdFk1RmFFY01XY3hEaktya0lMbjhOTDE2Ukw0WHZKMFhJRTcxTlBrVm5mL2piRkNJODFTb2pRdEZNWT0iLCJjb250ZW50IjoiXG48ZGl2IGNsYXNzPVwid3AtYmxvY2stamV0cGFjay1jb250YWN0LWZvcm0gaXMtc3R5bGUtb3V0bGluZWQgY3VzdG9tLWZvcm0tZ2FwXCI-W2NvbnRhY3QtZmllbGQgdHlwZT1cInRleHRcIiBsYWJlbD1cIlRleHRcIiBsYWJlbGNsYXNzZXM9XCJ3cC1ibG9jay1qZXRwYWNrLWxhYmVsXCIgbGFiZWxoaWRkZW5ieWJsb2NrdmlzaWJpbGl0eT1cIlwiIHBsYWNlaG9sZGVyPVwiXCIgbWluPVwiXCIgbWF4PVwiXCIgaW5wdXRjbGFzc2VzPVwid3AtYmxvY2stamV0cGFjay1pbnB1dFwiIHN0eWxldmFyaWF0aW9uYXR0cmlidXRlcz1cIlwiIHN0eWxldmFyaWF0aW9uY2xhc3Nlcz1cIlwiIHN0eWxldmFyaWF0aW9uc3R5bGVzPVwiXCIgZmllbGR3cmFwcGVyY2xhc3Nlcz1cIndwLWJsb2NrLWpldHBhY2stZmllbGQtdGV4dFwiXSZsdDtkaXYmZ3Q7XG5cbiZsdDsvZGl2Jmd0O1svY29udGFjdC1maWVsZF1cblxuPGRpdiBjbGFzcz1cIndwLWJsb2NrLWpldHBhY2stYnV0dG9uIGlzLXN0eWxlLW91dGxpbmUgLmNvbnRhaW5lci1jbGFzcyB7ICAgICBkaXNwbGF5OiBmbGV4OyAgICAgYWxpZ24taXRlbXM6IGNlbnRlcjsgICAgIGdhcDogMDsgLyogUmVtb3ZlcyBzcGFjZSAqLyB9IHdwLWJsb2NrLWJ1dHRvblwiIHN0eWxlPVwid2lkdGg6IDEwMCU7XCI-PGJ1dHRvbiBjbGFzcz1cIndwLWJsb2NrLWJ1dHRvbl9fbGluayBpcy1zdHlsZS1vdXRsaW5lIC5jb250YWluZXItY2xhc3MgeyAgICAgZGlzcGxheTogZmxleDsgICAgIGFsaWduLWl0ZW1zOiBjZW50ZXI7ICAgICBnYXA6IDA7IC8qIFJlbW92ZXMgc3BhY2UgKi8gfSBuby1ib3JkZXItcmFkaXVzXCIgc3R5bGU9XCJmb250LXNpemU6IDEwcHg7XCIgZGF0YS1pZC1hdHRyPVwicGxhY2Vob2xkZXJcIiB0eXBlPVwic3VibWl0XCIgZGF0YS13cC1jbGFzcy0taXMtc3VibWl0dGluZz1cInN0YXRlLmlzU3VibWl0dGluZ1wiIGRhdGEtd3AtYmluZC0tYXJpYS1kaXNhYmxlZD1cInN0YXRlLmlzQXJpYURpc2FibGVkXCI-U3VibWl0PHNwYW4gY2xhc3M9XCJzcGlubmVyXCIgYXJpYS1oaWRkZW49XCJ0cnVlXCI-PHN2ZyB3aWR0aD1cIjI0XCIgaGVpZ2h0PVwiMjRcIiB2aWV3Qm94PVwiMCAwIDI0IDI0XCIgeG1sbnM9XCJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2Z1wiPjxwYXRoIGQ9XCJNMTIsMUExMSwxMSwwLDEsMCwyMywxMiwxMSwxMSwwLDAsMCwxMiwxWm0wLDE5YTgsOCwwLDEsMSw4LThBOCw4LDAsMCwxLDEyLDIwWlwiIG9wYWNpdHk9XCIuMjVcIi8-PHBhdGggZD1cIk0xMC4xNCwxLjE2YTExLDExLDAsMCwwLTksOC45MkExLjU5LDEuNTksMCwwLDAsMi40NiwxMiwxLjUyLDEuNTIsMCwwLDAsNC4xMSwxMC43YTgsOCwwLDAsMSw2LjY2LTYuNjFBMS40MiwxLjQyLDAsMCwwLDEyLDIuNjloMEExLjU3LDEuNTcsMCwwLDAsMTAuMTQsMS4xNlpcIj48YW5pbWF0ZVRyYW5zZm9ybSBhdHRyaWJ1dGVOYW1lPVwidHJhbnNmb3JtXCIgdHlwZT1cInJvdGF0ZVwiIGR1cj1cIjAuNzVzXCIgdmFsdWVzPVwiMCAxMiAxMjszNjAgMTIgMTJcIiByZXBlYXRDb3VudD1cImluZGVmaW5pdGVcIi8-PC9wYXRoPjwvc3ZnPjxzcGFuIGNsYXNzPVwiaXMtdmlzdWFsbHktaGlkZGVuXCI-U3VibWl0dGluZyBmb3JtPC9zcGFuPjwvc3Bhbj48L2J1dHRvbj48L2Rpdj48L2Rpdj5cbiIsImhhc2giOiI1NjY5NWJjNmViYWQ1OTIwYjdjM2ZhYzcyODEwODQ5MzJhMzMzNGEzIiwic291cmNlIjp7ImVudHJ5X3RpdGxlIjoiTGl0ZXJhdHVyZSIsImVudHJ5X3BhZ2UiOjEsInNvdXJjZV9pZCI6MjcsInNvdXJjZV90eXBlIjoic2luZ2xlIiwicmVxdWVzdF91cmwiOiJodHRwczovL3NtYXJ0bWljcm9zY29weS5vcmcvbGl0ZXJhdHVyZSJ9LCJ2ZXJzaW9uIjoyLCJjaXBoZXIiOiJhZXMtMjU2LWdjbSJ9.jMxxtvL6k-TYIbsCWXqhY82KBhrdCmWaM25sfHGJCGs' />

<div class="wp-block-jetpack-contact-form is-style-outlined custom-form-gap">
<div class='contact-form__inset-label-wrap'  data-wp-interactive="jetpack/form" data-wp-context='{"fieldId":"g27-text","fieldType":"text","fieldLabel":"Text","fieldValue":"","fieldPlaceholder":"","fieldIsRequired":"","fieldErrorMessage":"","fieldExtra":[],"formHash":"56695bc6ebad5920b7c3fac7281084932a3334a3"}'  >

<div style="--jetpack--contact-form--notch-width: max(var(--jetpack--contact-form--input-padding-left, 16px), var(--jetpack--contact-form--border-radius));"  class='wp-block-jetpack-field-text grunion-field-text-wrap wp-block-jetpack-input-wrap grunion-field-wrap'  data-wp-init='callbacks.initializeField' data-wp-on--jetpack-form-reset='callbacks.initializeField' >

			<div class="notched-label">
				<div class="notched-label__leading" style=""></div>
				<div class="notched-label__notch" style="">
					<label
						for="g27-text"
						class=" notched-label__label wp-block-jetpack-label"
						style="--jetpack--contact-form--notch-width: max(var(--jetpack--contact-form--input-padding-left, 16px), var(--jetpack--contact-form--border-radius));"
					>
					<span class="grunion-label-text">Text</span></label>
				</div>
				<div class="notched-label__filler" style=""></div>
				<div class="notched-label__trailing" style=""></div>
			</div><input
					type='text'
					name='g27-text'
					id='g27-text'
					value=''

					data-wp-bind--aria-invalid='state.fieldHasErrors'
					data-wp-bind--value='state.getFieldValue'
					aria-errormessage='g27-text-text-error-message'
					data-wp-on--input='actions.onFieldChange'
					data-wp-on--blur='actions.onFieldBlur'
					data-wp-class--has-value='state.hasFieldValue'

					class='text wp-block-jetpack-input grunion-field' 
					 />
  
	</div>

			<div id="g27-text-text-error" class="contact-form__input-error" data-wp-class--has-errors="state.fieldHasErrors">
				<span class="contact-form__warning-icon">
					<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
						<path d="M8.50015 11.6402H7.50015V10.6402H8.50015V11.6402Z" />
						<path d="M7.50015 9.64018H8.50015V6.30684H7.50015V9.64018Z" />
						<path fill-rule="evenodd" clip-rule="evenodd" d="M6.98331 3.0947C7.42933 2.30177 8.57096 2.30177 9.01698 3.09469L13.8771 11.7349C14.3145 12.5126 13.7525 13.4735 12.8602 13.4735H3.14004C2.24774 13.4735 1.68575 12.5126 2.12321 11.7349L6.98331 3.0947ZM8.14541 3.58496C8.08169 3.47168 7.9186 3.47168 7.85488 3.58496L2.99478 12.2251C2.93229 12.3362 3.01257 12.4735 3.14004 12.4735H12.8602C12.9877 12.4735 13.068 12.3362 13.0055 12.2251L8.14541 3.58496Z" />
					</svg>
					<span class="visually-hidden">Warning</span>
				</span>
				<span data-wp-text="state.errorMessage" id="g27-text-text-error-message" role="alert" aria-live="assertive"></span>
			</div>	</div>


<div class="wp-block-jetpack-button is-style-outline .container-class {     display: flex;     align-items: center;     gap: 0; /* Removes space */ } wp-block-button" style="width: 100%;"><button class="wp-block-button__link is-style-outline .container-class {     display: flex;     align-items: center;     gap: 0; /* Removes space */ } no-border-radius" style="font-size: 10px;" data-id-attr="placeholder" type="submit" data-wp-class--is-submitting="state.isSubmitting" data-wp-bind--aria-disabled="state.isAriaDisabled">Submit<span class="spinner" aria-hidden="true"><svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12,1A11,11,0,1,0,23,12,11,11,0,0,0,12,1Zm0,19a8,8,0,1,1,8-8A8,8,0,0,1,12,20Z" opacity=".25"/><path d="M10.14,1.16a11,11,0,0,0-9,8.92A1.59,1.59,0,0,0,2.46,12,1.52,1.52,0,0,0,4.11,10.7a8,8,0,0,1,6.66-6.61A1.42,1.42,0,0,0,12,2.69h0A1.57,1.57,0,0,0,10.14,1.16Z"><animateTransform attributeName="transform" type="rotate" dur="0.75s" values="0 12 12;360 12 12" repeatCount="indefinite"/></path></svg><span class="is-visually-hidden">Submitting form</span></span></button></div></div>
		<input type='hidden' name='contact-form-id' value='27' />
		<input type='hidden' name='action' value='grunion-contact-form' />
		<input type='hidden' name='contact-form-hash' value='56695bc6ebad5920b7c3fac7281084932a3334a3' />
<p style="display: none !important;" class="akismet-fields-container" data-prefix="ak_"><label>&#916;<textarea name="ak_hp_textarea" cols="45" rows="8" maxlength="100"></textarea></label><input type="hidden" id="ak_js_1" name="ak_js" value="226"/><script>document.getElementById( "ak_js_1" ).setAttribute( "value", ( new Date() ).getTime() );</script></p></form>
</div></div>
</div>



<div class="wp-block-columns alignwide is-layout-flex wp-container-core-columns-is-layout-bcb7d8a1 wp-block-columns-is-layout-flex" style="padding-top:var(--wp--preset--spacing--20);padding-right:var(--wp--preset--spacing--20);padding-bottom:var(--wp--preset--spacing--20);padding-left:var(--wp--preset--spacing--20)">
<div class="wp-block-column is-layout-flow wp-block-column-is-layout-flow" style="flex-basis:100%">
<p class="has-text-align-justify">This page synchronizes an open Zotero library, which you can access <a href="https://www.zotero.org/groups/5384098/smart_microscopy_library" data-type="link" data-id="https://www.zotero.org/groups/5384098/smart_microscopy_library">here</a>. Anything missing? <a href="https://forum.image.sc/tag/smart-microscopy">Let us know in the imagesc forum</a> or <a href="/contact/" data-type="page" data-id="330">contact us</a>! <em>The formatting for  this page is a work in progress. Any feedback? <a href="/contact/" data-type="page" data-id="330">Let us know!</a></em></p>



<style>
    .collection label {
      display: inline-block;
      width: 100%; /* adjust as needed */
      font-weight: 500;
    }
  
    .collection select {
      width: 100%; /* or 100% if inside a flex/grid container */
    }
</style>

<!-- 📦 COLLECTION SELECTOR -->
 <!-- 📦 COLLECTION & TAG SELECTORS (horizontal layout) -->
<div id="filters">
    <div class="collection"  style="display: flex; gap: 2rem; margin-bottom: 1rem;">
        <label for="collection-filter" width=10% display=inline-block margin-bottom=10px>Filter by type: 
        <select id="collection-filter">
        <option value="all">All types</option>
        <option value="IYTK29JT">Articles</option>
        <option value="4I89N939">Books</option>
        <option value="HQUL7BI3">Comments</option>
        <option value="UQ77P7I5">Preprints</option>
        <option value="4KIL6FA6">Reviews</option>
        </label>
        </select>
    </div>
</div>
<!-- 📚 ZOTPRESS ENTRIES FOR EACH COLLECTION -->
<div id="zotpress-entries">

  <!-- All -->
  <div class="zotpress-section" data-collection="all">
    <div id="zotpress-4b2fcfdc2cecc977c6429237eba45b72" class="zp-Zotpress zp-Zotpress-Bib wp-block-group">

		<span class="ZP_API_USER_ID ZP_ATTR">5384098</span>
		<span class="ZP_ITEM_KEY ZP_ATTR"></span>
		<span class="ZP_COLLECTION_ID ZP_ATTR"></span>
		<span class="ZP_TAG_ID ZP_ATTR"></span>
		<span class="ZP_AUTHOR ZP_ATTR"></span>
		<span class="ZP_YEAR ZP_ATTR"></span>
        <span class="ZP_ITEMTYPE ZP_ATTR"></span>
		<span class="ZP_INCLUSIVE ZP_ATTR">1</span>
		<span class="ZP_STYLE ZP_ATTR">harvard1</span>
		<span class="ZP_LIMIT ZP_ATTR">50</span>
		<span class="ZP_SORTBY ZP_ATTR">date</span>
		<span class="ZP_ORDER ZP_ATTR">desc</span>
		<span class="ZP_TITLE ZP_ATTR"></span>
		<span class="ZP_SHOWIMAGE ZP_ATTR"></span>
		<span class="ZP_SHOWTAGS ZP_ATTR">1</span>
		<span class="ZP_DOWNLOADABLE ZP_ATTR">1</span>
		<span class="ZP_NOTES ZP_ATTR"></span>
		<span class="ZP_ABSTRACT ZP_ATTR"></span>
		<span class="ZP_CITEABLE ZP_ATTR"></span>
		<span class="ZP_TARGET ZP_ATTR">1</span>
		<span class="ZP_URLWRAP ZP_ATTR">title</span>
		<span class="ZP_FORCENUM ZP_ATTR"></span>
        <span class="ZP_HIGHLIGHT ZP_ATTR"></span>
        <span class="ZP_POSTID ZP_ATTR">27</span>
		<span class="ZOTPRESS_PLUGIN_URL ZP_ATTR"><a href="/wp-content/plugins/zotpress/">/wp-content/plugins/zotpress/</a></span>

		<div class="zp-List loading">
			<div class="zp-SEO-Content">
				<span class="ZP_JSON ZP_ATTR">%7B%22status%22%3A%22success%22%2C%22updateneeded%22%3Afalse%2C%22instance%22%3Afalse%2C%22meta%22%3A%7B%22request_last%22%3A50%2C%22request_next%22%3A50%2C%22used_cache%22%3Atrue%7D%2C%22data%22%3A%5B%7B%22key%22%3A%22BULWHJV6%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Leger%20et%20al.%22%2C%22parsedDate%22%3A%222025-12-14%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BLeger%2C%20D.%2C%20Ivanisevic%2C%20M.%20and%20Lenart%2C%20P.%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.biorxiv.org%5C%2Fcontent%5C%2F10.64898%5C%2F2025.12.11.693650v1%26%23039%3B%26gt%3BAutoSTED%3A%20An%20automated%20workflow%20for%20STED%20super-resolution%20imaging%20of%20cell%20nuclei%26lt%3B%5C%2Fa%26gt%3B.%26quot%3B%20bioRxiv.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.64898%5C%2F2025.12.11.693650.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22preprint%22%2C%22title%22%3A%22AutoSTED%3A%20An%20automated%20workflow%20for%20STED%20super-resolution%20imaging%20of%20cell%20nuclei%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Dorian%22%2C%22lastName%22%3A%22Leger%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Milena%22%2C%22lastName%22%3A%22Ivanisevic%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Peter%22%2C%22lastName%22%3A%22Lenart%22%7D%5D%2C%22abstractNote%22%3A%22The%20traditional%20microscopy%20workflow%20involves%20manual%20selection%20of%20objects%20one%20after%20the%20other.%20This%20is%20not%20only%20time%20consuming%2C%20but%20inevitably%20introduces%20human%20bias.%20Here%2C%20we%20describe%20a%20workflow%2C%20AutoSTED%20for%20identification%20of%20cell%20nuclei%20on%20wide-field%20fluorescence%20images%20and%20subsequent%20automated%20imaging%20of%20these%20nuclei%20at%20super-resolution%20by%20Stimulated%20Emission%20Depletion%20%28STED%29%20microscopy.%20We%20demonstrate%20that%20in%20an%20overnight%20run%20we%20are%20able%20to%20image%20hundreds%20of%20cells%20selected%20without%20bias.%20The%20resolution%20afforded%20by%20STED%20allowed%20straightforward%20identification%20and%20counting%20of%20individual%20nuclear%20pore%20complexes%20%28NPCs%29%2C%20and%20enabled%20us%20to%20automatically%20extract%20parameters%20such%20as%20NPC%20density%20under%20different%20experimental%20conditions.%20Together%2C%20we%20illustrate%20how%20automation%20enables%20optimal%20use%20of%20instrument%20time%20to%20produce%20large%20unbiased%20datasets.%22%2C%22genre%22%3A%22%22%2C%22repository%22%3A%22bioRxiv%22%2C%22archiveID%22%3A%22%22%2C%22date%22%3A%222025-12-14%22%2C%22DOI%22%3A%2210.64898%5C%2F2025.12.11.693650%22%2C%22citationKey%22%3A%22%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.biorxiv.org%5C%2Fcontent%5C%2F10.64898%5C%2F2025.12.11.693650v1%22%2C%22language%22%3A%22en%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222025-12-16T13%3A19%3A02Z%22%2C%22tags%22%3A%5B%5D%7D%7D%2C%7B%22key%22%3A%22NTMGRK8D%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Redondo%20and%20Paszek%22%2C%22parsedDate%22%3A%222025-12-01%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BRedondo%2C%20J.A.%20and%20Paszek%2C%20P.%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fcames.ippt.pan.pl%5C%2Findex.php%5C%2Fcames%5C%2Farticle%5C%2Fview%5C%2F1945%26%23039%3B%26gt%3BAI-Assisted%20Microscopy%20for%20Infection%20Biology%3A%20Advances%20in%20High-Content%20Imaging%20of%20Host-Pathogen%20Interactions%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BComputer%20Assisted%20Methods%20in%20Engineering%20and%20Science%26lt%3B%5C%2Fi%26gt%3B%20%5BPreprint%5D.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.24423%5C%2Fcames.2025.1945.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22AI-Assisted%20Microscopy%20for%20Infection%20Biology%3A%20Advances%20in%20High-Content%20Imaging%20of%20Host-Pathogen%20Interactions%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Juan%20Alfonso%22%2C%22lastName%22%3A%22Redondo%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Pawel%22%2C%22lastName%22%3A%22Paszek%22%7D%5D%2C%22abstractNote%22%3A%22Advances%20in%20high-content%20microscopy%20and%20artificial%20intelligence%20%28AI%29%20are%20transforming%20the%20quantitative%20study%20of%20infection%20biology.%20Automated%20imaging%20platforms%20now%20enable%20rapid%2C%20large-scale%20acquisition%20of%20host-pathogen%20interactions%20across%20thousands%20of%20cells%20and%20multiple%20experimental%20conditions.%20When%20combined%20with%20AI-based%20segmentation%2C%20these%20workflows%20extract%20infection-relevant%20features%20such%20as%20pathogen%20load%2C%20intracellular%20localization%2C%20and%20host%20response%20markers%20at%20single-cell%20resolution.%20Deep-learning%20models%20have%20proven%20especially%20powerful%2C%20outperforming%20classical%20threshold-based%20methods%20under%20different%20imaging%20conditions%2C%20reducing%20reliance%20on%20manual%20annotation%2C%20and%20detecting%20rare%20infection%20outcomes.%20Beyond%20robust%20image%20analysis%2C%20these%20approaches%20generate%20scalable%20and%20reproducible%20datasets%20that%20can%20be%20integrated%20with%20computational%20modelling%20and%20systems%20biology%2C%20providing%20predictive%20insight%20into%20infection%20dynamics.%20This%20review%20highlights%20recent%20progress%20in%20AI-assisted%20microscopy%20for%20bacterial%20infection%20and%20outlines%20future%20directions%20toward%20multimodal%20integration%2C%20clinical%20translation%2C%20and%20open-source%20tool%20development.%22%2C%22date%22%3A%222025-12-01%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.24423%5C%2Fcames.2025.1945%22%2C%22ISSN%22%3A%222956-5839%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fcames.ippt.pan.pl%5C%2Findex.php%5C%2Fcames%5C%2Farticle%5C%2Fview%5C%2F1945%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222025-12-09T11%3A53%3A38Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22artificial%20intelligence%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22cell-to-cell%20variability%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22cellular%20heterogeneity%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22deep%20learning%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22host-pathogen%20interactions%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22infection%20biology%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22machine%20learning%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22single-cell%20biology%22%2C%22type%22%3A1%7D%5D%7D%7D%2C%7B%22key%22%3A%22PGILC555%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Cosenza%20et%20al.%22%2C%22parsedDate%22%3A%222025-12%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BCosenza%2C%20M.R.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41586-025-09632-5%26%23039%3B%26gt%3BOrigins%20of%20chromosome%20instability%20unveiled%20by%20coupled%20imaging%20and%20genomics%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%26lt%3B%5C%2Fi%26gt%3B%2C%20648%288093%29%2C%20pp.%20383%5Cu2013393.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41586-025-09632-5.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Origins%20of%20chromosome%20instability%20unveiled%20by%20coupled%20imaging%20and%20genomics%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Marco%20Raffaele%22%2C%22lastName%22%3A%22Cosenza%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Alice%22%2C%22lastName%22%3A%22Gaiatto%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22B%5Cu00fc%5Cu015fra%22%2C%22lastName%22%3A%22Erarslan%20Uysal%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22%5Cu00c1lvaro%22%2C%22lastName%22%3A%22Andrades%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Nina%20Luisa%22%2C%22lastName%22%3A%22Sautter%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Marina%22%2C%22lastName%22%3A%22Simunovic%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Michael%20Adrian%22%2C%22lastName%22%3A%22Jendrusch%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Sonia%22%2C%22lastName%22%3A%22Zumalave%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Tobias%22%2C%22lastName%22%3A%22Rausch%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Aliaksandr%22%2C%22lastName%22%3A%22Halavatyi%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Eva-Maria%22%2C%22lastName%22%3A%22Geissen%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Joshua%20Lucas%22%2C%22lastName%22%3A%22Eigenmann%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Thomas%22%2C%22lastName%22%3A%22Weber%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Patrick%22%2C%22lastName%22%3A%22Hasenfeld%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Eva%22%2C%22lastName%22%3A%22Benito%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Catherine%22%2C%22lastName%22%3A%22Stober%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Isidro%22%2C%22lastName%22%3A%22Cortes-Ciriano%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Andreas%20E.%22%2C%22lastName%22%3A%22Kulozik%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Rainer%22%2C%22lastName%22%3A%22Pepperkok%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jan%20O.%22%2C%22lastName%22%3A%22Korbel%22%7D%5D%2C%22abstractNote%22%3A%22Somatic%20chromosome%20instability%20results%20in%20widespread%20structural%20and%20numerical%20chromosomal%20abnormalities%20%28CAs%29%20during%20cancer%20evolution1%5Cu20133.%20Although%20CAs%20have%20been%20linked%20to%20mitotic%20errors%20resulting%20in%20the%20emergence%20of%20nuclear%20atypia4%5Cu20137%2C%20the%20underlying%20processes%20and%20rates%20of%20spontaneous%20CA%20formation%20in%20human%20cells%20are%20underexplored.%20Here%20we%20introduce%20machine-learning-assisted%20genomics%20and%20imaging%20convergence%20%28MAGIC%29%5Cu2014an%20autonomously%20operated%20platform%20that%20integrates%20live-cell%20imaging%20of%20micronucleated%20cells%2C%20machine%20learning%20on-the-fly%20and%20single-cell%20genomics%20to%20systematically%20investigate%20CA%20formation.%20Applying%20MAGIC%20to%20near-diploid%2C%20non-transformed%20cell%20lines%2C%20we%20track%20de%20novo%20CAs%20over%20successive%20cell%20cycles%2C%20highlighting%20the%20common%20role%20of%20dicentric%20chromosomes%20as%20initiating%20events.%20We%20determine%20the%20baseline%20CA%20mutation%20rate%2C%20which%20approximately%20doubles%20in%20TP53-deficient%20cells%2C%20and%20observe%20that%20chromosome%20losses%20arise%20more%20frequently%20than%20gains.%20The%20targeted%20induction%20of%20DNA%20double-strand%20breaks%20along%20chromosome%20arms%20triggers%20distinct%20CA%20processes%2C%20revealing%20stable%20isochromosomes%2C%20coordinated%20segregation%20and%20amplification%20of%20isoacentric%20segments%20in%20multiples%20of%20two%2C%20as%20well%20as%20complex%20CA%20outcomes%2C%20influenced%20by%20the%20chromosomal%20break%20location.%20Our%20data%20contrast%20de%20novo%20CA%20spectra%20from%20somatic%20mutational%20landscapes%20after%20selection%20occurred.%20The%20experimentation%20enabled%20by%20MAGIC%20advances%20the%20dissection%20of%20DNA%20rearrangement%20processes%2C%20shedding%20light%20on%20fundamental%20determinants%20of%20chromosomal%20instability.%22%2C%22date%22%3A%222025-12%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs41586-025-09632-5%22%2C%22ISSN%22%3A%221476-4687%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41586-025-09632-5%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222025-12-16T13%3A11%3A56Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Chromosome%20segregation%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22Genomic%20instability%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22High-throughput%20screening%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22Imaging%22%2C%22type%22%3A1%7D%5D%7D%7D%2C%7B%22key%22%3A%22TWDLDCMT%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Tang%22%2C%22parsedDate%22%3A%222025-12%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BTang%2C%20L.%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-025-02986-8%26%23039%3B%26gt%3BBridging%20smart%20microscopy%20with%20genomics%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%20Methods%26lt%3B%5C%2Fi%26gt%3B%2C%2022%2812%29%2C%20pp.%202499%5Cu20132499.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41592-025-02986-8.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Bridging%20smart%20microscopy%20with%20genomics%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Lei%22%2C%22lastName%22%3A%22Tang%22%7D%5D%2C%22abstractNote%22%3A%22%22%2C%22date%22%3A%222025-12%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs41592-025-02986-8%22%2C%22ISSN%22%3A%221548-7105%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-025-02986-8%22%2C%22collections%22%3A%5B%22HQUL7BI3%22%5D%2C%22dateModified%22%3A%222025-12-16T13%3A11%3A51Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Chromosome%20segregation%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22Genomic%20analysis%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22Genomic%20instability%22%2C%22type%22%3A1%7D%5D%7D%7D%2C%7B%22key%22%3A%22WFEWMARH%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Berrevoets%20et%20al.%22%2C%22parsedDate%22%3A%222025-11-27%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BBerrevoets%2C%20E.S.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41467-025-65723-x%26%23039%3B%26gt%3BSmart%203D%20super-resolution%20microscopy%20reveals%20the%20architecture%20of%20the%20RNA%20scaffold%20in%20a%20nuclear%20body%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%20Communications%26lt%3B%5C%2Fi%26gt%3B%2C%2016%281%29%2C%20p.%2010689.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41467-025-65723-x.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Smart%203D%20super-resolution%20microscopy%20reveals%20the%20architecture%20of%20the%20RNA%20scaffold%20in%20a%20nuclear%20body%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Enya%20S.%22%2C%22lastName%22%3A%22Berrevoets%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Laurell%20F.%22%2C%22lastName%22%3A%22Kessler%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ashwin%22%2C%22lastName%22%3A%22Balakrishnan%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ellen%20Kazumi%22%2C%22lastName%22%3A%22Okuda%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Michaela%22%2C%22lastName%22%3A%22M%5Cu00fcller-McNicoll%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Bernd%22%2C%22lastName%22%3A%22Rieger%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Sjoerd%22%2C%22lastName%22%3A%22Stallinga%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Mike%22%2C%22lastName%22%3A%22Heilemann%22%7D%5D%2C%22abstractNote%22%3A%22Small%20subcellular%20organelles%20orchestrate%20key%20cellular%20functions.%20How%20biomolecules%20are%20spatially%20organized%20within%20these%20assemblies%20is%20poorly%20understood.%20Here%2C%20we%20report%20an%20automated%20super-resolution%20imaging%20and%20analysis%20workflow%20that%20integrates%20confocal%20microscopy%2C%20morphological%20object%20screening%2C%20targeted%203D%20super-resolution%20STED%20microscopy%20and%20quantitative%20image%20analysis.%20Using%20this%20smart%20microscopy%20workflow%2C%20we%20target%20the%203D%20organization%20of%20NEAT1%2C%20an%20architectural%20RNA%20that%20constitutes%20the%20structural%20backbone%20of%20paraspeckles%2C%20a%20membraneless%20nuclear%20organelle.%20Using%20site-specific%20labeling%2C%20morphological%20sorting%20and%20particle%20averaging%2C%20we%20reconstruct%20the%20morphological%20space%20of%20paraspeckles%20along%20their%20development%20cycle%20from%20over%2010%2C000%20individual%20particles.%20Applying%20spherical%20harmonics%20analysis%2C%20we%20report%20so-far%20unknown%20heterotypes%20of%20NEAT1%20RNA%20organization.%20By%20integrating%20multi-positional%20labeling%2C%20we%20determine%20the%20coarse%20conformation%20of%20NEAT1%20within%20the%20organelle%20and%20show%20that%20the%203%5Cu2019%20end%20forms%20a%20loop-like%20structure%20at%20the%20surface%20of%20the%20paraspeckle.%20Our%20study%20reveals%20key%20structural%20features%20of%20paraspeckle%20structure%20and%20growth%2C%20as%20well%20as%20the%20molecular%20organization%20of%20its%20scaffolding%20RNA.%22%2C%22date%22%3A%222025-11-27%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs41467-025-65723-x%22%2C%22ISSN%22%3A%222041-1723%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41467-025-65723-x%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222025-12-01T13%3A04%3A56Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Imaging%20techniques%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22Nanoscale%20biophysics%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22Super-resolution%20microscopy%22%2C%22type%22%3A1%7D%5D%7D%7D%2C%7B%22key%22%3A%222C2QZA49%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22London%20et%20al.%22%2C%22parsedDate%22%3A%222025-11-03%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BLondon%2C%20J.A.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.biorxiv.org%5C%2Fcontent%5C%2F10.1101%5C%2F2025.10.31.685671v1%26%23039%3B%26gt%3BA%20Python%20framework%20for%20magnetic%20tweezers%20real-time%20image%20processing%20and%20microscope%20control%26lt%3B%5C%2Fa%26gt%3B.%26quot%3B%20bioRxiv.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1101%5C%2F2025.10.31.685671.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22preprint%22%2C%22title%22%3A%22A%20Python%20framework%20for%20magnetic%20tweezers%20real-time%20image%20processing%20and%20microscope%20control%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22James%20A.%22%2C%22lastName%22%3A%22London%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Abhishek%20K.%22%2C%22lastName%22%3A%22Singh%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Teague%20C.%22%2C%22lastName%22%3A%22Svendsen%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Naciye%20Esma%22%2C%22lastName%22%3A%22Tirtom%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Zachary%20A.%22%2C%22lastName%22%3A%22Root%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Richard%22%2C%22lastName%22%3A%22Fishel%22%7D%5D%2C%22abstractNote%22%3A%22Magnetic%20tweezers%20are%20a%20popular%20biophysical%20instrument%20for%20manipulating%20and%20measuring%20single%20molecules.%20Most%20laboratories%20rely%20on%20custom-built%20setups%20tailored%20to%20specific%20experiments%2C%20resulting%20in%20a%20variety%20of%20hardware%20and%20software%20implementations.%20Typically%2C%20image%20acquisition%20and%20hardware%20control%20are%20automated%20via%20LabVIEW%20and%20specialized%20C%5C%2FC%2B%2B%5C%2FCUDA%20libraries%20for%20real-time%20video%20processing.%20Live%20processing%20eliminates%20the%20need%20to%20store%20raw%20video%2C%20enabling%20high%20throughput%2C%20fast%20acquisition%20rates%2C%20and%20simplified%20experimental%20workflows.%20However%2C%20no%20open-source%2C%20general-purpose%20software%20framework%20currently%20unifies%20these%20capabilities%20for%20magnetic%20tweezers%20experiments.%20Here%2C%20we%20introduce%20MagTrack%20and%20MagScope%2C%20open-source%20Python-based%20tools%20designed%20to%20fill%20this%20gap.%20MagTrack%20is%20an%20image-processing%20library%20that%20efficiently%20determines%20bead-positions%20from%20magnetic-tweezers%20videos%20using%20CPU%20and%5C%2For%20GPU%20computation.%20MagScope%20is%20a%20comprehensive%20software%20framework%20offering%20a%20graphical%20user%20interface%2C%20real-time%20hardware%20control%2C%20data%20acquisition%2C%20and%20video%20processing.%20It%20is%20built%20on%20a%20multiprocessing%20architecture%20for%20responsive%2C%20high-throughput%20computation.%20Together%2C%20MagTrack%20and%20MagScope%20offer%20a%20flexible%2C%20modern%2C%20and%20fully%20customizable%20open-source%20alternative%20to%20proprietary%20or%20fragmented%20systems%2C%20enabling%20laboratories%20to%20adapt%20and%20extend%20the%20framework%20according%20to%20their%20experimental%20and%20programming%20needs.%22%2C%22genre%22%3A%22%22%2C%22repository%22%3A%22bioRxiv%22%2C%22archiveID%22%3A%22%22%2C%22date%22%3A%222025-11-03%22%2C%22DOI%22%3A%2210.1101%5C%2F2025.10.31.685671%22%2C%22citationKey%22%3A%22%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.biorxiv.org%5C%2Fcontent%5C%2F10.1101%5C%2F2025.10.31.685671v1%22%2C%22language%22%3A%22en%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222025-12-01T13%3A16%3A49Z%22%2C%22tags%22%3A%5B%5D%7D%7D%2C%7B%22key%22%3A%223JF4LIGZ%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Mandal%20et%20al.%22%2C%22parsedDate%22%3A%222025-10-14%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BMandal%2C%20I.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41467-025-64105-7%26%23039%3B%26gt%3BEvaluating%20large%20language%20model%20agents%20for%20automation%20of%20atomic%20force%20microscopy%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%20Communications%26lt%3B%5C%2Fi%26gt%3B%2C%2016%281%29%2C%20p.%209104.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41467-025-64105-7.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Evaluating%20large%20language%20model%20agents%20for%20automation%20of%20atomic%20force%20microscopy%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Indrajeet%22%2C%22lastName%22%3A%22Mandal%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jitendra%22%2C%22lastName%22%3A%22Soni%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Mohd%22%2C%22lastName%22%3A%22Zaki%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Morten%20M.%22%2C%22lastName%22%3A%22Smedskjaer%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Katrin%22%2C%22lastName%22%3A%22Wondraczek%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Lothar%22%2C%22lastName%22%3A%22Wondraczek%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Nitya%20Nand%22%2C%22lastName%22%3A%22Gosvami%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22N.%20M.%20Anoop%22%2C%22lastName%22%3A%22Krishnan%22%7D%5D%2C%22abstractNote%22%3A%22Large%20language%20models%20%28LLMs%29%20are%20transforming%20laboratory%20automation%20by%20enabling%20self-driving%20laboratories%20%28SDLs%29%20that%20could%20accelerate%20materials%20research.%20However%2C%20current%20SDL%20implementations%20rely%20on%20rigid%20protocols%20that%20fail%20to%20capture%20the%20adaptability%20and%20intuition%20of%20expert%20scientists%20in%20dynamic%20experimental%20settings.%20Here%2C%20we%20show%20that%20LLM%20agents%20can%20automate%20atomic%20force%20microscopy%5Cu00a0%28AFM%29%20through%20our%20Artificially%20Intelligent%20Lab%20Assistant%20%28AILA%29%20framework.%20Further%2C%20we%20develop%20AFMBench%5Cu2014a%20comprehensive%20evaluation%20suite%20challenging%20LLM%20agents%20across%20the%20complete%20scientific%20workflow%20from%20experimental%20design%20to%20results%20analysis.%20We%20find%20that%20state-of-the-art%20LLMs%20struggle%20with%20basic%20tasks%20and%20coordination%20scenarios.%20Notably%2C%20models%20excelling%20at%20materials%20science%20question-answering%20perform%20poorly%20in%20laboratory%20settings%2C%20showing%20that%20domain%20knowledge%20does%20not%20translate%20to%20experimental%20capabilities.%20Additionally%2C%20we%20observe%20that%20LLM%20agents%20can%20deviate%20from%20instructions%2C%20a%20phenomenon%20referred%20to%20as%5Cu00a0sleepwalking%2C%20raising%20safety%20alignment%20concerns%20for%20SDL%20applications.%20Our%20ablations%20reveal%20that%20multi-agent%20frameworks%20significantly%20outperform%20single-agent%20approaches%2C%20though%20both%20remain%20sensitive%20to%20minor%20changes%20in%20instruction%20formatting%20or%20prompting.%20Finally%2C%20we%20evaluate%20AILA%5Cu2019s%20effectiveness%20in%20increasingly%20advanced%20experiments%5Cu2014AFM%20calibration%2C%20feature%20detection%2C%20mechanical%20property%20measurement%2C%20graphene%20layer%20counting%2C%20and%20indenter%20detection.%20These%20findings%20establish%20the%20necessity%20for%20benchmarking%20and%20robust%20safety%20protocols%20before%20deploying%20LLM%20agents%20as%20autonomous%20laboratory%20assistants%20across%20scientific%20disciplines.%22%2C%22date%22%3A%222025-10-14%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs41467-025-64105-7%22%2C%22ISSN%22%3A%222041-1723%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41467-025-64105-7%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222025-10-20T11%3A01%3A49Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Atomic%20force%20microscopy%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22Computer%20science%22%2C%22type%22%3A1%7D%5D%7D%7D%2C%7B%22key%22%3A%22KY9GPYEJ%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Hickey%20et%20al.%22%2C%22parsedDate%22%3A%222025-10-10%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BHickey%2C%20J.W.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.biorxiv.org%5C%2Fcontent%5C%2F10.1101%5C%2F2025.10.09.680291v1%26%23039%3B%26gt%3BFluid-Squid%3A%20DIY%20Multiplexed%20Imaging%20of%20Cells%20and%20Tissues%26lt%3B%5C%2Fa%26gt%3B.%26quot%3B%20bioRxiv.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1101%5C%2F2025.10.09.680291.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22preprint%22%2C%22title%22%3A%22Fluid-Squid%3A%20DIY%20Multiplexed%20Imaging%20of%20Cells%20and%20Tissues%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22John%20W.%22%2C%22lastName%22%3A%22Hickey%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Hongquan%22%2C%22lastName%22%3A%22Li%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Chiara%22%2C%22lastName%22%3A%22Caraccio%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22You%22%2C%22lastName%22%3A%22Yan%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Kevin%22%2C%22lastName%22%3A%22Marx%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Peter%22%2C%22lastName%22%3A%22Martin%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Hope%20T.%22%2C%22lastName%22%3A%22Leng%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22John%22%2C%22lastName%22%3A%22Fayiah%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Elvis%22%2C%22lastName%22%3A%22Towalid%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Bonnie%22%2C%22lastName%22%3A%22Dighero-Kemp%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Garry%20P.%22%2C%22lastName%22%3A%22Nolan%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Manu%22%2C%22lastName%22%3A%22Prakash%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22David%20R.%22%2C%22lastName%22%3A%22McIlwain%22%7D%5D%2C%22abstractNote%22%3A%22Recent%20advances%20in%20multiplexed%20single-cell%20characterization%20have%20revolutionized%20our%20insight%20into%20cell%20biology%2C%20but%20many%20available%20technologies%20remain%20limited%20by%20high%20costs%20or%20a%20lack%20of%20customizability.%20To%20address%20these%20challenges%2C%20we%20developed%20Fluid-Squid%2C%20a%20cost-effective%2C%20quantitative%20imaging%20platform%20that%20integrates%20automated%20fluidics%20to%20support%20customizable%2C%20do-it-yourself%20multiplexed%20imaging%20workflows.%20Using%20Fluid-Squid%2C%20we%20successfully%20imaged%20fresh%20frozen%20human%20intestinal%20tissues%20with%20a%2036-plex%20oligonucleotide-barcoded%20antibody%20panel%20and%20further%20demonstrated%20the%20feasibility%20of%20lyophilizing%20such%20multiplexed%20panels.%20We%20also%20adapted%20existing%20multiplexed%20imaging%20workflows%20to%20characterize%20individual%20cells%20to%20identify%20immune%20cell%20populations%2C%20phenotype%2C%20and%20antigen-specific%20cells%20from%20mouse%20splenocytes%20and%20human%20peripheral%20blood%20mononuclear%20cells%20%28PBMCs%29%20with%20a%2039-antibody%20panel.%20To%20further%20expand%20its%20utility%2C%20we%20developed%20a%20barcoding%20strategy%20that%20allows%20for%20the%20pooling%20and%20simultaneous%20staining%20of%20multiple%20samples%2C%20reducing%20time%2C%20costs%2C%20and%20batch%20effects%20in%20single-cell%20experiments.%20This%20approach%20facilitated%20rapid%20titration%20to%20optimize%20antibody%20concentrations%20and%20assess%20the%20impact%20of%20various%20blood%20preparation%20methods%20on%20cell%20type%20retention.%20Overall%2C%20our%20work%20provides%20a%20new%20open-source%20framework%20for%20automated%20fluidics%20and%20microscopy%20in%20a%20flexible%2C%20cost-effective%20platform%2C%20empowering%20adaptable%20multiplexed%20characterization%20of%20both%20single%20cells%20and%20tissues.%22%2C%22genre%22%3A%22%22%2C%22repository%22%3A%22bioRxiv%22%2C%22archiveID%22%3A%22%22%2C%22date%22%3A%222025-10-10%22%2C%22DOI%22%3A%2210.1101%5C%2F2025.10.09.680291%22%2C%22citationKey%22%3A%22%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.biorxiv.org%5C%2Fcontent%5C%2F10.1101%5C%2F2025.10.09.680291v1%22%2C%22language%22%3A%22en%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222025-12-01T13%3A16%3A11Z%22%2C%22tags%22%3A%5B%5D%7D%7D%2C%7B%22key%22%3A%22LVWHWJR5%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Friederich%20et%20al.%22%2C%22parsedDate%22%3A%222025-10-01%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BFriederich%2C%20N.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.degruyterbrill.com%5C%2Fdocument%5C%2Fdoi%5C%2F10.1515%5C%2Fauto-2025-0018%5C%2Fhtml%3Frecommended%3Dsidebar%26%23039%3B%26gt%3BEAP4EMSIG%20%5Cu2013%20enhancing%20event-driven%20microscopy%20for%20microfluidic%20single-cell%20analysis%3A%20EAP4EMSIG%20%5Cu2013%20Verbesserung%20der%20ereignisgesteuerten%20Mikroskopie%20f%5Cu00fcr%20die%20mikrofluidische%20Einzelzellanalyse%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3Bat%20-%20Automatisierungstechnik%26lt%3B%5C%2Fi%26gt%3B%2C%2073%2810%29%2C%20pp.%20808%5Cu2013823.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1515%5C%2Fauto-2025-0018.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22EAP4EMSIG%20%5Cu2013%20enhancing%20event-driven%20microscopy%20for%20microfluidic%20single-cell%20analysis%3A%20EAP4EMSIG%20%5Cu2013%20Verbesserung%20der%20ereignisgesteuerten%20Mikroskopie%20f%5Cu00fcr%20die%20mikrofluidische%20Einzelzellanalyse%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Nils%22%2C%22lastName%22%3A%22Friederich%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Angelo%20Jovin%20Yamachui%22%2C%22lastName%22%3A%22Sitcheu%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Annika%22%2C%22lastName%22%3A%22Nassal%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Erenus%22%2C%22lastName%22%3A%22Yildiz%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Matthias%22%2C%22lastName%22%3A%22Pesch%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Maximilian%22%2C%22lastName%22%3A%22Beichter%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Lukas%22%2C%22lastName%22%3A%22Scholtes%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Bahar%22%2C%22lastName%22%3A%22Akbaba%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Thomas%22%2C%22lastName%22%3A%22Lautenschlager%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Oliver%22%2C%22lastName%22%3A%22Neumann%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Dietrich%22%2C%22lastName%22%3A%22Kohlheyer%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Hanno%22%2C%22lastName%22%3A%22Scharr%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Johannes%22%2C%22lastName%22%3A%22Seiffarth%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Katharina%22%2C%22lastName%22%3A%22N%5Cu00f6h%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ralf%22%2C%22lastName%22%3A%22Mikut%22%7D%5D%2C%22abstractNote%22%3A%22Microfluidic%20Live-Cell%20Imaging%20%28MLCI%29%20yields%20data%20on%20microbial%20cell%20factories.%20However%2C%20continuous%20acquisition%20is%20challenging%20as%20high-throughput%20experiments%20often%20lack%20real-time%20insights%2C%20delaying%20responses%20to%20stochastic%20events.%20We%20introduce%20three%20components%20in%20the%20Experiment%20Automation%20Pipeline%20for%20Event-Driven%20Microscopy%20to%20Smart%20Microfluidic%20Single-Cell%20Analysis%20%28EAP4EMSIG%29%3A%20a%20fast%2C%20accurate%20Multi-Layer%20Perceptron%20%28MLP%29-based%20autofocusing%20method%20predicting%20the%20focus%20offset%2C%20an%20evaluation%20of%20real-time%20segmentation%20methods%20and%20a%20real-time%20data%20analysis%20dashboard.%20Our%20MLP-based%20autofocusing%20achieves%20a%20Mean%20Absolute%20Error%20%28MAE%29%20of%200.105%5Cu202f%5Cu00b5m%20with%20inference%20times%20of%2087%5Cu202fms.%20Among%20eleven%20evaluated%20Deep%20Learning%20%28DL%29%20segmentation%20methods%2C%20Cellpose%20reached%20a%20Panoptic%20Quality%20%28PQ%29%20of%2093.36%5Cu202f%25%2C%20while%20a%20distance-based%20method%20was%20fastest%20%28121%5Cu202fms%2C%20Panoptic%20Quality%2093.02%5Cu202f%25%29.%22%2C%22date%22%3A%222025-10-01%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1515%5C%2Fauto-2025-0018%22%2C%22ISSN%22%3A%222196-677X%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.degruyterbrill.com%5C%2Fdocument%5C%2Fdoi%5C%2F10.1515%5C%2Fauto-2025-0018%5C%2Fhtml%3Frecommended%3Dsidebar%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222025-10-27T14%3A30%3A37Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22MLP-based%20autofocusing%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22deep%20learning%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22experiment%20automation%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22live-cell%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22microfluidic%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22segmentation%22%2C%22type%22%3A1%7D%5D%7D%7D%2C%7B%22key%22%3A%22ZUJE3P8Y%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Lee%20et%20al.%22%2C%22parsedDate%22%3A%222025-10%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BLee%2C%20J.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs43587-025-00950-x%26%23039%3B%26gt%3BREV-ERB%5Cu03b1%20regulates%20brain%20NAD%2B%20levels%20and%20tauopathy%20via%20an%20NFIL3%5Cu2013CD38%20axis%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%20Aging%26lt%3B%5C%2Fi%26gt%3B%2C%205%2810%29%2C%20pp.%202070%5Cu20132085.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs43587-025-00950-x.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22REV-ERB%5Cu03b1%20regulates%20brain%20NAD%2B%20levels%20and%20tauopathy%20via%20an%20NFIL3%5Cu2013CD38%20axis%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jiyeon%22%2C%22lastName%22%3A%22Lee%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ryeonghwa%22%2C%22lastName%22%3A%22Kang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Sohui%22%2C%22lastName%22%3A%22Park%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ibrahim%20O.%22%2C%22lastName%22%3A%22Saliu%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Minsoo%22%2C%22lastName%22%3A%22Son%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jaymie%20R.%22%2C%22lastName%22%3A%22Voorhees%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Julie%20M.%22%2C%22lastName%22%3A%22Dimitry%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Elsa%20I.%22%2C%22lastName%22%3A%22Quillin%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Lauren%20N.%22%2C%22lastName%22%3A%22Woodie%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Brian%20V.%22%2C%22lastName%22%3A%22Lananna%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Li%22%2C%22lastName%22%3A%22Gan%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Young-Ah%22%2C%22lastName%22%3A%22Goo%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Guoyan%22%2C%22lastName%22%3A%22Zhao%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Mitchell%20A.%22%2C%22lastName%22%3A%22Lazar%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Thomas%20P.%22%2C%22lastName%22%3A%22Burris%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Erik%20S.%22%2C%22lastName%22%3A%22Musiek%22%7D%5D%2C%22abstractNote%22%3A%22Nicotinamide%20adenine%20dinucleotide%20%28NAD%2B%29%20is%20a%20critical%20metabolic%20co-enzyme%20implicated%20in%20brain%20aging%2C%20and%20augmenting%20NAD%2B%20levels%20in%20the%20aging%20brain%20is%20an%20attractive%20therapeutic%20strategy%20for%20neurodegeneration.%20However%2C%20the%20molecular%20mechanisms%20of%20brain%20NAD%2B%20regulation%20are%20incompletely%20understood.%20In%20cardiac%20tissue%2C%20the%20circadian%20nuclear%20receptor%20REV-ERB%5Cu03b1%20has%20been%20shown%20to%20regulate%20NAD%2B%20via%20control%20of%20the%20NAD%2B-producing%20enzyme%20NAMPT.%20Here%20we%20show%20that%20REV-ERB%5Cu03b1%20controls%20brain%20NAD%2B%20levels%20through%20a%20distinct%20pathway%20involving%20NFIL3-dependent%20suppression%20of%20the%20NAD%2B-consuming%20enzyme%20CD38%2C%20particularly%20in%20astrocytes.%20REV-ERB%5Cu03b1%20deletion%20does%20not%20affect%20NAMPT%20expression%20in%20the%20brain%20and%20has%20an%20opposite%20effect%20on%20NAD%2B%20levels%20as%20in%20the%20heart.%20Astrocytic%20REV-ERB%5Cu03b1%20deletion%20augments%20brain%20NAD%2B%20and%20prevents%20tauopathy%20in%20P301S%20mice.%20Our%20data%20reveal%20that%20REV-ERB%5Cu03b1%20regulates%20NAD%2B%20in%20a%20tissue-specific%20manner%20via%20opposing%20regulation%20of%20NAMPT%20versus%20CD38%20and%20define%20an%20astrocyte%20REV-ERB%5Cu03b1%5Cu2013NFIL3%5Cu2013CD38%20pathway%20controlling%20brain%20NAD%2B%20metabolism%20and%20neurodegeneration.%22%2C%22date%22%3A%222025-10%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs43587-025-00950-x%22%2C%22ISSN%22%3A%222662-8465%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs43587-025-00950-x%22%2C%22collections%22%3A%5B%5D%2C%22dateModified%22%3A%222025-10-20T11%3A01%3A39Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Ageing%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22Alzheimer%27s%20disease%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22Astrocyte%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22Circadian%20mechanisms%22%2C%22type%22%3A1%7D%5D%7D%7D%2C%7B%22key%22%3A%22KFNGL64Q%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Zhang%20et%20al.%22%2C%22parsedDate%22%3A%222025-09-08%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BZhang%2C%20N.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fpubs.rsc.org%5C%2Fen%5C%2Fcontent%5C%2Farticlelanding%5C%2F2025%5C%2Flc%5C%2Fd5lc00286a%26%23039%3B%26gt%3BModular%2C%20open-sourced%20multiplexing%20for%20democratizing%20spatial%20omics%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BLab%20on%20a%20Chip%26lt%3B%5C%2Fi%26gt%3B%20%5BPreprint%5D.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1039%5C%2FD5LC00286A.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Modular%2C%20open-sourced%20multiplexing%20for%20democratizing%20spatial%20omics%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Nicholas%22%2C%22lastName%22%3A%22Zhang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Zhou%22%2C%22lastName%22%3A%22Fang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Priyam%22%2C%22lastName%22%3A%22Kadakia%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jamie%22%2C%22lastName%22%3A%22Guo%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Dakshin%22%2C%22lastName%22%3A%22Vijay%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Manoj%22%2C%22lastName%22%3A%22Thapa%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Samuel%22%2C%22lastName%22%3A%22Dembowitz%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Arash%22%2C%22lastName%22%3A%22Grakoui%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ahmet%20F.%22%2C%22lastName%22%3A%22Coskun%22%7D%5D%2C%22abstractNote%22%3A%22Spatial%20omics%20technologies%20have%20revolutionized%20the%20field%20of%20biology%20by%20enabling%20the%20visualization%20of%20biomolecules%20within%20their%20native%20tissue%20context.%20However%2C%20the%20high%20costs%20associated%20with%20proprietary%20instrumentation%2C%20specialized%20reagents%2C%20and%20complex%20workflows%20have%20limited%20the%20broad%20application%20of%20these%20techniques.%20In%20this%20study%2C%20we%20introduce%20Python-based%20robotic%20imaging%20and%20staining%20for%20modular%20spatial%20omics%20%28PRISMS%29%2C%20an%20open-sourced%2C%20automated%20multiplexing%20pipeline%20compatible%20with%20several%20biospecimen%20targets%20and%20streamlined%20microscopy%20software%20tools.%20PRISMS%20utilizes%20a%20liquid%20handling%20robot%20with%20thermal%20control%20to%20enable%20the%20rapid%20and%20automated%20staining%20of%20RNA%20and%20protein%20samples.%20The%20modular%20sample%20holders%20and%20Python%20control%20facilitate%20high-throughput%2C%20single-molecule%20fluorescence%20imaging%20on%20widefield%20and%20confocal%20microscopes.%20We%20successfully%20demonstrated%20the%20versatility%20of%20PRISMS%20by%20imaging%20tissue%20slides%20and%20adherent%20cells.%20We%20demonstrate%20that%20PRISMS%20can%20be%20utilized%20to%20perform%20super-resolved%20imaging%2C%20such%20as%20super-resolution%20radial%20fluctuations%20%28SRRF%29.%20PRISMS%20is%20a%20powerful%20tool%20that%20can%20be%20used%20to%20democratize%20spatial%20omics%20by%20providing%20researchers%20with%20an%20accessible%2C%20reproducible%2C%20and%20cost-effective%20solution%20for%20multiplex%20imaging.%20Specifically%2C%20PRISMS%20is%20an%20open-source%2C%20automated%20multiplexing%20pipeline%20for%20spatial%20omics%2C%20compatible%20with%20several%20sample%20types%20and%20Nikon%20NIS%20Elements%20Basic%20Research%20software%2C%20as%20well%20as%20Python-based%20biodevices.%20It%20performs%20high-throughput%2C%20single-molecule%20fluorescence%20imaging%20both%20on%20widefield%20and%20confocal%20microscopes%2C%20and%20can%20be%20used%20to%20perform%20super-resolved%20imaging%2C%20such%20as%20SRRF.%20Overall%2C%20PRISMS%20is%20a%20powerful%20tool%20that%20can%20be%20used%20to%20democratize%20spatial%20omics%20by%20providing%20researchers%20with%20an%20accessible%2C%20reproducible%2C%20and%20cost-effective%20solution%20for%20multiplex%20imaging.%20This%20open-source%20platform%20will%20enable%20researchers%20to%20push%20the%20boundaries%20of%20spatial%20biology%20and%20make%20groundbreaking%20discoveries.%22%2C%22date%22%3A%222025-09-08%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1039%5C%2FD5LC00286A%22%2C%22ISSN%22%3A%221473-0189%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fpubs.rsc.org%5C%2Fen%5C%2Fcontent%5C%2Farticlelanding%5C%2F2025%5C%2Flc%5C%2Fd5lc00286a%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222025-09-16T08%3A46%3A28Z%22%2C%22tags%22%3A%5B%5D%7D%7D%2C%7B%22key%22%3A%22AS5J555G%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Alvelid%20et%20al.%22%2C%22parsedDate%22%3A%222025-09-01%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BAlvelid%2C%20J.%2C%20Koerfer%2C%20A.%20and%20Eggeling%2C%20C.%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.biorxiv.org%5C%2Fcontent%5C%2F10.1101%5C%2F2025.08.27.672674v1%26%23039%3B%26gt%3BEvent-triggered%20MINFLUX%20microscopy%3A%20smart%20microscopy%20to%20catch%20and%20follow%20rare%20events%26lt%3B%5C%2Fa%26gt%3B.%26quot%3B%20bioRxiv.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1101%5C%2F2025.08.27.672674.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22preprint%22%2C%22title%22%3A%22Event-triggered%20MINFLUX%20microscopy%3A%20smart%20microscopy%20to%20catch%20and%20follow%20rare%20events%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jonatan%22%2C%22lastName%22%3A%22Alvelid%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Agnes%22%2C%22lastName%22%3A%22Koerfer%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Christian%22%2C%22lastName%22%3A%22Eggeling%22%7D%5D%2C%22abstractNote%22%3A%22MINFLUX%20microscopy%20is%20a%20powerful%20microscopy%20method%20allowing%20for%20the%20characterization%20of%20molecular%20organization%20and%20dynamics%20with%20single%20nanometer%20spatial%20resolution%20and%20sub-hundred%20microseconds%20temporal%20resolution.%20However%2C%20acquisition%20times%20often%20span%20minutes%20to%20hours%20as%20a%20single%20fluorophore%20is%20measured%20at%20a%20time.%20Applying%20it%20to%20study%20cellular%20processes%20in%20living%20cells%20therefore%20requires%20careful%20consideration%20of%20where%20and%20when%20to%20apply%20MINFLUX%20data%20acquisition%2C%20a%20consideration%20where%20manual%20control%20limits%20its%20potential%20applications.%20Here%2C%20to%20overcome%20the%20limitations%20of%20acquisition%20speed%2C%20acquisition%20initiation%2C%20and%20data%20throughput%2C%20we%20present%20a%20smart%20microscopy%20method%20that%20uses%20confocal%20imaging%20as%20a%20monitoring%20method%2C%20runs%20real-time%20image%20analysis%2C%20and%20only%20applies%20MINFLUX%20data%20acquisition%20exactly%20where%20and%20when%20deemed%20necessary%20based%20on%20the%20analysis%20outcome.%20The%20method%2C%20event-triggered%20MINFLUX%2C%20is%20controlled%20through%20a%20custom-written%20and%20open%20source%20Python%20widget%20that%20automatically%20controls%20a%20commercial%20MINFLUX%20microscope.%20We%20apply%20this%20method%20to%20investigate%20molecular%20membrane%20dynamics%20and%20organization%20during%20three%20different%20cellular%20events%3A%20two-dimensional%20lipid%20dynamics%20at%20caveolae%3B%20three-dimensional%20membrane%20topography%20during%20dynamin-mediated%20endocytosis%3B%20and%20three-dimensional%20membrane%20fluidity%20and%20topography%20during%20budding%20site%20formation%20of%20HIV-1%20proteins.%20Thanks%20to%20rapid%20event%20detection%20and%20minimal%20regions%20of%20interest%20the%20method%20provides%20data%20that%20would%20be%20unfeasible%20or%20impossible%20to%20acquire%20through%20manual%20control%20of%20the%20microscope.%22%2C%22genre%22%3A%22%22%2C%22repository%22%3A%22bioRxiv%22%2C%22archiveID%22%3A%22%22%2C%22date%22%3A%222025-09-01%22%2C%22DOI%22%3A%2210.1101%5C%2F2025.08.27.672674%22%2C%22citationKey%22%3A%22%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.biorxiv.org%5C%2Fcontent%5C%2F10.1101%5C%2F2025.08.27.672674v1%22%2C%22language%22%3A%22en%22%2C%22collections%22%3A%5B%22UQ77P7I5%22%5D%2C%22dateModified%22%3A%222025-10-02T12%3A19%3A18Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Application%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22HV29U83C%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Meirovitch%20et%20al.%22%2C%22parsedDate%22%3A%222025-05-15%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BMeirovitch%2C%20Y.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.biorxiv.org%5C%2Fcontent%5C%2F10.1101%5C%2F2023.10.05.561103v4%26%23039%3B%26gt%3BSmartEM%3A%20machine-learning%20guided%20electron%20microscopy%26lt%3B%5C%2Fa%26gt%3B.%26quot%3B%20bioRxiv.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1101%5C%2F2023.10.05.561103.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22preprint%22%2C%22title%22%3A%22SmartEM%3A%20machine-learning%20guided%20electron%20microscopy%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Yaron%22%2C%22lastName%22%3A%22Meirovitch%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ishaan%20Singh%22%2C%22lastName%22%3A%22Chandok%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Core%20Francisco%22%2C%22lastName%22%3A%22Park%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Pavel%22%2C%22lastName%22%3A%22Potocek%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Lu%22%2C%22lastName%22%3A%22Mi%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Shashata%22%2C%22lastName%22%3A%22Sawmya%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Yicong%22%2C%22lastName%22%3A%22Li%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Thomas%20L.%22%2C%22lastName%22%3A%22Athey%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Vladislav%22%2C%22lastName%22%3A%22Susoy%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Neha%22%2C%22lastName%22%3A%22Karlupia%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Yuelong%22%2C%22lastName%22%3A%22Wu%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Daniel%20R.%22%2C%22lastName%22%3A%22Berger%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Richard%22%2C%22lastName%22%3A%22Schalek%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Caitlyn%20A.%22%2C%22lastName%22%3A%22Bishop%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Daniel%22%2C%22lastName%22%3A%22Xenes%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Hannah%22%2C%22lastName%22%3A%22Martinez%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jordan%22%2C%22lastName%22%3A%22Matelsky%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Brock%20A.%22%2C%22lastName%22%3A%22Wester%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Hanspeter%22%2C%22lastName%22%3A%22Pfister%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Remco%22%2C%22lastName%22%3A%22Schoenmakers%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Maurice%22%2C%22lastName%22%3A%22Peemen%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jeff%20W.%22%2C%22lastName%22%3A%22Lichtman%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Aravinthan%20D.%20T.%22%2C%22lastName%22%3A%22Samuel%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Nir%22%2C%22lastName%22%3A%22Shavit%22%7D%5D%2C%22abstractNote%22%3A%22Connectomics%20provides%20nanometer-resolution%2C%20synapse-level%20maps%20of%20neural%20circuits%20to%20understand%20brain%20activity%20and%20behavior.%20However%2C%20few%20researchers%20have%20access%20to%20the%20high-throughput%20electron%20microscopes%20necessary%20to%20generate%20enough%20data%20for%20whole%20brain%20or%20even%20whole%20circuit%20reconstruction.%20To%20date%2C%20machine-learning%20methods%20have%20been%20used%20after%20the%20collection%20of%20images%20by%20electron%20microscopy%20%28EM%29%20to%20accelerate%20and%20improve%20neuronal%20segmentation%2C%20synapse%20reconstruction%20and%20other%20data%20analysis.%20With%20the%20computational%20improvements%20in%20processing%20EM%20images%2C%20acquiring%20EM%20images%20has%20now%20become%20the%20rate-limiting%20step%20in%20automated%20connectomics.%20Here%2C%20in%20order%20to%20speed%20up%20EM%20imaging%2C%20we%20integrate%20machine-learning%20into%20real-time%20image%20acquisition%20in%20a%20single-beam%20scanning%20electron%20microscope.%20This%20SmartEM%20approach%20allows%20an%20electron%20microscope%20to%20perform%20intelligent%2C%20data-aware%20imaging%20of%20specimens.%20SmartEM%20saves%20time%20by%20allocating%20the%20proper%20imaging%20time%20for%20each%20region%20of%20interest%20%5Cu2013%20first%20scanning%20all%20pixels%20rapidly%2C%20then%20rescanning%20more%20slowly%20only%20the%20small%20subareas%20where%20a%20higher%20quality%20signal%20is%20required.%20We%20demonstrate%20that%20SmartEM%20achieves%20up%20to%20a%20~7-fold%20acceleration%20of%20image%20acquisition%20time%20for%20connectomic%20samples%20using%20a%20commercial%20single-beam%20SEM%20in%20samples%20from%20nematodes%2C%20mice%20and%20human%20brain.%20We%20apply%20this%20fast%20imaging%20method%20to%20reconstruct%20a%20portion%20of%20mouse%20cerebral%20cortex%20with%20an%20accuracy%20comparable%20to%20traditional%20electron%20microscopy.%22%2C%22genre%22%3A%22%22%2C%22repository%22%3A%22bioRxiv%22%2C%22archiveID%22%3A%22%22%2C%22date%22%3A%222025-05-15%22%2C%22DOI%22%3A%2210.1101%5C%2F2023.10.05.561103%22%2C%22citationKey%22%3A%22%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.biorxiv.org%5C%2Fcontent%5C%2F10.1101%5C%2F2023.10.05.561103v4%22%2C%22language%22%3A%22en%22%2C%22collections%22%3A%5B%22UQ77P7I5%22%5D%2C%22dateModified%22%3A%222025-10-06T10%3A44%3A14Z%22%2C%22tags%22%3A%5B%5D%7D%7D%2C%7B%22key%22%3A%22QJYYQESZ%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Hsieh%20et%20al.%22%2C%22parsedDate%22%3A%222025-04-17%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BHsieh%2C%20H.-C.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-025-02647-w%26%23039%3B%26gt%3BImaging%203D%20cell%20cultures%20with%20optical%20microscopy%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%20Methods%26lt%3B%5C%2Fi%26gt%3B%2C%20pp.%201%5Cu201324.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41592-025-02647-w.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Imaging%203D%20cell%20cultures%20with%20optical%20microscopy%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Huai-Ching%22%2C%22lastName%22%3A%22Hsieh%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Qinghua%22%2C%22lastName%22%3A%22Han%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22David%22%2C%22lastName%22%3A%22Brenes%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Kevin%20W.%22%2C%22lastName%22%3A%22Bishop%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Rui%22%2C%22lastName%22%3A%22Wang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Yuli%22%2C%22lastName%22%3A%22Wang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Chetan%22%2C%22lastName%22%3A%22Poudel%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Adam%20K.%22%2C%22lastName%22%3A%22Glaser%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Benjamin%20S.%22%2C%22lastName%22%3A%22Freedman%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Joshua%20C.%22%2C%22lastName%22%3A%22Vaughan%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Nancy%20L.%22%2C%22lastName%22%3A%22Allbritton%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jonathan%20T.%20C.%22%2C%22lastName%22%3A%22Liu%22%7D%5D%2C%22abstractNote%22%3A%22Three-dimensional%20%283D%29%20cell%20cultures%20have%20gained%20popularity%20in%20recent%20years%20due%20to%20their%20ability%20to%20represent%20complex%20tissues%20or%20organs%20more%20faithfully%20than%20conventional%20two-dimensional%20%282D%29%20cell%20culture.%20This%20article%20reviews%20the%20application%20of%20both%202D%20and%203D%20microscopy%20approaches%20for%20monitoring%20and%20studying%203D%20cell%20cultures.%20We%20first%20summarize%20the%20most%20popular%20optical%20microscopy%20methods%20that%20have%20been%20used%20with%203D%20cell%20cultures.%20We%20then%20discuss%20the%20general%20advantages%20and%20disadvantages%20of%20various%20microscopy%20techniques%20for%20several%20broad%20categories%20of%20investigation%20involving%203D%20cell%20cultures.%20Finally%2C%20we%20provide%20perspectives%20on%20key%20areas%20of%20technical%20need%20in%20which%20there%20are%20clear%20opportunities%20for%20innovation.%20Our%20goal%20is%20to%20guide%20microscope%20engineers%20and%20biomedical%20end%20users%20toward%20optimal%20imaging%20methods%20for%20specific%20investigational%20scenarios%20and%20to%20identify%20use%20cases%20in%20which%20additional%20innovations%20in%20high-resolution%20imaging%20could%20be%20helpful.%22%2C%22date%22%3A%222025-04-17%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs41592-025-02647-w%22%2C%22ISSN%22%3A%221548-7105%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-025-02647-w%22%2C%22collections%22%3A%5B%224KIL6FA6%22%5D%2C%22dateModified%22%3A%222025-04-29T14%3A05%3A14Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22QELVL8TC%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Yang%20et%20al.%22%2C%22parsedDate%22%3A%222025-04-14%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BYang%2C%20J.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttp%3A%5C%2F%5C%2Farxiv.org%5C%2Fabs%5C%2F2504.10281%26%23039%3B%26gt%3BZero-shot%20Autonomous%20Microscopy%20for%20Scalable%20and%20Intelligent%20Characterization%20of%202D%20Materials%26lt%3B%5C%2Fa%26gt%3B.%26quot%3B%20arXiv.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.48550%5C%2FarXiv.2504.10281.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22preprint%22%2C%22title%22%3A%22Zero-shot%20Autonomous%20Microscopy%20for%20Scalable%20and%20Intelligent%20Characterization%20of%202D%20Materials%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jingyun%22%2C%22lastName%22%3A%22Yang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ruoyan%20Avery%22%2C%22lastName%22%3A%22Yin%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Chi%22%2C%22lastName%22%3A%22Jiang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Yuepeng%22%2C%22lastName%22%3A%22Hu%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Xiaokai%22%2C%22lastName%22%3A%22Zhu%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Xingjian%22%2C%22lastName%22%3A%22Hu%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Sutharsika%22%2C%22lastName%22%3A%22Kumar%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Xiao%22%2C%22lastName%22%3A%22Wang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Xiaohua%22%2C%22lastName%22%3A%22Zhai%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Keran%22%2C%22lastName%22%3A%22Rong%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Yunyue%22%2C%22lastName%22%3A%22Zhu%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Tianyi%22%2C%22lastName%22%3A%22Zhang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Zongyou%22%2C%22lastName%22%3A%22Yin%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jing%22%2C%22lastName%22%3A%22Kong%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Neil%20Zhenqiang%22%2C%22lastName%22%3A%22Gong%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Zhichu%22%2C%22lastName%22%3A%22Ren%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Haozhe%22%2C%22lastName%22%3A%22Wang%22%7D%5D%2C%22abstractNote%22%3A%22Characterization%20of%20atomic-scale%20materials%20traditionally%20requires%20human%20experts%20with%20months%20to%20years%20of%20specialized%20training.%20Even%20for%20trained%20human%20operators%2C%20accurate%20and%20reliable%20characterization%20remains%20challenging%20when%20examining%20newly%20discovered%20materials%20such%20as%20two-dimensional%20%282D%29%20structures.%20This%20bottleneck%20drives%20demand%20for%20fully%20autonomous%20experimentation%20systems%20capable%20of%20comprehending%20research%20objectives%20without%20requiring%20large%20training%20datasets.%20In%20this%20work%2C%20we%20present%20ATOMIC%20%28Autonomous%20Technology%20for%20Optical%20Microscopy%20%26amp%3B%20Intelligent%20Characterization%29%2C%20an%20end-to-end%20framework%20that%20integrates%20foundation%20models%20to%20enable%20fully%20autonomous%2C%20zero-shot%20characterization%20of%202D%20materials.%20Our%20system%20integrates%20the%20vision%20foundation%20model%20%28i.e.%2C%20Segment%20Anything%20Model%29%2C%20large%20language%20models%20%28i.e.%2C%20ChatGPT%29%2C%20unsupervised%20clustering%2C%20and%20topological%20analysis%20to%20automate%20microscope%20control%2C%20sample%20scanning%2C%20image%20segmentation%2C%20and%20intelligent%20analysis%20through%20prompt%20engineering%2C%20eliminating%20the%20need%20for%20additional%20training.%20When%20analyzing%20typical%20MoS2%20samples%2C%20our%20approach%20achieves%2099.7%25%20segmentation%20accuracy%20for%20single%20layer%20identification%2C%20which%20is%20equivalent%20to%20that%20of%20human%20experts.%20In%20addition%2C%20the%20integrated%20model%20is%20able%20to%20detect%20grain%20boundary%20slits%20that%20are%20challenging%20to%20identify%20with%20human%20eyes.%20Furthermore%2C%20the%20system%20retains%20robust%20accuracy%20despite%20variable%20conditions%20including%20defocus%2C%20color%20temperature%20fluctuations%2C%20and%20exposure%20variations.%20It%20is%20applicable%20to%20a%20broad%20spectrum%20of%20common%202D%20materials-including%20graphene%2C%20MoS2%2C%20WSe2%2C%20SnSe-regardless%20of%20whether%20they%20were%20fabricated%20via%20chemical%20vapor%20deposition%20or%20mechanical%20exfoliation.%20This%20work%20represents%20the%20implementation%20of%20foundation%20models%20to%20achieve%20autonomous%20analysis%2C%20establishing%20a%20scalable%20and%20data-efficient%20characterization%20paradigm%20that%20fundamentally%20transforms%20the%20approach%20to%20nanoscale%20materials%20research.%22%2C%22genre%22%3A%22%22%2C%22repository%22%3A%22arXiv%22%2C%22archiveID%22%3A%22arXiv%3A2504.10281%22%2C%22date%22%3A%222025-04-14%22%2C%22DOI%22%3A%2210.48550%5C%2FarXiv.2504.10281%22%2C%22citationKey%22%3A%22%22%2C%22url%22%3A%22http%3A%5C%2F%5C%2Farxiv.org%5C%2Fabs%5C%2F2504.10281%22%2C%22language%22%3A%22%22%2C%22collections%22%3A%5B%22UQ77P7I5%22%5D%2C%22dateModified%22%3A%222025-04-29T14%3A01%3A27Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22CAZ4PPYG%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Friederich%20et%20al.%22%2C%22parsedDate%22%3A%222025-03-30%22%2C%22numChildren%22%3A1%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BFriederich%2C%20N.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttp%3A%5C%2F%5C%2Farxiv.org%5C%2Fabs%5C%2F2504.00047%26%23039%3B%26gt%3BEAP4EMSIG%20--%20Enhancing%20Event-Driven%20Microscopy%20for%20Microfluidic%20Single-Cell%20Analysis%26lt%3B%5C%2Fa%26gt%3B.%26quot%3B%20arXiv.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.48550%5C%2FarXiv.2504.00047.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22preprint%22%2C%22title%22%3A%22EAP4EMSIG%20--%20Enhancing%20Event-Driven%20Microscopy%20for%20Microfluidic%20Single-Cell%20Analysis%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Nils%22%2C%22lastName%22%3A%22Friederich%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Angelo%20Jovin%20Yamachui%22%2C%22lastName%22%3A%22Sitcheu%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Annika%22%2C%22lastName%22%3A%22Nassal%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Erenus%22%2C%22lastName%22%3A%22Yildiz%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Matthias%22%2C%22lastName%22%3A%22Pesch%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Maximilian%22%2C%22lastName%22%3A%22Beichter%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Lukas%22%2C%22lastName%22%3A%22Scholtes%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Bahar%22%2C%22lastName%22%3A%22Akbaba%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Thomas%22%2C%22lastName%22%3A%22Lautenschlager%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Oliver%22%2C%22lastName%22%3A%22Neumann%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Dietrich%22%2C%22lastName%22%3A%22Kohlheyer%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Hanno%22%2C%22lastName%22%3A%22Scharr%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Johannes%22%2C%22lastName%22%3A%22Seiffarth%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Katharina%22%2C%22lastName%22%3A%22N%5Cu00f6h%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ralf%22%2C%22lastName%22%3A%22Mikut%22%7D%5D%2C%22abstractNote%22%3A%22Microfluidic%20Live-Cell%20Imaging%20yields%20data%20on%20microbial%20cell%20factories.%20However%2C%20continuous%20acquisition%20is%20challenging%20as%20high-throughput%20experiments%20often%20lack%20realtime%20insights%2C%20delaying%20responses%20to%20stochastic%20events.%20We%20introduce%20three%20components%20in%20the%20Experiment%20Automation%20Pipeline%20for%20Event-Driven%20Microscopy%20to%20Smart%20Microfluidic%20Single-Cell%20Analysis%3A%20a%20fast%2C%20accurate%20Deep%20Learning%20autofocusing%20method%20predicting%20the%20focus%20offset%2C%20an%20evaluation%20of%20real-time%20segmentation%20methods%20and%20a%20realtime%20data%20analysis%20dashboard.%20Our%20autofocusing%20achieves%20a%20Mean%20Absolute%20Error%20of%200.0226%5C%5Ctextmu%20m%20with%20inference%20times%20below%2050~ms.%20Among%20eleven%20Deep%20Learning%20segmentation%20methods%2C%20Cellpose~3%20reached%20a%20Panoptic%20Quality%20of%2093.58%5C%5C%25%2C%20while%20a%20distance-based%20method%20is%20fastest%20%28121~ms%2C%20Panoptic%20Quality%2093.02%5C%5C%25%29.%20All%20six%20Deep%20Learning%20Foundation%20Models%20were%20unsuitable%20for%20real-time%20segmentation.%22%2C%22genre%22%3A%22%22%2C%22repository%22%3A%22arXiv%22%2C%22archiveID%22%3A%22arXiv%3A2504.00047%22%2C%22date%22%3A%222025-03-30%22%2C%22DOI%22%3A%2210.48550%5C%2FarXiv.2504.00047%22%2C%22citationKey%22%3A%22%22%2C%22url%22%3A%22http%3A%5C%2F%5C%2Farxiv.org%5C%2Fabs%5C%2F2504.00047%22%2C%22language%22%3A%22%22%2C%22collections%22%3A%5B%22UQ77P7I5%22%5D%2C%22dateModified%22%3A%222025-04-29T14%3A04%3A42Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Application%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22TCPZKI5S%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Kesavan%20and%20Bohra%22%2C%22parsedDate%22%3A%222025-03-04%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BKesavan%2C%20P.S.%20and%20Bohra%2C%20D.%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.biorxiv.org%5C%2Fcontent%5C%2F10.1101%5C%2F2025.02.25.639997v2%26%23039%3B%26gt%3Bdeepthought%3A%20domain%20driven%20design%20for%20microscopy%20with%20applications%20in%20DNA%20damage%20responses%26lt%3B%5C%2Fa%26gt%3B.%26quot%3B%20bioRxiv.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1101%5C%2F2025.02.25.639997.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22preprint%22%2C%22title%22%3A%22deepthought%3A%20domain%20driven%20design%20for%20microscopy%20with%20applications%20in%20DNA%20damage%20responses%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22P.%20S.%22%2C%22lastName%22%3A%22Kesavan%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Darshika%22%2C%22lastName%22%3A%22Bohra%22%7D%5D%2C%22abstractNote%22%3A%22Microscope%20systems%20have%20traditionally%20separated%20data%20acquisition%20and%20analysis%20into%20distinct%20phases%2C%20limiting%20real-time%20insights%20and%20experimental%20adaptability.%20While%20open-source%20tools%20exist%20for%20device%20control%2C%20there%20remains%20a%20need%20for%20integrated%20software%20frameworks%20that%20combine%20acquisition%20and%20analysis.%20Here%20we%20present%20deepthought%2C%20a%20framework%20built%20on%20domain-driven%20design%20principles%20that%20unifies%20microscope%20control%20and%20data%20analysis%20through%20a%20common%20domain%20language.%20This%20framework%20enables%20automated%20high-throughput%20imaging%20with%20real-time%20analysis%20capabilities%2C%20demonstrated%20through%20studies%20of%20DNA%20damage%20responses%20in%20live%20cells.%20By%20implementing%20deepthought%20on%20a%20standard%20wide-field%20microscope%2C%20we%20achieved%20sample%20sizes%20exceeding%2010%2C000%20cells%20per%20condition%20and%20identified%20rare%20cellular%20subpopulations%2C%20capabilities%20typically%20limited%20to%20specialized%20high-content%20systems.%20Our%20approach%20provides%20a%20foundation%20for%20developing%20context-aware%20microscopy%20software%20that%20can%20dynamically%20adjust%20experimental%20parameters%20based%20on%20real-time%20analysis%20of%20biological%20phenomena.%22%2C%22genre%22%3A%22%22%2C%22repository%22%3A%22bioRxiv%22%2C%22archiveID%22%3A%22%22%2C%22date%22%3A%222025-03-04%22%2C%22DOI%22%3A%2210.1101%5C%2F2025.02.25.639997%22%2C%22citationKey%22%3A%22%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.biorxiv.org%5C%2Fcontent%5C%2F10.1101%5C%2F2025.02.25.639997v2%22%2C%22language%22%3A%22en%22%2C%22collections%22%3A%5B%22UQ77P7I5%22%5D%2C%22dateModified%22%3A%222025-04-29T14%3A04%3A20Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Application%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22PA6N7DHN%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Daetwyler%20et%20al.%22%2C%22parsedDate%22%3A%222025-02-12%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BDaetwyler%2C%20S.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-025-02598-2%26%23039%3B%26gt%3BImaging%20of%20cellular%20dynamics%20from%20a%20whole%20organism%20to%20subcellular%20scale%20with%20self-driving%2C%20multiscale%20microscopy%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%20Methods%26lt%3B%5C%2Fi%26gt%3B%2C%20pp.%201%5Cu201310.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41592-025-02598-2.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Imaging%20of%20cellular%20dynamics%20from%20a%20whole%20organism%20to%20subcellular%20scale%20with%20self-driving%2C%20multiscale%20microscopy%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Stephan%22%2C%22lastName%22%3A%22Daetwyler%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Hanieh%22%2C%22lastName%22%3A%22Mazloom-Farsibaf%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Felix%20Y.%22%2C%22lastName%22%3A%22Zhou%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Dagan%22%2C%22lastName%22%3A%22Segal%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Etai%22%2C%22lastName%22%3A%22Sapoznik%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Bingying%22%2C%22lastName%22%3A%22Chen%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jill%20M.%22%2C%22lastName%22%3A%22Westcott%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Rolf%20A.%22%2C%22lastName%22%3A%22Brekken%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Gaudenz%22%2C%22lastName%22%3A%22Danuser%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Reto%22%2C%22lastName%22%3A%22Fiolka%22%7D%5D%2C%22abstractNote%22%3A%22Most%20biological%20processes%2C%20from%20development%20to%20pathogenesis%2C%20span%20multiple%20time%20and%20length%20scales.%20While%20light-sheet%20fluorescence%20microscopy%20has%20become%20a%20fast%20and%20efficient%20method%20for%20imaging%20organisms%2C%20cells%20and%20subcellular%20dynamics%2C%20simultaneous%20observations%20across%20all%20these%20scales%20have%20remained%20challenging.%20Moreover%2C%20continuous%20high-resolution%20imaging%20inside%20living%20organisms%20has%20mostly%20been%20limited%20to%20a%20few%20hours%2C%20as%20regions%20of%20interest%20quickly%20move%20out%20of%20view%20due%20to%20sample%20movement%20and%20growth.%20Here%2C%20we%20present%20a%20self-driving%2C%20multiresolution%20light-sheet%20microscope%20platform%20controlled%20by%20custom%20Python-based%20software%2C%20to%20simultaneously%20observe%20and%20quantify%20subcellular%20dynamics%20in%20the%20context%20of%20entire%20organisms%20in%20vitro%20and%20in%20vivo%20over%20hours%20of%20imaging.%20We%20apply%20the%20platform%20to%20the%20study%20of%20developmental%20processes%2C%20cancer%20invasion%20and%20metastasis%2C%20and%20we%20provide%20quantitative%20multiscale%20analysis%20of%20immune%5Cu2013cancer%20cell%20interactions%20in%20zebrafish%20xenografts.%22%2C%22date%22%3A%222025-02-12%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs41592-025-02598-2%22%2C%22ISSN%22%3A%221548-7105%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-025-02598-2%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222025-04-29T14%3A03%3A45Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Application%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22FPEQIBFZ%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Thomas-%20-Chemin%20et%20al.%22%2C%22parsedDate%22%3A%222025-02-11%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BThomas-%20-Chemin%2C%20O.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fpubs.acs.org%5C%2Fdoi%5C%2F10.1021%5C%2Facsnano.4c07729%26%23039%3B%26gt%3BAdvancing%20High-Throughput%20Cellular%20Atomic%20Force%20Microscopy%20with%20Automation%20and%20Artificial%20Intelligence%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BACS%20Nano%26lt%3B%5C%2Fi%26gt%3B%2C%2019%285%29%2C%20pp.%205045%5Cu20135062.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1021%5C%2Facsnano.4c07729.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Advancing%20High-Throughput%20Cellular%20Atomic%20Force%20Microscopy%20with%20Automation%20and%20Artificial%20Intelligence%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Oph%5Cu00e9lie%22%2C%22lastName%22%3A%22Thomas-%20-Chemin%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22S%5Cu00e9bastien%22%2C%22lastName%22%3A%22Janel%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Zeyd%22%2C%22lastName%22%3A%22Boumehdi%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Child%5Cu00e9rick%22%2C%22lastName%22%3A%22S%5Cu00e9verac%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Emmanuelle%22%2C%22lastName%22%3A%22Trevisiol%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Etienne%22%2C%22lastName%22%3A%22Dague%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Vincent%22%2C%22lastName%22%3A%22Dupr%5Cu00e9s%22%7D%5D%2C%22abstractNote%22%3A%22%22%2C%22date%22%3A%222025-02-11%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1021%5C%2Facsnano.4c07729%22%2C%22ISSN%22%3A%221936-0851%2C%201936-086X%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fpubs.acs.org%5C%2Fdoi%5C%2F10.1021%5C%2Facsnano.4c07729%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222025-04-29T14%3A01%3A41Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%225CPC59Y9%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Kim%20and%20Lee%22%2C%22parsedDate%22%3A%222025%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BKim%2C%20J.%20and%20Lee%2C%20S.J.%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Flinkinghub.elsevier.com%5C%2Fretrieve%5C%2Fpii%5C%2FS014381662500079X%26%23039%3B%26gt%3BAI-based%20autofocusing%20of%20red%20blood%20cells%20in%20digital%20in-line%20holographic%20microscopy%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BOptics%20and%20Lasers%20in%20Engineering%26lt%3B%5C%2Fi%26gt%3B%2C%20188%2C%20p.%20108892.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1016%5C%2Fj.optlaseng.2025.108892.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22AI-based%20autofocusing%20of%20red%20blood%20cells%20in%20digital%20in-line%20holographic%20microscopy%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jihwan%22%2C%22lastName%22%3A%22Kim%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Sang%20Joon%22%2C%22lastName%22%3A%22Lee%22%7D%5D%2C%22abstractNote%22%3A%22%22%2C%22date%22%3A%2205%5C%2F2025%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1016%5C%2Fj.optlaseng.2025.108892%22%2C%22ISSN%22%3A%2201438166%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Flinkinghub.elsevier.com%5C%2Fretrieve%5C%2Fpii%5C%2FS014381662500079X%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222025-04-29T14%3A03%3A11Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Application%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22Y6KMZ8RU%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Berrevoets%20et%20al.%22%2C%22parsedDate%22%3A%222024-11-29%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BBerrevoets%2C%20E.S.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282024%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.biorxiv.org%5C%2Fcontent%5C%2F10.1101%5C%2F2024.11.28.625872v1%26%23039%3B%26gt%3BSmart%203D%20super-resolution%20microscopy%20reveals%20the%20architecture%20of%20the%20RNA%20scaffold%20in%20a%20nuclear%20body%26lt%3B%5C%2Fa%26gt%3B.%26quot%3B%20bioRxiv.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1101%5C%2F2024.11.28.625872.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22preprint%22%2C%22title%22%3A%22Smart%203D%20super-resolution%20microscopy%20reveals%20the%20architecture%20of%20the%20RNA%20scaffold%20in%20a%20nuclear%20body%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Enya%20S.%22%2C%22lastName%22%3A%22Berrevoets%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Laurell%20F.%22%2C%22lastName%22%3A%22Kessler%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ashwin%22%2C%22lastName%22%3A%22Balakrishnan%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Michaela%22%2C%22lastName%22%3A%22M%5Cu00fcller-McNicoll%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Bernd%22%2C%22lastName%22%3A%22Rieger%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Sjoerd%22%2C%22lastName%22%3A%22Stallinga%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Mike%22%2C%22lastName%22%3A%22Heilemann%22%7D%5D%2C%22abstractNote%22%3A%22Small%20subcellular%20organelles%20orchestrate%20key%20cellular%20functions.%20How%20biomolecules%20are%20spatially%20organized%20within%20these%20assemblies%20is%20poorly%20understood.%20Here%2C%20we%20report%20an%20automated%20super-%20resolution%20imaging%20and%20analysis%20workflow%20that%20integrates%20confocal%20microscopy%2C%20morphological%20object%20screening%2C%20targeted%203D%20super-resolution%20STED%20microscopy%20and%20quantitative%20image%20analysis.%20Using%20this%20smart%20microscopy%20workflow%2C%20we%20targeted%20the%203D%20organization%20of%20an%20architectural%20RNA%20that%20constitutes%20the%20structural%20backbone%20of%20paraspeckles%2C%20a%20membraneless%20nuclear%20organelle.%20Using%20site-specific%20labeling%2C%20morphological%20sorting%20and%20particle%20averaging%2C%20we%20reconstructed%20the%20morphological%20space%20of%20paraspeckles%20along%20their%20development%20cycle%20from%20over%2010%2C000%20individual%20particles.%20Applying%20spherical%20harmonics%20analysis%2C%20we%20report%20so-far%20unknown%20heterotypes%20of%20RNA%20organization.%20By%20integrating%20multi-%20positional%20labeling%2C%20we%20determined%20the%20coarse%20conformation%20of%20the%20RNA%20within%20the%20organelle%20and%20found%20the%203%5Cu2019%20end%20forming%20a%20loop-like%20structure%20at%20the%20surface%20of%20the%20paraspeckle.%20Our%20study%20reveals%20key%20structural%20features%20of%20nuclear%20paraspeckle%20structure%20and%20growth%2C%20as%20well%20as%20on%20the%20molecular%20organization%20of%20the%20scaffold%20RNA.%22%2C%22genre%22%3A%22%22%2C%22repository%22%3A%22bioRxiv%22%2C%22archiveID%22%3A%22%22%2C%22date%22%3A%222024-11-29%22%2C%22DOI%22%3A%2210.1101%5C%2F2024.11.28.625872%22%2C%22citationKey%22%3A%22%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.biorxiv.org%5C%2Fcontent%5C%2F10.1101%5C%2F2024.11.28.625872v1%22%2C%22language%22%3A%22en%22%2C%22collections%22%3A%5B%22UQ77P7I5%22%5D%2C%22dateModified%22%3A%222024-12-03T09%3A48%3A43Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Application%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22DFIINUT8%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Brooks%20et%20al.%22%2C%22parsedDate%22%3A%222024-10-19%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BBrooks%2C%20S.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282024%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.biorxiv.org%5C%2Fcontent%5C%2F10.1101%5C%2F2024.10.17.618897v1%26%23039%3B%26gt%3BCelFDrive%3A%20Artificial%20Intelligence%20assisted%20microscopy%20for%20automated%20detection%20of%20rare%20events%26lt%3B%5C%2Fa%26gt%3B.%26quot%3B%20bioRxiv.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1101%5C%2F2024.10.17.618897.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22preprint%22%2C%22title%22%3A%22CelFDrive%3A%20Artificial%20Intelligence%20assisted%20microscopy%20for%20automated%20detection%20of%20rare%20events%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Scott%22%2C%22lastName%22%3A%22Brooks%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Sara%22%2C%22lastName%22%3A%22Toral-P%5Cu00e9rez%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22David%20S.%22%2C%22lastName%22%3A%22Corcoran%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Karl%22%2C%22lastName%22%3A%22Kilborn%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Brian%22%2C%22lastName%22%3A%22Bodensteiner%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Hella%22%2C%22lastName%22%3A%22Baumann%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Nigel%20J.%22%2C%22lastName%22%3A%22Burroughs%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Andrew%20D.%22%2C%22lastName%22%3A%22McAinsh%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Till%22%2C%22lastName%22%3A%22Bretschneider%22%7D%5D%2C%22abstractNote%22%3A%221%20Abstract%5CnSummary%20CelFDrive%20automates%20high-resolution%203D%20imaging%20cells%20of%20interest%20across%20a%20variety%20of%20fluorescence%20microscopes%2C%20integrating%20deep%20learning%20cell%20classification%20from%20auxiliary%20low%20resolution%20widefield%20images.%20CelFDrive%20enables%20efficient%20detection%20of%20rare%20events%20in%20large%20cell%20populations%2C%20such%20as%20the%20onset%20of%20cell%20division%2C%20and%20subsequent%20rapid%20switching%20to%203D%20imaging%20modes%2C%20increasing%20the%20speed%20for%20finding%20cells%20of%20interest%20by%20an%20order%20of%20magnitude.%5CnAvailability%20and%20Implementation%20CelFDrive%20is%20available%20freely%20for%20academic%20purposes%20at%20the%20CelFDrive%20GitHub%20repository.%20and%20can%20be%20installed%20on%20Windows%2C%20macOS%20or%20Linux-based%20machines%20with%20relevant%20conda%20environments%20%5B1%5D.%20To%20interact%20with%20microscopy%20hardware%20requires%20additional%20software%3B%20we%20use%20SlideBook%20software%20from%20Intelligent%20Imaging%20Innovations%20%283i%29%2C%20but%20CelFDrive%20can%20be%20deployed%20with%20any%20microscope%20control%20software%20that%20can%20interact%20with%20a%20Python%20environment.%20Graphical%20Processing%20Units%20%28GPUs%29%20are%20recommended%20to%20increase%20the%20speed%20of%20application%20but%20are%20not%20required.%20On%203i%20systems%20the%20software%20can%20be%20deployed%20with%20a%20range%20of%20microscopes%20including%20their%20Lattice%20LightSheet%20microscope%20%28LLSM%29%20and%20spinning%20disk%20confocal%20%28SDC%29.%5CnContact%20s.brooks.2%7Bat%7Dwarwick.ac.uk%22%2C%22genre%22%3A%22%22%2C%22repository%22%3A%22bioRxiv%22%2C%22archiveID%22%3A%22%22%2C%22date%22%3A%222024-10-19%22%2C%22DOI%22%3A%2210.1101%5C%2F2024.10.17.618897%22%2C%22citationKey%22%3A%22%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.biorxiv.org%5C%2Fcontent%5C%2F10.1101%5C%2F2024.10.17.618897v1%22%2C%22language%22%3A%22en%22%2C%22collections%22%3A%5B%22UQ77P7I5%22%5D%2C%22dateModified%22%3A%222025-04-29T14%3A04%3A51Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22BLTPJ7K8%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Roos%20et%20al.%22%2C%22parsedDate%22%3A%222024-09-18%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BRoos%2C%20J.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282024%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-024-02404-5%26%23039%3B%26gt%3BArkitekt%3A%20streaming%20analysis%20and%20real-time%20workflows%20for%20microscopy%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%20Methods%26lt%3B%5C%2Fi%26gt%3B%20%5BPreprint%5D.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41592-024-02404-5.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Arkitekt%3A%20streaming%20analysis%20and%20real-time%20workflows%20for%20microscopy%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Johannes%22%2C%22lastName%22%3A%22Roos%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22St%5Cu00e9phane%22%2C%22lastName%22%3A%22Bancelin%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Tom%22%2C%22lastName%22%3A%22Delaire%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Alexander%22%2C%22lastName%22%3A%22Wilhelmi%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Florian%22%2C%22lastName%22%3A%22Levet%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Maren%22%2C%22lastName%22%3A%22Engelhardt%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Virgile%22%2C%22lastName%22%3A%22Viasnoff%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22R%5Cu00e9mi%22%2C%22lastName%22%3A%22Galland%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22U.%20Valentin%22%2C%22lastName%22%3A%22N%5Cu00e4gerl%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jean-Baptiste%22%2C%22lastName%22%3A%22Sibarita%22%7D%5D%2C%22abstractNote%22%3A%22%22%2C%22date%22%3A%222024-09-18%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs41592-024-02404-5%22%2C%22ISSN%22%3A%221548-7091%2C%201548-7105%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-024-02404-5%22%2C%22collections%22%3A%5B%5D%2C%22dateModified%22%3A%222025-05-07T13%3A38%3A10Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22Y63EJFYD%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Roos%20et%20al.%22%2C%22parsedDate%22%3A%222024-09-18%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BRoos%2C%20J.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282024%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-024-02404-5%26%23039%3B%26gt%3BArkitekt%3A%20streaming%20analysis%20and%20real-time%20workflows%20for%20microscopy%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%20Methods%26lt%3B%5C%2Fi%26gt%3B%20%5BPreprint%5D.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41592-024-02404-5.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Arkitekt%3A%20streaming%20analysis%20and%20real-time%20workflows%20for%20microscopy%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Johannes%22%2C%22lastName%22%3A%22Roos%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22St%5Cu00e9phane%22%2C%22lastName%22%3A%22Bancelin%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Tom%22%2C%22lastName%22%3A%22Delaire%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Alexander%22%2C%22lastName%22%3A%22Wilhelmi%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Florian%22%2C%22lastName%22%3A%22Levet%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Maren%22%2C%22lastName%22%3A%22Engelhardt%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Virgile%22%2C%22lastName%22%3A%22Viasnoff%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22R%5Cu00e9mi%22%2C%22lastName%22%3A%22Galland%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22U.%20Valentin%22%2C%22lastName%22%3A%22N%5Cu00e4gerl%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jean-Baptiste%22%2C%22lastName%22%3A%22Sibarita%22%7D%5D%2C%22abstractNote%22%3A%22%22%2C%22date%22%3A%222024-09-18%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs41592-024-02404-5%22%2C%22ISSN%22%3A%221548-7091%2C%201548-7105%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-024-02404-5%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222024-09-20T10%3A51%3A09Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%225Z8J3KML%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Marin%20et%20al.%22%2C%22parsedDate%22%3A%222024-09-11%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BMarin%2C%20Z.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282024%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41592-024-02413-4%26%23039%3B%26gt%3BNavigate%3A%20an%20open-source%20platform%20for%20smart%20light-sheet%20microscopy%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%20Methods%26lt%3B%5C%2Fi%26gt%3B%20%5BPreprint%5D.%20Available%20at%3A%20%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Navigate%3A%20an%20open-source%20platform%20for%20smart%20light-sheet%20microscopy%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Zach%22%2C%22lastName%22%3A%22Marin%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Xiaoding%22%2C%22lastName%22%3A%22Wang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Dax%20W.%22%2C%22lastName%22%3A%22Collison%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Conor%22%2C%22lastName%22%3A%22McFadden%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jinlong%22%2C%22lastName%22%3A%22Lin%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Hazel%20M.%22%2C%22lastName%22%3A%22Borges%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Bingying%22%2C%22lastName%22%3A%22Chen%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Dushyant%22%2C%22lastName%22%3A%22Mehra%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Qionghua%22%2C%22lastName%22%3A%22Shen%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Seweryn%22%2C%22lastName%22%3A%22Ga%5Cu0142ecki%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Stephan%22%2C%22lastName%22%3A%22Daetwyler%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Steven%20J.%22%2C%22lastName%22%3A%22Sheppard%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Phu%22%2C%22lastName%22%3A%22Thien%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Baylee%20A.%22%2C%22lastName%22%3A%22Porter%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Suzanne%20D.%22%2C%22lastName%22%3A%22Conzen%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Douglas%20P.%22%2C%22lastName%22%3A%22Shepherd%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Reto%22%2C%22lastName%22%3A%22Fiolka%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Kevin%20M.%22%2C%22lastName%22%3A%22Dean%22%7D%5D%2C%22abstractNote%22%3A%22%22%2C%22date%22%3A%222024-09-11%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs41592-024-02413-4%22%2C%22ISSN%22%3A%221548-7091%2C%201548-7105%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41592-024-02413-4%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222025-04-29T13%3A11%3A58Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22C2H2QVG6%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Peters%20et%20al.%22%2C%22parsedDate%22%3A%222024-08-02%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BPeters%2C%20J.J.P.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282024%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.science.org%5C%2Fdoi%5C%2F10.1126%5C%2Fscience.ado8579%26%23039%3B%26gt%3BEvent-responsive%20scanning%20transmission%20electron%20microscopy%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BScience%26lt%3B%5C%2Fi%26gt%3B%2C%20385%286708%29%2C%20pp.%20549%5Cu2013553.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1126%5C%2Fscience.ado8579.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Event-responsive%20scanning%20transmission%20electron%20microscopy%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jonathan%20J.%20P.%22%2C%22lastName%22%3A%22Peters%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Bryan%20W.%22%2C%22lastName%22%3A%22Reed%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Yu%22%2C%22lastName%22%3A%22Jimbo%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Kanako%22%2C%22lastName%22%3A%22Noguchi%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Karin%20H.%22%2C%22lastName%22%3A%22M%5Cu00fcller%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Alexandra%22%2C%22lastName%22%3A%22Porter%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Daniel%20J.%22%2C%22lastName%22%3A%22Masiel%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Lewys%22%2C%22lastName%22%3A%22Jones%22%7D%5D%2C%22abstractNote%22%3A%22An%20ever-present%20limitation%20of%20transmission%20electron%20microscopy%20is%20the%20damage%20caused%20by%20high-energy%20electrons%20interacting%20with%20any%20sample.%20By%20reconsidering%20the%20fundamentals%20of%20imaging%2C%20we%20demonstrate%20an%20event-responsive%20approach%20to%20electron%20microscopy%20that%20delivers%20more%20information%20about%20the%20sample%20for%20a%20given%20beam%20current.%20Measuring%20the%20time%20to%20achieve%20an%20electron%20count%20threshold%20rather%20than%20waiting%20a%20predefined%20constant%20time%20improves%20the%20information%20obtained%20per%20electron.%20The%20microscope%20was%20made%20to%20respond%20to%20these%20events%20by%20blanking%20the%20beam%2C%20thus%20reducing%20the%20overall%20dose%20required.%20This%20approach%20automatically%20apportions%20dose%20to%20achieve%20a%20given%20signal-to-noise%20ratio%20in%20each%20pixel%2C%20eliminating%20excess%20dose%20that%20is%20associated%20with%20diminishing%20returns%20of%20information.%20We%20demonstrate%20the%20wide%20applicability%20of%20our%20approach%20to%20beam-sensitive%20materials%20by%20imaging%20biological%20tissue%20and%20zeolite.%22%2C%22date%22%3A%222024-08-02%22%2C%22language%22%3A%22%22%2C%22DOI%22%3A%2210.1126%5C%2Fscience.ado8579%22%2C%22ISSN%22%3A%22%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.science.org%5C%2Fdoi%5C%2F10.1126%5C%2Fscience.ado8579%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222025-04-29T14%3A01%3A58Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Application%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22KYCN7A4V%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Pomp%20et%20al.%22%2C%22parsedDate%22%3A%222024-03-21%22%2C%22numChildren%22%3A1%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BPomp%2C%20W.%2C%20Meeussen%2C%20J.V.W.%20and%20Lenstra%2C%20T.L.%20%282024%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.cell.com%5C%2Fmolecular-cell%5C%2Fabstract%5C%2FS1097-2765%2824%2900056-X%26%23039%3B%26gt%3BTranscription%20factor%20exchange%20enables%20prolonged%20transcriptional%20bursts%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BMolecular%20Cell%26lt%3B%5C%2Fi%26gt%3B%2C%2084%286%29%2C%20pp.%201036-1048.e9.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1016%5C%2Fj.molcel.2024.01.020.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Transcription%20factor%20exchange%20enables%20prolonged%20transcriptional%20bursts%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Wim%22%2C%22lastName%22%3A%22Pomp%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Joseph%20V.%20W.%22%2C%22lastName%22%3A%22Meeussen%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Tineke%20L.%22%2C%22lastName%22%3A%22Lenstra%22%7D%5D%2C%22abstractNote%22%3A%22%26lt%3Bh2%26gt%3BSummary%26lt%3B%5C%2Fh2%26gt%3B%26lt%3Bp%26gt%3BSingle-molecule%20imaging%20inside%20living%20cells%20has%20revealed%20that%20transcription%20factors%20%28TFs%29%20bind%20to%20DNA%20transiently%2C%20but%20a%20long-standing%20question%20is%20how%20this%20transient%20binding%20is%20related%20to%20transcription%20activation.%20Here%2C%20we%20devised%20a%20microscopy%20method%20to%20simultaneously%20measure%20transient%20TF%20binding%20at%20a%20single%20locus%20and%20the%20effect%20of%20these%20binding%20events%20on%20transcription.%20We%20show%20that%20DNA%20binding%20of%20the%20yeast%20TF%20Gal4%20activates%20transcription%20of%20a%20target%20gene%20within%20a%20few%20seconds%2C%20with%20at%20least%20%5Cu223c20%25%20efficiency%20and%20with%20a%20high%20initiation%20rate%20of%20%5Cu223c1%20RNA%5C%2Fs.%20Gal4%20DNA%20dissociation%20decreases%20transcription%20rapidly.%20Moreover%2C%20at%20a%20gene%20with%20multiple%20binding%20sites%2C%20individual%20Gal4%20molecules%20only%20rarely%20stay%20bound%20throughout%20the%20entire%20burst%20but%20instead%20frequently%20exchange%20during%20a%20burst%20to%20increase%20the%20transcriptional%20burst%20duration.%20Our%20results%20suggest%20a%20mechanism%20for%20enhancer%20regulation%20in%20more%20complex%20eukaryotes%2C%20where%20TF%20cooperativity%20and%20exchange%20enable%20robust%20and%20responsive%20transcription%20regulation.%26lt%3B%5C%2Fp%26gt%3B%22%2C%22date%22%3A%222024%5C%2F03%5C%2F21%22%2C%22language%22%3A%22English%22%2C%22DOI%22%3A%2210.1016%5C%2Fj.molcel.2024.01.020%22%2C%22ISSN%22%3A%221097-2765%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.cell.com%5C%2Fmolecular-cell%5C%2Fabstract%5C%2FS1097-2765%2824%2900056-X%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222024-05-24T09%3A49%3A39Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Application%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22QUM2DVR6%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Morgado%20et%20al.%22%2C%22parsedDate%22%3A%222024-03-06%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BMorgado%2C%20L.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282024%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fonlinelibrary.wiley.com%5C%2Fdoi%5C%2F10.1111%5C%2Fjmi.13282%26%23039%3B%26gt%3BThe%20rise%20of%20data%5Cu2010driven%20microscopy%20powered%20by%20machine%20learning%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BJournal%20of%20Microscopy%26lt%3B%5C%2Fi%26gt%3B%2C%20p.%20jmi.13282.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1111%5C%2Fjmi.13282.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22The%20rise%20of%20data%5Cu2010driven%20microscopy%20powered%20by%20machine%20learning%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Leonor%22%2C%22lastName%22%3A%22Morgado%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Estibaliz%22%2C%22lastName%22%3A%22G%5Cu00f3mez%5Cu2010de%5Cu2010Mariscal%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Hannah%20S.%22%2C%22lastName%22%3A%22Heil%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ricardo%22%2C%22lastName%22%3A%22Henriques%22%7D%5D%2C%22abstractNote%22%3A%22Abstract%5Cn%20%20%20%20%20%20%20%20%20%20%20%20Optical%20microscopy%20is%20an%20indispensable%20tool%20in%20life%20sciences%20research%2C%20but%20conventional%20techniques%20require%20compromises%20between%20imaging%20parameters%20like%20speed%2C%20resolution%2C%20field%20of%20view%20and%20phototoxicity.%20To%20overcome%20these%20limitations%2C%20data%5Cu2010driven%20microscopes%20incorporate%20feedback%20loops%20between%20data%20acquisition%20and%20analysis.%20This%20review%20overviews%20how%20machine%20learning%20enables%20automated%20image%20analysis%20to%20optimise%20microscopy%20in%20real%20time.%20We%20first%20introduce%20key%20data%5Cu2010driven%20microscopy%20concepts%20and%20machine%20learning%20methods%20relevant%20to%20microscopy%20image%20analysis.%20Subsequently%2C%20we%20highlight%20pioneering%20works%20and%20recent%20advances%20in%20integrating%20machine%20learning%20into%20microscopy%20acquisition%20workflows%2C%20including%20optimising%20illumination%2C%20switching%20modalities%20and%20acquisition%20rates%2C%20and%20triggering%20targeted%20experiments.%20We%20then%20discuss%20the%20remaining%20challenges%20and%20future%20outlook.%20Overall%2C%20intelligent%20microscopes%20that%20can%20sense%2C%20analyse%20and%20adapt%20promise%20to%20transform%20optical%20imaging%20by%20opening%20new%20experimental%20possibilities.%22%2C%22date%22%3A%222024-03-06%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1111%5C%2Fjmi.13282%22%2C%22ISSN%22%3A%220022-2720%2C%201365-2818%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fonlinelibrary.wiley.com%5C%2Fdoi%5C%2F10.1111%5C%2Fjmi.13282%22%2C%22collections%22%3A%5B%224KIL6FA6%22%5D%2C%22dateModified%22%3A%222024-04-23T10%3A06%3A06Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22FLSWKSGB%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Shroff%20et%20al.%22%2C%22parsedDate%22%3A%222024-01-02%22%2C%22numChildren%22%3A1%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BShroff%2C%20H.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282024%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttp%3A%5C%2F%5C%2Farxiv.org%5C%2Fabs%5C%2F2401.01438%26%23039%3B%26gt%3BLive-cell%20imaging%20powered%20by%20computation%26lt%3B%5C%2Fa%26gt%3B.%26quot%3B%20arXiv.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.48550%5C%2FarXiv.2401.01438.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22preprint%22%2C%22title%22%3A%22Live-cell%20imaging%20powered%20by%20computation%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Hari%22%2C%22lastName%22%3A%22Shroff%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ilaria%22%2C%22lastName%22%3A%22Testa%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Florian%22%2C%22lastName%22%3A%22Jug%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Suliana%22%2C%22lastName%22%3A%22Manley%22%7D%5D%2C%22abstractNote%22%3A%22The%20proliferation%20of%20microscopy%20methods%20for%20live-cell%20imaging%20offers%20many%20new%20possibilities%20for%20users%20but%20can%20also%20be%20challenging%20to%20navigate.%20We%20focus%20here%20on%20computational%20methods%20that%20promise%20to%20boost%20live-cell%20fluorescence%20microscopy%2C%20where%20intra-cellular%20dynamics%20and%20cell%20viability%20constrain%20measurements%20considerably.%20Considering%20the%20tradeoffs%20between%20signal-to-noise%20ratio%20%28SNR%29%2C%20spatial%20resolution%2C%20temporal%20resolution%2C%20and%20multi-color%20and%20multi-channel%20capacity%2C%20we%20review%20computational%20methods%20that%20can%20be%20layered%20on%20top%20of%20commonly%20used%20existing%20microscopies%2C%20as%20well%20as%20hybrid%20methods%20that%20integrate%20computation%20and%20microscope%20hardware.%22%2C%22genre%22%3A%22%22%2C%22repository%22%3A%22arXiv%22%2C%22archiveID%22%3A%22arXiv%3A2401.01438%22%2C%22date%22%3A%222024-01-02%22%2C%22DOI%22%3A%2210.48550%5C%2FarXiv.2401.01438%22%2C%22citationKey%22%3A%22%22%2C%22url%22%3A%22http%3A%5C%2F%5C%2Farxiv.org%5C%2Fabs%5C%2F2401.01438%22%2C%22language%22%3A%22%22%2C%22collections%22%3A%5B%224KIL6FA6%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A10%3A28Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Data%20Analysis%22%7D%5D%7D%7D%2C%7B%22key%22%3A%225Q2SZY6M%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Shi%20et%20al.%22%2C%22parsedDate%22%3A%222024-01-02%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BShi%2C%20Y.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282024%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-023-02126-0%26%23039%3B%26gt%3BSmart%20lattice%20light-sheet%20microscopy%20for%20imaging%20rare%20and%20complex%20cellular%20events%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%20Methods%26lt%3B%5C%2Fi%26gt%3B%2C%20pp.%201%5Cu201310.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41592-023-02126-0.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Smart%20lattice%20light-sheet%20microscopy%20for%20imaging%20rare%20and%20complex%20cellular%20events%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Yu%22%2C%22lastName%22%3A%22Shi%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jimmy%20S.%22%2C%22lastName%22%3A%22Tabet%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Daniel%20E.%22%2C%22lastName%22%3A%22Milkie%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Timothy%20A.%22%2C%22lastName%22%3A%22Daugird%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Chelsea%20Q.%22%2C%22lastName%22%3A%22Yang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Alex%20T.%22%2C%22lastName%22%3A%22Ritter%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Andrea%22%2C%22lastName%22%3A%22Giovannucci%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Wesley%20R.%22%2C%22lastName%22%3A%22Legant%22%7D%5D%2C%22abstractNote%22%3A%22Light-sheet%20microscopes%20enable%20rapid%20high-resolution%20imaging%20of%20biological%20specimens%3B%20however%2C%20biological%20processes%20span%20spatiotemporal%20scales.%20Moreover%2C%20long-term%20phenotypes%20are%20often%20instigated%20by%20rare%20or%20fleeting%20biological%20events%20that%20are%20difficult%20to%20capture%20with%20a%20single%20imaging%20modality.%20Here%2C%20to%20overcome%20this%20limitation%2C%20we%20present%20smartLLSM%2C%20a%20microscope%20that%20incorporates%20artificial%20intelligence-based%20instrument%20control%20to%20autonomously%20switch%20between%20epifluorescent%20inverted%20imaging%20and%20lattice%20light-sheet%20microscopy%20%28LLSM%29.%20We%20apply%20this%20approach%20to%20two%20unique%20processes%3A%20cell%20division%20and%20immune%20synapse%20formation.%20In%20each%20context%2C%20smartLLSM%20provides%20population-level%20statistics%20across%20thousands%20of%20cells%20and%20autonomously%20captures%20multicolor%20three-dimensional%20datasets%20or%20four-dimensional%20time-lapse%20movies%20of%20rare%20events%20at%20rates%20that%20dramatically%20exceed%20human%20capabilities.%20From%20this%2C%20we%20quantify%20the%20effects%20of%20Taxol%20dose%20on%20spindle%20structure%20and%20kinetochore%20dynamics%20in%20dividing%20cells%20and%20of%20antigen%20strength%20on%20cytotoxic%20T%20lymphocyte%20engagement%20and%20lytic%20granule%20polarization%20at%20the%20immune%20synapse.%20Overall%2C%20smartLLSM%20efficiently%20detects%20rare%20events%20within%20heterogeneous%20cell%20populations%20and%20records%20these%20processes%20with%20high%20spatiotemporal%20four-dimensional%20imaging%20over%20statistically%20significant%20replicates.%22%2C%22date%22%3A%222024-01-02%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs41592-023-02126-0%22%2C%22ISSN%22%3A%221548-7105%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-023-02126-0%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A04%3A12Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22FQ7UJRZE%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Lei%20et%20al.%22%2C%22parsedDate%22%3A%222023-12-05%22%2C%22numChildren%22%3A1%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BLei%2C%20W.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282023%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttp%3A%5C%2F%5C%2Farxiv.org%5C%2Fabs%5C%2F2310.18351%26%23039%3B%26gt%3BBioImage.IO%20Chatbot%3A%20A%20Personalized%20Assistant%20for%20BioImage%20Analysis%20Augmented%20by%20Community%20Knowledge%20Base%26lt%3B%5C%2Fa%26gt%3B.%26quot%3B%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.5281%5C%2Fzenodo.10032227.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22BioImage.IO%20Chatbot%3A%20A%20Personalized%20Assistant%20for%20BioImage%20Analysis%20Augmented%20by%20Community%20Knowledge%20Base%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Wanlu%22%2C%22lastName%22%3A%22Lei%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Caterina%22%2C%22lastName%22%3A%22Fuster-Barcel%5Cu00f3%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Arrate%22%2C%22lastName%22%3A%22Mu%5Cu00f1oz-Barrutia%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Wei%22%2C%22lastName%22%3A%22Ouyang%22%7D%5D%2C%22abstractNote%22%3A%22The%20rapidly%20expanding%20landscape%20of%20bioimage%20analysis%20tools%20presents%20a%20navigational%20challenge%20for%20both%20experts%20and%20newcomers.%20Traditional%20search%20methods%20often%20fall%20short%20in%20assisting%20users%20in%20this%20complex%20environment.%20To%20address%20this%2C%20we%20introduce%20the%20BioImage%24.%24IO%20Chatbot%2C%20an%20AI-driven%20conversational%20assistant%20tailored%20for%20the%20bioimage%20community.%20Built%20upon%20large%20language%20models%2C%20this%20chatbot%20provides%20personalized%2C%20context-aware%20answers%20by%20aggregating%20and%20interpreting%20information%20from%20diverse%20databases%2C%20tool-specific%20documentation%2C%20and%20structured%20data%20sources.%20Enhanced%20by%20a%20community-contributed%20knowledge%20base%20and%20fine-tuned%20retrieval%20methods%2C%20the%20BioImage%24.%24IO%20Chatbot%20offers%20not%20just%20a%20personalized%20interaction%20but%20also%20a%20knowledge-enriched%2C%20context-aware%20experience.%20It%20fundamentally%20transforms%20the%20way%20biologists%2C%20bioimage%20analysts%2C%20and%20developers%20navigate%20and%20utilize%20advanced%20bioimage%20analysis%20tools%2C%20setting%20a%20new%20standard%20for%20community-driven%2C%20accessible%20scientific%20research.%22%2C%22date%22%3A%222023-12-05%22%2C%22language%22%3A%22%22%2C%22DOI%22%3A%2210.5281%5C%2Fzenodo.10032227%22%2C%22ISSN%22%3A%22%22%2C%22url%22%3A%22http%3A%5C%2F%5C%2Farxiv.org%5C%2Fabs%5C%2F2310.18351%22%2C%22collections%22%3A%5B%22UQ77P7I5%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A01%3A15Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Data%20Analysis%22%7D%2C%7B%22tag%22%3A%22Data%20handling%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22PAJYPWZV%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Pylv%5Cu00e4n%5Cu00e4inen%20et%20al.%22%2C%22parsedDate%22%3A%222023-12-01%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BPylv%5Cu00e4n%5Cu00e4inen%2C%20J.W.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282023%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.sciencedirect.com%5C%2Fscience%5C%2Farticle%5C%2Fpii%5C%2FS0955067423001205%26%23039%3B%26gt%3BLive-cell%20imaging%20in%20the%20deep%20learning%20era%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BCurrent%20Opinion%20in%20Cell%20Biology%26lt%3B%5C%2Fi%26gt%3B%2C%2085%2C%20p.%20102271.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1016%5C%2Fj.ceb.2023.102271.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Live-cell%20imaging%20in%20the%20deep%20learning%20era%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Joanna%20W.%22%2C%22lastName%22%3A%22Pylv%5Cu00e4n%5Cu00e4inen%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Estibaliz%22%2C%22lastName%22%3A%22G%5Cu00f3mez-de-Mariscal%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ricardo%22%2C%22lastName%22%3A%22Henriques%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Guillaume%22%2C%22lastName%22%3A%22Jacquemet%22%7D%5D%2C%22abstractNote%22%3A%22Live%20imaging%20is%20a%20powerful%20tool%2C%20enabling%20scientists%20to%20observe%20living%20organisms%20in%20real%20time.%20In%20particular%2C%20when%20combined%20with%20fluorescence%20microscopy%2C%20live%20imaging%20allows%20the%20monitoring%20of%20cellular%20components%20with%20high%20sensitivity%20and%20specificity.%20Yet%2C%20due%20to%20critical%20challenges%20%28i.e.%2C%20drift%2C%20phototoxicity%2C%20dataset%20size%29%2C%20implementing%20live%20imaging%20and%20analyzing%20the%20resulting%20datasets%20is%20rarely%20straightforward.%20Over%20the%20past%20years%2C%20the%20development%20of%20bioimage%20analysis%20tools%2C%20including%20deep%20learning%2C%20is%20changing%20how%20we%20perform%20live%20imaging.%20Here%20we%20briefly%20cover%20important%20computational%20methods%20aiding%20live%20imaging%20and%20carrying%20out%20key%20tasks%20such%20as%20drift%20correction%2C%20denoising%2C%20super-resolution%20imaging%2C%20artificial%20labeling%2C%20tracking%2C%20and%20time%20series%20analysis.%20We%20also%20cover%20recent%20advances%20in%20self-driving%20microscopy.%22%2C%22date%22%3A%222023-12-01%22%2C%22language%22%3A%22%22%2C%22DOI%22%3A%2210.1016%5C%2Fj.ceb.2023.102271%22%2C%22ISSN%22%3A%220955-0674%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.sciencedirect.com%5C%2Fscience%5C%2Farticle%5C%2Fpii%5C%2FS0955067423001205%22%2C%22collections%22%3A%5B%224KIL6FA6%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A12%3A06Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Data%20Analysis%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22I3Y6H3S2%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Cabriel%20et%20al.%22%2C%22parsedDate%22%3A%222023-12%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BCabriel%2C%20C.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282023%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41566-023-01308-8%26%23039%3B%26gt%3BEvent-based%20vision%20sensor%20for%20fast%20and%20dense%20single-molecule%20localization%20microscopy%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%20Photonics%26lt%3B%5C%2Fi%26gt%3B%2C%2017%2812%29%2C%20pp.%201105%5Cu20131113.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41566-023-01308-8.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Event-based%20vision%20sensor%20for%20fast%20and%20dense%20single-molecule%20localization%20microscopy%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Cl%5Cu00e9ment%22%2C%22lastName%22%3A%22Cabriel%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Tual%22%2C%22lastName%22%3A%22Monfort%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Christian%20G.%22%2C%22lastName%22%3A%22Specht%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ignacio%22%2C%22lastName%22%3A%22Izeddin%22%7D%5D%2C%22abstractNote%22%3A%22Single-molecule%20localization%20microscopy%20%28SMLM%29%20enables%20crucial%20insights%20into%20cellular%20structures%20and%20processes%20to%20be%20revealed%20at%20the%20single-molecule%20level.%20However%2C%20SMLM%20is%20often%20hampered%20by%20limited%20temporal%20resolution%20and%20the%20fixed%20frame%20rate%20of%20the%20acquisition.%20Here%20we%20present%20a%20new%20approach%20to%20SMLM%20data%20acquisition%20and%20processing%20based%20on%20an%20affordable%20event-based%20sensor.%20This%20type%20of%20sensor%20reacts%20to%20changes%20in%20light%20intensity%2C%20rather%20than%20integrating%20photons%20during%20the%20exposure%20time%20of%20each%20frame.%20Each%20pixel%20works%20independently%20and%20returns%20a%20signal%20only%20when%20an%20intensity%20change%20is%20detected.%20Compared%20with%20video%20acquisition%20using%20traditional%20electron-multiplying%20charge-coupled%20device%20or%20scientific%20complementary%20metal%5Cu2013oxide%5Cu2013semiconductor%20cameras%2C%20the%20event-based%20sensor%20provides%20higher%20temporal%20resolution%20and%20throughput%20on%20the%20positions%20of%20blinking%20molecules.%20We%20demonstrate%20event-based%20SMLM%20super-resolution%20imaging%20on%20biological%20samples%20with%20spatial%20resolution%20on%20a%20par%20with%20the%20performance%20of%20electron-multiplying%20charge-coupled%20device%20or%20scientific%20complementary%20metal%5Cu2013oxide%5Cu2013semiconductor%20cameras%2C%20while%20registering%20only%20the%20on%20and%20off%20switching%20of%20blinking%20molecules.%20We%20use%20event-based%20SMLM%20to%20perform%20very%20dense%20single-molecule%20imaging%2C%20where%20frame-based%20cameras%20experience%20major%20limitations.%22%2C%22date%22%3A%222023-12%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs41566-023-01308-8%22%2C%22ISSN%22%3A%221749-4893%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41566-023-01308-8%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A13%3A50Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Application%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22CUSN9WPC%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Balasubramanian%20et%20al.%22%2C%22parsedDate%22%3A%222023-10-28%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BBalasubramanian%2C%20H.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282023%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs42003-023-05468-9%26%23039%3B%26gt%3BImagining%20the%20future%20of%20optical%20microscopy%3A%20everything%2C%20everywhere%2C%20all%20at%20once%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BCommunications%20Biology%26lt%3B%5C%2Fi%26gt%3B%2C%206%281%29%2C%20pp.%201%5Cu201312.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs42003-023-05468-9.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Imagining%20the%20future%20of%20optical%20microscopy%3A%20everything%2C%20everywhere%2C%20all%20at%20once%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Harikrushnan%22%2C%22lastName%22%3A%22Balasubramanian%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Chad%20M.%22%2C%22lastName%22%3A%22Hobson%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Teng-Leong%22%2C%22lastName%22%3A%22Chew%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jesse%20S.%22%2C%22lastName%22%3A%22Aaron%22%7D%5D%2C%22abstractNote%22%3A%22The%20optical%20microscope%20has%20revolutionized%20biology%20since%20at%20least%20the%2017th%20Century.%20Since%20then%2C%20it%20has%20progressed%20from%20a%20largely%20observational%20tool%20to%20a%20powerful%20bioanalytical%20platform.%20However%2C%20realizing%20its%20full%20potential%20to%20study%20live%20specimens%20is%20hindered%20by%20a%20daunting%20array%20of%20technical%20challenges.%20Here%2C%20we%20delve%20into%20the%20current%20state%20of%20live%20imaging%20to%20explore%20the%20barriers%20that%20must%20be%20overcome%20and%20the%20possibilities%20that%20lie%20ahead.%20We%20venture%20to%20envision%20a%20future%20where%20we%20can%20visualize%20and%20study%20everything%2C%20everywhere%2C%20all%20at%20once%20%5Cu2013%20from%20the%20intricate%20inner%20workings%20of%20a%20single%20cell%20to%20the%20dynamic%20interplay%20across%20entire%20organisms%2C%20and%20a%20world%20where%20scientists%20could%20access%20the%20necessary%20microscopy%20technologies%20anywhere.%22%2C%22date%22%3A%222023-10-28%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs42003-023-05468-9%22%2C%22ISSN%22%3A%222399-3642%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs42003-023-05468-9%22%2C%22collections%22%3A%5B%224KIL6FA6%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A12%3A30Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%2C%7B%22tag%22%3A%22Data%20Analysis%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22SD9WC6DX%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22G%5Cu00f3mez-de-Mariscal%20et%20al.%22%2C%22parsedDate%22%3A%222023-08-08%22%2C%22numChildren%22%3A1%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BG%5Cu00f3mez-de-Mariscal%2C%20E.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282023%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttp%3A%5C%2F%5C%2Farxiv.org%5C%2Fabs%5C%2F2308.04387%26%23039%3B%26gt%3BHarnessing%20Artificial%20Intelligence%20To%20Reduce%20Phototoxicity%20in%20Live%20Imaging%26lt%3B%5C%2Fa%26gt%3B.%26quot%3B%20arXiv.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.48550%5C%2FarXiv.2308.04387.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22preprint%22%2C%22title%22%3A%22Harnessing%20Artificial%20Intelligence%20To%20Reduce%20Phototoxicity%20in%20Live%20Imaging%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Estibaliz%22%2C%22lastName%22%3A%22G%5Cu00f3mez-de-Mariscal%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Mario%22%2C%22lastName%22%3A%22Del%20Rosario%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Joanna%20W.%22%2C%22lastName%22%3A%22Pylv%5Cu00e4n%5Cu00e4inen%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Guillaume%22%2C%22lastName%22%3A%22Jacquemet%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ricardo%22%2C%22lastName%22%3A%22Henriques%22%7D%5D%2C%22abstractNote%22%3A%22Fluorescence%20microscopy%2C%20widely%20used%20in%20the%20study%20of%20living%20cells%2C%20tissues%2C%20and%20organisms%2C%20often%20faces%20the%20challenge%20of%20photodamage.%20This%20is%20primarily%20caused%20by%20the%20interaction%20between%20light%20and%20biochemical%20components%20during%20the%20imaging%20process%2C%20leading%20to%20compromised%20accuracy%20and%20reliability%20of%20biological%20results.%20Methods%20necessitating%20extended%20high-intensity%20illumination%2C%20such%20as%20super-resolution%20microscopy%20or%20thick%20sample%20imaging%2C%20are%20particularly%20susceptible%20to%20this%20issue.%20As%20part%20of%20the%20solution%20to%20these%20problems%2C%20advanced%20imaging%20approaches%20involving%20artificial%20intelligence%20%28AI%29%20have%20been%20developed.%20Here%20we%20underscore%20the%20necessity%20of%20establishing%20constraints%20to%20maintain%20light-induced%20damage%20at%20levels%20that%20permit%20cells%20to%20sustain%20their%20live%20behaviour.%20From%20this%20perspective%2C%20data-driven%20live-cell%20imaging%20bears%20significant%20potential%20in%20aiding%20the%20development%20of%20AI-enhanced%20photodamage-aware%20microscopy.%20These%20technologies%20could%20streamline%20precise%20observations%20of%20natural%20biological%20dynamics%20while%20minimising%20phototoxicity%20risks.%22%2C%22genre%22%3A%22%22%2C%22repository%22%3A%22arXiv%22%2C%22archiveID%22%3A%22arXiv%3A2308.04387%22%2C%22date%22%3A%222023-08-08%22%2C%22DOI%22%3A%2210.48550%5C%2FarXiv.2308.04387%22%2C%22citationKey%22%3A%22%22%2C%22url%22%3A%22http%3A%5C%2F%5C%2Farxiv.org%5C%2Fabs%5C%2F2308.04387%22%2C%22language%22%3A%22%22%2C%22collections%22%3A%5B%22UQ77P7I5%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A02%3A29Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Data%20Analysis%22%7D%2C%7B%22tag%22%3A%22Data%20handling%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22Q2A9GDKT%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Chai%20et%20al.%22%2C%22parsedDate%22%3A%222023-07-19%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BChai%2C%20B.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282023%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttp%3A%5C%2F%5C%2Farxiv.org%5C%2Fabs%5C%2F2307.10500%26%23039%3B%26gt%3BOpportunities%20and%20challenges%20for%20deep%20learning%20in%20cell%20dynamics%20research%26lt%3B%5C%2Fa%26gt%3B.%26quot%3B%20arXiv.%20Available%20at%3A%20%28Accessed%3A%20February%2014%2C%202024%29.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22preprint%22%2C%22title%22%3A%22Opportunities%20and%20challenges%20for%20deep%20learning%20in%20cell%20dynamics%20research%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Binghao%22%2C%22lastName%22%3A%22Chai%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Christoforos%22%2C%22lastName%22%3A%22Efstathiou%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Haoran%22%2C%22lastName%22%3A%22Yue%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Viji%20M.%22%2C%22lastName%22%3A%22Draviam%22%7D%5D%2C%22abstractNote%22%3A%22With%20the%20growth%20of%20artificial%20intelligence%20%28AI%29%2C%20there%20has%20been%20an%20increase%20in%20the%20adoption%20of%20computer%20vision%20and%20deep%20learning%20%28DL%29%20techniques%20for%20the%20evaluation%20of%20microscopy%20images%20and%20movies.%20This%20adoption%20has%20not%20only%20addressed%20hurdles%20in%20quantitative%20analysis%20of%20dynamic%20cell%20biological%20processes%2C%20but%20it%20has%20also%20started%20supporting%20advances%20in%20drug%20development%2C%20precision%20medicine%20and%20genome-phenome%20mapping.%20Here%20we%20survey%20existing%20AI-based%20techniques%20and%20tools%2C%20and%20open-source%20datasets%2C%20with%20a%20specific%20focus%20on%20the%20computational%20tasks%20of%20segmentation%2C%20classification%2C%20and%20tracking%20of%20cellular%20and%20subcellular%20structures%20and%20dynamics.%20We%20summarise%20long-standing%20challenges%20in%20microscopy%20video%20analysis%20from%20the%20computational%20perspective%20and%20review%20emerging%20research%20frontiers%20and%20innovative%20applications%20for%20deep%20learning-guided%20automation%20for%20cell%20dynamics%20research.%22%2C%22genre%22%3A%22%22%2C%22repository%22%3A%22arXiv%22%2C%22archiveID%22%3A%22arXiv%3A2307.10500%22%2C%22date%22%3A%222023-07-19%22%2C%22DOI%22%3A%22%22%2C%22citationKey%22%3A%22%22%2C%22url%22%3A%22http%3A%5C%2F%5C%2Farxiv.org%5C%2Fabs%5C%2F2307.10500%22%2C%22language%22%3A%22%22%2C%22collections%22%3A%5B%22UQ77P7I5%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A02%3A23Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Data%20Analysis%22%7D%2C%7B%22tag%22%3A%22Data%20handling%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22BUIQZUCI%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Nature%20Methods%22%2C%22parsedDate%22%3A%222023-07%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BNature%20Methods%20%282023%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-023-01950-8%26%23039%3B%26gt%3BWhat%26%23039%3Bs%20next%20for%20bioimage%20analysis%3F%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%20Methods%26lt%3B%5C%2Fi%26gt%3B%2C%2020%287%29%2C%20pp.%20945%5Cu2013946.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41592-023-01950-8.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22What%27s%20next%20for%20bioimage%20analysis%3F%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22%22%2C%22lastName%22%3A%22Nature%20Methods%22%7D%5D%2C%22abstractNote%22%3A%22Advanced%20bioimage%20analysis%20tools%20are%20poised%20to%20disrupt%20the%20way%20in%20which%20microscopy%20images%20are%20acquired%20and%20analyzed.%20This%20Focus%20issue%20shares%20the%20hopes%20and%20opinions%20of%20experts%20on%20the%20near%20and%20distant%20future%20of%20image%20analysis.%22%2C%22date%22%3A%222023-07%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs41592-023-01950-8%22%2C%22ISSN%22%3A%221548-7105%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-023-01950-8%22%2C%22collections%22%3A%5B%22HQUL7BI3%22%5D%2C%22dateModified%22%3A%222025-04-29T13%3A12%3A34Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Data%20Analysis%22%7D%2C%7B%22tag%22%3A%22Data%20handling%22%7D%5D%7D%7D%2C%7B%22key%22%3A%2269VUKG2U%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Carpenter%20et%20al.%22%2C%22parsedDate%22%3A%222023-07%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BCarpenter%2C%20A.E.%2C%20Cimini%2C%20B.A.%20and%20Eliceiri%2C%20K.W.%20%282023%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-023-01912-0%26%23039%3B%26gt%3BSmart%20microscopes%20of%20the%20future%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%20Methods%26lt%3B%5C%2Fi%26gt%3B%2C%2020%287%29%2C%20pp.%20962%5Cu2013964.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41592-023-01912-0.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Smart%20microscopes%20of%20the%20future%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Anne%20E.%22%2C%22lastName%22%3A%22Carpenter%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Beth%20A.%22%2C%22lastName%22%3A%22Cimini%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Kevin%20W.%22%2C%22lastName%22%3A%22Eliceiri%22%7D%5D%2C%22abstractNote%22%3A%22We%20dream%20of%20a%20future%20where%20light%20microscopes%20have%20new%20capabilities%3A%20language-guided%20image%20acquisition%2C%20automatic%20image%20analysis%20based%20on%20extensive%20prior%20training%20from%20biologist%20experts%2C%20and%20language-guided%20image%20analysis%20for%20custom%20analyses.%20Most%20capabilities%20have%20reached%20the%20proof-of-principle%20stage%2C%20but%20implementation%20would%20be%20accelerated%20by%20efforts%20to%20gather%20appropriate%20training%20sets%20and%20make%20user-friendly%20interfaces.%22%2C%22date%22%3A%222023-07%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs41592-023-01912-0%22%2C%22ISSN%22%3A%221548-7105%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-023-01912-0%22%2C%22collections%22%3A%5B%22HQUL7BI3%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A09%3A00Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%2C%7B%22tag%22%3A%22Data%20Analysis%22%7D%2C%7B%22tag%22%3A%22Data%20handling%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22EAN93DZP%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Daetwyler%20and%20Fiolka%22%2C%22parsedDate%22%3A%222023-05-09%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BDaetwyler%2C%20S.%20and%20Fiolka%2C%20R.P.%20%282023%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs42003-023-04857-4%26%23039%3B%26gt%3BLight-sheets%20and%20smart%20microscopy%2C%20an%20exciting%20future%20is%20dawning%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BCommunications%20Biology%26lt%3B%5C%2Fi%26gt%3B%2C%206%281%29%2C%20pp.%201%5Cu201311.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs42003-023-04857-4.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Light-sheets%20and%20smart%20microscopy%2C%20an%20exciting%20future%20is%20dawning%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Stephan%22%2C%22lastName%22%3A%22Daetwyler%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Reto%20Paul%22%2C%22lastName%22%3A%22Fiolka%22%7D%5D%2C%22abstractNote%22%3A%22Light-sheet%20fluorescence%20microscopy%20has%20transformed%20our%20ability%20to%20visualize%20and%20quantitatively%20measure%20biological%20processes%20rapidly%20and%20over%20long%20time%20periods.%20In%20this%20review%2C%20we%20discuss%20current%20and%20future%20developments%20in%20light-sheet%20fluorescence%20microscopy%20that%20we%20expect%20to%20further%20expand%20its%20capabilities.%20This%20includes%20smart%20and%20adaptive%20imaging%20schemes%20to%20overcome%20traditional%20imaging%20trade-offs%2C%20i.e.%2C%20spatiotemporal%20resolution%2C%20field%20of%20view%20and%20sample%20health.%20In%20smart%20microscopy%2C%20a%20microscope%20will%20autonomously%20decide%20where%2C%20when%2C%20what%20and%20how%20to%20image.%20We%20further%20assess%20how%20image%20restoration%20techniques%20provide%20avenues%20to%20overcome%20these%20tradeoffs%20and%20how%20%5Cu201copen%20top%5Cu201d%20light-sheet%20microscopes%20may%20enable%20multi-modal%20imaging%20with%20high%20throughput.%20As%20such%2C%20we%20predict%20that%20light-sheet%20microscopy%20will%20fulfill%20an%20important%20role%20in%20biomedical%20and%20clinical%20imaging%20in%20the%20future.%22%2C%22date%22%3A%222023-05-09%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs42003-023-04857-4%22%2C%22ISSN%22%3A%222399-3642%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs42003-023-04857-4%22%2C%22collections%22%3A%5B%224KIL6FA6%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A12%3A20Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%2C%7B%22tag%22%3A%22Data%20Analysis%22%7D%2C%7B%22tag%22%3A%22Data%20handling%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22UL82WX7E%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Andr%5Cu00e9%20et%20al.%22%2C%22parsedDate%22%3A%222023-03-27%22%2C%22numChildren%22%3A1%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BAndr%5Cu00e9%2C%20O.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282023%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.cell.com%5C%2Fcell-reports-methods%5C%2Fabstract%5C%2FS2667-2375%2823%2900030-9%26%23039%3B%26gt%3BData-driven%20microscopy%20allows%20for%20automated%20context-specific%20acquisition%20of%20high-fidelity%20image%20data%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BCell%20Reports%20Methods%26lt%3B%5C%2Fi%26gt%3B%2C%203%283%29.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1016%5C%2Fj.crmeth.2023.100419.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Data-driven%20microscopy%20allows%20for%20automated%20context-specific%20acquisition%20of%20high-fidelity%20image%20data%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Oscar%22%2C%22lastName%22%3A%22Andr%5Cu00e9%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Johannes%20Kumra%22%2C%22lastName%22%3A%22Ahnlide%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Nils%22%2C%22lastName%22%3A%22Norlin%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Vinay%22%2C%22lastName%22%3A%22Swaminathan%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Pontus%22%2C%22lastName%22%3A%22Nordenfelt%22%7D%5D%2C%22abstractNote%22%3A%22%22%2C%22date%22%3A%222023-03-27%22%2C%22language%22%3A%22English%22%2C%22DOI%22%3A%2210.1016%5C%2Fj.crmeth.2023.100419%22%2C%22ISSN%22%3A%222667-2375%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.cell.com%5C%2Fcell-reports-methods%5C%2Fabstract%5C%2FS2667-2375%2823%2900030-9%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A06%3A21Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%2C%7B%22tag%22%3A%22Data%20Analysis%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22P474WSZT%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Volpe%20et%20al.%22%2C%22parsedDate%22%3A%222023-03-07%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BVolpe%2C%20G.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282023%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttp%3A%5C%2F%5C%2Farxiv.org%5C%2Fabs%5C%2F2303.03793%26%23039%3B%26gt%3BRoadmap%20on%20Deep%20Learning%20for%20Microscopy%26lt%3B%5C%2Fa%26gt%3B.%26quot%3B%20arXiv.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.48550%5C%2FarXiv.2303.03793.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22preprint%22%2C%22title%22%3A%22Roadmap%20on%20Deep%20Learning%20for%20Microscopy%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Giovanni%22%2C%22lastName%22%3A%22Volpe%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Carolina%22%2C%22lastName%22%3A%22W%5Cu00e4hlby%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Lei%22%2C%22lastName%22%3A%22Tian%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Michael%22%2C%22lastName%22%3A%22Hecht%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Artur%22%2C%22lastName%22%3A%22Yakimovich%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Kristina%22%2C%22lastName%22%3A%22Monakhova%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Laura%22%2C%22lastName%22%3A%22Waller%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ivo%20F.%22%2C%22lastName%22%3A%22Sbalzarini%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Christopher%20A.%22%2C%22lastName%22%3A%22Metzler%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Mingyang%22%2C%22lastName%22%3A%22Xie%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Kevin%22%2C%22lastName%22%3A%22Zhang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Isaac%20C.%20D.%22%2C%22lastName%22%3A%22Lenton%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Halina%22%2C%22lastName%22%3A%22Rubinsztein-Dunlop%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Daniel%22%2C%22lastName%22%3A%22Brunner%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Bijie%22%2C%22lastName%22%3A%22Bai%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Aydogan%22%2C%22lastName%22%3A%22Ozcan%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Daniel%22%2C%22lastName%22%3A%22Midtvedt%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Hao%22%2C%22lastName%22%3A%22Wang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Nata%5Cu0161a%22%2C%22lastName%22%3A%22Sladoje%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Joakim%22%2C%22lastName%22%3A%22Lindblad%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jason%20T.%22%2C%22lastName%22%3A%22Smith%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Marien%22%2C%22lastName%22%3A%22Ochoa%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Margarida%22%2C%22lastName%22%3A%22Barroso%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Xavier%22%2C%22lastName%22%3A%22Intes%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Tong%22%2C%22lastName%22%3A%22Qiu%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Li-Yu%22%2C%22lastName%22%3A%22Yu%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Sixian%22%2C%22lastName%22%3A%22You%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Yongtao%22%2C%22lastName%22%3A%22Liu%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Maxim%20A.%22%2C%22lastName%22%3A%22Ziatdinov%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Sergei%20V.%22%2C%22lastName%22%3A%22Kalinin%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Arlo%22%2C%22lastName%22%3A%22Sheridan%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Uri%22%2C%22lastName%22%3A%22Manor%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Elias%22%2C%22lastName%22%3A%22Nehme%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ofri%22%2C%22lastName%22%3A%22Goldenberg%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Yoav%22%2C%22lastName%22%3A%22Shechtman%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Henrik%20K.%22%2C%22lastName%22%3A%22Moberg%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Christoph%22%2C%22lastName%22%3A%22Langhammer%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Barbora%22%2C%22lastName%22%3A%22%5Cu0160pa%5Cu010dkov%5Cu00e1%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Saga%22%2C%22lastName%22%3A%22Helgadottir%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Benjamin%22%2C%22lastName%22%3A%22Midtvedt%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Aykut%22%2C%22lastName%22%3A%22Argun%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Tobias%22%2C%22lastName%22%3A%22Thalheim%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Frank%22%2C%22lastName%22%3A%22Cichos%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Stefano%22%2C%22lastName%22%3A%22Bo%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Lars%22%2C%22lastName%22%3A%22Hubatsch%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jesus%22%2C%22lastName%22%3A%22Pineda%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Carlo%22%2C%22lastName%22%3A%22Manzo%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Harshith%22%2C%22lastName%22%3A%22Bachimanchi%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Erik%22%2C%22lastName%22%3A%22Selander%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Antoni%22%2C%22lastName%22%3A%22Homs-Corbera%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Martin%22%2C%22lastName%22%3A%22Fr%5Cu00e4nzl%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Kevin%22%2C%22lastName%22%3A%22de%20Haan%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Yair%22%2C%22lastName%22%3A%22Rivenson%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Zofia%22%2C%22lastName%22%3A%22Korczak%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Caroline%20Beck%22%2C%22lastName%22%3A%22Adiels%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Mite%22%2C%22lastName%22%3A%22Mijalkov%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22D%5Cu00e1niel%22%2C%22lastName%22%3A%22Ver%5Cu00e9b%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Yu-Wei%22%2C%22lastName%22%3A%22Chang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Joana%20B.%22%2C%22lastName%22%3A%22Pereira%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Damian%22%2C%22lastName%22%3A%22Matuszewski%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Gustaf%22%2C%22lastName%22%3A%22Kylberg%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ida-Maria%22%2C%22lastName%22%3A%22Sintorn%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Juan%20C.%22%2C%22lastName%22%3A%22Caicedo%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Beth%20A.%22%2C%22lastName%22%3A%22Cimini%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Muyinatu%20A.%20Lediju%22%2C%22lastName%22%3A%22Bell%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Bruno%20M.%22%2C%22lastName%22%3A%22Saraiva%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Guillaume%22%2C%22lastName%22%3A%22Jacquemet%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ricardo%22%2C%22lastName%22%3A%22Henriques%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Wei%22%2C%22lastName%22%3A%22Ouyang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Trang%22%2C%22lastName%22%3A%22Le%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Estibaliz%22%2C%22lastName%22%3A%22G%5Cu00f3mez-de-Mariscal%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Daniel%22%2C%22lastName%22%3A%22Sage%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Arrate%22%2C%22lastName%22%3A%22Mu%5Cu00f1oz-Barrutia%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ebba%20Josefson%22%2C%22lastName%22%3A%22Lindqvist%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Johanna%22%2C%22lastName%22%3A%22Bergman%22%7D%5D%2C%22abstractNote%22%3A%22Through%20digital%20imaging%2C%20microscopy%20has%20evolved%20from%20primarily%20being%20a%20means%20for%20visual%20observation%20of%20life%20at%20the%20micro-%20and%20nano-scale%2C%20to%20a%20quantitative%20tool%20with%20ever-increasing%20resolution%20and%20throughput.%20Artificial%20intelligence%2C%20deep%20neural%20networks%2C%20and%20machine%20learning%20are%20all%20niche%20terms%20describing%20computational%20methods%20that%20have%20gained%20a%20pivotal%20role%20in%20microscopy-based%20research%20over%20the%20past%20decade.%20This%20Roadmap%20is%20written%20collectively%20by%20prominent%20researchers%20and%20encompasses%20selected%20aspects%20of%20how%20machine%20learning%20is%20applied%20to%20microscopy%20image%20data%2C%20with%20the%20aim%20of%20gaining%20scientific%20knowledge%20by%20improved%20image%20quality%2C%20automated%20detection%2C%20segmentation%2C%20classification%20and%20tracking%20of%20objects%2C%20and%20efficient%20merging%20of%20information%20from%20multiple%20imaging%20modalities.%20We%20aim%20to%20give%20the%20reader%20an%20overview%20of%20the%20key%20developments%20and%20an%20understanding%20of%20possibilities%20and%20limitations%20of%20machine%20learning%20for%20microscopy.%20It%20will%20be%20of%20interest%20to%20a%20wide%20cross-disciplinary%20audience%20in%20the%20physical%20sciences%20and%20life%20sciences.%22%2C%22genre%22%3A%22%22%2C%22repository%22%3A%22arXiv%22%2C%22archiveID%22%3A%22arXiv%3A2303.03793%22%2C%22date%22%3A%222023-03-07%22%2C%22DOI%22%3A%2210.48550%5C%2FarXiv.2303.03793%22%2C%22citationKey%22%3A%22%22%2C%22url%22%3A%22http%3A%5C%2F%5C%2Farxiv.org%5C%2Fabs%5C%2F2303.03793%22%2C%22language%22%3A%22%22%2C%22collections%22%3A%5B%224KIL6FA6%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A10%3A18Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Data%20Analysis%22%7D%2C%7B%22tag%22%3A%22Data%20handling%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22GZPGA3QN%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Madhusoodanan%22%2C%22parsedDate%22%3A%222023-02-07%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BMadhusoodanan%2C%20J.%20%282023%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fd41586-023-00336-2%26%23039%3B%26gt%3BSmart%20microscopes%20spot%20fleeting%20biology%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%26lt%3B%5C%2Fi%26gt%3B%2C%20614%287947%29%2C%20pp.%20378%5Cu2013380.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fd41586-023-00336-2.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Smart%20microscopes%20spot%20fleeting%20biology%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jyoti%22%2C%22lastName%22%3A%22Madhusoodanan%22%7D%5D%2C%22abstractNote%22%3A%22Automated%20microscopes%20that%20adapt%20to%20each%20sample%5Cu2019s%20quirks%20can%20capture%20elusive%20biological%20phenomena%20at%20high%20resolution.%22%2C%22date%22%3A%222023-02-07%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fd41586-023-00336-2%22%2C%22ISSN%22%3A%22%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fd41586-023-00336-2%22%2C%22collections%22%3A%5B%22HQUL7BI3%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A08%3A03Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22YPXBQ6F2%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Ma%20and%20Wang%22%2C%22parsedDate%22%3A%222023%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BMa%2C%20J.%20and%20Wang%2C%20B.%20%282023%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-023-01885-0%26%23039%3B%26gt%3BTowards%20foundation%20models%20of%20biological%20image%20segmentation%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%20Methods%26lt%3B%5C%2Fi%26gt%3B%2C%2020%287%29%2C%20pp.%20953%5Cu2013955.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41592-023-01885-0.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Towards%20foundation%20models%20of%20biological%20image%20segmentation%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jun%22%2C%22lastName%22%3A%22Ma%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Bo%22%2C%22lastName%22%3A%22Wang%22%7D%5D%2C%22abstractNote%22%3A%22%22%2C%22date%22%3A%2207%5C%2F2023%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs41592-023-01885-0%22%2C%22ISSN%22%3A%221548-7091%2C%201548-7105%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-023-01885-0%22%2C%22collections%22%3A%5B%22HQUL7BI3%22%5D%2C%22dateModified%22%3A%222025-04-29T14%3A02%3A16Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Data%20analysis%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22C6VDYKVG%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Merino-Casallo%20et%20al.%22%2C%22parsedDate%22%3A%222022-12-31%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BMerino-Casallo%2C%20F.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282022%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1080%5C%2F19336918.2022.2055520%26%23039%3B%26gt%3BUnravelling%20cell%20migration%3A%20defining%20movement%20from%20the%20cell%20surface%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BCell%20Adhesion%20%26amp%3B%20Migration%26lt%3B%5C%2Fi%26gt%3B%2C%2016%281%29%2C%20pp.%2025%5Cu201364.%20Available%20at%3A%20%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Unravelling%20cell%20migration%3A%20defining%20movement%20from%20the%20cell%20surface%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Francisco%22%2C%22lastName%22%3A%22Merino-Casallo%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Maria%20Jose%22%2C%22lastName%22%3A%22Gomez-Benito%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Silvia%22%2C%22lastName%22%3A%22Hervas-Raluy%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jose%20Manuel%22%2C%22lastName%22%3A%22Garcia-Aznar%22%7D%5D%2C%22abstractNote%22%3A%22Cell%20motility%20is%20essential%20for%20life%20and%20development.%20Unfortunately%2C%20cell%20migration%20is%20also%20linked%20to%20several%20pathological%20processes%2C%20such%20as%20cancer%20metastasis.%20Cells%5Cu2019%20ability%20to%20migrate%20relies%20on%20many%20actors.%20Cells%20change%20their%20migratory%20strategy%20based%20on%20their%20phenotype%20and%20the%20properties%20of%20the%20surrounding%20microenvironment.%20Cell%20migration%20is%2C%20therefore%2C%20an%20extremely%20complex%20phenomenon.%20Researchers%20have%20investigated%20cell%20motility%20for%20more%20than%20a%20century.%20Recent%20discoveries%20have%20uncovered%20some%20of%20the%20mysteries%20associated%20with%20the%20mechanisms%20involved%20in%20cell%20migration%2C%20such%20as%20intracellular%20signaling%20and%20cell%20mechanics.%20These%20findings%20involve%20different%20players%2C%20including%20transmembrane%20receptors%2C%20adhesive%20complexes%2C%20cytoskeletal%20components%20%2C%20the%20nucleus%2C%20and%20the%20extracellular%20matrix.%20This%20review%20aims%20to%20give%20a%20global%20overview%20of%20our%20current%20understanding%20of%20cell%20migration.%22%2C%22date%22%3A%222022-12-31%22%2C%22language%22%3A%22%22%2C%22DOI%22%3A%2210.1080%5C%2F19336918.2022.2055520%22%2C%22ISSN%22%3A%221933-6918%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1080%5C%2F19336918.2022.2055520%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A15%3A47Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Application%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22W39S3ZSK%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Lan%20et%20al.%22%2C%22parsedDate%22%3A%222022-12-01%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BLan%2C%20T.-H.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282022%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.sciencedirect.com%5C%2Fscience%5C%2Farticle%5C%2Fpii%5C%2FS0168952522001408%26%23039%3B%26gt%3BOptogenetics%20for%20transcriptional%20programming%20and%20genetic%20engineering%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BTrends%20in%20Genetics%26lt%3B%5C%2Fi%26gt%3B%2C%2038%2812%29%2C%20pp.%201253%5Cu20131270.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1016%5C%2Fj.tig.2022.05.014.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Optogenetics%20for%20transcriptional%20programming%20and%20genetic%20engineering%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Tien-Hung%22%2C%22lastName%22%3A%22Lan%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Lian%22%2C%22lastName%22%3A%22He%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Yun%22%2C%22lastName%22%3A%22Huang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Yubin%22%2C%22lastName%22%3A%22Zhou%22%7D%5D%2C%22abstractNote%22%3A%22Optogenetics%20combines%20genetics%20and%20biophotonics%20to%20enable%20noninvasive%20control%20of%20biological%20processes%20with%20high%20spatiotemporal%20precision.%20When%20engineered%20into%20protein%20machineries%20that%20govern%20the%20cellular%20information%20flow%20as%20depicted%20in%20the%20central%20dogma%2C%20multiple%20genetically%20encoded%20non-opsin%20photosensory%20modules%20have%20been%20harnessed%20to%20modulate%20gene%20transcription%2C%20DNA%20or%20RNA%20modifications%2C%20DNA%20recombination%2C%20and%20genome%20engineering%20by%20utilizing%20photons%20emitting%20in%20the%20wide%20range%20of%20200%5Cu20131000%20nm.%20We%20present%20herein%20generally%20applicable%20modular%20strategies%20for%20optogenetic%20engineering%20and%20highlight%20latest%20advances%20in%20the%20broad%20applications%20of%20opsin-free%20optogenetics%20to%20program%20transcriptional%20outputs%20and%20precisely%20manipulate%20the%20mammalian%20genome%2C%20epigenome%2C%20and%20epitranscriptome.%20We%20also%20discuss%20current%20challenges%20and%20future%20trends%20in%20opsin-free%20optogenetics%2C%20which%20has%20been%20rapidly%20evolving%20to%20meet%20the%20growing%20needs%20in%20synthetic%20biology%20and%20genetics%20research.%22%2C%22date%22%3A%222022-12-01%22%2C%22language%22%3A%22%22%2C%22DOI%22%3A%2210.1016%5C%2Fj.tig.2022.05.014%22%2C%22ISSN%22%3A%220168-9525%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.sciencedirect.com%5C%2Fscience%5C%2Farticle%5C%2Fpii%5C%2FS0168952522001408%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A15%3A23Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Application%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22QKZ278FZ%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Alvelid%20et%20al.%22%2C%22parsedDate%22%3A%222022-10%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BAlvelid%2C%20J.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282022%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-022-01588-y%26%23039%3B%26gt%3BEvent-triggered%20STED%20imaging%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%20Methods%26lt%3B%5C%2Fi%26gt%3B%2C%2019%2810%29%2C%20pp.%201268%5Cu20131275.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41592-022-01588-y.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Event-triggered%20STED%20imaging%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jonatan%22%2C%22lastName%22%3A%22Alvelid%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Martina%22%2C%22lastName%22%3A%22Damenti%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Chiara%22%2C%22lastName%22%3A%22Sgattoni%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ilaria%22%2C%22lastName%22%3A%22Testa%22%7D%5D%2C%22abstractNote%22%3A%22Monitoring%20the%20proteins%20and%20lipids%20that%20mediate%20all%20cellular%20processes%20requires%20imaging%20methods%20with%20increased%20spatial%20and%20temporal%20resolution.%20STED%20%28stimulated%20emission%20depletion%29%20nanoscopy%20enables%20fast%20imaging%20of%20nanoscale%20structures%20in%20living%20cells%20but%20is%20limited%20by%20photobleaching.%20Here%2C%20we%20present%20event-triggered%20STED%2C%20an%20automated%20multiscale%20method%20capable%20of%20rapidly%20initiating%20two-dimensional%20%282D%29%20and%203D%20STED%20imaging%20after%20detecting%20cellular%20events%20such%20as%20protein%20recruitment%2C%20vesicle%20trafficking%20and%20second%20messengers%20activity%20using%20biosensors.%20STED%20is%20applied%20in%20the%20vicinity%20of%20detected%20events%20to%20maximize%20the%20temporal%20resolution.%20We%20imaged%20synaptic%20vesicle%20dynamics%20at%20up%20to%2024%5Cu2009Hz%2C%2040%5Cu2009ms%20after%20local%20calcium%20activity%3B%20endocytosis%20and%20exocytosis%20events%20at%20up%20to%2011%5Cu2009Hz%2C%2040%5Cu2009ms%20after%20local%20protein%20recruitment%20or%20pH%20changes%3B%20and%20the%20interaction%20between%20endosomal%20vesicles%20at%20up%20to%203%5Cu2009Hz%2C%2070%5Cu2009ms%20after%20approaching%20one%20another.%20Event-triggered%20STED%20extends%20the%20capabilities%20of%20live%20nanoscale%20imaging%2C%20enabling%20novel%20biological%20observations%20in%20real%20time.%22%2C%22date%22%3A%222022-10%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs41592-022-01588-y%22%2C%22ISSN%22%3A%221548-7105%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-022-01588-y%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A06%3A30Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%2267C3RZY6%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Mahecic%20et%20al.%22%2C%22parsedDate%22%3A%222022-10%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BMahecic%2C%20D.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282022%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-022-01589-x%26%23039%3B%26gt%3BEvent-driven%20acquisition%20for%20content-enriched%20microscopy%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%20Methods%26lt%3B%5C%2Fi%26gt%3B%2C%2019%2810%29%2C%20pp.%201262%5Cu20131267.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41592-022-01589-x.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Event-driven%20acquisition%20for%20content-enriched%20microscopy%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Dora%22%2C%22lastName%22%3A%22Mahecic%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Willi%20L.%22%2C%22lastName%22%3A%22Stepp%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Chen%22%2C%22lastName%22%3A%22Zhang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Juliette%22%2C%22lastName%22%3A%22Griffi%5Cu00e9%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Martin%22%2C%22lastName%22%3A%22Weigert%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Suliana%22%2C%22lastName%22%3A%22Manley%22%7D%5D%2C%22abstractNote%22%3A%22A%20common%20goal%20of%20fluorescence%20microscopy%20is%20to%20collect%20data%20on%20specific%20biological%20events.%20Yet%2C%20the%20event-specific%20content%20that%20can%20be%20collected%20from%20a%20sample%20is%20limited%2C%20especially%20for%20rare%20or%20stochastic%20processes.%20This%20is%20due%20in%20part%20to%20photobleaching%20and%20phototoxicity%2C%20which%20constrain%20imaging%20speed%20and%20duration.%20We%20developed%20an%20event-driven%20acquisition%20framework%2C%20in%20which%20neural-network-based%20recognition%20of%20specific%20biological%20events%20triggers%20real-time%20control%20in%20an%20instant%20structured%20illumination%20microscope.%20Our%20setup%20adapts%20acquisitions%20on-the-fly%20by%20switching%20between%20a%20slow%20imaging%20rate%20while%20detecting%20the%20onset%20of%20events%2C%20and%20a%20fast%20imaging%20rate%20during%20their%20progression.%20Thus%2C%20we%20capture%20mitochondrial%20and%20bacterial%20divisions%20at%20imaging%20rates%20that%20match%20their%20dynamic%20timescales%2C%20while%20extending%20overall%20imaging%20durations.%20Because%20event-driven%20acquisition%20allows%20the%20microscope%20to%20respond%20specifically%20to%20complex%20biological%20events%2C%20it%20acquires%20data%20enriched%20in%20relevant%20content.%22%2C%22date%22%3A%222022-10%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs41592-022-01589-x%22%2C%22ISSN%22%3A%221548-7105%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-022-01589-x%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A04%3A55Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22KK2INVGX%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Mol%20and%20Vlijm%22%2C%22parsedDate%22%3A%222022-09-30%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BMol%2C%20F.N.%20and%20Vlijm%2C%20R.%20%282022%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.biorxiv.org%5C%2Fcontent%5C%2F10.1101%5C%2F2022.09.29.510126v1%26%23039%3B%26gt%3BAutomated%20STED%20nanoscopy%20for%20high-throughput%20imaging%20of%20cellular%20structures%26lt%3B%5C%2Fa%26gt%3B.%26quot%3B%20bioRxiv.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1101%5C%2F2022.09.29.510126.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22preprint%22%2C%22title%22%3A%22Automated%20STED%20nanoscopy%20for%20high-throughput%20imaging%20of%20cellular%20structures%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Frank%20N.%22%2C%22lastName%22%3A%22Mol%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Rifka%22%2C%22lastName%22%3A%22Vlijm%22%7D%5D%2C%22abstractNote%22%3A%22STimulated%20Emission%20Depletion%20%28STED%29%20nanoscopy%20uniquely%20combines%20a%20high%20spatial%20resolution%20%2820-50nm%20in%20cells%29%20with%20relatively%20fast%20imaging%20%28frame%20rate%20of%20%5Cu223c1-30Hz%29%2C%20straightforward%20sample%20preparation%20and%20direct%20image%20output%20%28no%20postprocessing%20required%29.%20Although%20these%20characteristics%20in%20principle%20make%20STED%20very%20suitable%20for%20high-throughput%20imaging%2C%20only%20few%20steps%20towards%20automation%20have%20been%20made.%20Here%2C%20we%20have%20developed%20fully%20automated%20STED%20imaging%2C%20eliminating%20all%20manual%20steps%20including%20the%20selection%20and%20characterisation%20of%20the%20relevant%20%28cellular%29%20regions%2C%20sample%20focusing%20and%20positioning%2C%20and%20microscope%20adjustments.%20This%20automatic%20STED%20image%20acquisition%20increases%20the%20data%20output%20by%20roughly%20two%20orders%20of%20magnitude%2C%20resulting%20in%20a%20more%20efficient%20use%20of%20the%20high-end%20microscope%2C%20and%20the%20ability%20to%20detect%20and%20characterise%20objects%20that%20are%20only%20present%20in%20a%20small%20subset%20of%20the%20sample.%22%2C%22genre%22%3A%22%22%2C%22repository%22%3A%22bioRxiv%22%2C%22archiveID%22%3A%22%22%2C%22date%22%3A%222022-09-30%22%2C%22DOI%22%3A%2210.1101%5C%2F2022.09.29.510126%22%2C%22citationKey%22%3A%22%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.biorxiv.org%5C%2Fcontent%5C%2F10.1101%5C%2F2022.09.29.510126v1%22%2C%22language%22%3A%22en%22%2C%22collections%22%3A%5B%22UQ77P7I5%22%5D%2C%22dateModified%22%3A%222024-11-15T14%3A17%3A37Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Application%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%223GEXFHP8%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Dibrov%22%2C%22parsedDate%22%3A%222022-08-12%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BDibrov%2C%20A.%20%282022%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Ftud.qucosa.de%5C%2Fid%5C%2Fqucosa%3A80379%26%23039%3B%26gt%3BTowards%20Smarter%20Fluorescence%20Microscopy%3A%20Enabling%20Adaptive%20Acquisition%20Strategies%20With%20Optimized%20Photon%20Budget%26lt%3B%5C%2Fa%26gt%3B.%26quot%3B%20Available%20at%3A%20%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Towards%20Smarter%20Fluorescence%20Microscopy%3A%20Enabling%20Adaptive%20Acquisition%20Strategies%20With%20Optimized%20Photon%20Budget%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Alexandr%22%2C%22lastName%22%3A%22Dibrov%22%7D%5D%2C%22abstractNote%22%3A%22%22%2C%22date%22%3A%222022%5C%2F08%5C%2F12%22%2C%22language%22%3A%22eng%22%2C%22DOI%22%3A%22%22%2C%22ISSN%22%3A%22%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Ftud.qucosa.de%5C%2Fid%5C%2Fqucosa%3A80379%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222025-04-29T13%3A08%3A58Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22MZKL54LI%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Chiron%20et%20al.%22%2C%22parsedDate%22%3A%222022-07-08%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BChiron%2C%20L.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282022%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41598-022-15207-5%26%23039%3B%26gt%3BCyberSco.Py%20an%20open-source%20software%20for%20event-based%2C%20conditional%20microscopy%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BScientific%20Reports%26lt%3B%5C%2Fi%26gt%3B%2C%2012%281%29%2C%20p.%2011579.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41598-022-15207-5.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22CyberSco.Py%20an%20open-source%20software%20for%20event-based%2C%20conditional%20microscopy%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Lionel%22%2C%22lastName%22%3A%22Chiron%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Matthias%22%2C%22lastName%22%3A%22Le%20Bec%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22C%5Cu00e9line%22%2C%22lastName%22%3A%22Cordier%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Sylvain%22%2C%22lastName%22%3A%22Pouzet%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Dimitrije%22%2C%22lastName%22%3A%22Milunov%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Alvaro%22%2C%22lastName%22%3A%22Banderas%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jean-Marc%22%2C%22lastName%22%3A%22Di%20Meglio%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Benoit%22%2C%22lastName%22%3A%22Sorre%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Pascal%22%2C%22lastName%22%3A%22Hersen%22%7D%5D%2C%22abstractNote%22%3A%22Timelapse%20fluorescence%20microscopy%20imaging%20is%20routinely%20used%20in%20quantitative%20cell%20biology.%20However%2C%20microscopes%20could%20become%20much%20more%20powerful%20investigation%20systems%20if%20they%20were%20endowed%20with%20simple%20unsupervised%20decision-making%20algorithms%20to%20transform%20them%20into%20fully%20responsive%20and%20automated%20measurement%20devices.%20Here%2C%20we%20report%20CyberSco.Py%2C%20Python%20software%20for%20advanced%20automated%20timelapse%20experiments.%20We%20provide%20proof-of-principle%20of%20a%20user-friendly%20framework%20that%20increases%20the%20tunability%20and%20flexibility%20when%20setting%20up%20and%20running%20fluorescence%20timelapse%20microscopy%20experiments.%20Importantly%2C%20CyberSco.Py%20combines%20real-time%20image%20analysis%20with%20automation%20capability%2C%20which%20allows%20users%20to%20create%20conditional%2C%20event-based%20experiments%20in%20which%20the%20imaging%20acquisition%20parameters%20and%20the%20status%20of%20various%20devices%20can%20be%20changed%20automatically%20based%20on%20the%20image%20analysis.%20We%20exemplify%20the%20relevance%20of%20CyberSco.Py%20to%20cell%20biology%20using%20several%20use%20case%20experiments%20with%20budding%20yeast.%20We%20anticipate%20that%20CyberSco.Py%20could%20be%20used%20to%20address%20the%20growing%20need%20for%20smart%20microscopy%20systems%20to%20implement%20more%20informative%20quantitative%20cell%20biology%20experiments.%22%2C%22date%22%3A%222022-07-08%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs41598-022-15207-5%22%2C%22ISSN%22%3A%222045-2322%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41598-022-15207-5%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A05%3A50Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%5D%7D</span>

				<div id="zp-ID-27-5384098-BULWHJV6" data-zp-author-date='Leger-et-al.-2025-12-14' data-zp-date-author='2025-12-14-Leger-et-al.' data-zp-date='2025-12-14' data-zp-year='2025' data-zp-itemtype='preprint' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Leger, D., Ivanisevic, M. and Lenart, P. (2025) "<a class='zp-ItemURL' target='_blank' href='https://www.biorxiv.org/content/10.64898/2025.12.11.693650v1'>AutoSTED: An automated workflow for STED super-resolution imaging of cell nuclei</a>." bioRxiv. Available at: <a href="https://doi.org/10.64898/2025.12.11.693650" rel="nofollow">https://doi.org/10.64898/2025.12.11.693650</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-NTMGRK8D" data-zp-author-date='Redondo-and-Paszek-2025-12-01' data-zp-date-author='2025-12-01-Redondo-and-Paszek' data-zp-date='2025-12-01' data-zp-year='2025' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Redondo, J.A. and Paszek, P. (2025) "<a class='zp-ItemURL' target='_blank' href='https://cames.ippt.pan.pl/index.php/cames/article/view/1945'>AI-Assisted Microscopy for Infection Biology: Advances in High-Content Imaging of Host-Pathogen Interactions</a>," <i>Computer Assisted Methods in Engineering and Science</i> [Preprint]. Available at: <a href="https://doi.org/10.24423/cames.2025.1945" rel="nofollow">https://doi.org/10.24423/cames.2025.1945</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-PGILC555" data-zp-author-date='Cosenza-et-al.-2025-12' data-zp-date-author='2025-12-Cosenza-et-al.' data-zp-date='2025-12' data-zp-year='2025' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Cosenza, M.R. <i>et al.</i> (2025) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s41586-025-09632-5'>Origins of chromosome instability unveiled by coupled imaging and genomics</a>," <i>Nature</i>, 648(8093), pp. 383–393. Available at: <a href="https://doi.org/10.1038/s41586-025-09632-5" rel="nofollow">https://doi.org/10.1038/s41586-025-09632-5</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-TWDLDCMT" data-zp-author-date='Tang-2025-12' data-zp-date-author='2025-12-Tang' data-zp-date='2025-12' data-zp-year='2025' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Tang, L. (2025) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s41592-025-02986-8'>Bridging smart microscopy with genomics</a>," <i>Nature Methods</i>, 22(12), pp. 2499–2499. Available at: <a href="https://doi.org/10.1038/s41592-025-02986-8" rel="nofollow">https://doi.org/10.1038/s41592-025-02986-8</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-WFEWMARH" data-zp-author-date='Berrevoets-et-al.-2025-11-27' data-zp-date-author='2025-11-27-Berrevoets-et-al.' data-zp-date='2025-11-27' data-zp-year='2025' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Berrevoets, E.S. <i>et al.</i> (2025) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s41467-025-65723-x'>Smart 3D super-resolution microscopy reveals the architecture of the RNA scaffold in a nuclear body</a>," <i>Nature Communications</i>, 16(1), p. 10689. Available at: <a href="https://doi.org/10.1038/s41467-025-65723-x" rel="nofollow">https://doi.org/10.1038/s41467-025-65723-x</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-2C2QZA49" data-zp-author-date='London-et-al.-2025-11-03' data-zp-date-author='2025-11-03-London-et-al.' data-zp-date='2025-11-03' data-zp-year='2025' data-zp-itemtype='preprint' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">London, J.A. <i>et al.</i> (2025) "<a class='zp-ItemURL' target='_blank' href='https://www.biorxiv.org/content/10.1101/2025.10.31.685671v1'>A Python framework for magnetic tweezers real-time image processing and microscope control</a>." bioRxiv. Available at: <a href="https://doi.org/10.1101/2025.10.31.685671" rel="nofollow">https://doi.org/10.1101/2025.10.31.685671</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-3JF4LIGZ" data-zp-author-date='Mandal-et-al.-2025-10-14' data-zp-date-author='2025-10-14-Mandal-et-al.' data-zp-date='2025-10-14' data-zp-year='2025' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Mandal, I. <i>et al.</i> (2025) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s41467-025-64105-7'>Evaluating large language model agents for automation of atomic force microscopy</a>," <i>Nature Communications</i>, 16(1), p. 9104. Available at: <a href="https://doi.org/10.1038/s41467-025-64105-7" rel="nofollow">https://doi.org/10.1038/s41467-025-64105-7</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-KY9GPYEJ" data-zp-author-date='Hickey-et-al.-2025-10-10' data-zp-date-author='2025-10-10-Hickey-et-al.' data-zp-date='2025-10-10' data-zp-year='2025' data-zp-itemtype='preprint' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Hickey, J.W. <i>et al.</i> (2025) "<a class='zp-ItemURL' target='_blank' href='https://www.biorxiv.org/content/10.1101/2025.10.09.680291v1'>Fluid-Squid: DIY Multiplexed Imaging of Cells and Tissues</a>." bioRxiv. Available at: <a href="https://doi.org/10.1101/2025.10.09.680291" rel="nofollow">https://doi.org/10.1101/2025.10.09.680291</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-LVWHWJR5" data-zp-author-date='Friederich-et-al.-2025-10-01' data-zp-date-author='2025-10-01-Friederich-et-al.' data-zp-date='2025-10-01' data-zp-year='2025' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Friederich, N. <i>et al.</i> (2025) "<a class='zp-ItemURL' target='_blank' href='https://www.degruyterbrill.com/document/doi/10.1515/auto-2025-0018/html?recommended=sidebar'>EAP4EMSIG – enhancing event-driven microscopy for microfluidic single-cell analysis: EAP4EMSIG – Verbesserung der ereignisgesteuerten Mikroskopie für die mikrofluidische Einzelzellanalyse</a>," <i>at - Automatisierungstechnik</i>, 73(10), pp. 808–823. Available at: <a href="https://doi.org/10.1515/auto-2025-0018" rel="nofollow">https://doi.org/10.1515/auto-2025-0018</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-ZUJE3P8Y" data-zp-author-date='Lee-et-al.-2025-10' data-zp-date-author='2025-10-Lee-et-al.' data-zp-date='2025-10' data-zp-year='2025' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Lee, J. <i>et al.</i> (2025) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s43587-025-00950-x'>REV-ERBα regulates brain NAD+ levels and tauopathy via an NFIL3–CD38 axis</a>," <i>Nature Aging</i>, 5(10), pp. 2070–2085. Available at: <a href="https://doi.org/10.1038/s43587-025-00950-x" rel="nofollow">https://doi.org/10.1038/s43587-025-00950-x</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-KFNGL64Q" data-zp-author-date='Zhang-et-al.-2025-09-08' data-zp-date-author='2025-09-08-Zhang-et-al.' data-zp-date='2025-09-08' data-zp-year='2025' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Zhang, N. <i>et al.</i> (2025) "<a class='zp-ItemURL' target='_blank' href='https://pubs.rsc.org/en/content/articlelanding/2025/lc/d5lc00286a'>Modular, open-sourced multiplexing for democratizing spatial omics</a>," <i>Lab on a Chip</i> [Preprint]. Available at: <a href="https://doi.org/10.1039/D5LC00286A" rel="nofollow">https://doi.org/10.1039/D5LC00286A</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-AS5J555G" data-zp-author-date='Alvelid-et-al.-2025-09-01' data-zp-date-author='2025-09-01-Alvelid-et-al.' data-zp-date='2025-09-01' data-zp-year='2025' data-zp-itemtype='preprint' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Alvelid, J., Koerfer, A. and Eggeling, C. (2025) "<a class='zp-ItemURL' target='_blank' href='https://www.biorxiv.org/content/10.1101/2025.08.27.672674v1'>Event-triggered MINFLUX microscopy: smart microscopy to catch and follow rare events</a>." bioRxiv. Available at: <a href="https://doi.org/10.1101/2025.08.27.672674" rel="nofollow">https://doi.org/10.1101/2025.08.27.672674</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-HV29U83C" data-zp-author-date='Meirovitch-et-al.-2025-05-15' data-zp-date-author='2025-05-15-Meirovitch-et-al.' data-zp-date='2025-05-15' data-zp-year='2025' data-zp-itemtype='preprint' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Meirovitch, Y. <i>et al.</i> (2025) "<a class='zp-ItemURL' target='_blank' href='https://www.biorxiv.org/content/10.1101/2023.10.05.561103v4'>SmartEM: machine-learning guided electron microscopy</a>." bioRxiv. Available at: <a href="https://doi.org/10.1101/2023.10.05.561103" rel="nofollow">https://doi.org/10.1101/2023.10.05.561103</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-QJYYQESZ" data-zp-author-date='Hsieh-et-al.-2025-04-17' data-zp-date-author='2025-04-17-Hsieh-et-al.' data-zp-date='2025-04-17' data-zp-year='2025' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Hsieh, H.-C. <i>et al.</i> (2025) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s41592-025-02647-w'>Imaging 3D cell cultures with optical microscopy</a>," <i>Nature Methods</i>, pp. 1–24. Available at: <a href="https://doi.org/10.1038/s41592-025-02647-w" rel="nofollow">https://doi.org/10.1038/s41592-025-02647-w</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-QELVL8TC" data-zp-author-date='Yang-et-al.-2025-04-14' data-zp-date-author='2025-04-14-Yang-et-al.' data-zp-date='2025-04-14' data-zp-year='2025' data-zp-itemtype='preprint' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Yang, J. <i>et al.</i> (2025) "<a class='zp-ItemURL' target='_blank' href='http://arxiv.org/abs/2504.10281'>Zero-shot Autonomous Microscopy for Scalable and Intelligent Characterization of 2D Materials</a>." arXiv. Available at: <a href="https://doi.org/10.48550/arXiv.2504.10281" rel="nofollow">https://doi.org/10.48550/arXiv.2504.10281</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-CAZ4PPYG" data-zp-author-date='Friederich-et-al.-2025-03-30' data-zp-date-author='2025-03-30-Friederich-et-al.' data-zp-date='2025-03-30' data-zp-year='2025' data-zp-itemtype='preprint' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Friederich, N. <i>et al.</i> (2025) "<a class='zp-ItemURL' target='_blank' href='http://arxiv.org/abs/2504.00047'>EAP4EMSIG -- Enhancing Event-Driven Microscopy for Microfluidic Single-Cell Analysis</a>." arXiv. Available at: <a href="https://doi.org/10.48550/arXiv.2504.00047" rel="nofollow">https://doi.org/10.48550/arXiv.2504.00047</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-TCPZKI5S" data-zp-author-date='Kesavan-and-Bohra-2025-03-04' data-zp-date-author='2025-03-04-Kesavan-and-Bohra' data-zp-date='2025-03-04' data-zp-year='2025' data-zp-itemtype='preprint' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Kesavan, P.S. and Bohra, D. (2025) "<a class='zp-ItemURL' target='_blank' href='https://www.biorxiv.org/content/10.1101/2025.02.25.639997v2'>deepthought: domain driven design for microscopy with applications in DNA damage responses</a>." bioRxiv. Available at: <a href="https://doi.org/10.1101/2025.02.25.639997" rel="nofollow">https://doi.org/10.1101/2025.02.25.639997</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-PA6N7DHN" data-zp-author-date='Daetwyler-et-al.-2025-02-12' data-zp-date-author='2025-02-12-Daetwyler-et-al.' data-zp-date='2025-02-12' data-zp-year='2025' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Daetwyler, S. <i>et al.</i> (2025) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s41592-025-02598-2'>Imaging of cellular dynamics from a whole organism to subcellular scale with self-driving, multiscale microscopy</a>," <i>Nature Methods</i>, pp. 1–10. Available at: <a href="https://doi.org/10.1038/s41592-025-02598-2" rel="nofollow">https://doi.org/10.1038/s41592-025-02598-2</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-FPEQIBFZ" data-zp-author-date='Thomas---Chemin-et-al.-2025-02-11' data-zp-date-author='2025-02-11-Thomas---Chemin-et-al.' data-zp-date='2025-02-11' data-zp-year='2025' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Thomas- -Chemin, O. <i>et al.</i> (2025) "<a class='zp-ItemURL' target='_blank' href='https://pubs.acs.org/doi/10.1021/acsnano.4c07729'>Advancing High-Throughput Cellular Atomic Force Microscopy with Automation and Artificial Intelligence</a>," <i>ACS Nano</i>, 19(5), pp. 5045–5062. Available at: <a href="https://doi.org/10.1021/acsnano.4c07729" rel="nofollow">https://doi.org/10.1021/acsnano.4c07729</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-5CPC59Y9" data-zp-author-date='Kim-and-Lee-2025' data-zp-date-author='2025-Kim-and-Lee' data-zp-date='2025' data-zp-year='2025' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Kim, J. and Lee, S.J. (2025) "<a class='zp-ItemURL' target='_blank' href='https://linkinghub.elsevier.com/retrieve/pii/S014381662500079X'>AI-based autofocusing of red blood cells in digital in-line holographic microscopy</a>," <i>Optics and Lasers in Engineering</i>, 188, p. 108892. Available at: <a href="https://doi.org/10.1016/j.optlaseng.2025.108892" rel="nofollow">https://doi.org/10.1016/j.optlaseng.2025.108892</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-Y6KMZ8RU" data-zp-author-date='Berrevoets-et-al.-2024-11-29' data-zp-date-author='2024-11-29-Berrevoets-et-al.' data-zp-date='2024-11-29' data-zp-year='2024' data-zp-itemtype='preprint' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Berrevoets, E.S. <i>et al.</i> (2024) "<a class='zp-ItemURL' target='_blank' href='https://www.biorxiv.org/content/10.1101/2024.11.28.625872v1'>Smart 3D super-resolution microscopy reveals the architecture of the RNA scaffold in a nuclear body</a>." bioRxiv. Available at: <a href="https://doi.org/10.1101/2024.11.28.625872" rel="nofollow">https://doi.org/10.1101/2024.11.28.625872</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-DFIINUT8" data-zp-author-date='Brooks-et-al.-2024-10-19' data-zp-date-author='2024-10-19-Brooks-et-al.' data-zp-date='2024-10-19' data-zp-year='2024' data-zp-itemtype='preprint' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Brooks, S. <i>et al.</i> (2024) "<a class='zp-ItemURL' target='_blank' href='https://www.biorxiv.org/content/10.1101/2024.10.17.618897v1'>CelFDrive: Artificial Intelligence assisted microscopy for automated detection of rare events</a>." bioRxiv. Available at: <a href="https://doi.org/10.1101/2024.10.17.618897" rel="nofollow">https://doi.org/10.1101/2024.10.17.618897</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-BLTPJ7K8" data-zp-author-date='Roos-et-al.-2024-09-18' data-zp-date-author='2024-09-18-Roos-et-al.' data-zp-date='2024-09-18' data-zp-year='2024' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Roos, J. <i>et al.</i> (2024) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s41592-024-02404-5'>Arkitekt: streaming analysis and real-time workflows for microscopy</a>," <i>Nature Methods</i> [Preprint]. Available at: <a href="https://doi.org/10.1038/s41592-024-02404-5" rel="nofollow">https://doi.org/10.1038/s41592-024-02404-5</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-Y63EJFYD" data-zp-author-date='Roos-et-al.-2024-09-18' data-zp-date-author='2024-09-18-Roos-et-al.' data-zp-date='2024-09-18' data-zp-year='2024' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Roos, J. <i>et al.</i> (2024) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s41592-024-02404-5'>Arkitekt: streaming analysis and real-time workflows for microscopy</a>," <i>Nature Methods</i> [Preprint]. Available at: <a href="https://doi.org/10.1038/s41592-024-02404-5" rel="nofollow">https://doi.org/10.1038/s41592-024-02404-5</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-5Z8J3KML" data-zp-author-date='Marin-et-al.-2024-09-11' data-zp-date-author='2024-09-11-Marin-et-al.' data-zp-date='2024-09-11' data-zp-year='2024' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Marin, Z. <i>et al.</i> (2024) "<a class='zp-ItemURL' target='_blank' href='https://doi.org/10.1038/s41592-024-02413-4'>Navigate: an open-source platform for smart light-sheet microscopy</a>," <i>Nature Methods</i> [Preprint]. Available at: </div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-C2H2QVG6" data-zp-author-date='Peters-et-al.-2024-08-02' data-zp-date-author='2024-08-02-Peters-et-al.' data-zp-date='2024-08-02' data-zp-year='2024' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Peters, J.J.P. <i>et al.</i> (2024) "<a class='zp-ItemURL' target='_blank' href='https://www.science.org/doi/10.1126/science.ado8579'>Event-responsive scanning transmission electron microscopy</a>," <i>Science</i>, 385(6708), pp. 549–553. Available at: <a href="https://doi.org/10.1126/science.ado8579" rel="nofollow">https://doi.org/10.1126/science.ado8579</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-KYCN7A4V" data-zp-author-date='Pomp-et-al.-2024-03-21' data-zp-date-author='2024-03-21-Pomp-et-al.' data-zp-date='2024-03-21' data-zp-year='2024' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Pomp, W., Meeussen, J.V.W. and Lenstra, T.L. (2024) "<a class='zp-ItemURL' target='_blank' href='https://www.cell.com/molecular-cell/abstract/S1097-2765(24)00056-X'>Transcription factor exchange enables prolonged transcriptional bursts</a>," <i>Molecular Cell</i>, 84(6), pp. 1036-1048.e9. Available at: <a href="https://doi.org/10.1016/j.molcel.2024.01.020" rel="nofollow">https://doi.org/10.1016/j.molcel.2024.01.020</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-QUM2DVR6" data-zp-author-date='Morgado-et-al.-2024-03-06' data-zp-date-author='2024-03-06-Morgado-et-al.' data-zp-date='2024-03-06' data-zp-year='2024' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Morgado, L. <i>et al.</i> (2024) "<a class='zp-ItemURL' target='_blank' href='https://onlinelibrary.wiley.com/doi/10.1111/jmi.13282'>The rise of data‐driven microscopy powered by machine learning</a>," <i>Journal of Microscopy</i>, p. jmi.13282. Available at: <a href="https://doi.org/10.1111/jmi.13282" rel="nofollow">https://doi.org/10.1111/jmi.13282</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-FLSWKSGB" data-zp-author-date='Shroff-et-al.-2024-01-02' data-zp-date-author='2024-01-02-Shroff-et-al.' data-zp-date='2024-01-02' data-zp-year='2024' data-zp-itemtype='preprint' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Shroff, H. <i>et al.</i> (2024) "<a class='zp-ItemURL' target='_blank' href='http://arxiv.org/abs/2401.01438'>Live-cell imaging powered by computation</a>." arXiv. Available at: <a href="https://doi.org/10.48550/arXiv.2401.01438" rel="nofollow">https://doi.org/10.48550/arXiv.2401.01438</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-5Q2SZY6M" data-zp-author-date='Shi-et-al.-2024-01-02' data-zp-date-author='2024-01-02-Shi-et-al.' data-zp-date='2024-01-02' data-zp-year='2024' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Shi, Y. <i>et al.</i> (2024) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s41592-023-02126-0'>Smart lattice light-sheet microscopy for imaging rare and complex cellular events</a>," <i>Nature Methods</i>, pp. 1–10. Available at: <a href="https://doi.org/10.1038/s41592-023-02126-0" rel="nofollow">https://doi.org/10.1038/s41592-023-02126-0</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-FQ7UJRZE" data-zp-author-date='Lei-et-al.-2023-12-05' data-zp-date-author='2023-12-05-Lei-et-al.' data-zp-date='2023-12-05' data-zp-year='2023' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Lei, W. <i>et al.</i> (2023) "<a class='zp-ItemURL' target='_blank' href='http://arxiv.org/abs/2310.18351'>BioImage.IO Chatbot: A Personalized Assistant for BioImage Analysis Augmented by Community Knowledge Base</a>." Available at: <a href="https://doi.org/10.5281/zenodo.10032227" rel="nofollow">https://doi.org/10.5281/zenodo.10032227</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-PAJYPWZV" data-zp-author-date='Pylvänäinen-et-al.-2023-12-01' data-zp-date-author='2023-12-01-Pylvänäinen-et-al.' data-zp-date='2023-12-01' data-zp-year='2023' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Pylvänäinen, J.W. <i>et al.</i> (2023) "<a class='zp-ItemURL' target='_blank' href='https://www.sciencedirect.com/science/article/pii/S0955067423001205'>Live-cell imaging in the deep learning era</a>," <i>Current Opinion in Cell Biology</i>, 85, p. 102271. Available at: <a href="https://doi.org/10.1016/j.ceb.2023.102271" rel="nofollow">https://doi.org/10.1016/j.ceb.2023.102271</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-I3Y6H3S2" data-zp-author-date='Cabriel-et-al.-2023-12' data-zp-date-author='2023-12-Cabriel-et-al.' data-zp-date='2023-12' data-zp-year='2023' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Cabriel, C. <i>et al.</i> (2023) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s41566-023-01308-8'>Event-based vision sensor for fast and dense single-molecule localization microscopy</a>," <i>Nature Photonics</i>, 17(12), pp. 1105–1113. Available at: <a href="https://doi.org/10.1038/s41566-023-01308-8" rel="nofollow">https://doi.org/10.1038/s41566-023-01308-8</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-CUSN9WPC" data-zp-author-date='Balasubramanian-et-al.-2023-10-28' data-zp-date-author='2023-10-28-Balasubramanian-et-al.' data-zp-date='2023-10-28' data-zp-year='2023' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Balasubramanian, H. <i>et al.</i> (2023) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s42003-023-05468-9'>Imagining the future of optical microscopy: everything, everywhere, all at once</a>," <i>Communications Biology</i>, 6(1), pp. 1–12. Available at: <a href="https://doi.org/10.1038/s42003-023-05468-9" rel="nofollow">https://doi.org/10.1038/s42003-023-05468-9</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-SD9WC6DX" data-zp-author-date='Gómez-de-Mariscal-et-al.-2023-08-08' data-zp-date-author='2023-08-08-Gómez-de-Mariscal-et-al.' data-zp-date='2023-08-08' data-zp-year='2023' data-zp-itemtype='preprint' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Gómez-de-Mariscal, E. <i>et al.</i> (2023) "<a class='zp-ItemURL' target='_blank' href='http://arxiv.org/abs/2308.04387'>Harnessing Artificial Intelligence To Reduce Phototoxicity in Live Imaging</a>." arXiv. Available at: <a href="https://doi.org/10.48550/arXiv.2308.04387" rel="nofollow">https://doi.org/10.48550/arXiv.2308.04387</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-Q2A9GDKT" data-zp-author-date='Chai-et-al.-2023-07-19' data-zp-date-author='2023-07-19-Chai-et-al.' data-zp-date='2023-07-19' data-zp-year='2023' data-zp-itemtype='preprint' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Chai, B. <i>et al.</i> (2023) "<a class='zp-ItemURL' target='_blank' href='http://arxiv.org/abs/2307.10500'>Opportunities and challenges for deep learning in cell dynamics research</a>." arXiv. Available at: (Accessed: February 14, 2024).</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-BUIQZUCI" data-zp-author-date='Nature-Methods-2023-07' data-zp-date-author='2023-07-Nature-Methods' data-zp-date='2023-07' data-zp-year='2023' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Nature Methods (2023) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s41592-023-01950-8'>What's next for bioimage analysis?</a>," <i>Nature Methods</i>, 20(7), pp. 945–946. Available at: <a href="https://doi.org/10.1038/s41592-023-01950-8" rel="nofollow">https://doi.org/10.1038/s41592-023-01950-8</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-69VUKG2U" data-zp-author-date='Carpenter-et-al.-2023-07' data-zp-date-author='2023-07-Carpenter-et-al.' data-zp-date='2023-07' data-zp-year='2023' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Carpenter, A.E., Cimini, B.A. and Eliceiri, K.W. (2023) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s41592-023-01912-0'>Smart microscopes of the future</a>," <i>Nature Methods</i>, 20(7), pp. 962–964. Available at: <a href="https://doi.org/10.1038/s41592-023-01912-0" rel="nofollow">https://doi.org/10.1038/s41592-023-01912-0</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-EAN93DZP" data-zp-author-date='Daetwyler-and-Fiolka-2023-05-09' data-zp-date-author='2023-05-09-Daetwyler-and-Fiolka' data-zp-date='2023-05-09' data-zp-year='2023' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Daetwyler, S. and Fiolka, R.P. (2023) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s42003-023-04857-4'>Light-sheets and smart microscopy, an exciting future is dawning</a>," <i>Communications Biology</i>, 6(1), pp. 1–11. Available at: <a href="https://doi.org/10.1038/s42003-023-04857-4" rel="nofollow">https://doi.org/10.1038/s42003-023-04857-4</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-UL82WX7E" data-zp-author-date='André-et-al.-2023-03-27' data-zp-date-author='2023-03-27-André-et-al.' data-zp-date='2023-03-27' data-zp-year='2023' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">André, O. <i>et al.</i> (2023) "<a class='zp-ItemURL' target='_blank' href='https://www.cell.com/cell-reports-methods/abstract/S2667-2375(23)00030-9'>Data-driven microscopy allows for automated context-specific acquisition of high-fidelity image data</a>," <i>Cell Reports Methods</i>, 3(3). Available at: <a href="https://doi.org/10.1016/j.crmeth.2023.100419" rel="nofollow">https://doi.org/10.1016/j.crmeth.2023.100419</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-P474WSZT" data-zp-author-date='Volpe-et-al.-2023-03-07' data-zp-date-author='2023-03-07-Volpe-et-al.' data-zp-date='2023-03-07' data-zp-year='2023' data-zp-itemtype='preprint' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Volpe, G. <i>et al.</i> (2023) "<a class='zp-ItemURL' target='_blank' href='http://arxiv.org/abs/2303.03793'>Roadmap on Deep Learning for Microscopy</a>." arXiv. Available at: <a href="https://doi.org/10.48550/arXiv.2303.03793" rel="nofollow">https://doi.org/10.48550/arXiv.2303.03793</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-GZPGA3QN" data-zp-author-date='Madhusoodanan-2023-02-07' data-zp-date-author='2023-02-07-Madhusoodanan' data-zp-date='2023-02-07' data-zp-year='2023' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Madhusoodanan, J. (2023) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/d41586-023-00336-2'>Smart microscopes spot fleeting biology</a>," <i>Nature</i>, 614(7947), pp. 378–380. Available at: <a href="https://doi.org/10.1038/d41586-023-00336-2" rel="nofollow">https://doi.org/10.1038/d41586-023-00336-2</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-YPXBQ6F2" data-zp-author-date='Ma-and-Wang-2023' data-zp-date-author='2023-Ma-and-Wang' data-zp-date='2023' data-zp-year='2023' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Ma, J. and Wang, B. (2023) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s41592-023-01885-0'>Towards foundation models of biological image segmentation</a>," <i>Nature Methods</i>, 20(7), pp. 953–955. Available at: <a href="https://doi.org/10.1038/s41592-023-01885-0" rel="nofollow">https://doi.org/10.1038/s41592-023-01885-0</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-C6VDYKVG" data-zp-author-date='Merino-Casallo-et-al.-2022-12-31' data-zp-date-author='2022-12-31-Merino-Casallo-et-al.' data-zp-date='2022-12-31' data-zp-year='2022' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Merino-Casallo, F. <i>et al.</i> (2022) "<a class='zp-ItemURL' target='_blank' href='https://doi.org/10.1080/19336918.2022.2055520'>Unravelling cell migration: defining movement from the cell surface</a>," <i>Cell Adhesion & Migration</i>, 16(1), pp. 25–64. Available at: </div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-W39S3ZSK" data-zp-author-date='Lan-et-al.-2022-12-01' data-zp-date-author='2022-12-01-Lan-et-al.' data-zp-date='2022-12-01' data-zp-year='2022' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Lan, T.-H. <i>et al.</i> (2022) "<a class='zp-ItemURL' target='_blank' href='https://www.sciencedirect.com/science/article/pii/S0168952522001408'>Optogenetics for transcriptional programming and genetic engineering</a>," <i>Trends in Genetics</i>, 38(12), pp. 1253–1270. Available at: <a href="https://doi.org/10.1016/j.tig.2022.05.014" rel="nofollow">https://doi.org/10.1016/j.tig.2022.05.014</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-QKZ278FZ" data-zp-author-date='Alvelid-et-al.-2022-10' data-zp-date-author='2022-10-Alvelid-et-al.' data-zp-date='2022-10' data-zp-year='2022' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Alvelid, J. <i>et al.</i> (2022) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s41592-022-01588-y'>Event-triggered STED imaging</a>," <i>Nature Methods</i>, 19(10), pp. 1268–1275. Available at: <a href="https://doi.org/10.1038/s41592-022-01588-y" rel="nofollow">https://doi.org/10.1038/s41592-022-01588-y</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-67C3RZY6" data-zp-author-date='Mahecic-et-al.-2022-10' data-zp-date-author='2022-10-Mahecic-et-al.' data-zp-date='2022-10' data-zp-year='2022' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Mahecic, D. <i>et al.</i> (2022) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s41592-022-01589-x'>Event-driven acquisition for content-enriched microscopy</a>," <i>Nature Methods</i>, 19(10), pp. 1262–1267. Available at: <a href="https://doi.org/10.1038/s41592-022-01589-x" rel="nofollow">https://doi.org/10.1038/s41592-022-01589-x</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-KK2INVGX" data-zp-author-date='Mol-and-Vlijm-2022-09-30' data-zp-date-author='2022-09-30-Mol-and-Vlijm' data-zp-date='2022-09-30' data-zp-year='2022' data-zp-itemtype='preprint' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Mol, F.N. and Vlijm, R. (2022) "<a class='zp-ItemURL' target='_blank' href='https://www.biorxiv.org/content/10.1101/2022.09.29.510126v1'>Automated STED nanoscopy for high-throughput imaging of cellular structures</a>." bioRxiv. Available at: <a href="https://doi.org/10.1101/2022.09.29.510126" rel="nofollow">https://doi.org/10.1101/2022.09.29.510126</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-3GEXFHP8" data-zp-author-date='Dibrov-2022-08-12' data-zp-date-author='2022-08-12-Dibrov' data-zp-date='2022-08-12' data-zp-year='2022' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Dibrov, A. (2022) "<a class='zp-ItemURL' target='_blank' href='https://tud.qucosa.de/id/qucosa:80379'>Towards Smarter Fluorescence Microscopy: Enabling Adaptive Acquisition Strategies With Optimized Photon Budget</a>." Available at: </div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-MZKL54LI" data-zp-author-date='Chiron-et-al.-2022-07-08' data-zp-date-author='2022-07-08-Chiron-et-al.' data-zp-date='2022-07-08' data-zp-year='2022' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Chiron, L. <i>et al.</i> (2022) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s41598-022-15207-5'>CyberSco.Py an open-source software for event-based, conditional microscopy</a>," <i>Scientific Reports</i>, 12(1), p. 11579. Available at: <a href="https://doi.org/10.1038/s41598-022-15207-5" rel="nofollow">https://doi.org/10.1038/s41598-022-15207-5</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->
			</div><!-- .zp-zp-SEO-Content -->
		</div><!-- .zp-List -->
	</div><!--.zp-Zotpress-->


    <div class="zotpress-table-container"></div>
  </div>

  <!-- Individual Collections -->
  <div class="zotpress-section" data-collection="IYTK29JT">
    <div id="zotpress-132eaea631fc7ef3d8c38e6e9875b814" class="zp-Zotpress zp-Zotpress-Bib wp-block-group">

		<span class="ZP_API_USER_ID ZP_ATTR">5384098</span>
		<span class="ZP_ITEM_KEY ZP_ATTR"></span>
		<span class="ZP_COLLECTION_ID ZP_ATTR">IYTK29JT</span>
		<span class="ZP_TAG_ID ZP_ATTR"></span>
		<span class="ZP_AUTHOR ZP_ATTR"></span>
		<span class="ZP_YEAR ZP_ATTR"></span>
        <span class="ZP_ITEMTYPE ZP_ATTR"></span>
		<span class="ZP_INCLUSIVE ZP_ATTR">1</span>
		<span class="ZP_STYLE ZP_ATTR">harvard1</span>
		<span class="ZP_LIMIT ZP_ATTR">50</span>
		<span class="ZP_SORTBY ZP_ATTR">date</span>
		<span class="ZP_ORDER ZP_ATTR">desc</span>
		<span class="ZP_TITLE ZP_ATTR"></span>
		<span class="ZP_SHOWIMAGE ZP_ATTR"></span>
		<span class="ZP_SHOWTAGS ZP_ATTR">1</span>
		<span class="ZP_DOWNLOADABLE ZP_ATTR">1</span>
		<span class="ZP_NOTES ZP_ATTR"></span>
		<span class="ZP_ABSTRACT ZP_ATTR"></span>
		<span class="ZP_CITEABLE ZP_ATTR"></span>
		<span class="ZP_TARGET ZP_ATTR">1</span>
		<span class="ZP_URLWRAP ZP_ATTR">title</span>
		<span class="ZP_FORCENUM ZP_ATTR"></span>
        <span class="ZP_HIGHLIGHT ZP_ATTR"></span>
        <span class="ZP_POSTID ZP_ATTR">27</span>
		<span class="ZOTPRESS_PLUGIN_URL ZP_ATTR"><a href="/wp-content/plugins/zotpress/">/wp-content/plugins/zotpress/</a></span>

		<div class="zp-List loading">
			<div class="zp-SEO-Content">
				<span class="ZP_JSON ZP_ATTR">%7B%22status%22%3A%22success%22%2C%22updateneeded%22%3Afalse%2C%22instance%22%3Afalse%2C%22meta%22%3A%7B%22request_last%22%3A0%2C%22request_next%22%3A0%2C%22used_cache%22%3Atrue%7D%2C%22data%22%3A%5B%7B%22key%22%3A%22BULWHJV6%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Leger%20et%20al.%22%2C%22parsedDate%22%3A%222025-12-14%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BLeger%2C%20D.%2C%20Ivanisevic%2C%20M.%20and%20Lenart%2C%20P.%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.biorxiv.org%5C%2Fcontent%5C%2F10.64898%5C%2F2025.12.11.693650v1%26%23039%3B%26gt%3BAutoSTED%3A%20An%20automated%20workflow%20for%20STED%20super-resolution%20imaging%20of%20cell%20nuclei%26lt%3B%5C%2Fa%26gt%3B.%26quot%3B%20bioRxiv.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.64898%5C%2F2025.12.11.693650.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22preprint%22%2C%22title%22%3A%22AutoSTED%3A%20An%20automated%20workflow%20for%20STED%20super-resolution%20imaging%20of%20cell%20nuclei%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Dorian%22%2C%22lastName%22%3A%22Leger%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Milena%22%2C%22lastName%22%3A%22Ivanisevic%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Peter%22%2C%22lastName%22%3A%22Lenart%22%7D%5D%2C%22abstractNote%22%3A%22The%20traditional%20microscopy%20workflow%20involves%20manual%20selection%20of%20objects%20one%20after%20the%20other.%20This%20is%20not%20only%20time%20consuming%2C%20but%20inevitably%20introduces%20human%20bias.%20Here%2C%20we%20describe%20a%20workflow%2C%20AutoSTED%20for%20identification%20of%20cell%20nuclei%20on%20wide-field%20fluorescence%20images%20and%20subsequent%20automated%20imaging%20of%20these%20nuclei%20at%20super-resolution%20by%20Stimulated%20Emission%20Depletion%20%28STED%29%20microscopy.%20We%20demonstrate%20that%20in%20an%20overnight%20run%20we%20are%20able%20to%20image%20hundreds%20of%20cells%20selected%20without%20bias.%20The%20resolution%20afforded%20by%20STED%20allowed%20straightforward%20identification%20and%20counting%20of%20individual%20nuclear%20pore%20complexes%20%28NPCs%29%2C%20and%20enabled%20us%20to%20automatically%20extract%20parameters%20such%20as%20NPC%20density%20under%20different%20experimental%20conditions.%20Together%2C%20we%20illustrate%20how%20automation%20enables%20optimal%20use%20of%20instrument%20time%20to%20produce%20large%20unbiased%20datasets.%22%2C%22genre%22%3A%22%22%2C%22repository%22%3A%22bioRxiv%22%2C%22archiveID%22%3A%22%22%2C%22date%22%3A%222025-12-14%22%2C%22DOI%22%3A%2210.64898%5C%2F2025.12.11.693650%22%2C%22citationKey%22%3A%22%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.biorxiv.org%5C%2Fcontent%5C%2F10.64898%5C%2F2025.12.11.693650v1%22%2C%22language%22%3A%22en%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222025-12-16T13%3A19%3A02Z%22%2C%22tags%22%3A%5B%5D%7D%7D%2C%7B%22key%22%3A%22NTMGRK8D%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Redondo%20and%20Paszek%22%2C%22parsedDate%22%3A%222025-12-01%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BRedondo%2C%20J.A.%20and%20Paszek%2C%20P.%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fcames.ippt.pan.pl%5C%2Findex.php%5C%2Fcames%5C%2Farticle%5C%2Fview%5C%2F1945%26%23039%3B%26gt%3BAI-Assisted%20Microscopy%20for%20Infection%20Biology%3A%20Advances%20in%20High-Content%20Imaging%20of%20Host-Pathogen%20Interactions%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BComputer%20Assisted%20Methods%20in%20Engineering%20and%20Science%26lt%3B%5C%2Fi%26gt%3B%20%5BPreprint%5D.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.24423%5C%2Fcames.2025.1945.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22AI-Assisted%20Microscopy%20for%20Infection%20Biology%3A%20Advances%20in%20High-Content%20Imaging%20of%20Host-Pathogen%20Interactions%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Juan%20Alfonso%22%2C%22lastName%22%3A%22Redondo%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Pawel%22%2C%22lastName%22%3A%22Paszek%22%7D%5D%2C%22abstractNote%22%3A%22Advances%20in%20high-content%20microscopy%20and%20artificial%20intelligence%20%28AI%29%20are%20transforming%20the%20quantitative%20study%20of%20infection%20biology.%20Automated%20imaging%20platforms%20now%20enable%20rapid%2C%20large-scale%20acquisition%20of%20host-pathogen%20interactions%20across%20thousands%20of%20cells%20and%20multiple%20experimental%20conditions.%20When%20combined%20with%20AI-based%20segmentation%2C%20these%20workflows%20extract%20infection-relevant%20features%20such%20as%20pathogen%20load%2C%20intracellular%20localization%2C%20and%20host%20response%20markers%20at%20single-cell%20resolution.%20Deep-learning%20models%20have%20proven%20especially%20powerful%2C%20outperforming%20classical%20threshold-based%20methods%20under%20different%20imaging%20conditions%2C%20reducing%20reliance%20on%20manual%20annotation%2C%20and%20detecting%20rare%20infection%20outcomes.%20Beyond%20robust%20image%20analysis%2C%20these%20approaches%20generate%20scalable%20and%20reproducible%20datasets%20that%20can%20be%20integrated%20with%20computational%20modelling%20and%20systems%20biology%2C%20providing%20predictive%20insight%20into%20infection%20dynamics.%20This%20review%20highlights%20recent%20progress%20in%20AI-assisted%20microscopy%20for%20bacterial%20infection%20and%20outlines%20future%20directions%20toward%20multimodal%20integration%2C%20clinical%20translation%2C%20and%20open-source%20tool%20development.%22%2C%22date%22%3A%222025-12-01%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.24423%5C%2Fcames.2025.1945%22%2C%22ISSN%22%3A%222956-5839%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fcames.ippt.pan.pl%5C%2Findex.php%5C%2Fcames%5C%2Farticle%5C%2Fview%5C%2F1945%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222025-12-09T11%3A53%3A38Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22artificial%20intelligence%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22cell-to-cell%20variability%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22cellular%20heterogeneity%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22deep%20learning%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22host-pathogen%20interactions%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22infection%20biology%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22machine%20learning%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22single-cell%20biology%22%2C%22type%22%3A1%7D%5D%7D%7D%2C%7B%22key%22%3A%22PGILC555%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Cosenza%20et%20al.%22%2C%22parsedDate%22%3A%222025-12%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BCosenza%2C%20M.R.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41586-025-09632-5%26%23039%3B%26gt%3BOrigins%20of%20chromosome%20instability%20unveiled%20by%20coupled%20imaging%20and%20genomics%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%26lt%3B%5C%2Fi%26gt%3B%2C%20648%288093%29%2C%20pp.%20383%5Cu2013393.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41586-025-09632-5.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Origins%20of%20chromosome%20instability%20unveiled%20by%20coupled%20imaging%20and%20genomics%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Marco%20Raffaele%22%2C%22lastName%22%3A%22Cosenza%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Alice%22%2C%22lastName%22%3A%22Gaiatto%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22B%5Cu00fc%5Cu015fra%22%2C%22lastName%22%3A%22Erarslan%20Uysal%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22%5Cu00c1lvaro%22%2C%22lastName%22%3A%22Andrades%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Nina%20Luisa%22%2C%22lastName%22%3A%22Sautter%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Marina%22%2C%22lastName%22%3A%22Simunovic%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Michael%20Adrian%22%2C%22lastName%22%3A%22Jendrusch%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Sonia%22%2C%22lastName%22%3A%22Zumalave%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Tobias%22%2C%22lastName%22%3A%22Rausch%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Aliaksandr%22%2C%22lastName%22%3A%22Halavatyi%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Eva-Maria%22%2C%22lastName%22%3A%22Geissen%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Joshua%20Lucas%22%2C%22lastName%22%3A%22Eigenmann%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Thomas%22%2C%22lastName%22%3A%22Weber%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Patrick%22%2C%22lastName%22%3A%22Hasenfeld%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Eva%22%2C%22lastName%22%3A%22Benito%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Catherine%22%2C%22lastName%22%3A%22Stober%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Isidro%22%2C%22lastName%22%3A%22Cortes-Ciriano%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Andreas%20E.%22%2C%22lastName%22%3A%22Kulozik%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Rainer%22%2C%22lastName%22%3A%22Pepperkok%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jan%20O.%22%2C%22lastName%22%3A%22Korbel%22%7D%5D%2C%22abstractNote%22%3A%22Somatic%20chromosome%20instability%20results%20in%20widespread%20structural%20and%20numerical%20chromosomal%20abnormalities%20%28CAs%29%20during%20cancer%20evolution1%5Cu20133.%20Although%20CAs%20have%20been%20linked%20to%20mitotic%20errors%20resulting%20in%20the%20emergence%20of%20nuclear%20atypia4%5Cu20137%2C%20the%20underlying%20processes%20and%20rates%20of%20spontaneous%20CA%20formation%20in%20human%20cells%20are%20underexplored.%20Here%20we%20introduce%20machine-learning-assisted%20genomics%20and%20imaging%20convergence%20%28MAGIC%29%5Cu2014an%20autonomously%20operated%20platform%20that%20integrates%20live-cell%20imaging%20of%20micronucleated%20cells%2C%20machine%20learning%20on-the-fly%20and%20single-cell%20genomics%20to%20systematically%20investigate%20CA%20formation.%20Applying%20MAGIC%20to%20near-diploid%2C%20non-transformed%20cell%20lines%2C%20we%20track%20de%20novo%20CAs%20over%20successive%20cell%20cycles%2C%20highlighting%20the%20common%20role%20of%20dicentric%20chromosomes%20as%20initiating%20events.%20We%20determine%20the%20baseline%20CA%20mutation%20rate%2C%20which%20approximately%20doubles%20in%20TP53-deficient%20cells%2C%20and%20observe%20that%20chromosome%20losses%20arise%20more%20frequently%20than%20gains.%20The%20targeted%20induction%20of%20DNA%20double-strand%20breaks%20along%20chromosome%20arms%20triggers%20distinct%20CA%20processes%2C%20revealing%20stable%20isochromosomes%2C%20coordinated%20segregation%20and%20amplification%20of%20isoacentric%20segments%20in%20multiples%20of%20two%2C%20as%20well%20as%20complex%20CA%20outcomes%2C%20influenced%20by%20the%20chromosomal%20break%20location.%20Our%20data%20contrast%20de%20novo%20CA%20spectra%20from%20somatic%20mutational%20landscapes%20after%20selection%20occurred.%20The%20experimentation%20enabled%20by%20MAGIC%20advances%20the%20dissection%20of%20DNA%20rearrangement%20processes%2C%20shedding%20light%20on%20fundamental%20determinants%20of%20chromosomal%20instability.%22%2C%22date%22%3A%222025-12%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs41586-025-09632-5%22%2C%22ISSN%22%3A%221476-4687%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41586-025-09632-5%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222025-12-16T13%3A11%3A56Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Chromosome%20segregation%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22Genomic%20instability%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22High-throughput%20screening%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22Imaging%22%2C%22type%22%3A1%7D%5D%7D%7D%2C%7B%22key%22%3A%22WFEWMARH%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Berrevoets%20et%20al.%22%2C%22parsedDate%22%3A%222025-11-27%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BBerrevoets%2C%20E.S.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41467-025-65723-x%26%23039%3B%26gt%3BSmart%203D%20super-resolution%20microscopy%20reveals%20the%20architecture%20of%20the%20RNA%20scaffold%20in%20a%20nuclear%20body%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%20Communications%26lt%3B%5C%2Fi%26gt%3B%2C%2016%281%29%2C%20p.%2010689.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41467-025-65723-x.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Smart%203D%20super-resolution%20microscopy%20reveals%20the%20architecture%20of%20the%20RNA%20scaffold%20in%20a%20nuclear%20body%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Enya%20S.%22%2C%22lastName%22%3A%22Berrevoets%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Laurell%20F.%22%2C%22lastName%22%3A%22Kessler%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ashwin%22%2C%22lastName%22%3A%22Balakrishnan%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ellen%20Kazumi%22%2C%22lastName%22%3A%22Okuda%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Michaela%22%2C%22lastName%22%3A%22M%5Cu00fcller-McNicoll%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Bernd%22%2C%22lastName%22%3A%22Rieger%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Sjoerd%22%2C%22lastName%22%3A%22Stallinga%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Mike%22%2C%22lastName%22%3A%22Heilemann%22%7D%5D%2C%22abstractNote%22%3A%22Small%20subcellular%20organelles%20orchestrate%20key%20cellular%20functions.%20How%20biomolecules%20are%20spatially%20organized%20within%20these%20assemblies%20is%20poorly%20understood.%20Here%2C%20we%20report%20an%20automated%20super-resolution%20imaging%20and%20analysis%20workflow%20that%20integrates%20confocal%20microscopy%2C%20morphological%20object%20screening%2C%20targeted%203D%20super-resolution%20STED%20microscopy%20and%20quantitative%20image%20analysis.%20Using%20this%20smart%20microscopy%20workflow%2C%20we%20target%20the%203D%20organization%20of%20NEAT1%2C%20an%20architectural%20RNA%20that%20constitutes%20the%20structural%20backbone%20of%20paraspeckles%2C%20a%20membraneless%20nuclear%20organelle.%20Using%20site-specific%20labeling%2C%20morphological%20sorting%20and%20particle%20averaging%2C%20we%20reconstruct%20the%20morphological%20space%20of%20paraspeckles%20along%20their%20development%20cycle%20from%20over%2010%2C000%20individual%20particles.%20Applying%20spherical%20harmonics%20analysis%2C%20we%20report%20so-far%20unknown%20heterotypes%20of%20NEAT1%20RNA%20organization.%20By%20integrating%20multi-positional%20labeling%2C%20we%20determine%20the%20coarse%20conformation%20of%20NEAT1%20within%20the%20organelle%20and%20show%20that%20the%203%5Cu2019%20end%20forms%20a%20loop-like%20structure%20at%20the%20surface%20of%20the%20paraspeckle.%20Our%20study%20reveals%20key%20structural%20features%20of%20paraspeckle%20structure%20and%20growth%2C%20as%20well%20as%20the%20molecular%20organization%20of%20its%20scaffolding%20RNA.%22%2C%22date%22%3A%222025-11-27%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs41467-025-65723-x%22%2C%22ISSN%22%3A%222041-1723%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41467-025-65723-x%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222025-12-01T13%3A04%3A56Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Imaging%20techniques%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22Nanoscale%20biophysics%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22Super-resolution%20microscopy%22%2C%22type%22%3A1%7D%5D%7D%7D%2C%7B%22key%22%3A%222C2QZA49%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22London%20et%20al.%22%2C%22parsedDate%22%3A%222025-11-03%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BLondon%2C%20J.A.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.biorxiv.org%5C%2Fcontent%5C%2F10.1101%5C%2F2025.10.31.685671v1%26%23039%3B%26gt%3BA%20Python%20framework%20for%20magnetic%20tweezers%20real-time%20image%20processing%20and%20microscope%20control%26lt%3B%5C%2Fa%26gt%3B.%26quot%3B%20bioRxiv.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1101%5C%2F2025.10.31.685671.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22preprint%22%2C%22title%22%3A%22A%20Python%20framework%20for%20magnetic%20tweezers%20real-time%20image%20processing%20and%20microscope%20control%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22James%20A.%22%2C%22lastName%22%3A%22London%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Abhishek%20K.%22%2C%22lastName%22%3A%22Singh%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Teague%20C.%22%2C%22lastName%22%3A%22Svendsen%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Naciye%20Esma%22%2C%22lastName%22%3A%22Tirtom%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Zachary%20A.%22%2C%22lastName%22%3A%22Root%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Richard%22%2C%22lastName%22%3A%22Fishel%22%7D%5D%2C%22abstractNote%22%3A%22Magnetic%20tweezers%20are%20a%20popular%20biophysical%20instrument%20for%20manipulating%20and%20measuring%20single%20molecules.%20Most%20laboratories%20rely%20on%20custom-built%20setups%20tailored%20to%20specific%20experiments%2C%20resulting%20in%20a%20variety%20of%20hardware%20and%20software%20implementations.%20Typically%2C%20image%20acquisition%20and%20hardware%20control%20are%20automated%20via%20LabVIEW%20and%20specialized%20C%5C%2FC%2B%2B%5C%2FCUDA%20libraries%20for%20real-time%20video%20processing.%20Live%20processing%20eliminates%20the%20need%20to%20store%20raw%20video%2C%20enabling%20high%20throughput%2C%20fast%20acquisition%20rates%2C%20and%20simplified%20experimental%20workflows.%20However%2C%20no%20open-source%2C%20general-purpose%20software%20framework%20currently%20unifies%20these%20capabilities%20for%20magnetic%20tweezers%20experiments.%20Here%2C%20we%20introduce%20MagTrack%20and%20MagScope%2C%20open-source%20Python-based%20tools%20designed%20to%20fill%20this%20gap.%20MagTrack%20is%20an%20image-processing%20library%20that%20efficiently%20determines%20bead-positions%20from%20magnetic-tweezers%20videos%20using%20CPU%20and%5C%2For%20GPU%20computation.%20MagScope%20is%20a%20comprehensive%20software%20framework%20offering%20a%20graphical%20user%20interface%2C%20real-time%20hardware%20control%2C%20data%20acquisition%2C%20and%20video%20processing.%20It%20is%20built%20on%20a%20multiprocessing%20architecture%20for%20responsive%2C%20high-throughput%20computation.%20Together%2C%20MagTrack%20and%20MagScope%20offer%20a%20flexible%2C%20modern%2C%20and%20fully%20customizable%20open-source%20alternative%20to%20proprietary%20or%20fragmented%20systems%2C%20enabling%20laboratories%20to%20adapt%20and%20extend%20the%20framework%20according%20to%20their%20experimental%20and%20programming%20needs.%22%2C%22genre%22%3A%22%22%2C%22repository%22%3A%22bioRxiv%22%2C%22archiveID%22%3A%22%22%2C%22date%22%3A%222025-11-03%22%2C%22DOI%22%3A%2210.1101%5C%2F2025.10.31.685671%22%2C%22citationKey%22%3A%22%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.biorxiv.org%5C%2Fcontent%5C%2F10.1101%5C%2F2025.10.31.685671v1%22%2C%22language%22%3A%22en%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222025-12-01T13%3A16%3A49Z%22%2C%22tags%22%3A%5B%5D%7D%7D%2C%7B%22key%22%3A%223JF4LIGZ%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Mandal%20et%20al.%22%2C%22parsedDate%22%3A%222025-10-14%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BMandal%2C%20I.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41467-025-64105-7%26%23039%3B%26gt%3BEvaluating%20large%20language%20model%20agents%20for%20automation%20of%20atomic%20force%20microscopy%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%20Communications%26lt%3B%5C%2Fi%26gt%3B%2C%2016%281%29%2C%20p.%209104.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41467-025-64105-7.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Evaluating%20large%20language%20model%20agents%20for%20automation%20of%20atomic%20force%20microscopy%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Indrajeet%22%2C%22lastName%22%3A%22Mandal%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jitendra%22%2C%22lastName%22%3A%22Soni%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Mohd%22%2C%22lastName%22%3A%22Zaki%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Morten%20M.%22%2C%22lastName%22%3A%22Smedskjaer%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Katrin%22%2C%22lastName%22%3A%22Wondraczek%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Lothar%22%2C%22lastName%22%3A%22Wondraczek%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Nitya%20Nand%22%2C%22lastName%22%3A%22Gosvami%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22N.%20M.%20Anoop%22%2C%22lastName%22%3A%22Krishnan%22%7D%5D%2C%22abstractNote%22%3A%22Large%20language%20models%20%28LLMs%29%20are%20transforming%20laboratory%20automation%20by%20enabling%20self-driving%20laboratories%20%28SDLs%29%20that%20could%20accelerate%20materials%20research.%20However%2C%20current%20SDL%20implementations%20rely%20on%20rigid%20protocols%20that%20fail%20to%20capture%20the%20adaptability%20and%20intuition%20of%20expert%20scientists%20in%20dynamic%20experimental%20settings.%20Here%2C%20we%20show%20that%20LLM%20agents%20can%20automate%20atomic%20force%20microscopy%5Cu00a0%28AFM%29%20through%20our%20Artificially%20Intelligent%20Lab%20Assistant%20%28AILA%29%20framework.%20Further%2C%20we%20develop%20AFMBench%5Cu2014a%20comprehensive%20evaluation%20suite%20challenging%20LLM%20agents%20across%20the%20complete%20scientific%20workflow%20from%20experimental%20design%20to%20results%20analysis.%20We%20find%20that%20state-of-the-art%20LLMs%20struggle%20with%20basic%20tasks%20and%20coordination%20scenarios.%20Notably%2C%20models%20excelling%20at%20materials%20science%20question-answering%20perform%20poorly%20in%20laboratory%20settings%2C%20showing%20that%20domain%20knowledge%20does%20not%20translate%20to%20experimental%20capabilities.%20Additionally%2C%20we%20observe%20that%20LLM%20agents%20can%20deviate%20from%20instructions%2C%20a%20phenomenon%20referred%20to%20as%5Cu00a0sleepwalking%2C%20raising%20safety%20alignment%20concerns%20for%20SDL%20applications.%20Our%20ablations%20reveal%20that%20multi-agent%20frameworks%20significantly%20outperform%20single-agent%20approaches%2C%20though%20both%20remain%20sensitive%20to%20minor%20changes%20in%20instruction%20formatting%20or%20prompting.%20Finally%2C%20we%20evaluate%20AILA%5Cu2019s%20effectiveness%20in%20increasingly%20advanced%20experiments%5Cu2014AFM%20calibration%2C%20feature%20detection%2C%20mechanical%20property%20measurement%2C%20graphene%20layer%20counting%2C%20and%20indenter%20detection.%20These%20findings%20establish%20the%20necessity%20for%20benchmarking%20and%20robust%20safety%20protocols%20before%20deploying%20LLM%20agents%20as%20autonomous%20laboratory%20assistants%20across%20scientific%20disciplines.%22%2C%22date%22%3A%222025-10-14%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs41467-025-64105-7%22%2C%22ISSN%22%3A%222041-1723%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41467-025-64105-7%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222025-10-20T11%3A01%3A49Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Atomic%20force%20microscopy%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22Computer%20science%22%2C%22type%22%3A1%7D%5D%7D%7D%2C%7B%22key%22%3A%22KY9GPYEJ%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Hickey%20et%20al.%22%2C%22parsedDate%22%3A%222025-10-10%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BHickey%2C%20J.W.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.biorxiv.org%5C%2Fcontent%5C%2F10.1101%5C%2F2025.10.09.680291v1%26%23039%3B%26gt%3BFluid-Squid%3A%20DIY%20Multiplexed%20Imaging%20of%20Cells%20and%20Tissues%26lt%3B%5C%2Fa%26gt%3B.%26quot%3B%20bioRxiv.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1101%5C%2F2025.10.09.680291.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22preprint%22%2C%22title%22%3A%22Fluid-Squid%3A%20DIY%20Multiplexed%20Imaging%20of%20Cells%20and%20Tissues%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22John%20W.%22%2C%22lastName%22%3A%22Hickey%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Hongquan%22%2C%22lastName%22%3A%22Li%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Chiara%22%2C%22lastName%22%3A%22Caraccio%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22You%22%2C%22lastName%22%3A%22Yan%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Kevin%22%2C%22lastName%22%3A%22Marx%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Peter%22%2C%22lastName%22%3A%22Martin%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Hope%20T.%22%2C%22lastName%22%3A%22Leng%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22John%22%2C%22lastName%22%3A%22Fayiah%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Elvis%22%2C%22lastName%22%3A%22Towalid%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Bonnie%22%2C%22lastName%22%3A%22Dighero-Kemp%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Garry%20P.%22%2C%22lastName%22%3A%22Nolan%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Manu%22%2C%22lastName%22%3A%22Prakash%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22David%20R.%22%2C%22lastName%22%3A%22McIlwain%22%7D%5D%2C%22abstractNote%22%3A%22Recent%20advances%20in%20multiplexed%20single-cell%20characterization%20have%20revolutionized%20our%20insight%20into%20cell%20biology%2C%20but%20many%20available%20technologies%20remain%20limited%20by%20high%20costs%20or%20a%20lack%20of%20customizability.%20To%20address%20these%20challenges%2C%20we%20developed%20Fluid-Squid%2C%20a%20cost-effective%2C%20quantitative%20imaging%20platform%20that%20integrates%20automated%20fluidics%20to%20support%20customizable%2C%20do-it-yourself%20multiplexed%20imaging%20workflows.%20Using%20Fluid-Squid%2C%20we%20successfully%20imaged%20fresh%20frozen%20human%20intestinal%20tissues%20with%20a%2036-plex%20oligonucleotide-barcoded%20antibody%20panel%20and%20further%20demonstrated%20the%20feasibility%20of%20lyophilizing%20such%20multiplexed%20panels.%20We%20also%20adapted%20existing%20multiplexed%20imaging%20workflows%20to%20characterize%20individual%20cells%20to%20identify%20immune%20cell%20populations%2C%20phenotype%2C%20and%20antigen-specific%20cells%20from%20mouse%20splenocytes%20and%20human%20peripheral%20blood%20mononuclear%20cells%20%28PBMCs%29%20with%20a%2039-antibody%20panel.%20To%20further%20expand%20its%20utility%2C%20we%20developed%20a%20barcoding%20strategy%20that%20allows%20for%20the%20pooling%20and%20simultaneous%20staining%20of%20multiple%20samples%2C%20reducing%20time%2C%20costs%2C%20and%20batch%20effects%20in%20single-cell%20experiments.%20This%20approach%20facilitated%20rapid%20titration%20to%20optimize%20antibody%20concentrations%20and%20assess%20the%20impact%20of%20various%20blood%20preparation%20methods%20on%20cell%20type%20retention.%20Overall%2C%20our%20work%20provides%20a%20new%20open-source%20framework%20for%20automated%20fluidics%20and%20microscopy%20in%20a%20flexible%2C%20cost-effective%20platform%2C%20empowering%20adaptable%20multiplexed%20characterization%20of%20both%20single%20cells%20and%20tissues.%22%2C%22genre%22%3A%22%22%2C%22repository%22%3A%22bioRxiv%22%2C%22archiveID%22%3A%22%22%2C%22date%22%3A%222025-10-10%22%2C%22DOI%22%3A%2210.1101%5C%2F2025.10.09.680291%22%2C%22citationKey%22%3A%22%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.biorxiv.org%5C%2Fcontent%5C%2F10.1101%5C%2F2025.10.09.680291v1%22%2C%22language%22%3A%22en%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222025-12-01T13%3A16%3A11Z%22%2C%22tags%22%3A%5B%5D%7D%7D%2C%7B%22key%22%3A%22LVWHWJR5%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Friederich%20et%20al.%22%2C%22parsedDate%22%3A%222025-10-01%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BFriederich%2C%20N.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.degruyterbrill.com%5C%2Fdocument%5C%2Fdoi%5C%2F10.1515%5C%2Fauto-2025-0018%5C%2Fhtml%3Frecommended%3Dsidebar%26%23039%3B%26gt%3BEAP4EMSIG%20%5Cu2013%20enhancing%20event-driven%20microscopy%20for%20microfluidic%20single-cell%20analysis%3A%20EAP4EMSIG%20%5Cu2013%20Verbesserung%20der%20ereignisgesteuerten%20Mikroskopie%20f%5Cu00fcr%20die%20mikrofluidische%20Einzelzellanalyse%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3Bat%20-%20Automatisierungstechnik%26lt%3B%5C%2Fi%26gt%3B%2C%2073%2810%29%2C%20pp.%20808%5Cu2013823.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1515%5C%2Fauto-2025-0018.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22EAP4EMSIG%20%5Cu2013%20enhancing%20event-driven%20microscopy%20for%20microfluidic%20single-cell%20analysis%3A%20EAP4EMSIG%20%5Cu2013%20Verbesserung%20der%20ereignisgesteuerten%20Mikroskopie%20f%5Cu00fcr%20die%20mikrofluidische%20Einzelzellanalyse%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Nils%22%2C%22lastName%22%3A%22Friederich%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Angelo%20Jovin%20Yamachui%22%2C%22lastName%22%3A%22Sitcheu%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Annika%22%2C%22lastName%22%3A%22Nassal%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Erenus%22%2C%22lastName%22%3A%22Yildiz%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Matthias%22%2C%22lastName%22%3A%22Pesch%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Maximilian%22%2C%22lastName%22%3A%22Beichter%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Lukas%22%2C%22lastName%22%3A%22Scholtes%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Bahar%22%2C%22lastName%22%3A%22Akbaba%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Thomas%22%2C%22lastName%22%3A%22Lautenschlager%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Oliver%22%2C%22lastName%22%3A%22Neumann%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Dietrich%22%2C%22lastName%22%3A%22Kohlheyer%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Hanno%22%2C%22lastName%22%3A%22Scharr%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Johannes%22%2C%22lastName%22%3A%22Seiffarth%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Katharina%22%2C%22lastName%22%3A%22N%5Cu00f6h%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ralf%22%2C%22lastName%22%3A%22Mikut%22%7D%5D%2C%22abstractNote%22%3A%22Microfluidic%20Live-Cell%20Imaging%20%28MLCI%29%20yields%20data%20on%20microbial%20cell%20factories.%20However%2C%20continuous%20acquisition%20is%20challenging%20as%20high-throughput%20experiments%20often%20lack%20real-time%20insights%2C%20delaying%20responses%20to%20stochastic%20events.%20We%20introduce%20three%20components%20in%20the%20Experiment%20Automation%20Pipeline%20for%20Event-Driven%20Microscopy%20to%20Smart%20Microfluidic%20Single-Cell%20Analysis%20%28EAP4EMSIG%29%3A%20a%20fast%2C%20accurate%20Multi-Layer%20Perceptron%20%28MLP%29-based%20autofocusing%20method%20predicting%20the%20focus%20offset%2C%20an%20evaluation%20of%20real-time%20segmentation%20methods%20and%20a%20real-time%20data%20analysis%20dashboard.%20Our%20MLP-based%20autofocusing%20achieves%20a%20Mean%20Absolute%20Error%20%28MAE%29%20of%200.105%5Cu202f%5Cu00b5m%20with%20inference%20times%20of%2087%5Cu202fms.%20Among%20eleven%20evaluated%20Deep%20Learning%20%28DL%29%20segmentation%20methods%2C%20Cellpose%20reached%20a%20Panoptic%20Quality%20%28PQ%29%20of%2093.36%5Cu202f%25%2C%20while%20a%20distance-based%20method%20was%20fastest%20%28121%5Cu202fms%2C%20Panoptic%20Quality%2093.02%5Cu202f%25%29.%22%2C%22date%22%3A%222025-10-01%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1515%5C%2Fauto-2025-0018%22%2C%22ISSN%22%3A%222196-677X%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.degruyterbrill.com%5C%2Fdocument%5C%2Fdoi%5C%2F10.1515%5C%2Fauto-2025-0018%5C%2Fhtml%3Frecommended%3Dsidebar%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222025-10-27T14%3A30%3A37Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22MLP-based%20autofocusing%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22deep%20learning%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22experiment%20automation%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22live-cell%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22microfluidic%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22segmentation%22%2C%22type%22%3A1%7D%5D%7D%7D%2C%7B%22key%22%3A%22KFNGL64Q%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Zhang%20et%20al.%22%2C%22parsedDate%22%3A%222025-09-08%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BZhang%2C%20N.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fpubs.rsc.org%5C%2Fen%5C%2Fcontent%5C%2Farticlelanding%5C%2F2025%5C%2Flc%5C%2Fd5lc00286a%26%23039%3B%26gt%3BModular%2C%20open-sourced%20multiplexing%20for%20democratizing%20spatial%20omics%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BLab%20on%20a%20Chip%26lt%3B%5C%2Fi%26gt%3B%20%5BPreprint%5D.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1039%5C%2FD5LC00286A.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Modular%2C%20open-sourced%20multiplexing%20for%20democratizing%20spatial%20omics%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Nicholas%22%2C%22lastName%22%3A%22Zhang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Zhou%22%2C%22lastName%22%3A%22Fang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Priyam%22%2C%22lastName%22%3A%22Kadakia%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jamie%22%2C%22lastName%22%3A%22Guo%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Dakshin%22%2C%22lastName%22%3A%22Vijay%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Manoj%22%2C%22lastName%22%3A%22Thapa%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Samuel%22%2C%22lastName%22%3A%22Dembowitz%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Arash%22%2C%22lastName%22%3A%22Grakoui%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ahmet%20F.%22%2C%22lastName%22%3A%22Coskun%22%7D%5D%2C%22abstractNote%22%3A%22Spatial%20omics%20technologies%20have%20revolutionized%20the%20field%20of%20biology%20by%20enabling%20the%20visualization%20of%20biomolecules%20within%20their%20native%20tissue%20context.%20However%2C%20the%20high%20costs%20associated%20with%20proprietary%20instrumentation%2C%20specialized%20reagents%2C%20and%20complex%20workflows%20have%20limited%20the%20broad%20application%20of%20these%20techniques.%20In%20this%20study%2C%20we%20introduce%20Python-based%20robotic%20imaging%20and%20staining%20for%20modular%20spatial%20omics%20%28PRISMS%29%2C%20an%20open-sourced%2C%20automated%20multiplexing%20pipeline%20compatible%20with%20several%20biospecimen%20targets%20and%20streamlined%20microscopy%20software%20tools.%20PRISMS%20utilizes%20a%20liquid%20handling%20robot%20with%20thermal%20control%20to%20enable%20the%20rapid%20and%20automated%20staining%20of%20RNA%20and%20protein%20samples.%20The%20modular%20sample%20holders%20and%20Python%20control%20facilitate%20high-throughput%2C%20single-molecule%20fluorescence%20imaging%20on%20widefield%20and%20confocal%20microscopes.%20We%20successfully%20demonstrated%20the%20versatility%20of%20PRISMS%20by%20imaging%20tissue%20slides%20and%20adherent%20cells.%20We%20demonstrate%20that%20PRISMS%20can%20be%20utilized%20to%20perform%20super-resolved%20imaging%2C%20such%20as%20super-resolution%20radial%20fluctuations%20%28SRRF%29.%20PRISMS%20is%20a%20powerful%20tool%20that%20can%20be%20used%20to%20democratize%20spatial%20omics%20by%20providing%20researchers%20with%20an%20accessible%2C%20reproducible%2C%20and%20cost-effective%20solution%20for%20multiplex%20imaging.%20Specifically%2C%20PRISMS%20is%20an%20open-source%2C%20automated%20multiplexing%20pipeline%20for%20spatial%20omics%2C%20compatible%20with%20several%20sample%20types%20and%20Nikon%20NIS%20Elements%20Basic%20Research%20software%2C%20as%20well%20as%20Python-based%20biodevices.%20It%20performs%20high-throughput%2C%20single-molecule%20fluorescence%20imaging%20both%20on%20widefield%20and%20confocal%20microscopes%2C%20and%20can%20be%20used%20to%20perform%20super-resolved%20imaging%2C%20such%20as%20SRRF.%20Overall%2C%20PRISMS%20is%20a%20powerful%20tool%20that%20can%20be%20used%20to%20democratize%20spatial%20omics%20by%20providing%20researchers%20with%20an%20accessible%2C%20reproducible%2C%20and%20cost-effective%20solution%20for%20multiplex%20imaging.%20This%20open-source%20platform%20will%20enable%20researchers%20to%20push%20the%20boundaries%20of%20spatial%20biology%20and%20make%20groundbreaking%20discoveries.%22%2C%22date%22%3A%222025-09-08%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1039%5C%2FD5LC00286A%22%2C%22ISSN%22%3A%221473-0189%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fpubs.rsc.org%5C%2Fen%5C%2Fcontent%5C%2Farticlelanding%5C%2F2025%5C%2Flc%5C%2Fd5lc00286a%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222025-09-16T08%3A46%3A28Z%22%2C%22tags%22%3A%5B%5D%7D%7D%2C%7B%22key%22%3A%22PA6N7DHN%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Daetwyler%20et%20al.%22%2C%22parsedDate%22%3A%222025-02-12%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BDaetwyler%2C%20S.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-025-02598-2%26%23039%3B%26gt%3BImaging%20of%20cellular%20dynamics%20from%20a%20whole%20organism%20to%20subcellular%20scale%20with%20self-driving%2C%20multiscale%20microscopy%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%20Methods%26lt%3B%5C%2Fi%26gt%3B%2C%20pp.%201%5Cu201310.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41592-025-02598-2.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Imaging%20of%20cellular%20dynamics%20from%20a%20whole%20organism%20to%20subcellular%20scale%20with%20self-driving%2C%20multiscale%20microscopy%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Stephan%22%2C%22lastName%22%3A%22Daetwyler%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Hanieh%22%2C%22lastName%22%3A%22Mazloom-Farsibaf%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Felix%20Y.%22%2C%22lastName%22%3A%22Zhou%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Dagan%22%2C%22lastName%22%3A%22Segal%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Etai%22%2C%22lastName%22%3A%22Sapoznik%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Bingying%22%2C%22lastName%22%3A%22Chen%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jill%20M.%22%2C%22lastName%22%3A%22Westcott%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Rolf%20A.%22%2C%22lastName%22%3A%22Brekken%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Gaudenz%22%2C%22lastName%22%3A%22Danuser%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Reto%22%2C%22lastName%22%3A%22Fiolka%22%7D%5D%2C%22abstractNote%22%3A%22Most%20biological%20processes%2C%20from%20development%20to%20pathogenesis%2C%20span%20multiple%20time%20and%20length%20scales.%20While%20light-sheet%20fluorescence%20microscopy%20has%20become%20a%20fast%20and%20efficient%20method%20for%20imaging%20organisms%2C%20cells%20and%20subcellular%20dynamics%2C%20simultaneous%20observations%20across%20all%20these%20scales%20have%20remained%20challenging.%20Moreover%2C%20continuous%20high-resolution%20imaging%20inside%20living%20organisms%20has%20mostly%20been%20limited%20to%20a%20few%20hours%2C%20as%20regions%20of%20interest%20quickly%20move%20out%20of%20view%20due%20to%20sample%20movement%20and%20growth.%20Here%2C%20we%20present%20a%20self-driving%2C%20multiresolution%20light-sheet%20microscope%20platform%20controlled%20by%20custom%20Python-based%20software%2C%20to%20simultaneously%20observe%20and%20quantify%20subcellular%20dynamics%20in%20the%20context%20of%20entire%20organisms%20in%20vitro%20and%20in%20vivo%20over%20hours%20of%20imaging.%20We%20apply%20the%20platform%20to%20the%20study%20of%20developmental%20processes%2C%20cancer%20invasion%20and%20metastasis%2C%20and%20we%20provide%20quantitative%20multiscale%20analysis%20of%20immune%5Cu2013cancer%20cell%20interactions%20in%20zebrafish%20xenografts.%22%2C%22date%22%3A%222025-02-12%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs41592-025-02598-2%22%2C%22ISSN%22%3A%221548-7105%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-025-02598-2%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222025-04-29T14%3A03%3A45Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Application%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22FPEQIBFZ%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Thomas-%20-Chemin%20et%20al.%22%2C%22parsedDate%22%3A%222025-02-11%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BThomas-%20-Chemin%2C%20O.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fpubs.acs.org%5C%2Fdoi%5C%2F10.1021%5C%2Facsnano.4c07729%26%23039%3B%26gt%3BAdvancing%20High-Throughput%20Cellular%20Atomic%20Force%20Microscopy%20with%20Automation%20and%20Artificial%20Intelligence%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BACS%20Nano%26lt%3B%5C%2Fi%26gt%3B%2C%2019%285%29%2C%20pp.%205045%5Cu20135062.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1021%5C%2Facsnano.4c07729.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Advancing%20High-Throughput%20Cellular%20Atomic%20Force%20Microscopy%20with%20Automation%20and%20Artificial%20Intelligence%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Oph%5Cu00e9lie%22%2C%22lastName%22%3A%22Thomas-%20-Chemin%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22S%5Cu00e9bastien%22%2C%22lastName%22%3A%22Janel%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Zeyd%22%2C%22lastName%22%3A%22Boumehdi%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Child%5Cu00e9rick%22%2C%22lastName%22%3A%22S%5Cu00e9verac%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Emmanuelle%22%2C%22lastName%22%3A%22Trevisiol%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Etienne%22%2C%22lastName%22%3A%22Dague%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Vincent%22%2C%22lastName%22%3A%22Dupr%5Cu00e9s%22%7D%5D%2C%22abstractNote%22%3A%22%22%2C%22date%22%3A%222025-02-11%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1021%5C%2Facsnano.4c07729%22%2C%22ISSN%22%3A%221936-0851%2C%201936-086X%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fpubs.acs.org%5C%2Fdoi%5C%2F10.1021%5C%2Facsnano.4c07729%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222025-04-29T14%3A01%3A41Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%225CPC59Y9%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Kim%20and%20Lee%22%2C%22parsedDate%22%3A%222025%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BKim%2C%20J.%20and%20Lee%2C%20S.J.%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Flinkinghub.elsevier.com%5C%2Fretrieve%5C%2Fpii%5C%2FS014381662500079X%26%23039%3B%26gt%3BAI-based%20autofocusing%20of%20red%20blood%20cells%20in%20digital%20in-line%20holographic%20microscopy%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BOptics%20and%20Lasers%20in%20Engineering%26lt%3B%5C%2Fi%26gt%3B%2C%20188%2C%20p.%20108892.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1016%5C%2Fj.optlaseng.2025.108892.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22AI-based%20autofocusing%20of%20red%20blood%20cells%20in%20digital%20in-line%20holographic%20microscopy%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jihwan%22%2C%22lastName%22%3A%22Kim%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Sang%20Joon%22%2C%22lastName%22%3A%22Lee%22%7D%5D%2C%22abstractNote%22%3A%22%22%2C%22date%22%3A%2205%5C%2F2025%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1016%5C%2Fj.optlaseng.2025.108892%22%2C%22ISSN%22%3A%2201438166%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Flinkinghub.elsevier.com%5C%2Fretrieve%5C%2Fpii%5C%2FS014381662500079X%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222025-04-29T14%3A03%3A11Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Application%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22Y63EJFYD%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Roos%20et%20al.%22%2C%22parsedDate%22%3A%222024-09-18%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BRoos%2C%20J.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282024%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-024-02404-5%26%23039%3B%26gt%3BArkitekt%3A%20streaming%20analysis%20and%20real-time%20workflows%20for%20microscopy%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%20Methods%26lt%3B%5C%2Fi%26gt%3B%20%5BPreprint%5D.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41592-024-02404-5.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Arkitekt%3A%20streaming%20analysis%20and%20real-time%20workflows%20for%20microscopy%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Johannes%22%2C%22lastName%22%3A%22Roos%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22St%5Cu00e9phane%22%2C%22lastName%22%3A%22Bancelin%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Tom%22%2C%22lastName%22%3A%22Delaire%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Alexander%22%2C%22lastName%22%3A%22Wilhelmi%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Florian%22%2C%22lastName%22%3A%22Levet%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Maren%22%2C%22lastName%22%3A%22Engelhardt%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Virgile%22%2C%22lastName%22%3A%22Viasnoff%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22R%5Cu00e9mi%22%2C%22lastName%22%3A%22Galland%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22U.%20Valentin%22%2C%22lastName%22%3A%22N%5Cu00e4gerl%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jean-Baptiste%22%2C%22lastName%22%3A%22Sibarita%22%7D%5D%2C%22abstractNote%22%3A%22%22%2C%22date%22%3A%222024-09-18%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs41592-024-02404-5%22%2C%22ISSN%22%3A%221548-7091%2C%201548-7105%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-024-02404-5%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222024-09-20T10%3A51%3A09Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%225Z8J3KML%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Marin%20et%20al.%22%2C%22parsedDate%22%3A%222024-09-11%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BMarin%2C%20Z.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282024%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41592-024-02413-4%26%23039%3B%26gt%3BNavigate%3A%20an%20open-source%20platform%20for%20smart%20light-sheet%20microscopy%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%20Methods%26lt%3B%5C%2Fi%26gt%3B%20%5BPreprint%5D.%20Available%20at%3A%20%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Navigate%3A%20an%20open-source%20platform%20for%20smart%20light-sheet%20microscopy%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Zach%22%2C%22lastName%22%3A%22Marin%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Xiaoding%22%2C%22lastName%22%3A%22Wang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Dax%20W.%22%2C%22lastName%22%3A%22Collison%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Conor%22%2C%22lastName%22%3A%22McFadden%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jinlong%22%2C%22lastName%22%3A%22Lin%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Hazel%20M.%22%2C%22lastName%22%3A%22Borges%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Bingying%22%2C%22lastName%22%3A%22Chen%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Dushyant%22%2C%22lastName%22%3A%22Mehra%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Qionghua%22%2C%22lastName%22%3A%22Shen%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Seweryn%22%2C%22lastName%22%3A%22Ga%5Cu0142ecki%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Stephan%22%2C%22lastName%22%3A%22Daetwyler%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Steven%20J.%22%2C%22lastName%22%3A%22Sheppard%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Phu%22%2C%22lastName%22%3A%22Thien%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Baylee%20A.%22%2C%22lastName%22%3A%22Porter%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Suzanne%20D.%22%2C%22lastName%22%3A%22Conzen%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Douglas%20P.%22%2C%22lastName%22%3A%22Shepherd%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Reto%22%2C%22lastName%22%3A%22Fiolka%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Kevin%20M.%22%2C%22lastName%22%3A%22Dean%22%7D%5D%2C%22abstractNote%22%3A%22%22%2C%22date%22%3A%222024-09-11%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs41592-024-02413-4%22%2C%22ISSN%22%3A%221548-7091%2C%201548-7105%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41592-024-02413-4%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222025-04-29T13%3A11%3A58Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22C2H2QVG6%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Peters%20et%20al.%22%2C%22parsedDate%22%3A%222024-08-02%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BPeters%2C%20J.J.P.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282024%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.science.org%5C%2Fdoi%5C%2F10.1126%5C%2Fscience.ado8579%26%23039%3B%26gt%3BEvent-responsive%20scanning%20transmission%20electron%20microscopy%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BScience%26lt%3B%5C%2Fi%26gt%3B%2C%20385%286708%29%2C%20pp.%20549%5Cu2013553.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1126%5C%2Fscience.ado8579.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Event-responsive%20scanning%20transmission%20electron%20microscopy%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jonathan%20J.%20P.%22%2C%22lastName%22%3A%22Peters%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Bryan%20W.%22%2C%22lastName%22%3A%22Reed%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Yu%22%2C%22lastName%22%3A%22Jimbo%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Kanako%22%2C%22lastName%22%3A%22Noguchi%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Karin%20H.%22%2C%22lastName%22%3A%22M%5Cu00fcller%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Alexandra%22%2C%22lastName%22%3A%22Porter%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Daniel%20J.%22%2C%22lastName%22%3A%22Masiel%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Lewys%22%2C%22lastName%22%3A%22Jones%22%7D%5D%2C%22abstractNote%22%3A%22An%20ever-present%20limitation%20of%20transmission%20electron%20microscopy%20is%20the%20damage%20caused%20by%20high-energy%20electrons%20interacting%20with%20any%20sample.%20By%20reconsidering%20the%20fundamentals%20of%20imaging%2C%20we%20demonstrate%20an%20event-responsive%20approach%20to%20electron%20microscopy%20that%20delivers%20more%20information%20about%20the%20sample%20for%20a%20given%20beam%20current.%20Measuring%20the%20time%20to%20achieve%20an%20electron%20count%20threshold%20rather%20than%20waiting%20a%20predefined%20constant%20time%20improves%20the%20information%20obtained%20per%20electron.%20The%20microscope%20was%20made%20to%20respond%20to%20these%20events%20by%20blanking%20the%20beam%2C%20thus%20reducing%20the%20overall%20dose%20required.%20This%20approach%20automatically%20apportions%20dose%20to%20achieve%20a%20given%20signal-to-noise%20ratio%20in%20each%20pixel%2C%20eliminating%20excess%20dose%20that%20is%20associated%20with%20diminishing%20returns%20of%20information.%20We%20demonstrate%20the%20wide%20applicability%20of%20our%20approach%20to%20beam-sensitive%20materials%20by%20imaging%20biological%20tissue%20and%20zeolite.%22%2C%22date%22%3A%222024-08-02%22%2C%22language%22%3A%22%22%2C%22DOI%22%3A%2210.1126%5C%2Fscience.ado8579%22%2C%22ISSN%22%3A%22%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.science.org%5C%2Fdoi%5C%2F10.1126%5C%2Fscience.ado8579%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222025-04-29T14%3A01%3A58Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Application%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22KYCN7A4V%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Pomp%20et%20al.%22%2C%22parsedDate%22%3A%222024-03-21%22%2C%22numChildren%22%3A1%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BPomp%2C%20W.%2C%20Meeussen%2C%20J.V.W.%20and%20Lenstra%2C%20T.L.%20%282024%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.cell.com%5C%2Fmolecular-cell%5C%2Fabstract%5C%2FS1097-2765%2824%2900056-X%26%23039%3B%26gt%3BTranscription%20factor%20exchange%20enables%20prolonged%20transcriptional%20bursts%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BMolecular%20Cell%26lt%3B%5C%2Fi%26gt%3B%2C%2084%286%29%2C%20pp.%201036-1048.e9.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1016%5C%2Fj.molcel.2024.01.020.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Transcription%20factor%20exchange%20enables%20prolonged%20transcriptional%20bursts%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Wim%22%2C%22lastName%22%3A%22Pomp%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Joseph%20V.%20W.%22%2C%22lastName%22%3A%22Meeussen%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Tineke%20L.%22%2C%22lastName%22%3A%22Lenstra%22%7D%5D%2C%22abstractNote%22%3A%22%26lt%3Bh2%26gt%3BSummary%26lt%3B%5C%2Fh2%26gt%3B%26lt%3Bp%26gt%3BSingle-molecule%20imaging%20inside%20living%20cells%20has%20revealed%20that%20transcription%20factors%20%28TFs%29%20bind%20to%20DNA%20transiently%2C%20but%20a%20long-standing%20question%20is%20how%20this%20transient%20binding%20is%20related%20to%20transcription%20activation.%20Here%2C%20we%20devised%20a%20microscopy%20method%20to%20simultaneously%20measure%20transient%20TF%20binding%20at%20a%20single%20locus%20and%20the%20effect%20of%20these%20binding%20events%20on%20transcription.%20We%20show%20that%20DNA%20binding%20of%20the%20yeast%20TF%20Gal4%20activates%20transcription%20of%20a%20target%20gene%20within%20a%20few%20seconds%2C%20with%20at%20least%20%5Cu223c20%25%20efficiency%20and%20with%20a%20high%20initiation%20rate%20of%20%5Cu223c1%20RNA%5C%2Fs.%20Gal4%20DNA%20dissociation%20decreases%20transcription%20rapidly.%20Moreover%2C%20at%20a%20gene%20with%20multiple%20binding%20sites%2C%20individual%20Gal4%20molecules%20only%20rarely%20stay%20bound%20throughout%20the%20entire%20burst%20but%20instead%20frequently%20exchange%20during%20a%20burst%20to%20increase%20the%20transcriptional%20burst%20duration.%20Our%20results%20suggest%20a%20mechanism%20for%20enhancer%20regulation%20in%20more%20complex%20eukaryotes%2C%20where%20TF%20cooperativity%20and%20exchange%20enable%20robust%20and%20responsive%20transcription%20regulation.%26lt%3B%5C%2Fp%26gt%3B%22%2C%22date%22%3A%222024%5C%2F03%5C%2F21%22%2C%22language%22%3A%22English%22%2C%22DOI%22%3A%2210.1016%5C%2Fj.molcel.2024.01.020%22%2C%22ISSN%22%3A%221097-2765%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.cell.com%5C%2Fmolecular-cell%5C%2Fabstract%5C%2FS1097-2765%2824%2900056-X%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222024-05-24T09%3A49%3A39Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Application%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%225Q2SZY6M%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Shi%20et%20al.%22%2C%22parsedDate%22%3A%222024-01-02%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BShi%2C%20Y.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282024%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-023-02126-0%26%23039%3B%26gt%3BSmart%20lattice%20light-sheet%20microscopy%20for%20imaging%20rare%20and%20complex%20cellular%20events%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%20Methods%26lt%3B%5C%2Fi%26gt%3B%2C%20pp.%201%5Cu201310.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41592-023-02126-0.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Smart%20lattice%20light-sheet%20microscopy%20for%20imaging%20rare%20and%20complex%20cellular%20events%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Yu%22%2C%22lastName%22%3A%22Shi%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jimmy%20S.%22%2C%22lastName%22%3A%22Tabet%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Daniel%20E.%22%2C%22lastName%22%3A%22Milkie%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Timothy%20A.%22%2C%22lastName%22%3A%22Daugird%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Chelsea%20Q.%22%2C%22lastName%22%3A%22Yang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Alex%20T.%22%2C%22lastName%22%3A%22Ritter%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Andrea%22%2C%22lastName%22%3A%22Giovannucci%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Wesley%20R.%22%2C%22lastName%22%3A%22Legant%22%7D%5D%2C%22abstractNote%22%3A%22Light-sheet%20microscopes%20enable%20rapid%20high-resolution%20imaging%20of%20biological%20specimens%3B%20however%2C%20biological%20processes%20span%20spatiotemporal%20scales.%20Moreover%2C%20long-term%20phenotypes%20are%20often%20instigated%20by%20rare%20or%20fleeting%20biological%20events%20that%20are%20difficult%20to%20capture%20with%20a%20single%20imaging%20modality.%20Here%2C%20to%20overcome%20this%20limitation%2C%20we%20present%20smartLLSM%2C%20a%20microscope%20that%20incorporates%20artificial%20intelligence-based%20instrument%20control%20to%20autonomously%20switch%20between%20epifluorescent%20inverted%20imaging%20and%20lattice%20light-sheet%20microscopy%20%28LLSM%29.%20We%20apply%20this%20approach%20to%20two%20unique%20processes%3A%20cell%20division%20and%20immune%20synapse%20formation.%20In%20each%20context%2C%20smartLLSM%20provides%20population-level%20statistics%20across%20thousands%20of%20cells%20and%20autonomously%20captures%20multicolor%20three-dimensional%20datasets%20or%20four-dimensional%20time-lapse%20movies%20of%20rare%20events%20at%20rates%20that%20dramatically%20exceed%20human%20capabilities.%20From%20this%2C%20we%20quantify%20the%20effects%20of%20Taxol%20dose%20on%20spindle%20structure%20and%20kinetochore%20dynamics%20in%20dividing%20cells%20and%20of%20antigen%20strength%20on%20cytotoxic%20T%20lymphocyte%20engagement%20and%20lytic%20granule%20polarization%20at%20the%20immune%20synapse.%20Overall%2C%20smartLLSM%20efficiently%20detects%20rare%20events%20within%20heterogeneous%20cell%20populations%20and%20records%20these%20processes%20with%20high%20spatiotemporal%20four-dimensional%20imaging%20over%20statistically%20significant%20replicates.%22%2C%22date%22%3A%222024-01-02%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs41592-023-02126-0%22%2C%22ISSN%22%3A%221548-7105%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-023-02126-0%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A04%3A12Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22I3Y6H3S2%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Cabriel%20et%20al.%22%2C%22parsedDate%22%3A%222023-12%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BCabriel%2C%20C.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282023%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41566-023-01308-8%26%23039%3B%26gt%3BEvent-based%20vision%20sensor%20for%20fast%20and%20dense%20single-molecule%20localization%20microscopy%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%20Photonics%26lt%3B%5C%2Fi%26gt%3B%2C%2017%2812%29%2C%20pp.%201105%5Cu20131113.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41566-023-01308-8.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Event-based%20vision%20sensor%20for%20fast%20and%20dense%20single-molecule%20localization%20microscopy%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Cl%5Cu00e9ment%22%2C%22lastName%22%3A%22Cabriel%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Tual%22%2C%22lastName%22%3A%22Monfort%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Christian%20G.%22%2C%22lastName%22%3A%22Specht%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ignacio%22%2C%22lastName%22%3A%22Izeddin%22%7D%5D%2C%22abstractNote%22%3A%22Single-molecule%20localization%20microscopy%20%28SMLM%29%20enables%20crucial%20insights%20into%20cellular%20structures%20and%20processes%20to%20be%20revealed%20at%20the%20single-molecule%20level.%20However%2C%20SMLM%20is%20often%20hampered%20by%20limited%20temporal%20resolution%20and%20the%20fixed%20frame%20rate%20of%20the%20acquisition.%20Here%20we%20present%20a%20new%20approach%20to%20SMLM%20data%20acquisition%20and%20processing%20based%20on%20an%20affordable%20event-based%20sensor.%20This%20type%20of%20sensor%20reacts%20to%20changes%20in%20light%20intensity%2C%20rather%20than%20integrating%20photons%20during%20the%20exposure%20time%20of%20each%20frame.%20Each%20pixel%20works%20independently%20and%20returns%20a%20signal%20only%20when%20an%20intensity%20change%20is%20detected.%20Compared%20with%20video%20acquisition%20using%20traditional%20electron-multiplying%20charge-coupled%20device%20or%20scientific%20complementary%20metal%5Cu2013oxide%5Cu2013semiconductor%20cameras%2C%20the%20event-based%20sensor%20provides%20higher%20temporal%20resolution%20and%20throughput%20on%20the%20positions%20of%20blinking%20molecules.%20We%20demonstrate%20event-based%20SMLM%20super-resolution%20imaging%20on%20biological%20samples%20with%20spatial%20resolution%20on%20a%20par%20with%20the%20performance%20of%20electron-multiplying%20charge-coupled%20device%20or%20scientific%20complementary%20metal%5Cu2013oxide%5Cu2013semiconductor%20cameras%2C%20while%20registering%20only%20the%20on%20and%20off%20switching%20of%20blinking%20molecules.%20We%20use%20event-based%20SMLM%20to%20perform%20very%20dense%20single-molecule%20imaging%2C%20where%20frame-based%20cameras%20experience%20major%20limitations.%22%2C%22date%22%3A%222023-12%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs41566-023-01308-8%22%2C%22ISSN%22%3A%221749-4893%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41566-023-01308-8%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A13%3A50Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Application%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22UL82WX7E%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Andr%5Cu00e9%20et%20al.%22%2C%22parsedDate%22%3A%222023-03-27%22%2C%22numChildren%22%3A1%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BAndr%5Cu00e9%2C%20O.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282023%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.cell.com%5C%2Fcell-reports-methods%5C%2Fabstract%5C%2FS2667-2375%2823%2900030-9%26%23039%3B%26gt%3BData-driven%20microscopy%20allows%20for%20automated%20context-specific%20acquisition%20of%20high-fidelity%20image%20data%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BCell%20Reports%20Methods%26lt%3B%5C%2Fi%26gt%3B%2C%203%283%29.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1016%5C%2Fj.crmeth.2023.100419.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Data-driven%20microscopy%20allows%20for%20automated%20context-specific%20acquisition%20of%20high-fidelity%20image%20data%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Oscar%22%2C%22lastName%22%3A%22Andr%5Cu00e9%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Johannes%20Kumra%22%2C%22lastName%22%3A%22Ahnlide%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Nils%22%2C%22lastName%22%3A%22Norlin%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Vinay%22%2C%22lastName%22%3A%22Swaminathan%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Pontus%22%2C%22lastName%22%3A%22Nordenfelt%22%7D%5D%2C%22abstractNote%22%3A%22%22%2C%22date%22%3A%222023-03-27%22%2C%22language%22%3A%22English%22%2C%22DOI%22%3A%2210.1016%5C%2Fj.crmeth.2023.100419%22%2C%22ISSN%22%3A%222667-2375%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.cell.com%5C%2Fcell-reports-methods%5C%2Fabstract%5C%2FS2667-2375%2823%2900030-9%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A06%3A21Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%2C%7B%22tag%22%3A%22Data%20Analysis%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22C6VDYKVG%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Merino-Casallo%20et%20al.%22%2C%22parsedDate%22%3A%222022-12-31%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BMerino-Casallo%2C%20F.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282022%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1080%5C%2F19336918.2022.2055520%26%23039%3B%26gt%3BUnravelling%20cell%20migration%3A%20defining%20movement%20from%20the%20cell%20surface%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BCell%20Adhesion%20%26amp%3B%20Migration%26lt%3B%5C%2Fi%26gt%3B%2C%2016%281%29%2C%20pp.%2025%5Cu201364.%20Available%20at%3A%20%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Unravelling%20cell%20migration%3A%20defining%20movement%20from%20the%20cell%20surface%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Francisco%22%2C%22lastName%22%3A%22Merino-Casallo%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Maria%20Jose%22%2C%22lastName%22%3A%22Gomez-Benito%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Silvia%22%2C%22lastName%22%3A%22Hervas-Raluy%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jose%20Manuel%22%2C%22lastName%22%3A%22Garcia-Aznar%22%7D%5D%2C%22abstractNote%22%3A%22Cell%20motility%20is%20essential%20for%20life%20and%20development.%20Unfortunately%2C%20cell%20migration%20is%20also%20linked%20to%20several%20pathological%20processes%2C%20such%20as%20cancer%20metastasis.%20Cells%5Cu2019%20ability%20to%20migrate%20relies%20on%20many%20actors.%20Cells%20change%20their%20migratory%20strategy%20based%20on%20their%20phenotype%20and%20the%20properties%20of%20the%20surrounding%20microenvironment.%20Cell%20migration%20is%2C%20therefore%2C%20an%20extremely%20complex%20phenomenon.%20Researchers%20have%20investigated%20cell%20motility%20for%20more%20than%20a%20century.%20Recent%20discoveries%20have%20uncovered%20some%20of%20the%20mysteries%20associated%20with%20the%20mechanisms%20involved%20in%20cell%20migration%2C%20such%20as%20intracellular%20signaling%20and%20cell%20mechanics.%20These%20findings%20involve%20different%20players%2C%20including%20transmembrane%20receptors%2C%20adhesive%20complexes%2C%20cytoskeletal%20components%20%2C%20the%20nucleus%2C%20and%20the%20extracellular%20matrix.%20This%20review%20aims%20to%20give%20a%20global%20overview%20of%20our%20current%20understanding%20of%20cell%20migration.%22%2C%22date%22%3A%222022-12-31%22%2C%22language%22%3A%22%22%2C%22DOI%22%3A%2210.1080%5C%2F19336918.2022.2055520%22%2C%22ISSN%22%3A%221933-6918%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1080%5C%2F19336918.2022.2055520%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A15%3A47Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Application%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22W39S3ZSK%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Lan%20et%20al.%22%2C%22parsedDate%22%3A%222022-12-01%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BLan%2C%20T.-H.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282022%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.sciencedirect.com%5C%2Fscience%5C%2Farticle%5C%2Fpii%5C%2FS0168952522001408%26%23039%3B%26gt%3BOptogenetics%20for%20transcriptional%20programming%20and%20genetic%20engineering%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BTrends%20in%20Genetics%26lt%3B%5C%2Fi%26gt%3B%2C%2038%2812%29%2C%20pp.%201253%5Cu20131270.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1016%5C%2Fj.tig.2022.05.014.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Optogenetics%20for%20transcriptional%20programming%20and%20genetic%20engineering%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Tien-Hung%22%2C%22lastName%22%3A%22Lan%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Lian%22%2C%22lastName%22%3A%22He%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Yun%22%2C%22lastName%22%3A%22Huang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Yubin%22%2C%22lastName%22%3A%22Zhou%22%7D%5D%2C%22abstractNote%22%3A%22Optogenetics%20combines%20genetics%20and%20biophotonics%20to%20enable%20noninvasive%20control%20of%20biological%20processes%20with%20high%20spatiotemporal%20precision.%20When%20engineered%20into%20protein%20machineries%20that%20govern%20the%20cellular%20information%20flow%20as%20depicted%20in%20the%20central%20dogma%2C%20multiple%20genetically%20encoded%20non-opsin%20photosensory%20modules%20have%20been%20harnessed%20to%20modulate%20gene%20transcription%2C%20DNA%20or%20RNA%20modifications%2C%20DNA%20recombination%2C%20and%20genome%20engineering%20by%20utilizing%20photons%20emitting%20in%20the%20wide%20range%20of%20200%5Cu20131000%20nm.%20We%20present%20herein%20generally%20applicable%20modular%20strategies%20for%20optogenetic%20engineering%20and%20highlight%20latest%20advances%20in%20the%20broad%20applications%20of%20opsin-free%20optogenetics%20to%20program%20transcriptional%20outputs%20and%20precisely%20manipulate%20the%20mammalian%20genome%2C%20epigenome%2C%20and%20epitranscriptome.%20We%20also%20discuss%20current%20challenges%20and%20future%20trends%20in%20opsin-free%20optogenetics%2C%20which%20has%20been%20rapidly%20evolving%20to%20meet%20the%20growing%20needs%20in%20synthetic%20biology%20and%20genetics%20research.%22%2C%22date%22%3A%222022-12-01%22%2C%22language%22%3A%22%22%2C%22DOI%22%3A%2210.1016%5C%2Fj.tig.2022.05.014%22%2C%22ISSN%22%3A%220168-9525%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.sciencedirect.com%5C%2Fscience%5C%2Farticle%5C%2Fpii%5C%2FS0168952522001408%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A15%3A23Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Application%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22QKZ278FZ%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Alvelid%20et%20al.%22%2C%22parsedDate%22%3A%222022-10%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BAlvelid%2C%20J.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282022%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-022-01588-y%26%23039%3B%26gt%3BEvent-triggered%20STED%20imaging%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%20Methods%26lt%3B%5C%2Fi%26gt%3B%2C%2019%2810%29%2C%20pp.%201268%5Cu20131275.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41592-022-01588-y.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Event-triggered%20STED%20imaging%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jonatan%22%2C%22lastName%22%3A%22Alvelid%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Martina%22%2C%22lastName%22%3A%22Damenti%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Chiara%22%2C%22lastName%22%3A%22Sgattoni%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ilaria%22%2C%22lastName%22%3A%22Testa%22%7D%5D%2C%22abstractNote%22%3A%22Monitoring%20the%20proteins%20and%20lipids%20that%20mediate%20all%20cellular%20processes%20requires%20imaging%20methods%20with%20increased%20spatial%20and%20temporal%20resolution.%20STED%20%28stimulated%20emission%20depletion%29%20nanoscopy%20enables%20fast%20imaging%20of%20nanoscale%20structures%20in%20living%20cells%20but%20is%20limited%20by%20photobleaching.%20Here%2C%20we%20present%20event-triggered%20STED%2C%20an%20automated%20multiscale%20method%20capable%20of%20rapidly%20initiating%20two-dimensional%20%282D%29%20and%203D%20STED%20imaging%20after%20detecting%20cellular%20events%20such%20as%20protein%20recruitment%2C%20vesicle%20trafficking%20and%20second%20messengers%20activity%20using%20biosensors.%20STED%20is%20applied%20in%20the%20vicinity%20of%20detected%20events%20to%20maximize%20the%20temporal%20resolution.%20We%20imaged%20synaptic%20vesicle%20dynamics%20at%20up%20to%2024%5Cu2009Hz%2C%2040%5Cu2009ms%20after%20local%20calcium%20activity%3B%20endocytosis%20and%20exocytosis%20events%20at%20up%20to%2011%5Cu2009Hz%2C%2040%5Cu2009ms%20after%20local%20protein%20recruitment%20or%20pH%20changes%3B%20and%20the%20interaction%20between%20endosomal%20vesicles%20at%20up%20to%203%5Cu2009Hz%2C%2070%5Cu2009ms%20after%20approaching%20one%20another.%20Event-triggered%20STED%20extends%20the%20capabilities%20of%20live%20nanoscale%20imaging%2C%20enabling%20novel%20biological%20observations%20in%20real%20time.%22%2C%22date%22%3A%222022-10%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs41592-022-01588-y%22%2C%22ISSN%22%3A%221548-7105%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-022-01588-y%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A06%3A30Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%2267C3RZY6%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Mahecic%20et%20al.%22%2C%22parsedDate%22%3A%222022-10%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BMahecic%2C%20D.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282022%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-022-01589-x%26%23039%3B%26gt%3BEvent-driven%20acquisition%20for%20content-enriched%20microscopy%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%20Methods%26lt%3B%5C%2Fi%26gt%3B%2C%2019%2810%29%2C%20pp.%201262%5Cu20131267.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41592-022-01589-x.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Event-driven%20acquisition%20for%20content-enriched%20microscopy%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Dora%22%2C%22lastName%22%3A%22Mahecic%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Willi%20L.%22%2C%22lastName%22%3A%22Stepp%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Chen%22%2C%22lastName%22%3A%22Zhang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Juliette%22%2C%22lastName%22%3A%22Griffi%5Cu00e9%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Martin%22%2C%22lastName%22%3A%22Weigert%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Suliana%22%2C%22lastName%22%3A%22Manley%22%7D%5D%2C%22abstractNote%22%3A%22A%20common%20goal%20of%20fluorescence%20microscopy%20is%20to%20collect%20data%20on%20specific%20biological%20events.%20Yet%2C%20the%20event-specific%20content%20that%20can%20be%20collected%20from%20a%20sample%20is%20limited%2C%20especially%20for%20rare%20or%20stochastic%20processes.%20This%20is%20due%20in%20part%20to%20photobleaching%20and%20phototoxicity%2C%20which%20constrain%20imaging%20speed%20and%20duration.%20We%20developed%20an%20event-driven%20acquisition%20framework%2C%20in%20which%20neural-network-based%20recognition%20of%20specific%20biological%20events%20triggers%20real-time%20control%20in%20an%20instant%20structured%20illumination%20microscope.%20Our%20setup%20adapts%20acquisitions%20on-the-fly%20by%20switching%20between%20a%20slow%20imaging%20rate%20while%20detecting%20the%20onset%20of%20events%2C%20and%20a%20fast%20imaging%20rate%20during%20their%20progression.%20Thus%2C%20we%20capture%20mitochondrial%20and%20bacterial%20divisions%20at%20imaging%20rates%20that%20match%20their%20dynamic%20timescales%2C%20while%20extending%20overall%20imaging%20durations.%20Because%20event-driven%20acquisition%20allows%20the%20microscope%20to%20respond%20specifically%20to%20complex%20biological%20events%2C%20it%20acquires%20data%20enriched%20in%20relevant%20content.%22%2C%22date%22%3A%222022-10%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs41592-022-01589-x%22%2C%22ISSN%22%3A%221548-7105%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-022-01589-x%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A04%3A55Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%223GEXFHP8%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Dibrov%22%2C%22parsedDate%22%3A%222022-08-12%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BDibrov%2C%20A.%20%282022%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Ftud.qucosa.de%5C%2Fid%5C%2Fqucosa%3A80379%26%23039%3B%26gt%3BTowards%20Smarter%20Fluorescence%20Microscopy%3A%20Enabling%20Adaptive%20Acquisition%20Strategies%20With%20Optimized%20Photon%20Budget%26lt%3B%5C%2Fa%26gt%3B.%26quot%3B%20Available%20at%3A%20%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Towards%20Smarter%20Fluorescence%20Microscopy%3A%20Enabling%20Adaptive%20Acquisition%20Strategies%20With%20Optimized%20Photon%20Budget%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Alexandr%22%2C%22lastName%22%3A%22Dibrov%22%7D%5D%2C%22abstractNote%22%3A%22%22%2C%22date%22%3A%222022%5C%2F08%5C%2F12%22%2C%22language%22%3A%22eng%22%2C%22DOI%22%3A%22%22%2C%22ISSN%22%3A%22%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Ftud.qucosa.de%5C%2Fid%5C%2Fqucosa%3A80379%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222025-04-29T13%3A08%3A58Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22MZKL54LI%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Chiron%20et%20al.%22%2C%22parsedDate%22%3A%222022-07-08%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BChiron%2C%20L.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282022%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41598-022-15207-5%26%23039%3B%26gt%3BCyberSco.Py%20an%20open-source%20software%20for%20event-based%2C%20conditional%20microscopy%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BScientific%20Reports%26lt%3B%5C%2Fi%26gt%3B%2C%2012%281%29%2C%20p.%2011579.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41598-022-15207-5.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22CyberSco.Py%20an%20open-source%20software%20for%20event-based%2C%20conditional%20microscopy%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Lionel%22%2C%22lastName%22%3A%22Chiron%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Matthias%22%2C%22lastName%22%3A%22Le%20Bec%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22C%5Cu00e9line%22%2C%22lastName%22%3A%22Cordier%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Sylvain%22%2C%22lastName%22%3A%22Pouzet%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Dimitrije%22%2C%22lastName%22%3A%22Milunov%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Alvaro%22%2C%22lastName%22%3A%22Banderas%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jean-Marc%22%2C%22lastName%22%3A%22Di%20Meglio%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Benoit%22%2C%22lastName%22%3A%22Sorre%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Pascal%22%2C%22lastName%22%3A%22Hersen%22%7D%5D%2C%22abstractNote%22%3A%22Timelapse%20fluorescence%20microscopy%20imaging%20is%20routinely%20used%20in%20quantitative%20cell%20biology.%20However%2C%20microscopes%20could%20become%20much%20more%20powerful%20investigation%20systems%20if%20they%20were%20endowed%20with%20simple%20unsupervised%20decision-making%20algorithms%20to%20transform%20them%20into%20fully%20responsive%20and%20automated%20measurement%20devices.%20Here%2C%20we%20report%20CyberSco.Py%2C%20Python%20software%20for%20advanced%20automated%20timelapse%20experiments.%20We%20provide%20proof-of-principle%20of%20a%20user-friendly%20framework%20that%20increases%20the%20tunability%20and%20flexibility%20when%20setting%20up%20and%20running%20fluorescence%20timelapse%20microscopy%20experiments.%20Importantly%2C%20CyberSco.Py%20combines%20real-time%20image%20analysis%20with%20automation%20capability%2C%20which%20allows%20users%20to%20create%20conditional%2C%20event-based%20experiments%20in%20which%20the%20imaging%20acquisition%20parameters%20and%20the%20status%20of%20various%20devices%20can%20be%20changed%20automatically%20based%20on%20the%20image%20analysis.%20We%20exemplify%20the%20relevance%20of%20CyberSco.Py%20to%20cell%20biology%20using%20several%20use%20case%20experiments%20with%20budding%20yeast.%20We%20anticipate%20that%20CyberSco.Py%20could%20be%20used%20to%20address%20the%20growing%20need%20for%20smart%20microscopy%20systems%20to%20implement%20more%20informative%20quantitative%20cell%20biology%20experiments.%22%2C%22date%22%3A%222022-07-08%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs41598-022-15207-5%22%2C%22ISSN%22%3A%222045-2322%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41598-022-15207-5%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A05%3A50Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22GRHGAX2R%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Beghin%20et%20al.%22%2C%22parsedDate%22%3A%222022-07%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BBeghin%2C%20A.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282022%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-022-01508-0%26%23039%3B%26gt%3BAutomated%20high-speed%203D%20imaging%20of%20organoid%20cultures%20with%20multi-scale%20phenotypic%20quantification%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%20Methods%26lt%3B%5C%2Fi%26gt%3B%2C%2019%287%29%2C%20pp.%20881%5Cu2013892.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41592-022-01508-0.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Automated%20high-speed%203D%20imaging%20of%20organoid%20cultures%20with%20multi-scale%20phenotypic%20quantification%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Anne%22%2C%22lastName%22%3A%22Beghin%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Gianluca%22%2C%22lastName%22%3A%22Grenci%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Geetika%22%2C%22lastName%22%3A%22Sahni%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Su%22%2C%22lastName%22%3A%22Guo%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Harini%22%2C%22lastName%22%3A%22Rajendiran%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Tom%22%2C%22lastName%22%3A%22Delaire%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Saburnisha%20Binte%22%2C%22lastName%22%3A%22Mohamad%20Raffi%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Damien%22%2C%22lastName%22%3A%22Blanc%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Richard%22%2C%22lastName%22%3A%22de%20Mets%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Hui%20Ting%22%2C%22lastName%22%3A%22Ong%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Xareni%22%2C%22lastName%22%3A%22Galindo%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Anais%22%2C%22lastName%22%3A%22Monet%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Vidhyalakshmi%22%2C%22lastName%22%3A%22Acharya%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Victor%22%2C%22lastName%22%3A%22Racine%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Florian%22%2C%22lastName%22%3A%22Levet%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Remi%22%2C%22lastName%22%3A%22Galland%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jean-Baptiste%22%2C%22lastName%22%3A%22Sibarita%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Virgile%22%2C%22lastName%22%3A%22Viasnoff%22%7D%5D%2C%22abstractNote%22%3A%22Current%20imaging%20approaches%20limit%20the%20ability%20to%20perform%20multi-scale%20characterization%20of%20three-dimensional%20%283D%29%20organotypic%20cultures%20%28organoids%29%20in%20large%20numbers.%20Here%2C%20we%20present%20an%20automated%20multi-scale%203D%20imaging%20platform%20synergizing%20high-density%20organoid%20cultures%20with%20rapid%20and%20live%203D%20single-objective%20light-sheet%20imaging.%20It%20is%20composed%20of%20disposable%20microfabricated%20organoid%20culture%20chips%2C%20termed%20JeWells%2C%20with%20embedded%20optical%20components%20and%20a%20laser%20beam-steering%20unit%20coupled%20to%20a%20commercial%20inverted%20microscope.%20It%20permits%20streamlining%20organoid%20culture%20and%20high-content%203D%20imaging%20on%20a%20single%20user-friendly%20instrument%20with%20minimal%20manipulations%20and%20a%20throughput%20of%20300%20organoids%20per%20hour.%20We%20demonstrate%20that%20the%20large%20number%20of%203D%20stacks%20that%20can%20be%20collected%20via%20our%20platform%20allows%20training%20deep%20learning-based%20algorithms%20to%20quantify%20morphogenetic%20organizations%20of%20organoids%20at%20multi-scales%2C%20ranging%20from%20the%20subcellular%20scale%20to%20the%20whole%20organoid%20level.%20We%20validated%20the%20versatility%20and%20robustness%20of%20our%20approach%20on%20intestine%2C%20hepatic%2C%20neuroectoderm%20organoids%20and%20oncospheres.%22%2C%22date%22%3A%222022-07%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs41592-022-01508-0%22%2C%22ISSN%22%3A%221548-7105%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-022-01508-0%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222025-05-07T09%3A47%3A14Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Application%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%2289WA4U6K%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Fox%20et%20al.%22%2C%22parsedDate%22%3A%222022-04-22%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BFox%2C%20Z.R.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282022%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41467-022-29888-z%26%23039%3B%26gt%3BEnabling%20reactive%20microscopy%20with%20MicroMator%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%20Communications%26lt%3B%5C%2Fi%26gt%3B%2C%2013%281%29%2C%20p.%202199.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41467-022-29888-z.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Enabling%20reactive%20microscopy%20with%20MicroMator%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Zachary%20R.%22%2C%22lastName%22%3A%22Fox%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Steven%22%2C%22lastName%22%3A%22Fletcher%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Achille%22%2C%22lastName%22%3A%22Fraisse%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Chetan%22%2C%22lastName%22%3A%22Aditya%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Sebasti%5Cu00e1n%22%2C%22lastName%22%3A%22Sosa-Carrillo%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Julienne%22%2C%22lastName%22%3A%22Petit%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22S%5Cu00e9bastien%22%2C%22lastName%22%3A%22Gilles%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Fran%5Cu00e7ois%22%2C%22lastName%22%3A%22Bertaux%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jakob%22%2C%22lastName%22%3A%22Ruess%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Gregory%22%2C%22lastName%22%3A%22Batt%22%7D%5D%2C%22abstractNote%22%3A%22Microscopy%20image%20analysis%20has%20recently%20made%20enormous%20progress%20both%20in%20terms%20of%20accuracy%20and%20speed%20thanks%20to%20machine%20learning%20methods%20and%20improved%20computational%20resources.%20This%20greatly%20facilitates%20the%20online%20adaptation%20of%20microscopy%20experimental%20plans%20using%20real-time%20information%20of%20the%20observed%20systems%20and%20their%20environments.%20Applications%20in%20which%20reactiveness%20is%20needed%20are%20multifarious.%20Here%20we%20report%20MicroMator%2C%20an%20open%20and%20flexible%20software%20for%20defining%20and%20driving%20reactive%20microscopy%20experiments.%20It%20provides%20a%20Python%20software%20environment%20and%20an%20extensible%20set%20of%20modules%20that%20greatly%20facilitate%20the%20definition%20of%20events%20with%20triggers%20and%20effects%20interacting%20with%20the%20experiment.%20We%20provide%20a%20pedagogic%20example%20performing%20dynamic%20adaptation%20of%20fluorescence%20illumination%20on%20bacteria%2C%20and%20demonstrate%20MicroMator%5Cu2019s%20potential%20via%20two%20challenging%20case%20studies%20in%20yeast%20to%20single-cell%20control%20and%20single-cell%20recombination%2C%20both%20requiring%20real-time%20tracking%20and%20light%20targeting%20at%20the%20single-cell%20level.%22%2C%22date%22%3A%222022-04-22%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs41467-022-29888-z%22%2C%22ISSN%22%3A%222041-1723%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41467-022-29888-z%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A05%3A22Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%2268YQKGG2%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Balluet%20et%20al.%22%2C%22parsedDate%22%3A%222022%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BBalluet%2C%20M.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282022%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fonlinelibrary.wiley.com%5C%2Fdoi%5C%2Fabs%5C%2F10.1111%5C%2Fjmi.13062%26%23039%3B%26gt%3BNeural%20network%20fast-classifies%20biological%20images%20through%20features%20selecting%20to%20power%20automated%20microscopy%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BJournal%20of%20Microscopy%26lt%3B%5C%2Fi%26gt%3B%2C%20285%281%29%2C%20pp.%203%5Cu201319.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1111%5C%2Fjmi.13062.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Neural%20network%20fast-classifies%20biological%20images%20through%20features%20selecting%20to%20power%20automated%20microscopy%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ma%5Cu00ebl%22%2C%22lastName%22%3A%22Balluet%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Florian%22%2C%22lastName%22%3A%22Sizaire%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Youssef%22%2C%22lastName%22%3A%22El%20Habouz%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Thomas%22%2C%22lastName%22%3A%22Walter%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22J%5Cu00e9r%5Cu00e9my%22%2C%22lastName%22%3A%22Pont%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Baptiste%22%2C%22lastName%22%3A%22Giroux%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Otmane%22%2C%22lastName%22%3A%22Bouchareb%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Marc%22%2C%22lastName%22%3A%22Tramier%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jacques%22%2C%22lastName%22%3A%22Pecreaux%22%7D%5D%2C%22abstractNote%22%3A%22Artificial%20intelligence%20is%20nowadays%20used%20for%20cell%20detection%20and%20classification%20in%20optical%20microscopy%20during%20post-acquisition%20analysis.%20The%20microscopes%20are%20now%20fully%20automated%20and%20next%20expected%20to%20be%20smart%20by%20making%20acquisition%20decisions%20based%20on%20the%20images.%20It%20calls%20for%20analysing%20them%20on%20the%20fly.%20Biology%20further%20imposes%20training%20on%20a%20reduced%20data%20set%20due%20to%20cost%20and%20time%20to%20prepare%20the%20samples%20and%20have%20the%20data%20sets%20annotated%20by%20experts.%20We%20propose%20a%20real-time%20image%20processing%20compliant%20with%20these%20specifications%20by%20balancing%20accurate%20detection%20and%20execution%20performance.%20We%20characterised%20the%20images%20using%20a%20generic%2C%20high-dimensional%20feature%20extractor.%20We%20then%20classified%20the%20images%20using%20machine%20learning%20to%20understand%20the%20contribution%20of%20each%20feature%20in%20decision%20and%20execution%20time.%20We%20found%20that%20the%20non-linear-classifier%20random%20forests%20outperformed%20Fisher%26%23039%3Bs%20linear%20discriminant.%20More%20importantly%2C%20the%20most%20discriminant%20and%20time-consuming%20features%20could%20be%20excluded%20without%20significant%20accuracy%20loss%2C%20offering%20a%20substantial%20gain%20in%20execution%20time.%20It%20suggests%20a%20feature-group%20redundancy%20likely%20related%20to%20the%20biology%20of%20the%20observed%20cells.%20We%20offer%20a%20method%20to%20select%20fast%20and%20discriminant%20features.%20In%20our%20assay%2C%20a%2079.6%202.4%25%20accurate%20classification%20of%20a%20cell%20took%2068.7%203.5%20ms%20%28mean%20SD%2C%205-fold%20cross-validation%20nested%20in%2010%20bootstrap%20repeats%29%2C%20corresponding%20to%2014%20cells%20per%20second%2C%20dispatched%20into%20eight%20phases%20of%20the%20cell%20cycle%2C%20using%2012%20feature%20groups%20and%20operating%20a%20consumer%20market%20ARM-based%20embedded%20system.%20A%20simple%20neural%20network%20offered%20similar%20performances%20paving%20the%20way%20to%20faster%20training%20and%20classification%2C%20using%20parallel%20execution%20on%20a%20general-purpose%20graphic%20processing%20unit.%20Finally%2C%20this%20strategy%20is%20also%20usable%20for%20deep%20neural%20networks%20paving%20the%20way%20to%20optimizing%20these%20algorithms%20for%20smart%20microscopy.%22%2C%22date%22%3A%222022%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1111%5C%2Fjmi.13062%22%2C%22ISSN%22%3A%221365-2818%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fonlinelibrary.wiley.com%5C%2Fdoi%5C%2Fabs%5C%2F10.1111%5C%2Fjmi.13062%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A05%3A58Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22732U9I9P%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Ouyang%20et%20al.%22%2C%22parsedDate%22%3A%222022%22%2C%22numChildren%22%3A1%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BOuyang%2C%20W.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282022%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fonlinelibrary.wiley.com%5C%2Fdoi%5C%2Fabs%5C%2F10.1002%5C%2Fadbi.202101063%26%23039%3B%26gt%3BAn%20Open-Source%20Modular%20Framework%20for%20Automated%20Pipetting%20and%20Imaging%20Applications%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BAdvanced%20Biology%26lt%3B%5C%2Fi%26gt%3B%2C%206%284%29%2C%20p.%202101063.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1002%5C%2Fadbi.202101063.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22An%20Open-Source%20Modular%20Framework%20for%20Automated%20Pipetting%20and%20Imaging%20Applications%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Wei%22%2C%22lastName%22%3A%22Ouyang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Richard%20W.%22%2C%22lastName%22%3A%22Bowman%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Haoran%22%2C%22lastName%22%3A%22Wang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Kaspar%20E.%22%2C%22lastName%22%3A%22Bumke%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Joel%20T.%22%2C%22lastName%22%3A%22Collins%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ola%22%2C%22lastName%22%3A%22Spjuth%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jordi%22%2C%22lastName%22%3A%22Carreras-Puigvert%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Benedict%22%2C%22lastName%22%3A%22Diederich%22%7D%5D%2C%22abstractNote%22%3A%22The%20number%20of%20samples%20in%20biological%20experiments%20is%20continuously%20increasing%2C%20but%20complex%20protocols%20and%20human%20error%20in%20many%20cases%20lead%20to%20suboptimal%20data%20quality%20and%20hence%20difficulties%20in%20reproducing%20scientific%20findings.%20Laboratory%20automation%20can%20alleviate%20many%20of%20these%20problems%20by%20precisely%20reproducing%20machine-readable%20protocols.%20These%20instruments%20generally%20require%20high%20up-front%20investments%2C%20and%20due%20to%20the%20lack%20of%20open%20application%20programming%20interfaces%20%28APIs%29%2C%20they%20are%20notoriously%20difficult%20for%20scientists%20to%20customize%20and%20control%20outside%20of%20the%20vendor-supplied%20software.%20Here%2C%20automated%2C%20high-throughput%20experiments%20are%20demonstrated%20for%20interdisciplinary%20research%20in%20life%20science%20that%20can%20be%20replicated%20on%20a%20modest%20budget%2C%20using%20open%20tools%20to%20ensure%20reproducibility%20by%20combining%20the%20tools%20OpenFlexure%2C%20Opentrons%2C%20ImJoy%2C%20and%20UC2.%20This%20automated%20sample%20preparation%20and%20imaging%20pipeline%20can%20easily%20be%20replicated%20and%20established%20in%20many%20laboratories%20as%20well%20as%20in%20educational%20contexts%20through%20easy-to-understand%20algorithms%20and%20easy-to-build%20microscopes.%20Additionally%2C%20the%20creation%20of%20feedback%20loops%2C%20with%20later%20pipetting%20or%20imaging%20steps%20depending%20on%20the%20analysis%20of%20previously%20acquired%20images%2C%20enables%20the%20realization%20of%20fully%20autonomous%20%5Cu201csmart%5Cu201d%20microscopy%20experiments.%20All%20documents%20and%20source%20files%20are%20publicly%20available%20to%20prove%20the%20concept%20of%20smart%20lab%20automation%20using%20inexpensive%2C%20open%20tools.%20It%20is%20believed%20this%20democratizes%20access%20to%20the%20power%20and%20repeatability%20of%20automated%20experiments.%22%2C%22date%22%3A%222022%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1002%5C%2Fadbi.202101063%22%2C%22ISSN%22%3A%222701-0198%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fonlinelibrary.wiley.com%5C%2Fdoi%5C%2Fabs%5C%2F10.1002%5C%2Fadbi.202101063%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A04%3A32Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22S26EVKC2%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Bouvette%20et%20al.%22%2C%22parsedDate%22%3A%222021-10%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BBouvette%2C%20J.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282021%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fieeexplore.ieee.org%5C%2Fdocument%5C%2F9762104%26%23039%3B%26gt%3BSmartscope%3A%20AI-driven%20grid%20navigation%20for%20high-throughput%20cryo-EM%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20in%20%26lt%3Bi%26gt%3B2021%20IEEE%20Applied%20Imagery%20Pattern%20Recognition%20Workshop%20%28AIPR%29%26lt%3B%5C%2Fi%26gt%3B.%20%26lt%3Bi%26gt%3B2021%20IEEE%20Applied%20Imagery%20Pattern%20Recognition%20Workshop%20%28AIPR%29%26lt%3B%5C%2Fi%26gt%3B%2C%20pp.%201%5Cu20136.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1109%5C%2FAIPR52630.2021.9762104.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22conferencePaper%22%2C%22title%22%3A%22Smartscope%3A%20AI-driven%20grid%20navigation%20for%20high-throughput%20cryo-EM%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jonathan%22%2C%22lastName%22%3A%22Bouvette%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Qinwen%22%2C%22lastName%22%3A%22Huang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Alberto%22%2C%22lastName%22%3A%22Bartesaghi%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Mario%20J.%22%2C%22lastName%22%3A%22Borgnia%22%7D%5D%2C%22abstractNote%22%3A%22Specimen%20optimization%20is%20currently%20one%20of%20the%20main%20limiting%20steps%20in%20the%20cryo-electron%20microscopy%20%28EM%29%20structure%20determination%20pipeline.%20The%20ideal%20specimen%20is%20a%20molecule-thin%20layer%20of%20macromolecules%20in%20solution%20frozen%20on%20top%20of%20a%20holey%20membrane%20stabilized%20by%20a%20metal%20support%20grid.%20During%20screening%2C%20experienced%20microscopists%20visualize%20the%20specimen%20at%20increasing%20magnifications%20by%20navigating%20to%20areas%20that%20are%20most%20likely%20to%20provide%20information%20useful%20to%20guide%20the%20optimization.%20Iterating%20this%20procedure%20over%20different%20experimental%20conditions%2C%20eventually%20results%20in%20grids%20that%20are%20suitable%20for%20high-resolution%20imaging.%20While%20automation%20has%20led%20to%20increased%20throughput%20of%20data%20collection%20in%20single%20particle%20cryo-EM%2C%20specimen%20screening%20is%20still%20a%20largely%20manual%20and%20time-consuming%20task%20where%20data%20coherence%20and%20intermediate%20readouts%20are%20not%20frequently%20recorded.%20Here%2C%20we%20present%20Smartscope%2C%20a%20framework%20to%20simplify%20and%20automate%20the%20screening%20process%20of%20cryo-EM%20grids.%20By%20abstracting%20the%20intermediate%20steps%20of%20specimen%20navigation%2C%20Smartscope%20saves%20metadata%20into%20a%20database%20and%20presents%20the%20results%20to%20the%20user%20through%20an%20interactive%2C%20user-friendly%20web%20based%20interface.%20Grid%20squares%20and%20holes%20in%20the%20substrate%20are%20automatically%20detected%20and%20labeled%20using%20neural%20network-based%20approaches%20that%20si-multaneously%20detect%20and%20classify%20squares%20with%20high%20accuracy%2C%20and%20precisely%20recover%20the%20position%20of%20holes%20within%20grid%20squares.%20Moreover%2C%20Smartscope%26%23039%3Bs%20web%20interface%20can%20also%20be%20used%20as%20a%20platform%20for%20automated%20data%20collection%20as%20it%20allows%20the%20quick%20selection%20of%20areas%20for%20imaging%2C%20thus%20significantly%20reducing%20setup%20time.%20By%20unifying%20data%20management%20for%20proper%20bookkeeping%20and%20using%20AI-based%20routines%20for%20autonomous%20grid%20navigation%2C%20Smartscope%20offers%20a%20convenient%20platform%20that%20minimizes%20human%20intervention%20and%20optimizes%20microscope%20usage%2C%20thus%20significantly%20improving%20the%20throughput%20of%20cryo-EM%20structure%20determination.%22%2C%22date%22%3A%222021-10%22%2C%22proceedingsTitle%22%3A%222021%20IEEE%20Applied%20Imagery%20Pattern%20Recognition%20Workshop%20%28AIPR%29%22%2C%22conferenceName%22%3A%222021%20IEEE%20Applied%20Imagery%20Pattern%20Recognition%20Workshop%20%28AIPR%29%22%2C%22language%22%3A%22%22%2C%22DOI%22%3A%2210.1109%5C%2FAIPR52630.2021.9762104%22%2C%22ISBN%22%3A%22%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fieeexplore.ieee.org%5C%2Fdocument%5C%2F9762104%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A06%3A55Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22WRFE42Q8%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Fan%20et%20al.%22%2C%22parsedDate%22%3A%222021-09-03%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BFan%2C%20Y.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282021%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1186%5C%2Fs43074-021-00040-2%26%23039%3B%26gt%3BSmart%20computational%20light%20microscopes%20%28SCLMs%29%20of%20smart%20computational%20imaging%20laboratory%20%28SCILab%29%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BPhotoniX%26lt%3B%5C%2Fi%26gt%3B%2C%202%281%29%2C%20p.%2019.%20Available%20at%3A%20%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Smart%20computational%20light%20microscopes%20%28SCLMs%29%20of%20smart%20computational%20imaging%20laboratory%20%28SCILab%29%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Yao%22%2C%22lastName%22%3A%22Fan%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jiaji%22%2C%22lastName%22%3A%22Li%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Linpeng%22%2C%22lastName%22%3A%22Lu%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jiasong%22%2C%22lastName%22%3A%22Sun%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Yan%22%2C%22lastName%22%3A%22Hu%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jialin%22%2C%22lastName%22%3A%22Zhang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Zhuoshi%22%2C%22lastName%22%3A%22Li%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Qian%22%2C%22lastName%22%3A%22Shen%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Bowen%22%2C%22lastName%22%3A%22Wang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Runnan%22%2C%22lastName%22%3A%22Zhang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Qian%22%2C%22lastName%22%3A%22Chen%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Chao%22%2C%22lastName%22%3A%22Zuo%22%7D%5D%2C%22abstractNote%22%3A%22Computational%20microscopy%2C%20as%20a%20subfield%20of%20computational%20imaging%2C%20combines%20optical%20manipulation%20and%20image%20algorithmic%20reconstruction%20to%20recover%20multi-dimensional%20microscopic%20images%20or%20information%20of%20micro-objects.%20In%20recent%20years%2C%20the%20revolution%20in%20light-emitting%20diodes%20%28LEDs%29%2C%20low-cost%20consumer%20image%20sensors%2C%20modern%20digital%20computers%2C%20and%20smartphones%20provide%20fertile%20opportunities%20for%20the%20rapid%20development%20of%20computational%20microscopy.%20Consequently%2C%20diverse%20forms%20of%20computational%20microscopy%20have%20been%20invented%2C%20including%20digital%20holographic%20microscopy%20%28DHM%29%2C%20transport%20of%20intensity%20equation%20%28TIE%29%2C%20differential%20phase%20contrast%20%28DPC%29%20microscopy%2C%20lens-free%20on-chip%20holography%2C%20and%20Fourier%20ptychographic%20microscopy%20%28FPM%29.%20These%20computational%20microscopy%20techniques%20not%20only%20provide%20high-resolution%2C%20label-free%2C%20quantitative%20phase%20imaging%20capability%20but%20also%20decipher%20new%20and%20advanced%20biomedical%20research%20and%20industrial%20applications.%20Nevertheless%2C%20most%20computational%20microscopy%20techniques%20are%20still%20at%20an%20early%20stage%20of%20%5Cu201cproof%20of%20concept%5Cu201d%20or%20%5Cu201cproof%20of%20prototype%5Cu201d%20%28based%20on%20commercially%20available%20microscope%20platforms%29.%20Translating%20those%20concepts%20to%20stand-alone%20optical%20instruments%20for%20practical%20use%20is%20an%20essential%20step%20for%20the%20promotion%20and%20adoption%20of%20computational%20microscopy%20by%20the%20wider%20bio-medicine%2C%20industry%2C%20and%20education%20community.%20In%20this%20paper%2C%20we%20present%20four%20smart%20computational%20light%20microscopes%20%28SCLMs%29%20developed%20by%20our%20laboratory%2C%20i.e.%2C%20smart%20computational%20imaging%20laboratory%20%28SCILab%29%20of%20Nanjing%20University%20of%20Science%20and%20Technology%20%28NJUST%29%2C%20China.%20These%20microscopes%20are%20empowered%20by%20advanced%20computational%20microscopy%20techniques%2C%20including%20digital%20holography%2C%20TIE%2C%20DPC%2C%20lensless%20holography%2C%20and%20FPM%2C%20which%20not%20only%20enables%20multi-modal%20contrast-enhanced%20observations%20for%20unstained%20specimens%2C%20but%20also%20can%20recover%20their%20three-dimensional%20profiles%20quantitatively.%20We%20introduce%20their%20basic%20principles%2C%20hardware%20configurations%2C%20reconstruction%20algorithms%2C%20and%20software%20design%2C%20quantify%20their%20imaging%20performance%2C%20and%20illustrate%20their%20typical%20applications%20for%20cell%20analysis%2C%20medical%20diagnosis%2C%20and%20microlens%20characterization.%22%2C%22date%22%3A%222021-09-03%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1186%5C%2Fs43074-021-00040-2%22%2C%22ISSN%22%3A%222662-1991%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1186%5C%2Fs43074-021-00040-2%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A15%3A11Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Application%22%7D%5D%7D%7D%2C%7B%22key%22%3A%224WSR74TL%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Levental%20et%20al.%22%2C%22parsedDate%22%3A%222021-09%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BLevental%2C%20M.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282021%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fieeexplore.ieee.org%5C%2Fdocument%5C%2F9582285%26%23039%3B%26gt%3BUltrafast%20Focus%20Detection%20for%20Automated%20Microscopy%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20in%20%26lt%3Bi%26gt%3B2021%20IEEE%2017th%20International%20Conference%20on%20eScience%20%28eScience%29%26lt%3B%5C%2Fi%26gt%3B.%20%26lt%3Bi%26gt%3B2021%20IEEE%2017th%20International%20Conference%20on%20eScience%20%28eScience%29%26lt%3B%5C%2Fi%26gt%3B%2C%20pp.%20237%5Cu2013238.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1109%5C%2FeScience51609.2021.00039.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22conferencePaper%22%2C%22title%22%3A%22Ultrafast%20Focus%20Detection%20for%20Automated%20Microscopy%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Maksim%22%2C%22lastName%22%3A%22Levental%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ryan%22%2C%22lastName%22%3A%22Chard%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Kyle%22%2C%22lastName%22%3A%22Chard%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ian%22%2C%22lastName%22%3A%22Foster%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Gregg%20A.%22%2C%22lastName%22%3A%22Wildenberg%22%7D%5D%2C%22abstractNote%22%3A%22Recent%20advances%20in%20scientific%20instrumentation%20technology%20have%20led%20to%20an%20increase%20in%20the%20volumes%20and%20velocities%20of%20data%20being%20generated%20in%20typical%20laboratories.%20Such%20advanced%20instruments%20now%20necessitate%20commensurately%20advanced%20computing%20resources%20and%20techniques%20to%20realize%20their%20full%20potential.%20In%20particular%2C%20such%20as%20in%20the%20case%20of%20quality%20control%20for%20automated%20microscopes%20collecting%20large%20volumes%20of%20images.%20We%20propose%20a%20faster%20than%20real-time%20%5Cu201cout-of-focus%5Cu201d%20detection%20algorithm%20for%20electron%20microscopy%20images.%20Our%20technique%2C%20Multi-scale%20Histologic%20Feature%20Detection%20%28MHFD%29%2C%20adapts%20classical%20computer%20vision%20techniques%20and%20is%20based%20on%20detecting%20various%20fine-grained%20histologic%20features.%20We%20also%20study%20the%20inherent%20parallelism%20in%20the%20technique%20in%20order%20to%20employ%20GPU%20accelerators.%20We%20present%20preliminary%20tests%20that%20demonstrate%20the%20efficacy%20of%20MHFD%20and%20discuss%20applying%20our%20solution%20to%20images%20produced%20by%20a%20scanning%20electron%20microscope%20in%20an%20active%20connectomics%20laboratory.%22%2C%22date%22%3A%222021-09%22%2C%22proceedingsTitle%22%3A%222021%20IEEE%2017th%20International%20Conference%20on%20eScience%20%28eScience%29%22%2C%22conferenceName%22%3A%222021%20IEEE%2017th%20International%20Conference%20on%20eScience%20%28eScience%29%22%2C%22language%22%3A%22%22%2C%22DOI%22%3A%2210.1109%5C%2FeScience51609.2021.00039%22%2C%22ISBN%22%3A%22%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fieeexplore.ieee.org%5C%2Fdocument%5C%2F9582285%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A06%3A43Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22QTZV78RC%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Moreno%20et%20al.%22%2C%22parsedDate%22%3A%222021-08-14%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BMoreno%2C%20X.C.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282021%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fjoss.theoj.org%5C%2Fpapers%5C%2F10.21105%5C%2Fjoss.03394%26%23039%3B%26gt%3BImSwitch%3A%20Generalizing%20microscope%20control%20in%20Python%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BJournal%20of%20Open%20Source%20Software%26lt%3B%5C%2Fi%26gt%3B%2C%206%2864%29%2C%20p.%203394.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.21105%5C%2Fjoss.03394.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22ImSwitch%3A%20Generalizing%20microscope%20control%20in%20Python%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Xavier%20Casas%22%2C%22lastName%22%3A%22Moreno%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Staffan%22%2C%22lastName%22%3A%22Al-Kadhimi%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jonatan%22%2C%22lastName%22%3A%22Alvelid%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Andreas%22%2C%22lastName%22%3A%22Bod%5Cu00e9n%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ilaria%22%2C%22lastName%22%3A%22Testa%22%7D%5D%2C%22abstractNote%22%3A%22Moreno%20et%20al.%2C%20%282021%29.%20ImSwitch%3A%20Generalizing%20microscope%20control%20in%20Python.%20Journal%20of%20Open%20Source%20Software%2C%206%2864%29%2C%203394%2C%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.21105%5C%2Fjoss.03394%22%2C%22date%22%3A%222021%5C%2F08%5C%2F14%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.21105%5C%2Fjoss.03394%22%2C%22ISSN%22%3A%222475-9066%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fjoss.theoj.org%5C%2Fpapers%5C%2F10.21105%5C%2Fjoss.03394%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A04%3A44Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22UKSYWZTP%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22He%22%2C%22parsedDate%22%3A%222020-01-24%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BHe%2C%20J.%20%282020%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fnbn-resolving.org%5C%2Furn%3Anbn%3Ade%3Absz%3A14-qucosa2-379998%26%23039%3B%26gt%3BTowards%20a%20smarter%20light%20sheet%20microscope%26lt%3B%5C%2Fa%26gt%3B.%26quot%3B%20Available%20at%3A%20%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Towards%20a%20smarter%20light%20sheet%20microscope%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jiaye%22%2C%22lastName%22%3A%22He%22%7D%5D%2C%22abstractNote%22%3A%22%22%2C%22date%22%3A%222020%5C%2F01%5C%2F24%22%2C%22language%22%3A%22eng%22%2C%22DOI%22%3A%22%22%2C%22ISSN%22%3A%22%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fnbn-resolving.org%5C%2Furn%3Anbn%3Ade%3Absz%3A14-qucosa2-379998%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222025-04-29T13%3A10%3A04Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Data%20Analysis%22%7D%2C%7B%22tag%22%3A%22Data%20handling%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22YLT2L6YA%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Seo%20et%20al.%22%2C%22parsedDate%22%3A%222020%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BSeo%2C%20H.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282020%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fonlinelibrary.wiley.com%5C%2Fdoi%5C%2Fabs%5C%2F10.1002%5C%2Fmp.13649%26%23039%3B%26gt%3BMachine%20learning%20techniques%20for%20biomedical%20image%20segmentation%3A%20An%20overview%20of%20technical%20aspects%20and%20introduction%20to%20state-of-art%20applications%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BMedical%20Physics%26lt%3B%5C%2Fi%26gt%3B%2C%2047%285%29%2C%20pp.%20e148%5Cu2013e167.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1002%5C%2Fmp.13649.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Machine%20learning%20techniques%20for%20biomedical%20image%20segmentation%3A%20An%20overview%20of%20technical%20aspects%20and%20introduction%20to%20state-of-art%20applications%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Hyunseok%22%2C%22lastName%22%3A%22Seo%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Masoud%22%2C%22lastName%22%3A%22Badiei%20Khuzani%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Varun%22%2C%22lastName%22%3A%22Vasudevan%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Charles%22%2C%22lastName%22%3A%22Huang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Hongyi%22%2C%22lastName%22%3A%22Ren%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ruoxiu%22%2C%22lastName%22%3A%22Xiao%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Xiao%22%2C%22lastName%22%3A%22Jia%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Lei%22%2C%22lastName%22%3A%22Xing%22%7D%5D%2C%22abstractNote%22%3A%22In%20recent%20years%2C%20significant%20progress%20has%20been%20made%20in%20developing%20more%20accurate%20and%20efficient%20machine%20learning%20algorithms%20for%20segmentation%20of%20medical%20and%20natural%20images.%20In%20this%20review%20article%2C%20we%20highlight%20the%20imperative%20role%20of%20machine%20learning%20algorithms%20in%20enabling%20efficient%20and%20accurate%20segmentation%20in%20the%20field%20of%20medical%20imaging.%20We%20specifically%20focus%20on%20several%20key%20studies%20pertaining%20to%20the%20application%20of%20machine%20learning%20methods%20to%20biomedical%20image%20segmentation.%20We%20review%20classical%20machine%20learning%20algorithms%20such%20as%20Markov%20random%20fields%2C%20k-means%20clustering%2C%20random%20forest%2C%20etc.%20Although%20such%20classical%20learning%20models%20are%20often%20less%20accurate%20compared%20to%20the%20deep-learning%20techniques%2C%20they%20are%20often%20more%20sample%20efficient%20and%20have%20a%20less%20complex%20structure.%20We%20also%20review%20different%20deep-learning%20architectures%2C%20such%20as%20the%20artificial%20neural%20networks%20%28ANNs%29%2C%20the%20convolutional%20neural%20networks%20%28CNNs%29%2C%20and%20the%20recurrent%20neural%20networks%20%28RNNs%29%2C%20and%20present%20the%20segmentation%20results%20attained%20by%20those%20learning%20models%20that%20were%20published%20in%20the%20past%203%20yr.%20We%20highlight%20the%20successes%20and%20limitations%20of%20each%20machine%20learning%20paradigm.%20In%20addition%2C%20we%20discuss%20several%20challenges%20related%20to%20the%20training%20of%20different%20machine%20learning%20models%2C%20and%20we%20present%20some%20heuristics%20to%20address%20those%20challenges.%22%2C%22date%22%3A%222020%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1002%5C%2Fmp.13649%22%2C%22ISSN%22%3A%222473-4209%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fonlinelibrary.wiley.com%5C%2Fdoi%5C%2Fabs%5C%2F10.1002%5C%2Fmp.13649%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A01%3A34Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Data%20Analysis%22%7D%2C%7B%22tag%22%3A%22Data%20handling%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22VAV4CAQ8%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Collinson%22%2C%22parsedDate%22%3A%222019-08%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BCollinson%2C%20L.M.%20%282019%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.cambridge.org%5C%2Fcore%5C%2Fjournals%5C%2Fmicroscopy-and-microanalysis%5C%2Farticle%5C%2Fsmart-microscopy-automation-of-clem-using-in-situ-fluorescence-detection%5C%2F0343ACFB2745C642F1ACACE64E7F70C9%26%23039%3B%26gt%3BSmart%20Microscopy%3A%20Automation%20of%20CLEM%20using%20In%20situ%20Fluorescence%20Detection%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BMicroscopy%20and%20Microanalysis%26lt%3B%5C%2Fi%26gt%3B%2C%2025%28S2%29%2C%20pp.%201018%5Cu20131019.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1017%5C%2FS1431927619005828.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Smart%20Microscopy%3A%20Automation%20of%20CLEM%20using%20In%20situ%20Fluorescence%20Detection%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Lucy%20M.%22%2C%22lastName%22%3A%22Collinson%22%7D%5D%2C%22abstractNote%22%3A%22%5C%2F%5C%2Fstatic.cambridge.org%5C%2Fcontent%5C%2Fid%5C%2Furn%253Acambridge.org%253Aid%253Aarticle%253AS1431927619005828%5C%2Fresource%5C%2Fname%5C%2FfirstPage-S1431927619005828a.jpg%22%2C%22date%22%3A%222019%5C%2F08%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1017%5C%2FS1431927619005828%22%2C%22ISSN%22%3A%221431-9276%2C%201435-8115%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.cambridge.org%5C%2Fcore%5C%2Fjournals%5C%2Fmicroscopy-and-microanalysis%5C%2Farticle%5C%2Fsmart-microscopy-automation-of-clem-using-in-situ-fluorescence-detection%5C%2F0343ACFB2745C642F1ACACE64E7F70C9%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A15%3A02Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Application%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22ZTH5I6CE%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Liu%20et%20al.%22%2C%22parsedDate%22%3A%222018-04-20%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BLiu%2C%20T.-L.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282018%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.science.org%5C%2Fdoi%5C%2F10.1126%5C%2Fscience.aaq1392%26%23039%3B%26gt%3BObserving%20the%20cell%20in%20its%20native%20state%3A%20Imaging%20subcellular%20dynamics%20in%20multicellular%20organisms%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BScience%26lt%3B%5C%2Fi%26gt%3B%2C%20360%286386%29%2C%20p.%20eaaq1392.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1126%5C%2Fscience.aaq1392.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Observing%20the%20cell%20in%20its%20native%20state%3A%20Imaging%20subcellular%20dynamics%20in%20multicellular%20organisms%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Tsung-Li%22%2C%22lastName%22%3A%22Liu%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Srigokul%22%2C%22lastName%22%3A%22Upadhyayula%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Daniel%20E.%22%2C%22lastName%22%3A%22Milkie%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ved%22%2C%22lastName%22%3A%22Singh%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Kai%22%2C%22lastName%22%3A%22Wang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ian%20A.%22%2C%22lastName%22%3A%22Swinburne%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Kishore%20R.%22%2C%22lastName%22%3A%22Mosaliganti%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Zach%20M.%22%2C%22lastName%22%3A%22Collins%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Tom%20W.%22%2C%22lastName%22%3A%22Hiscock%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jamien%22%2C%22lastName%22%3A%22Shea%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Abraham%20Q.%22%2C%22lastName%22%3A%22Kohrman%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Taylor%20N.%22%2C%22lastName%22%3A%22Medwig%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Daphne%22%2C%22lastName%22%3A%22Dambournet%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ryan%22%2C%22lastName%22%3A%22Forster%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Brian%22%2C%22lastName%22%3A%22Cunniff%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Yuan%22%2C%22lastName%22%3A%22Ruan%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Hanako%22%2C%22lastName%22%3A%22Yashiro%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Steffen%22%2C%22lastName%22%3A%22Scholpp%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Elliot%20M.%22%2C%22lastName%22%3A%22Meyerowitz%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Dirk%22%2C%22lastName%22%3A%22Hockemeyer%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22David%20G.%22%2C%22lastName%22%3A%22Drubin%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Benjamin%20L.%22%2C%22lastName%22%3A%22Martin%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22David%20Q.%22%2C%22lastName%22%3A%22Matus%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Minoru%22%2C%22lastName%22%3A%22Koyama%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Sean%20G.%22%2C%22lastName%22%3A%22Megason%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Tom%22%2C%22lastName%22%3A%22Kirchhausen%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Eric%22%2C%22lastName%22%3A%22Betzig%22%7D%5D%2C%22abstractNote%22%3A%22True%20physiological%20imaging%20of%20subcellular%20dynamics%20requires%20studying%20cells%20within%20their%20parent%20organisms%2C%20where%20all%20the%20environmental%20cues%20that%20drive%20gene%20expression%2C%20and%20hence%20the%20phenotypes%20that%20we%20actually%20observe%2C%20are%20present.%20A%20complete%20understanding%20also%20requires%20volumetric%20imaging%20of%20the%20cell%20and%20its%20surroundings%20at%20high%20spatiotemporal%20resolution%2C%20without%20inducing%20undue%20stress%20on%20either.%20We%20combined%20lattice%20light-sheet%20microscopy%20with%20adaptive%20optics%20to%20achieve%2C%20across%20large%20multicellular%20volumes%2C%20noninvasive%20aberration-free%20imaging%20of%20subcellular%20processes%2C%20including%20endocytosis%2C%20organelle%20remodeling%20during%20mitosis%2C%20and%20the%20migration%20of%20axons%2C%20immune%20cells%2C%20and%20metastatic%20cancer%20cells%20in%20vivo.%20The%20technology%20reveals%20the%20phenotypic%20diversity%20within%20cells%20across%20different%20organisms%20and%20developmental%20stages%20and%20may%20offer%20insights%20into%20how%20cells%20harness%20their%20intrinsic%20variability%20to%20adapt%20to%20different%20physiological%20environments.%22%2C%22date%22%3A%222018-04-20%22%2C%22language%22%3A%22%22%2C%22DOI%22%3A%2210.1126%5C%2Fscience.aaq1392%22%2C%22ISSN%22%3A%22%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.science.org%5C%2Fdoi%5C%2F10.1126%5C%2Fscience.aaq1392%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222025-05-07T09%3A46%3A32Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Application%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22EYAZIWXL%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Ouyang%20and%20Zimmer%22%2C%22parsedDate%22%3A%222017-08-01%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BOuyang%2C%20W.%20and%20Zimmer%2C%20C.%20%282017%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.sciencedirect.com%5C%2Fscience%5C%2Farticle%5C%2Fpii%5C%2FS2452310017300537%26%23039%3B%26gt%3BThe%20imaging%20tsunami%3A%20Computational%20opportunities%20and%20challenges%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BCurrent%20Opinion%20in%20Systems%20Biology%26lt%3B%5C%2Fi%26gt%3B%2C%204%2C%20pp.%20105%5Cu2013113.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1016%5C%2Fj.coisb.2017.07.011.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22The%20imaging%20tsunami%3A%20Computational%20opportunities%20and%20challenges%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Wei%22%2C%22lastName%22%3A%22Ouyang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Christophe%22%2C%22lastName%22%3A%22Zimmer%22%7D%5D%2C%22abstractNote%22%3A%22Microscopy%20has%20fueled%20biological%20discoveries%20for%20centuries%2C%20but%20technical%20progress%20during%20the%20last%20decades%20has%20greatly%20expanded%20the%20type%20and%20quantity%20of%20biological%20information%20that%20can%20be%20revealed%20by%20imaging.%20Advances%20in%20instrumentation%2C%20labeling%20and%20computation%20are%20driving%20an%20imaging%20data%20tsunami%20on%20par%20with%20that%20of%20DNA%20sequencing.%20We%20illustrate%20this%20trend%20on%20five%20data%20intensive%20microscopy%20techniques%20and%20discuss%20some%20of%20the%20challenges%20and%20opportunities%20that%20massive%20imaging%20data%20raise%20for%20our%20understanding%20of%20biological%20systems.%22%2C%22date%22%3A%222017-08-01%22%2C%22language%22%3A%22%22%2C%22DOI%22%3A%2210.1016%5C%2Fj.coisb.2017.07.011%22%2C%22ISSN%22%3A%222452-3100%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.sciencedirect.com%5C%2Fscience%5C%2Farticle%5C%2Fpii%5C%2FS2452310017300537%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A01%3A23Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Data%20Analysis%22%7D%2C%7B%22tag%22%3A%22Data%20handling%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22A82CIX6S%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Royer%20et%20al.%22%2C%22parsedDate%22%3A%222016-12%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BRoyer%2C%20L.A.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282016%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fnbt.3708%26%23039%3B%26gt%3BAdaptive%20light-sheet%20microscopy%20for%20long-term%2C%20high-resolution%20imaging%20in%20living%20organisms%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%20Biotechnology%26lt%3B%5C%2Fi%26gt%3B%2C%2034%2812%29%2C%20pp.%201267%5Cu20131278.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fnbt.3708.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Adaptive%20light-sheet%20microscopy%20for%20long-term%2C%20high-resolution%20imaging%20in%20living%20organisms%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Lo%5Cu00efc%20A.%22%2C%22lastName%22%3A%22Royer%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22William%20C.%22%2C%22lastName%22%3A%22Lemon%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Raghav%20K.%22%2C%22lastName%22%3A%22Chhetri%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Yinan%22%2C%22lastName%22%3A%22Wan%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Michael%22%2C%22lastName%22%3A%22Coleman%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Eugene%20W.%22%2C%22lastName%22%3A%22Myers%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Philipp%20J.%22%2C%22lastName%22%3A%22Keller%22%7D%5D%2C%22abstractNote%22%3A%22Adaptive%20light-sheet%20microscopy%20improves%20imaging%20of%20live%20organisms%20by%20correcting%20for%20optical%20aberrations%20in%20real%20time.%22%2C%22date%22%3A%222016-12%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fnbt.3708%22%2C%22ISSN%22%3A%221546-1696%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fnbt.3708%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222025-05-07T09%3A46%3A59Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Application%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22N76BLUWJ%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Chakrova%20et%20al.%22%2C%22parsedDate%22%3A%222016-10-01%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BChakrova%2C%20N.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282016%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fopg.optica.org%5C%2Fboe%5C%2Fabstract.cfm%3Furi%3Dboe-7-10-4263%26%23039%3B%26gt%3BAdaptive%20illumination%20reduces%20photobleaching%20in%20structured%20illumination%20microscopy%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BBiomedical%20Optics%20Express%26lt%3B%5C%2Fi%26gt%3B%2C%207%2810%29%2C%20pp.%204263%5Cu20134274.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1364%5C%2FBOE.7.004263.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Adaptive%20illumination%20reduces%20photobleaching%20in%20structured%20illumination%20microscopy%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Nadya%22%2C%22lastName%22%3A%22Chakrova%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Alicia%20Soler%22%2C%22lastName%22%3A%22Canton%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Christophe%22%2C%22lastName%22%3A%22Danelon%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Sjoerd%22%2C%22lastName%22%3A%22Stallinga%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Bernd%22%2C%22lastName%22%3A%22Rieger%22%7D%5D%2C%22abstractNote%22%3A%22Photobleaching%20is%20a%20major%20factor%20limiting%20the%20observation%20time%20in%20fluorescence%20microscopy.%20We%20achieve%20photobleaching%20reduction%20in%20structured%20illumination%20microscopy%20%28SIM%29%20by%20locally%20adjusting%20the%20illumination%20intensities%20according%20to%20the%20sample.%20Adaptive%20SIM%20is%20enabled%20by%20a%20digital%20micro-mirror%20device%20%28DMD%29%2C%20which%20provides%20a%20projection%20of%20the%20grayscale%20illumination%20patterns.%20We%20demonstrate%20a%20reduction%20in%20photobleaching%20by%20a%20factor%20of%20three%20in%20adaptive%20SIM%20compared%20to%20the%20non-adaptive%20SIM%20based%20on%20a%20spot%20grid%20scanning%20approach.%20Our%20proof-of-principle%20experiments%20show%20great%20potential%20for%20DMD-based%20microscopes%20to%20become%20a%20more%20useful%20tool%20in%20live-cell%20SIM%20imaging.%22%2C%22date%22%3A%222016%5C%2F10%5C%2F01%22%2C%22language%22%3A%22EN%22%2C%22DOI%22%3A%2210.1364%5C%2FBOE.7.004263%22%2C%22ISSN%22%3A%222156-7085%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fopg.optica.org%5C%2Fboe%5C%2Fabstract.cfm%3Furi%3Dboe-7-10-4263%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222024-11-15T14%3A18%3A00Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Application%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22WZRYEDHE%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Jones%20et%20al.%22%2C%22parsedDate%22%3A%222015-07-10%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BJones%2C%20L.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282015%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1186%5C%2Fs40679-015-0008-4%26%23039%3B%26gt%3BSmart%20Align%5Cu2014a%20new%20tool%20for%20robust%20non-rigid%20registration%20of%20scanning%20microscope%20data%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BAdvanced%20Structural%20and%20Chemical%20Imaging%26lt%3B%5C%2Fi%26gt%3B%2C%201%281%29%2C%20p.%208.%20Available%20at%3A%20%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Smart%20Align%5Cu2014a%20new%20tool%20for%20robust%20non-rigid%20registration%20of%20scanning%20microscope%20data%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Lewys%22%2C%22lastName%22%3A%22Jones%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Hao%22%2C%22lastName%22%3A%22Yang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Timothy%20J.%22%2C%22lastName%22%3A%22Pennycook%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Matthew%20S.%20J.%22%2C%22lastName%22%3A%22Marshall%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Sandra%22%2C%22lastName%22%3A%22Van%20Aert%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Nigel%20D.%22%2C%22lastName%22%3A%22Browning%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Martin%20R.%22%2C%22lastName%22%3A%22Castell%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Peter%20D.%22%2C%22lastName%22%3A%22Nellist%22%7D%5D%2C%22abstractNote%22%3A%22Many%20microscopic%20investigations%20of%20materials%20may%20benefit%20from%20the%20recording%20of%20multiple%20successive%20images.%20This%20can%20include%20techniques%20common%20to%20several%20types%20of%20microscopy%20such%20as%20frame%20averaging%20to%20improve%20signal-to-noise%20ratios%20%28SNR%29%20or%20time%20series%20to%20study%20dynamic%20processes%20or%20more%20specific%20applications.%20In%20the%20scanning%20transmission%20electron%20microscope%2C%20this%20might%20include%20focal%20series%20for%20optical%20sectioning%20or%20aberration%20measurement%2C%20beam%20damage%20studies%20or%20camera-length%20series%20to%20study%20the%20effects%20of%20strain%3B%20whilst%20in%20the%20scanning%20tunnelling%20microscope%2C%20this%20might%20include%20bias-voltage%20series%20to%20probe%20local%20electronic%20structure.%20Whatever%20the%20application%2C%20such%20investigations%20must%20begin%20with%20the%20careful%20alignment%20of%20these%20data%20stacks%2C%20an%20operation%20that%20is%20not%20always%20trivial.%20In%20addition%2C%20the%20presence%20of%20low-frequency%20scanning%20distortions%20can%20introduce%20intra-image%20shifts%20to%20the%20data.%20Here%2C%20we%20describe%20an%20improved%20automated%20method%20of%20performing%20non-rigid%20registration%20customised%20for%20the%20challenges%20unique%20to%20scanned%20microscope%20data%20specifically%20addressing%20the%20issues%20of%20low-SNR%20data%2C%20images%20containing%20a%20large%20proportion%20of%20crystalline%20material%20and%5C%2For%20local%20features%20of%20interest%20such%20as%20dislocations%20or%20edges.%20Careful%20attention%20has%20been%20paid%20to%20artefact%20testing%20of%20the%20non-rigid%20registration%20method%20used%2C%20and%20the%20importance%20of%20this%20registration%20for%20the%20quantitative%20interpretation%20of%20feature%20intensities%20and%20positions%20is%20evaluated.%22%2C%22date%22%3A%222015-07-10%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1186%5C%2Fs40679-015-0008-4%22%2C%22ISSN%22%3A%222198-0926%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1186%5C%2Fs40679-015-0008-4%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A05%3A08Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22SCXQYP4A%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Conrad%20et%20al.%22%2C%22parsedDate%22%3A%222011-03%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BConrad%2C%20C.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282011%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fnmeth.1558%26%23039%3B%26gt%3BMicropilot%3A%20automation%20of%20fluorescence%20microscopy%5Cu2013based%20imaging%20for%20systems%20biology%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%20Methods%26lt%3B%5C%2Fi%26gt%3B%2C%208%283%29%2C%20pp.%20246%5Cu2013249.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fnmeth.1558.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Micropilot%3A%20automation%20of%20fluorescence%20microscopy%5Cu2013based%20imaging%20for%20systems%20biology%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Christian%22%2C%22lastName%22%3A%22Conrad%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Annelie%22%2C%22lastName%22%3A%22W%5Cu00fcnsche%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Tze%20Heng%22%2C%22lastName%22%3A%22Tan%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jutta%22%2C%22lastName%22%3A%22Bulkescher%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Frank%22%2C%22lastName%22%3A%22Sieckmann%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Fatima%22%2C%22lastName%22%3A%22Verissimo%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Arthur%22%2C%22lastName%22%3A%22Edelstein%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Thomas%22%2C%22lastName%22%3A%22Walter%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Urban%22%2C%22lastName%22%3A%22Liebel%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Rainer%22%2C%22lastName%22%3A%22Pepperkok%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jan%22%2C%22lastName%22%3A%22Ellenberg%22%7D%5D%2C%22abstractNote%22%3A%22Microscope%20control%20software%20allowed%20automatic%20machine%20learning-based%20detection%20of%20rare%20events%20in%20living%20cells%20and%20unattended%20operation%20of%20complex%20imaging%20assays.%20Performance%20was%20demonstrated%20by%20detailed%20analysis%20of%20processes%20during%20transient%20mitotic%20stages.%22%2C%22date%22%3A%222011-03%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fnmeth.1558%22%2C%22ISSN%22%3A%221548-7105%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fnmeth.1558%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A05%3A41Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22NVUKKSUV%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Caarls%20et%20al.%22%2C%22parsedDate%22%3A%222011-01%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BCaarls%2C%20W.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282011%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fonlinelibrary.wiley.com%5C%2Fdoi%5C%2F10.1111%5C%2Fj.1365-2818.2010.03413.x%26%23039%3B%26gt%3BMinimizing%20light%20exposure%20with%20the%20programmable%20array%20microscope%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BJournal%20of%20Microscopy%26lt%3B%5C%2Fi%26gt%3B%2C%20241%281%29%2C%20pp.%20101%5Cu2013110.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1111%5C%2Fj.1365-2818.2010.03413.x.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Minimizing%20light%20exposure%20with%20the%20programmable%20array%20microscope%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22W.%22%2C%22lastName%22%3A%22Caarls%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22B.%22%2C%22lastName%22%3A%22Rieger%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22A.h.b.%22%2C%22lastName%22%3A%22De%20Vries%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22D.j.%22%2C%22lastName%22%3A%22Arndt-Jovin%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22T.m.%22%2C%22lastName%22%3A%22Jovin%22%7D%5D%2C%22abstractNote%22%3A%22Summary%20The%20exposure%20of%20fluorophores%20to%20intense%20illumination%20in%20a%20microscope%20often%20results%20in%20photobleaching%20and%20phototoxicity%2C%20thus%20constituting%20a%20major%20limiting%20factor%20in%20time%20lapse%20live%20cell%20or%20single%20molecule%20imaging.%20Laser%20scanning%20confocal%20microscopes%20are%20particularly%20prone%20to%20this%20problem%2C%20inasmuch%20as%20they%20require%20high%20irradiances%20to%20compensate%20for%20the%20inherently%20low%20duty%20cycle%20of%20point%20scanning%20systems.%20In%20the%20attempt%20to%20maintain%20adequate%20speed%20and%20signal-to-noise%20ratios%2C%20the%20fluorophores%20are%20often%20driven%20into%20saturation%2C%20thereby%20generating%20a%20nonlinear%20response.%20One%20approach%20for%20reducing%20photodegradation%20in%20the%20laser%20scanning%20confocal%20microscope%20is%20represented%20by%20controlled%20light%20exposure%20microscopy%2C%20introduced%20by%20Manders%20and%20colleagues.%20The%20strategy%20is%20to%20reduce%20the%20illumination%20intensity%20in%20both%20background%20areas%20%28devoid%20of%20information%29%20as%20well%20as%20in%20bright%20foreground%20regions%2C%20for%20which%20an%20adequate%20signal-to-noise%20ratio%20can%20be%20achieved%20with%20lower%20excitation%20levels%20than%20those%20required%20for%20the%20less%20intense%20foreground%20pixels%5C%2Fvoxels.%20Such%20a%20variable%20illumination%20scheme%20can%20also%20be%20exploited%20in%20widefield%20microscopes%20that%20employ%20lower%20irradiance%20but%20higher%20illumination%20duty%20cycles.%20We%20report%20here%20on%20the%20adaptation%20of%20the%20controlled%20light%20exposure%20microscopy%20principle%20to%20the%20programmable%20array%20microscope%2C%20which%20achieves%20optical%20sectioning%20by%20use%20of%20a%20spatial%20light%20modulator%20%28SLM%29%20in%20an%20image%20plane%20as%20a%20programmable%20mask%20for%20illumination%20and%20conjugate%20%28and%20nonconjugate%29%20detection.%20By%20incorporating%20the%20basic%20controlled%20light%20exposure%20microscopy%20concept%20for%20minimizing%20exposure%2C%20we%20have%20obtained%20a%20reduction%20in%20the%20rate%20of%20photobleaching%20of%20up%20to%20%3F5-fold%2C%20while%20maintaining%20an%20image%20quality%20comparable%20to%20regular%20imaging%20with%20the%20programmable%20array%20microscope.%22%2C%22date%22%3A%222011-01%22%2C%22language%22%3A%22%22%2C%22DOI%22%3A%2210.1111%5C%2Fj.1365-2818.2010.03413.x%22%2C%22ISSN%22%3A%220022-2720%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fonlinelibrary.wiley.com%5C%2Fdoi%5C%2F10.1111%5C%2Fj.1365-2818.2010.03413.x%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222024-11-15T14%3A18%3A18Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Application%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22UJ5FWVJD%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Stagg%20et%20al.%22%2C%22parsedDate%22%3A%222006-08-01%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BStagg%2C%20S.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282006%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1017%5C%2FS1431927606062969%26%23039%3B%26gt%3BA%20Case%20Study%20in%20Automated%20Molecular%20Microscopy%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BMicroscopy%20and%20Microanalysis%26lt%3B%5C%2Fi%26gt%3B%2C%2012%28S02%29%2C%20pp.%201096%5Cu20131097.%20Available%20at%3A%20%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22A%20Case%20Study%20in%20Automated%20Molecular%20Microscopy%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22S%22%2C%22lastName%22%3A%22Stagg%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22J%22%2C%22lastName%22%3A%22Pulokas%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22D%22%2C%22lastName%22%3A%22Fellmann%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22A%22%2C%22lastName%22%3A%22Cheng%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22J%22%2C%22lastName%22%3A%22Quispe%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22S%22%2C%22lastName%22%3A%22Mallick%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22R%22%2C%22lastName%22%3A%22Avila%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22CS%22%2C%22lastName%22%3A%22Potter%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22B%22%2C%22lastName%22%3A%22Carragher%22%7D%5D%2C%22abstractNote%22%3A%22Extended%20abstract%20of%20a%20paper%20presented%20at%20Microscopy%20and%20Microanalysis%202006%20in%20Chicago%2C%20Illinois%2C%20USA%2C%20July%2030%20%5Cu2013%20August%203%2C%202006%22%2C%22date%22%3A%222006-08-01%22%2C%22language%22%3A%22%22%2C%22DOI%22%3A%2210.1017%5C%2FS1431927606062969%22%2C%22ISSN%22%3A%221431-9276%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1017%5C%2FS1431927606062969%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A03%3A58Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22N4UTEB9B%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Joy%22%2C%22parsedDate%22%3A%222002%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BJoy%2C%20D.C.%20%282002%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fonlinelibrary.wiley.com%5C%2Fdoi%5C%2Fabs%5C%2F10.1046%5C%2Fj.1365-2818.2002.01062.x%26%23039%3B%26gt%3BSMART%20%5Cu2013%20a%20program%20to%20measure%20SEM%20resolution%20and%20imaging%20performance%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BJournal%20of%20Microscopy%26lt%3B%5C%2Fi%26gt%3B%2C%20208%281%29%2C%20pp.%2024%5Cu201334.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1046%5C%2Fj.1365-2818.2002.01062.x.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22SMART%20%5Cu2013%20a%20program%20to%20measure%20SEM%20resolution%20and%20imaging%20performance%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22D.%20C.%22%2C%22lastName%22%3A%22Joy%22%7D%5D%2C%22abstractNote%22%3A%22It%20is%20important%20to%20be%20able%20to%20measure%20the%20parameters%2C%20such%20as%20spatial%20resolution%2C%20astigmastism%2C%20signal-to-noise%20ratio%2C%20and%20drift%20and%20instability%2C%20that%20characterize%20the%20performance%20of%20a%20scanning%20electron%20microscope.%20These%20quantities%20can%20be%20determined%20most%20reliably%20by%20a%20Fourier%20analysis%20of%20digital%20micrographs%20from%20the%20instrument%2C%20recorded%20under%20conditions%20of%20interest.%20A%20program%20designed%20to%20implement%20all%20of%20the%20necessary%20steps%20in%20an%20automated%20manner%20has%20been%20developed%20as%20a%20%5Cu2018macro%5Cu2019%20for%20the%20popular%2C%20and%20freely%20available%2C%20NIH%20Image%20and%20SCION%20Image%20programs.%22%2C%22date%22%3A%222002%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1046%5C%2Fj.1365-2818.2002.01062.x%22%2C%22ISSN%22%3A%221365-2818%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fonlinelibrary.wiley.com%5C%2Fdoi%5C%2Fabs%5C%2F10.1046%5C%2Fj.1365-2818.2002.01062.x%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A01%3A04Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Data%20Analysis%22%7D%2C%7B%22tag%22%3A%22Data%20handling%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22L32UDJ7R%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Yavuz%20et%20al.%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BYavuz%2C%20S.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%28no%20date%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fonlinelibrary.wiley.com%5C%2Fdoi%5C%2Fabs%5C%2F10.1002%5C%2F2211-5463.70161%26%23039%3B%26gt%3BAutomated%20FRAP%20microscopy%20for%20high-throughput%20analysis%20of%20protein%20dynamics%20in%20chromatin%20organization%20and%20transcription%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BFEBS%20Open%20Bio%26lt%3B%5C%2Fi%26gt%3B%2C%20n%5C%2Fa%28n%5C%2Fa%29.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1002%5C%2F2211-5463.70161.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Automated%20FRAP%20microscopy%20for%20high-throughput%20analysis%20of%20protein%20dynamics%20in%20chromatin%20organization%20and%20transcription%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Sel%5Cu00e7uk%22%2C%22lastName%22%3A%22Yavuz%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Bart%22%2C%22lastName%22%3A%22Geverts%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Johan%20A.%22%2C%22lastName%22%3A%22Slotman%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Andrea%22%2C%22lastName%22%3A%22Sacchetti%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Stefan%22%2C%22lastName%22%3A%22Prekovic%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Martin%20E.%22%2C%22lastName%22%3A%22van%20Royen%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Adriaan%20B.%22%2C%22lastName%22%3A%22Houtsmuller%22%7D%5D%2C%22abstractNote%22%3A%22Fluorescence%20recovery%20after%20photobleaching%20%28FRAP%29%20is%20a%20quantitative%20technique%20to%20study%20the%20dynamics%20of%20fluorescently%20tagged%20proteins%20in%20living%20cells.%20Current%20FRAP%20workflows%20are%20limited%20in%20throughput%20because%20of%20the%20requirement%20for%20human%20interaction.%20Here%2C%20we%20present%20RoboMic%2C%20a%20fully%20automated%20confocal%20microscopy%20platform%20for%20high-throughput%20imaging%20assays%20such%20as%20FRAP.%20We%20demonstrate%20its%20capabilities%20using%20two%20complementary%20approaches%3A%20sequential%20FRAP%20%28sFRAP%29%20and%20a%20novel%20parallel%20FRAP%20%28pFRAP%29.%20The%20latter%20enables%20simultaneous%20photobleaching%20and%20monitoring%20of%20multiple%20cells%20within%20one%20imaging%20cycle%2C%20increasing%20throughput%20by%20approximately%20five-%20to%2010-fold%20while%20maintaining%20spatiotemporal%20resolution.%20The%20protocol%20consists%20of%20microscope%20control%20software%20for%20automated%2C%20AI-based%20selection%20and%20segmentation%20of%20cell%20nuclei%2C%20sub-nuclear%20ROI%20definition%2C%20photobleaching%2C%20and%20time-lapse%20imaging.%20As%20proof%20of%20concept%2C%20we%20examined%20the%20nuclear%20dynamics%20of%20the%20androgen%20receptor%20and%20the%20cohesin%20complex%20under%20diverse%20conditions%2C%20demonstrating%20that%20RoboMic%20generates%20robust%20and%20reproducible%20data.%20In%20a%20single%20session%2C%20the%20platform%20yields%20hundreds%20of%20FRAP%20measurements%2C%20thereby%20increasing%20statistical%20power%20and%20scalability%20for%20large-scale%20studies%20of%20protein%20mobility.%20While%20we%20focus%20here%20on%20FRAP%2C%20RoboMic%20can%20be%20readily%20applied%20to%20a%20wide%20range%20of%20quantitative%20functional%20imaging%20assays.%22%2C%22date%22%3A%22%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1002%5C%2F2211-5463.70161%22%2C%22ISSN%22%3A%222211-5463%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fonlinelibrary.wiley.com%5C%2Fdoi%5C%2Fabs%5C%2F10.1002%5C%2F2211-5463.70161%22%2C%22collections%22%3A%5B%22IYTK29JT%22%5D%2C%22dateModified%22%3A%222025-12-16T13%3A19%3A07Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22FRAP%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22androgen%20receptor%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22automated%20microscopy%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22cohesin%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22dynamic%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22transcription%22%2C%22type%22%3A1%7D%5D%7D%7D%5D%7D</span>

				<div id="zp-ID-27-5384098-BULWHJV6" data-zp-author-date='Leger-et-al.-2025-12-14' data-zp-date-author='2025-12-14-Leger-et-al.' data-zp-date='2025-12-14' data-zp-year='2025' data-zp-itemtype='preprint' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Leger, D., Ivanisevic, M. and Lenart, P. (2025) "<a class='zp-ItemURL' target='_blank' href='https://www.biorxiv.org/content/10.64898/2025.12.11.693650v1'>AutoSTED: An automated workflow for STED super-resolution imaging of cell nuclei</a>." bioRxiv. Available at: <a href="https://doi.org/10.64898/2025.12.11.693650" rel="nofollow">https://doi.org/10.64898/2025.12.11.693650</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-NTMGRK8D" data-zp-author-date='Redondo-and-Paszek-2025-12-01' data-zp-date-author='2025-12-01-Redondo-and-Paszek' data-zp-date='2025-12-01' data-zp-year='2025' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Redondo, J.A. and Paszek, P. (2025) "<a class='zp-ItemURL' target='_blank' href='https://cames.ippt.pan.pl/index.php/cames/article/view/1945'>AI-Assisted Microscopy for Infection Biology: Advances in High-Content Imaging of Host-Pathogen Interactions</a>," <i>Computer Assisted Methods in Engineering and Science</i> [Preprint]. Available at: <a href="https://doi.org/10.24423/cames.2025.1945" rel="nofollow">https://doi.org/10.24423/cames.2025.1945</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-PGILC555" data-zp-author-date='Cosenza-et-al.-2025-12' data-zp-date-author='2025-12-Cosenza-et-al.' data-zp-date='2025-12' data-zp-year='2025' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Cosenza, M.R. <i>et al.</i> (2025) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s41586-025-09632-5'>Origins of chromosome instability unveiled by coupled imaging and genomics</a>," <i>Nature</i>, 648(8093), pp. 383–393. Available at: <a href="https://doi.org/10.1038/s41586-025-09632-5" rel="nofollow">https://doi.org/10.1038/s41586-025-09632-5</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-WFEWMARH" data-zp-author-date='Berrevoets-et-al.-2025-11-27' data-zp-date-author='2025-11-27-Berrevoets-et-al.' data-zp-date='2025-11-27' data-zp-year='2025' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Berrevoets, E.S. <i>et al.</i> (2025) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s41467-025-65723-x'>Smart 3D super-resolution microscopy reveals the architecture of the RNA scaffold in a nuclear body</a>," <i>Nature Communications</i>, 16(1), p. 10689. Available at: <a href="https://doi.org/10.1038/s41467-025-65723-x" rel="nofollow">https://doi.org/10.1038/s41467-025-65723-x</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-2C2QZA49" data-zp-author-date='London-et-al.-2025-11-03' data-zp-date-author='2025-11-03-London-et-al.' data-zp-date='2025-11-03' data-zp-year='2025' data-zp-itemtype='preprint' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">London, J.A. <i>et al.</i> (2025) "<a class='zp-ItemURL' target='_blank' href='https://www.biorxiv.org/content/10.1101/2025.10.31.685671v1'>A Python framework for magnetic tweezers real-time image processing and microscope control</a>." bioRxiv. Available at: <a href="https://doi.org/10.1101/2025.10.31.685671" rel="nofollow">https://doi.org/10.1101/2025.10.31.685671</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-3JF4LIGZ" data-zp-author-date='Mandal-et-al.-2025-10-14' data-zp-date-author='2025-10-14-Mandal-et-al.' data-zp-date='2025-10-14' data-zp-year='2025' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Mandal, I. <i>et al.</i> (2025) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s41467-025-64105-7'>Evaluating large language model agents for automation of atomic force microscopy</a>," <i>Nature Communications</i>, 16(1), p. 9104. Available at: <a href="https://doi.org/10.1038/s41467-025-64105-7" rel="nofollow">https://doi.org/10.1038/s41467-025-64105-7</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-KY9GPYEJ" data-zp-author-date='Hickey-et-al.-2025-10-10' data-zp-date-author='2025-10-10-Hickey-et-al.' data-zp-date='2025-10-10' data-zp-year='2025' data-zp-itemtype='preprint' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Hickey, J.W. <i>et al.</i> (2025) "<a class='zp-ItemURL' target='_blank' href='https://www.biorxiv.org/content/10.1101/2025.10.09.680291v1'>Fluid-Squid: DIY Multiplexed Imaging of Cells and Tissues</a>." bioRxiv. Available at: <a href="https://doi.org/10.1101/2025.10.09.680291" rel="nofollow">https://doi.org/10.1101/2025.10.09.680291</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-LVWHWJR5" data-zp-author-date='Friederich-et-al.-2025-10-01' data-zp-date-author='2025-10-01-Friederich-et-al.' data-zp-date='2025-10-01' data-zp-year='2025' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Friederich, N. <i>et al.</i> (2025) "<a class='zp-ItemURL' target='_blank' href='https://www.degruyterbrill.com/document/doi/10.1515/auto-2025-0018/html?recommended=sidebar'>EAP4EMSIG – enhancing event-driven microscopy for microfluidic single-cell analysis: EAP4EMSIG – Verbesserung der ereignisgesteuerten Mikroskopie für die mikrofluidische Einzelzellanalyse</a>," <i>at - Automatisierungstechnik</i>, 73(10), pp. 808–823. Available at: <a href="https://doi.org/10.1515/auto-2025-0018" rel="nofollow">https://doi.org/10.1515/auto-2025-0018</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-KFNGL64Q" data-zp-author-date='Zhang-et-al.-2025-09-08' data-zp-date-author='2025-09-08-Zhang-et-al.' data-zp-date='2025-09-08' data-zp-year='2025' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Zhang, N. <i>et al.</i> (2025) "<a class='zp-ItemURL' target='_blank' href='https://pubs.rsc.org/en/content/articlelanding/2025/lc/d5lc00286a'>Modular, open-sourced multiplexing for democratizing spatial omics</a>," <i>Lab on a Chip</i> [Preprint]. Available at: <a href="https://doi.org/10.1039/D5LC00286A" rel="nofollow">https://doi.org/10.1039/D5LC00286A</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-PA6N7DHN" data-zp-author-date='Daetwyler-et-al.-2025-02-12' data-zp-date-author='2025-02-12-Daetwyler-et-al.' data-zp-date='2025-02-12' data-zp-year='2025' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Daetwyler, S. <i>et al.</i> (2025) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s41592-025-02598-2'>Imaging of cellular dynamics from a whole organism to subcellular scale with self-driving, multiscale microscopy</a>," <i>Nature Methods</i>, pp. 1–10. Available at: <a href="https://doi.org/10.1038/s41592-025-02598-2" rel="nofollow">https://doi.org/10.1038/s41592-025-02598-2</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-FPEQIBFZ" data-zp-author-date='Thomas---Chemin-et-al.-2025-02-11' data-zp-date-author='2025-02-11-Thomas---Chemin-et-al.' data-zp-date='2025-02-11' data-zp-year='2025' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Thomas- -Chemin, O. <i>et al.</i> (2025) "<a class='zp-ItemURL' target='_blank' href='https://pubs.acs.org/doi/10.1021/acsnano.4c07729'>Advancing High-Throughput Cellular Atomic Force Microscopy with Automation and Artificial Intelligence</a>," <i>ACS Nano</i>, 19(5), pp. 5045–5062. Available at: <a href="https://doi.org/10.1021/acsnano.4c07729" rel="nofollow">https://doi.org/10.1021/acsnano.4c07729</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-5CPC59Y9" data-zp-author-date='Kim-and-Lee-2025' data-zp-date-author='2025-Kim-and-Lee' data-zp-date='2025' data-zp-year='2025' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Kim, J. and Lee, S.J. (2025) "<a class='zp-ItemURL' target='_blank' href='https://linkinghub.elsevier.com/retrieve/pii/S014381662500079X'>AI-based autofocusing of red blood cells in digital in-line holographic microscopy</a>," <i>Optics and Lasers in Engineering</i>, 188, p. 108892. Available at: <a href="https://doi.org/10.1016/j.optlaseng.2025.108892" rel="nofollow">https://doi.org/10.1016/j.optlaseng.2025.108892</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-Y63EJFYD" data-zp-author-date='Roos-et-al.-2024-09-18' data-zp-date-author='2024-09-18-Roos-et-al.' data-zp-date='2024-09-18' data-zp-year='2024' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Roos, J. <i>et al.</i> (2024) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s41592-024-02404-5'>Arkitekt: streaming analysis and real-time workflows for microscopy</a>," <i>Nature Methods</i> [Preprint]. Available at: <a href="https://doi.org/10.1038/s41592-024-02404-5" rel="nofollow">https://doi.org/10.1038/s41592-024-02404-5</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-5Z8J3KML" data-zp-author-date='Marin-et-al.-2024-09-11' data-zp-date-author='2024-09-11-Marin-et-al.' data-zp-date='2024-09-11' data-zp-year='2024' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Marin, Z. <i>et al.</i> (2024) "<a class='zp-ItemURL' target='_blank' href='https://doi.org/10.1038/s41592-024-02413-4'>Navigate: an open-source platform for smart light-sheet microscopy</a>," <i>Nature Methods</i> [Preprint]. Available at: </div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-C2H2QVG6" data-zp-author-date='Peters-et-al.-2024-08-02' data-zp-date-author='2024-08-02-Peters-et-al.' data-zp-date='2024-08-02' data-zp-year='2024' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Peters, J.J.P. <i>et al.</i> (2024) "<a class='zp-ItemURL' target='_blank' href='https://www.science.org/doi/10.1126/science.ado8579'>Event-responsive scanning transmission electron microscopy</a>," <i>Science</i>, 385(6708), pp. 549–553. Available at: <a href="https://doi.org/10.1126/science.ado8579" rel="nofollow">https://doi.org/10.1126/science.ado8579</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-KYCN7A4V" data-zp-author-date='Pomp-et-al.-2024-03-21' data-zp-date-author='2024-03-21-Pomp-et-al.' data-zp-date='2024-03-21' data-zp-year='2024' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Pomp, W., Meeussen, J.V.W. and Lenstra, T.L. (2024) "<a class='zp-ItemURL' target='_blank' href='https://www.cell.com/molecular-cell/abstract/S1097-2765(24)00056-X'>Transcription factor exchange enables prolonged transcriptional bursts</a>," <i>Molecular Cell</i>, 84(6), pp. 1036-1048.e9. Available at: <a href="https://doi.org/10.1016/j.molcel.2024.01.020" rel="nofollow">https://doi.org/10.1016/j.molcel.2024.01.020</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-5Q2SZY6M" data-zp-author-date='Shi-et-al.-2024-01-02' data-zp-date-author='2024-01-02-Shi-et-al.' data-zp-date='2024-01-02' data-zp-year='2024' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Shi, Y. <i>et al.</i> (2024) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s41592-023-02126-0'>Smart lattice light-sheet microscopy for imaging rare and complex cellular events</a>," <i>Nature Methods</i>, pp. 1–10. Available at: <a href="https://doi.org/10.1038/s41592-023-02126-0" rel="nofollow">https://doi.org/10.1038/s41592-023-02126-0</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-I3Y6H3S2" data-zp-author-date='Cabriel-et-al.-2023-12' data-zp-date-author='2023-12-Cabriel-et-al.' data-zp-date='2023-12' data-zp-year='2023' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Cabriel, C. <i>et al.</i> (2023) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s41566-023-01308-8'>Event-based vision sensor for fast and dense single-molecule localization microscopy</a>," <i>Nature Photonics</i>, 17(12), pp. 1105–1113. Available at: <a href="https://doi.org/10.1038/s41566-023-01308-8" rel="nofollow">https://doi.org/10.1038/s41566-023-01308-8</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-UL82WX7E" data-zp-author-date='André-et-al.-2023-03-27' data-zp-date-author='2023-03-27-André-et-al.' data-zp-date='2023-03-27' data-zp-year='2023' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">André, O. <i>et al.</i> (2023) "<a class='zp-ItemURL' target='_blank' href='https://www.cell.com/cell-reports-methods/abstract/S2667-2375(23)00030-9'>Data-driven microscopy allows for automated context-specific acquisition of high-fidelity image data</a>," <i>Cell Reports Methods</i>, 3(3). Available at: <a href="https://doi.org/10.1016/j.crmeth.2023.100419" rel="nofollow">https://doi.org/10.1016/j.crmeth.2023.100419</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-C6VDYKVG" data-zp-author-date='Merino-Casallo-et-al.-2022-12-31' data-zp-date-author='2022-12-31-Merino-Casallo-et-al.' data-zp-date='2022-12-31' data-zp-year='2022' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Merino-Casallo, F. <i>et al.</i> (2022) "<a class='zp-ItemURL' target='_blank' href='https://doi.org/10.1080/19336918.2022.2055520'>Unravelling cell migration: defining movement from the cell surface</a>," <i>Cell Adhesion & Migration</i>, 16(1), pp. 25–64. Available at: </div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-W39S3ZSK" data-zp-author-date='Lan-et-al.-2022-12-01' data-zp-date-author='2022-12-01-Lan-et-al.' data-zp-date='2022-12-01' data-zp-year='2022' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Lan, T.-H. <i>et al.</i> (2022) "<a class='zp-ItemURL' target='_blank' href='https://www.sciencedirect.com/science/article/pii/S0168952522001408'>Optogenetics for transcriptional programming and genetic engineering</a>," <i>Trends in Genetics</i>, 38(12), pp. 1253–1270. Available at: <a href="https://doi.org/10.1016/j.tig.2022.05.014" rel="nofollow">https://doi.org/10.1016/j.tig.2022.05.014</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-QKZ278FZ" data-zp-author-date='Alvelid-et-al.-2022-10' data-zp-date-author='2022-10-Alvelid-et-al.' data-zp-date='2022-10' data-zp-year='2022' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Alvelid, J. <i>et al.</i> (2022) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s41592-022-01588-y'>Event-triggered STED imaging</a>," <i>Nature Methods</i>, 19(10), pp. 1268–1275. Available at: <a href="https://doi.org/10.1038/s41592-022-01588-y" rel="nofollow">https://doi.org/10.1038/s41592-022-01588-y</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-67C3RZY6" data-zp-author-date='Mahecic-et-al.-2022-10' data-zp-date-author='2022-10-Mahecic-et-al.' data-zp-date='2022-10' data-zp-year='2022' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Mahecic, D. <i>et al.</i> (2022) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s41592-022-01589-x'>Event-driven acquisition for content-enriched microscopy</a>," <i>Nature Methods</i>, 19(10), pp. 1262–1267. Available at: <a href="https://doi.org/10.1038/s41592-022-01589-x" rel="nofollow">https://doi.org/10.1038/s41592-022-01589-x</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-3GEXFHP8" data-zp-author-date='Dibrov-2022-08-12' data-zp-date-author='2022-08-12-Dibrov' data-zp-date='2022-08-12' data-zp-year='2022' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Dibrov, A. (2022) "<a class='zp-ItemURL' target='_blank' href='https://tud.qucosa.de/id/qucosa:80379'>Towards Smarter Fluorescence Microscopy: Enabling Adaptive Acquisition Strategies With Optimized Photon Budget</a>." Available at: </div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-MZKL54LI" data-zp-author-date='Chiron-et-al.-2022-07-08' data-zp-date-author='2022-07-08-Chiron-et-al.' data-zp-date='2022-07-08' data-zp-year='2022' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Chiron, L. <i>et al.</i> (2022) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s41598-022-15207-5'>CyberSco.Py an open-source software for event-based, conditional microscopy</a>," <i>Scientific Reports</i>, 12(1), p. 11579. Available at: <a href="https://doi.org/10.1038/s41598-022-15207-5" rel="nofollow">https://doi.org/10.1038/s41598-022-15207-5</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-GRHGAX2R" data-zp-author-date='Beghin-et-al.-2022-07' data-zp-date-author='2022-07-Beghin-et-al.' data-zp-date='2022-07' data-zp-year='2022' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Beghin, A. <i>et al.</i> (2022) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s41592-022-01508-0'>Automated high-speed 3D imaging of organoid cultures with multi-scale phenotypic quantification</a>," <i>Nature Methods</i>, 19(7), pp. 881–892. Available at: <a href="https://doi.org/10.1038/s41592-022-01508-0" rel="nofollow">https://doi.org/10.1038/s41592-022-01508-0</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-89WA4U6K" data-zp-author-date='Fox-et-al.-2022-04-22' data-zp-date-author='2022-04-22-Fox-et-al.' data-zp-date='2022-04-22' data-zp-year='2022' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Fox, Z.R. <i>et al.</i> (2022) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s41467-022-29888-z'>Enabling reactive microscopy with MicroMator</a>," <i>Nature Communications</i>, 13(1), p. 2199. Available at: <a href="https://doi.org/10.1038/s41467-022-29888-z" rel="nofollow">https://doi.org/10.1038/s41467-022-29888-z</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-68YQKGG2" data-zp-author-date='Balluet-et-al.-2022' data-zp-date-author='2022-Balluet-et-al.' data-zp-date='2022' data-zp-year='2022' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Balluet, M. <i>et al.</i> (2022) "<a class='zp-ItemURL' target='_blank' href='https://onlinelibrary.wiley.com/doi/abs/10.1111/jmi.13062'>Neural network fast-classifies biological images through features selecting to power automated microscopy</a>," <i>Journal of Microscopy</i>, 285(1), pp. 3–19. Available at: <a href="https://doi.org/10.1111/jmi.13062" rel="nofollow">https://doi.org/10.1111/jmi.13062</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-732U9I9P" data-zp-author-date='Ouyang-et-al.-2022' data-zp-date-author='2022-Ouyang-et-al.' data-zp-date='2022' data-zp-year='2022' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Ouyang, W. <i>et al.</i> (2022) "<a class='zp-ItemURL' target='_blank' href='https://onlinelibrary.wiley.com/doi/abs/10.1002/adbi.202101063'>An Open-Source Modular Framework for Automated Pipetting and Imaging Applications</a>," <i>Advanced Biology</i>, 6(4), p. 2101063. Available at: <a href="https://doi.org/10.1002/adbi.202101063" rel="nofollow">https://doi.org/10.1002/adbi.202101063</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-S26EVKC2" data-zp-author-date='Bouvette-et-al.-2021-10' data-zp-date-author='2021-10-Bouvette-et-al.' data-zp-date='2021-10' data-zp-year='2021' data-zp-itemtype='conferencePaper' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Bouvette, J. <i>et al.</i> (2021) "<a class='zp-ItemURL' target='_blank' href='https://ieeexplore.ieee.org/document/9762104'>Smartscope: AI-driven grid navigation for high-throughput cryo-EM</a>," in <i>2021 IEEE Applied Imagery Pattern Recognition Workshop (AIPR)</i>. <i>2021 IEEE Applied Imagery Pattern Recognition Workshop (AIPR)</i>, pp. 1–6. Available at: <a href="https://doi.org/10.1109/AIPR52630.2021.9762104" rel="nofollow">https://doi.org/10.1109/AIPR52630.2021.9762104</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-WRFE42Q8" data-zp-author-date='Fan-et-al.-2021-09-03' data-zp-date-author='2021-09-03-Fan-et-al.' data-zp-date='2021-09-03' data-zp-year='2021' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Fan, Y. <i>et al.</i> (2021) "<a class='zp-ItemURL' target='_blank' href='https://doi.org/10.1186/s43074-021-00040-2'>Smart computational light microscopes (SCLMs) of smart computational imaging laboratory (SCILab)</a>," <i>PhotoniX</i>, 2(1), p. 19. Available at: </div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-4WSR74TL" data-zp-author-date='Levental-et-al.-2021-09' data-zp-date-author='2021-09-Levental-et-al.' data-zp-date='2021-09' data-zp-year='2021' data-zp-itemtype='conferencePaper' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Levental, M. <i>et al.</i> (2021) "<a class='zp-ItemURL' target='_blank' href='https://ieeexplore.ieee.org/document/9582285'>Ultrafast Focus Detection for Automated Microscopy</a>," in <i>2021 IEEE 17th International Conference on eScience (eScience)</i>. <i>2021 IEEE 17th International Conference on eScience (eScience)</i>, pp. 237–238. Available at: <a href="https://doi.org/10.1109/eScience51609.2021.00039" rel="nofollow">https://doi.org/10.1109/eScience51609.2021.00039</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-QTZV78RC" data-zp-author-date='Moreno-et-al.-2021-08-14' data-zp-date-author='2021-08-14-Moreno-et-al.' data-zp-date='2021-08-14' data-zp-year='2021' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Moreno, X.C. <i>et al.</i> (2021) "<a class='zp-ItemURL' target='_blank' href='https://joss.theoj.org/papers/10.21105/joss.03394'>ImSwitch: Generalizing microscope control in Python</a>," <i>Journal of Open Source Software</i>, 6(64), p. 3394. Available at: <a href="https://doi.org/10.21105/joss.03394" rel="nofollow">https://doi.org/10.21105/joss.03394</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-UKSYWZTP" data-zp-author-date='He-2020-01-24' data-zp-date-author='2020-01-24-He' data-zp-date='2020-01-24' data-zp-year='2020' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">He, J. (2020) "<a class='zp-ItemURL' target='_blank' href='https://nbn-resolving.org/urn:nbn:de:bsz:14-qucosa2-379998'>Towards a smarter light sheet microscope</a>." Available at: </div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-YLT2L6YA" data-zp-author-date='Seo-et-al.-2020' data-zp-date-author='2020-Seo-et-al.' data-zp-date='2020' data-zp-year='2020' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Seo, H. <i>et al.</i> (2020) "<a class='zp-ItemURL' target='_blank' href='https://onlinelibrary.wiley.com/doi/abs/10.1002/mp.13649'>Machine learning techniques for biomedical image segmentation: An overview of technical aspects and introduction to state-of-art applications</a>," <i>Medical Physics</i>, 47(5), pp. e148–e167. Available at: <a href="https://doi.org/10.1002/mp.13649" rel="nofollow">https://doi.org/10.1002/mp.13649</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-VAV4CAQ8" data-zp-author-date='Collinson-2019-08' data-zp-date-author='2019-08-Collinson' data-zp-date='2019-08' data-zp-year='2019' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Collinson, L.M. (2019) "<a class='zp-ItemURL' target='_blank' href='https://www.cambridge.org/core/journals/microscopy-and-microanalysis/article/smart-microscopy-automation-of-clem-using-in-situ-fluorescence-detection/0343ACFB2745C642F1ACACE64E7F70C9'>Smart Microscopy: Automation of CLEM using In situ Fluorescence Detection</a>," <i>Microscopy and Microanalysis</i>, 25(S2), pp. 1018–1019. Available at: <a href="https://doi.org/10.1017/S1431927619005828" rel="nofollow">https://doi.org/10.1017/S1431927619005828</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-ZTH5I6CE" data-zp-author-date='Liu-et-al.-2018-04-20' data-zp-date-author='2018-04-20-Liu-et-al.' data-zp-date='2018-04-20' data-zp-year='2018' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Liu, T.-L. <i>et al.</i> (2018) "<a class='zp-ItemURL' target='_blank' href='https://www.science.org/doi/10.1126/science.aaq1392'>Observing the cell in its native state: Imaging subcellular dynamics in multicellular organisms</a>," <i>Science</i>, 360(6386), p. eaaq1392. Available at: <a href="https://doi.org/10.1126/science.aaq1392" rel="nofollow">https://doi.org/10.1126/science.aaq1392</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-EYAZIWXL" data-zp-author-date='Ouyang-and-Zimmer-2017-08-01' data-zp-date-author='2017-08-01-Ouyang-and-Zimmer' data-zp-date='2017-08-01' data-zp-year='2017' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Ouyang, W. and Zimmer, C. (2017) "<a class='zp-ItemURL' target='_blank' href='https://www.sciencedirect.com/science/article/pii/S2452310017300537'>The imaging tsunami: Computational opportunities and challenges</a>," <i>Current Opinion in Systems Biology</i>, 4, pp. 105–113. Available at: <a href="https://doi.org/10.1016/j.coisb.2017.07.011" rel="nofollow">https://doi.org/10.1016/j.coisb.2017.07.011</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-A82CIX6S" data-zp-author-date='Royer-et-al.-2016-12' data-zp-date-author='2016-12-Royer-et-al.' data-zp-date='2016-12' data-zp-year='2016' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Royer, L.A. <i>et al.</i> (2016) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/nbt.3708'>Adaptive light-sheet microscopy for long-term, high-resolution imaging in living organisms</a>," <i>Nature Biotechnology</i>, 34(12), pp. 1267–1278. Available at: <a href="https://doi.org/10.1038/nbt.3708" rel="nofollow">https://doi.org/10.1038/nbt.3708</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-N76BLUWJ" data-zp-author-date='Chakrova-et-al.-2016-10-01' data-zp-date-author='2016-10-01-Chakrova-et-al.' data-zp-date='2016-10-01' data-zp-year='2016' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Chakrova, N. <i>et al.</i> (2016) "<a class='zp-ItemURL' target='_blank' href='https://opg.optica.org/boe/abstract.cfm?uri=boe-7-10-4263'>Adaptive illumination reduces photobleaching in structured illumination microscopy</a>," <i>Biomedical Optics Express</i>, 7(10), pp. 4263–4274. Available at: <a href="https://doi.org/10.1364/BOE.7.004263" rel="nofollow">https://doi.org/10.1364/BOE.7.004263</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-WZRYEDHE" data-zp-author-date='Jones-et-al.-2015-07-10' data-zp-date-author='2015-07-10-Jones-et-al.' data-zp-date='2015-07-10' data-zp-year='2015' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Jones, L. <i>et al.</i> (2015) "<a class='zp-ItemURL' target='_blank' href='https://doi.org/10.1186/s40679-015-0008-4'>Smart Align—a new tool for robust non-rigid registration of scanning microscope data</a>," <i>Advanced Structural and Chemical Imaging</i>, 1(1), p. 8. Available at: </div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-SCXQYP4A" data-zp-author-date='Conrad-et-al.-2011-03' data-zp-date-author='2011-03-Conrad-et-al.' data-zp-date='2011-03' data-zp-year='2011' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Conrad, C. <i>et al.</i> (2011) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/nmeth.1558'>Micropilot: automation of fluorescence microscopy–based imaging for systems biology</a>," <i>Nature Methods</i>, 8(3), pp. 246–249. Available at: <a href="https://doi.org/10.1038/nmeth.1558" rel="nofollow">https://doi.org/10.1038/nmeth.1558</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-NVUKKSUV" data-zp-author-date='Caarls-et-al.-2011-01' data-zp-date-author='2011-01-Caarls-et-al.' data-zp-date='2011-01' data-zp-year='2011' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Caarls, W. <i>et al.</i> (2011) "<a class='zp-ItemURL' target='_blank' href='https://onlinelibrary.wiley.com/doi/10.1111/j.1365-2818.2010.03413.x'>Minimizing light exposure with the programmable array microscope</a>," <i>Journal of Microscopy</i>, 241(1), pp. 101–110. Available at: <a href="https://doi.org/10.1111/j.1365-2818.2010.03413.x" rel="nofollow">https://doi.org/10.1111/j.1365-2818.2010.03413.x</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-UJ5FWVJD" data-zp-author-date='Stagg-et-al.-2006-08-01' data-zp-date-author='2006-08-01-Stagg-et-al.' data-zp-date='2006-08-01' data-zp-year='2006' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Stagg, S. <i>et al.</i> (2006) "<a class='zp-ItemURL' target='_blank' href='https://doi.org/10.1017/S1431927606062969'>A Case Study in Automated Molecular Microscopy</a>," <i>Microscopy and Microanalysis</i>, 12(S02), pp. 1096–1097. Available at: </div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-N4UTEB9B" data-zp-author-date='Joy-2002' data-zp-date-author='2002-Joy' data-zp-date='2002' data-zp-year='2002' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Joy, D.C. (2002) "<a class='zp-ItemURL' target='_blank' href='https://onlinelibrary.wiley.com/doi/abs/10.1046/j.1365-2818.2002.01062.x'>SMART – a program to measure SEM resolution and imaging performance</a>," <i>Journal of Microscopy</i>, 208(1), pp. 24–34. Available at: <a href="https://doi.org/10.1046/j.1365-2818.2002.01062.x" rel="nofollow">https://doi.org/10.1046/j.1365-2818.2002.01062.x</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-L32UDJ7R" data-zp-author-date='Yavuz-et-al.-0000-00-00' data-zp-date-author='0000-00-00-Yavuz-et-al.' data-zp-date='0000-00-00' data-zp-year='0000' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Yavuz, S. <i>et al.</i> (no date) "<a class='zp-ItemURL' target='_blank' href='https://onlinelibrary.wiley.com/doi/abs/10.1002/2211-5463.70161'>Automated FRAP microscopy for high-throughput analysis of protein dynamics in chromatin organization and transcription</a>," <i>FEBS Open Bio</i>, n/a(n/a). Available at: <a href="https://doi.org/10.1002/2211-5463.70161" rel="nofollow">https://doi.org/10.1002/2211-5463.70161</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->
			</div><!-- .zp-zp-SEO-Content -->
		</div><!-- .zp-List -->
	</div><!--.zp-Zotpress-->


    <div class="zotpress-table-container"></div>
  </div>
  <div class="zotpress-section" data-collection="4I89N939">
    <div id="zotpress-fa72f429703fe89c14c48f1739683632" class="zp-Zotpress zp-Zotpress-Bib wp-block-group">

		<span class="ZP_API_USER_ID ZP_ATTR">5384098</span>
		<span class="ZP_ITEM_KEY ZP_ATTR"></span>
		<span class="ZP_COLLECTION_ID ZP_ATTR">4I89N939</span>
		<span class="ZP_TAG_ID ZP_ATTR"></span>
		<span class="ZP_AUTHOR ZP_ATTR"></span>
		<span class="ZP_YEAR ZP_ATTR"></span>
        <span class="ZP_ITEMTYPE ZP_ATTR"></span>
		<span class="ZP_INCLUSIVE ZP_ATTR">1</span>
		<span class="ZP_STYLE ZP_ATTR">harvard1</span>
		<span class="ZP_LIMIT ZP_ATTR">50</span>
		<span class="ZP_SORTBY ZP_ATTR">date</span>
		<span class="ZP_ORDER ZP_ATTR">desc</span>
		<span class="ZP_TITLE ZP_ATTR"></span>
		<span class="ZP_SHOWIMAGE ZP_ATTR"></span>
		<span class="ZP_SHOWTAGS ZP_ATTR">1</span>
		<span class="ZP_DOWNLOADABLE ZP_ATTR">1</span>
		<span class="ZP_NOTES ZP_ATTR"></span>
		<span class="ZP_ABSTRACT ZP_ATTR"></span>
		<span class="ZP_CITEABLE ZP_ATTR"></span>
		<span class="ZP_TARGET ZP_ATTR">1</span>
		<span class="ZP_URLWRAP ZP_ATTR">title</span>
		<span class="ZP_FORCENUM ZP_ATTR"></span>
        <span class="ZP_HIGHLIGHT ZP_ATTR"></span>
        <span class="ZP_POSTID ZP_ATTR">27</span>
		<span class="ZOTPRESS_PLUGIN_URL ZP_ATTR"><a href="/wp-content/plugins/zotpress/">/wp-content/plugins/zotpress/</a></span>

		<div class="zp-List loading">
			<div class="zp-SEO-Content">

			</div><!-- .zp-zp-SEO-Content -->
		</div><!-- .zp-List -->
	</div><!--.zp-Zotpress-->


    <div class="zotpress-table-container"></div>
  </div>
  <div class="zotpress-section" data-collection="HQUL7BI3">
    <div id="zotpress-6ea79a1b7935ea7bb2400fb2b6c9c5e8" class="zp-Zotpress zp-Zotpress-Bib wp-block-group">

		<span class="ZP_API_USER_ID ZP_ATTR">5384098</span>
		<span class="ZP_ITEM_KEY ZP_ATTR"></span>
		<span class="ZP_COLLECTION_ID ZP_ATTR">HQUL7BI3</span>
		<span class="ZP_TAG_ID ZP_ATTR"></span>
		<span class="ZP_AUTHOR ZP_ATTR"></span>
		<span class="ZP_YEAR ZP_ATTR"></span>
        <span class="ZP_ITEMTYPE ZP_ATTR"></span>
		<span class="ZP_INCLUSIVE ZP_ATTR">1</span>
		<span class="ZP_STYLE ZP_ATTR">harvard1</span>
		<span class="ZP_LIMIT ZP_ATTR">50</span>
		<span class="ZP_SORTBY ZP_ATTR">date</span>
		<span class="ZP_ORDER ZP_ATTR">desc</span>
		<span class="ZP_TITLE ZP_ATTR"></span>
		<span class="ZP_SHOWIMAGE ZP_ATTR"></span>
		<span class="ZP_SHOWTAGS ZP_ATTR">1</span>
		<span class="ZP_DOWNLOADABLE ZP_ATTR">1</span>
		<span class="ZP_NOTES ZP_ATTR"></span>
		<span class="ZP_ABSTRACT ZP_ATTR"></span>
		<span class="ZP_CITEABLE ZP_ATTR"></span>
		<span class="ZP_TARGET ZP_ATTR">1</span>
		<span class="ZP_URLWRAP ZP_ATTR">title</span>
		<span class="ZP_FORCENUM ZP_ATTR"></span>
        <span class="ZP_HIGHLIGHT ZP_ATTR"></span>
        <span class="ZP_POSTID ZP_ATTR">27</span>
		<span class="ZOTPRESS_PLUGIN_URL ZP_ATTR"><a href="/wp-content/plugins/zotpress/">/wp-content/plugins/zotpress/</a></span>

		<div class="zp-List loading">
			<div class="zp-SEO-Content">
				<span class="ZP_JSON ZP_ATTR">%7B%22status%22%3A%22success%22%2C%22updateneeded%22%3Afalse%2C%22instance%22%3Afalse%2C%22meta%22%3A%7B%22request_last%22%3A0%2C%22request_next%22%3A0%2C%22used_cache%22%3Atrue%7D%2C%22data%22%3A%5B%7B%22key%22%3A%22TWDLDCMT%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Tang%22%2C%22parsedDate%22%3A%222025-12%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BTang%2C%20L.%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-025-02986-8%26%23039%3B%26gt%3BBridging%20smart%20microscopy%20with%20genomics%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%20Methods%26lt%3B%5C%2Fi%26gt%3B%2C%2022%2812%29%2C%20pp.%202499%5Cu20132499.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41592-025-02986-8.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Bridging%20smart%20microscopy%20with%20genomics%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Lei%22%2C%22lastName%22%3A%22Tang%22%7D%5D%2C%22abstractNote%22%3A%22%22%2C%22date%22%3A%222025-12%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs41592-025-02986-8%22%2C%22ISSN%22%3A%221548-7105%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-025-02986-8%22%2C%22collections%22%3A%5B%22HQUL7BI3%22%5D%2C%22dateModified%22%3A%222025-12-16T13%3A11%3A51Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Chromosome%20segregation%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22Genomic%20analysis%22%2C%22type%22%3A1%7D%2C%7B%22tag%22%3A%22Genomic%20instability%22%2C%22type%22%3A1%7D%5D%7D%7D%2C%7B%22key%22%3A%22BUIQZUCI%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Nature%20Methods%22%2C%22parsedDate%22%3A%222023-07%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BNature%20Methods%20%282023%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-023-01950-8%26%23039%3B%26gt%3BWhat%26%23039%3Bs%20next%20for%20bioimage%20analysis%3F%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%20Methods%26lt%3B%5C%2Fi%26gt%3B%2C%2020%287%29%2C%20pp.%20945%5Cu2013946.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41592-023-01950-8.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22What%27s%20next%20for%20bioimage%20analysis%3F%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22%22%2C%22lastName%22%3A%22Nature%20Methods%22%7D%5D%2C%22abstractNote%22%3A%22Advanced%20bioimage%20analysis%20tools%20are%20poised%20to%20disrupt%20the%20way%20in%20which%20microscopy%20images%20are%20acquired%20and%20analyzed.%20This%20Focus%20issue%20shares%20the%20hopes%20and%20opinions%20of%20experts%20on%20the%20near%20and%20distant%20future%20of%20image%20analysis.%22%2C%22date%22%3A%222023-07%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs41592-023-01950-8%22%2C%22ISSN%22%3A%221548-7105%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-023-01950-8%22%2C%22collections%22%3A%5B%22HQUL7BI3%22%5D%2C%22dateModified%22%3A%222025-04-29T13%3A12%3A34Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Data%20Analysis%22%7D%2C%7B%22tag%22%3A%22Data%20handling%22%7D%5D%7D%7D%2C%7B%22key%22%3A%2269VUKG2U%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Carpenter%20et%20al.%22%2C%22parsedDate%22%3A%222023-07%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BCarpenter%2C%20A.E.%2C%20Cimini%2C%20B.A.%20and%20Eliceiri%2C%20K.W.%20%282023%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-023-01912-0%26%23039%3B%26gt%3BSmart%20microscopes%20of%20the%20future%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%20Methods%26lt%3B%5C%2Fi%26gt%3B%2C%2020%287%29%2C%20pp.%20962%5Cu2013964.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41592-023-01912-0.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Smart%20microscopes%20of%20the%20future%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Anne%20E.%22%2C%22lastName%22%3A%22Carpenter%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Beth%20A.%22%2C%22lastName%22%3A%22Cimini%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Kevin%20W.%22%2C%22lastName%22%3A%22Eliceiri%22%7D%5D%2C%22abstractNote%22%3A%22We%20dream%20of%20a%20future%20where%20light%20microscopes%20have%20new%20capabilities%3A%20language-guided%20image%20acquisition%2C%20automatic%20image%20analysis%20based%20on%20extensive%20prior%20training%20from%20biologist%20experts%2C%20and%20language-guided%20image%20analysis%20for%20custom%20analyses.%20Most%20capabilities%20have%20reached%20the%20proof-of-principle%20stage%2C%20but%20implementation%20would%20be%20accelerated%20by%20efforts%20to%20gather%20appropriate%20training%20sets%20and%20make%20user-friendly%20interfaces.%22%2C%22date%22%3A%222023-07%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs41592-023-01912-0%22%2C%22ISSN%22%3A%221548-7105%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-023-01912-0%22%2C%22collections%22%3A%5B%22HQUL7BI3%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A09%3A00Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%2C%7B%22tag%22%3A%22Data%20Analysis%22%7D%2C%7B%22tag%22%3A%22Data%20handling%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22GZPGA3QN%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Madhusoodanan%22%2C%22parsedDate%22%3A%222023-02-07%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BMadhusoodanan%2C%20J.%20%282023%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fd41586-023-00336-2%26%23039%3B%26gt%3BSmart%20microscopes%20spot%20fleeting%20biology%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%26lt%3B%5C%2Fi%26gt%3B%2C%20614%287947%29%2C%20pp.%20378%5Cu2013380.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fd41586-023-00336-2.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Smart%20microscopes%20spot%20fleeting%20biology%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jyoti%22%2C%22lastName%22%3A%22Madhusoodanan%22%7D%5D%2C%22abstractNote%22%3A%22Automated%20microscopes%20that%20adapt%20to%20each%20sample%5Cu2019s%20quirks%20can%20capture%20elusive%20biological%20phenomena%20at%20high%20resolution.%22%2C%22date%22%3A%222023-02-07%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fd41586-023-00336-2%22%2C%22ISSN%22%3A%22%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fd41586-023-00336-2%22%2C%22collections%22%3A%5B%22HQUL7BI3%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A08%3A03Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22YPXBQ6F2%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Ma%20and%20Wang%22%2C%22parsedDate%22%3A%222023%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BMa%2C%20J.%20and%20Wang%2C%20B.%20%282023%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-023-01885-0%26%23039%3B%26gt%3BTowards%20foundation%20models%20of%20biological%20image%20segmentation%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%20Methods%26lt%3B%5C%2Fi%26gt%3B%2C%2020%287%29%2C%20pp.%20953%5Cu2013955.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41592-023-01885-0.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Towards%20foundation%20models%20of%20biological%20image%20segmentation%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jun%22%2C%22lastName%22%3A%22Ma%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Bo%22%2C%22lastName%22%3A%22Wang%22%7D%5D%2C%22abstractNote%22%3A%22%22%2C%22date%22%3A%2207%5C%2F2023%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs41592-023-01885-0%22%2C%22ISSN%22%3A%221548-7091%2C%201548-7105%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-023-01885-0%22%2C%22collections%22%3A%5B%22HQUL7BI3%22%5D%2C%22dateModified%22%3A%222025-04-29T14%3A02%3A16Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Data%20analysis%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22E525TMMZ%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Pinkard%20and%20Waller%22%2C%22parsedDate%22%3A%222022%22%2C%22numChildren%22%3A1%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BPinkard%2C%20H.%20and%20Waller%2C%20L.%20%282022%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-022-01566-4%26%23039%3B%26gt%3BMicroscopes%20are%20coming%20for%20your%20job%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%20Methods%26lt%3B%5C%2Fi%26gt%3B%2C%2019%2810%29%2C%20pp.%201175%5Cu20131176.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41592-022-01566-4.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Microscopes%20are%20coming%20for%20your%20job%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Henry%22%2C%22lastName%22%3A%22Pinkard%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Laura%22%2C%22lastName%22%3A%22Waller%22%7D%5D%2C%22abstractNote%22%3A%22%22%2C%22date%22%3A%2210%5C%2F2022%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs41592-022-01566-4%22%2C%22ISSN%22%3A%221548-7091%2C%201548-7105%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-022-01566-4%22%2C%22collections%22%3A%5B%22HQUL7BI3%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A07%3A52Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%2248VNXAQ6%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Kugler%22%2C%22parsedDate%22%3A%222021-04-24%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BKugler%2C%20E.%20%282021%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Ffocalplane.biologists.com%5C%2F2021%5C%2F04%5C%2F24%5C%2Flsfm-series-partv-the-future%5C%2F%26%23039%3B%26gt%3BPart%20V%20The%20future%3A%20The%20hope%20of%20smart%20microscopes%20and%20phantoms%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BFocalPlane%26lt%3B%5C%2Fi%26gt%3B%20%5BPreprint%5D.%20Available%20at%3A%20%28Accessed%3A%20January%2031%2C%202024%29.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Part%20V%20The%20future%3A%20The%20hope%20of%20smart%20microscopes%20and%20phantoms%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Elisabeth%22%2C%22lastName%22%3A%22Kugler%22%7D%5D%2C%22abstractNote%22%3A%22Part%20V%20The%20future%3A%20The%20hope%20of%20smart%20microscopes%20and%20phantoms%20-%20Tools%22%2C%22date%22%3A%222021-04-24T09%3A00%3A00%2B00%3A00%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%22%22%2C%22ISSN%22%3A%22%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Ffocalplane.biologists.com%5C%2F2021%5C%2F04%5C%2F24%5C%2Flsfm-series-partv-the-future%5C%2F%22%2C%22collections%22%3A%5B%22HQUL7BI3%22%5D%2C%22dateModified%22%3A%222025-04-29T14%3A04%3A04Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22YWWRYJ74%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Eisenstein%22%2C%22parsedDate%22%3A%222020-11%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BEisenstein%2C%20M.%20%282020%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-020-00988-2%26%23039%3B%26gt%3BSmart%20solutions%20for%20automated%20imaging%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%20Methods%26lt%3B%5C%2Fi%26gt%3B%2C%2017%2811%29%2C%20pp.%201075%5Cu20131079.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41592-020-00988-2.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Smart%20solutions%20for%20automated%20imaging%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Michael%22%2C%22lastName%22%3A%22Eisenstein%22%7D%5D%2C%22abstractNote%22%3A%22Algorithms%20trained%20to%20interpret%20microscope%20data%20can%20greatly%20extend%20the%20information%20that%20can%20be%20derived%20from%20the%20resulting%20images%2C%20or%20even%20optimize%20how%20imaging%20experiments%20are%20conducted.%22%2C%22date%22%3A%222020-11%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs41592-020-00988-2%22%2C%22ISSN%22%3A%221548-7105%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-020-00988-2%22%2C%22collections%22%3A%5B%22HQUL7BI3%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A08%3A47Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22CN3KALYL%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Strack%22%2C%22parsedDate%22%3A%222020-01%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BStrack%2C%20R.%20%282020%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-019-0708-0%26%23039%3B%26gt%3BSmarter%20microscopes%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%20Methods%26lt%3B%5C%2Fi%26gt%3B%2C%2017%281%29%2C%20pp.%2023%5Cu201323.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41592-019-0708-0.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Smarter%20microscopes%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Rita%22%2C%22lastName%22%3A%22Strack%22%7D%5D%2C%22abstractNote%22%3A%22%22%2C%22date%22%3A%222020-01%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs41592-019-0708-0%22%2C%22ISSN%22%3A%221548-7105%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-019-0708-0%22%2C%22collections%22%3A%5B%22HQUL7BI3%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A07%3A28Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%2C%7B%22tag%22%3A%22Data%20Analysis%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22RUG44S9D%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Huisken%22%2C%22parsedDate%22%3A%222017-07-01%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BHuisken%2C%20J.%20%282017%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1017%5C%2FS1431927617006493%26%23039%3B%26gt%3BSmart%20Microscopy%20for%20Multi-Scale%20Developmental%20Biology%20in%20Real-Time%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BMicroscopy%20and%20Microanalysis%26lt%3B%5C%2Fi%26gt%3B%2C%2023%28S1%29%2C%20pp.%201166%5Cu20131167.%20Available%20at%3A%20%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Smart%20Microscopy%20for%20Multi-Scale%20Developmental%20Biology%20in%20Real-Time%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jan%22%2C%22lastName%22%3A%22Huisken%22%7D%5D%2C%22abstractNote%22%3A%22%22%2C%22date%22%3A%222017-07-01%22%2C%22language%22%3A%22%22%2C%22DOI%22%3A%2210.1017%5C%2FS1431927617006493%22%2C%22ISSN%22%3A%221431-9276%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1017%5C%2FS1431927617006493%22%2C%22collections%22%3A%5B%22HQUL7BI3%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A08%3A26Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22UF3MXTXS%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Scherf%20and%20Huisken%22%2C%22parsedDate%22%3A%222015-08%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BScherf%2C%20N.%20and%20Huisken%2C%20J.%20%282015%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fnbt.3310%26%23039%3B%26gt%3BThe%20smart%20and%20gentle%20microscope%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%20Biotechnology%26lt%3B%5C%2Fi%26gt%3B%2C%2033%288%29%2C%20pp.%20815%5Cu2013818.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fnbt.3310.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22The%20smart%20and%20gentle%20microscope%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Nico%22%2C%22lastName%22%3A%22Scherf%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jan%22%2C%22lastName%22%3A%22Huisken%22%7D%5D%2C%22abstractNote%22%3A%22New%20instruments%20are%20needed%20to%20realize%20the%20potential%20of%20quantitative%20and%20systematic%20imaging%20of%20living%20samples.%20But%20what%20would%20such%20a%20microscope%20look%20like%3F%22%2C%22date%22%3A%222015-08%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fnbt.3310%22%2C%22ISSN%22%3A%221546-1696%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fnbt.3310%22%2C%22collections%22%3A%5B%22HQUL7BI3%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A07%3A44Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22RN8R7ZQK%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Glaser%22%2C%22parsedDate%22%3A%222012-12%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BGlaser%2C%20V.%20%282012%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.liebertpub.com%5C%2Fdoi%5C%2Ffull%5C%2F10.1089%5C%2Fgen.32.21.13%26%23039%3B%26gt%3BClose-Up%20View%20of%20Life%20Science%20Microscopy%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BGenetic%20Engineering%20%26amp%3B%20Biotechnology%20News%26lt%3B%5C%2Fi%26gt%3B%2C%2032%2821%29%2C%20pp.%2030%5Cu201332.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1089%5C%2Fgen.32.21.13.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Close-Up%20View%20of%20Life%20Science%20Microscopy%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Vicki%22%2C%22lastName%22%3A%22Glaser%22%7D%5D%2C%22abstractNote%22%3A%22%22%2C%22date%22%3A%222012-12%22%2C%22language%22%3A%22%22%2C%22DOI%22%3A%2210.1089%5C%2Fgen.32.21.13%22%2C%22ISSN%22%3A%221935-472X%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.liebertpub.com%5C%2Fdoi%5C%2Ffull%5C%2F10.1089%5C%2Fgen.32.21.13%22%2C%22collections%22%3A%5B%22HQUL7BI3%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A08%3A35Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22TYNYJA9G%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Prewitt%22%2C%22parsedDate%22%3A%221979-11%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BPrewitt%2C%20J.M.S.%20%281979%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fieeexplore.ieee.org%5C%2Fdocument%5C%2F762545%26%23039%3B%26gt%3BIntelligent%20microscopes%3A%20a%20scientific%20poem%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20in%20%26lt%3Bi%26gt%3BCOMPSAC%2079.%20Proceedings.%20Computer%20Software%20and%20The%20IEEE%20Computer%20Society%26%23039%3Bs%20Third%20International%20Applications%20Conference%2C%201979.%26lt%3B%5C%2Fi%26gt%3B%20%26lt%3Bi%26gt%3BCOMPSAC%2079.%20Proceedings.%20Computer%20Software%20and%20The%20IEEE%20Computer%20Society%26%23039%3Bs%20Third%20International%20Applications%20Conference%2C%201979.%26lt%3B%5C%2Fi%26gt%3B%2C%20pp.%20477%5Cu2013487.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1109%5C%2FCMPSAC.1979.762545.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22conferencePaper%22%2C%22title%22%3A%22Intelligent%20microscopes%3A%20a%20scientific%20poem%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22J.M.S.%22%2C%22lastName%22%3A%22Prewitt%22%7D%5D%2C%22abstractNote%22%3A%22This%20paper%2C%20written%20in%20the%20form%20of%20a%20scientific%20poem%2C%20reviews%20the%20current%20status%20of%20automated%20intel%20ligent%20microscopes%20based%20on%20computer%20technology.%20The%20basic%20concepts%20of%20image%20analysis%20for%20cytology%20and%20histology%20are%20presented%20and%20illustrated.%20Limitations%20of%20commercial%20devices%20and%20research%20endeavors%20are%20examined%2C%20and%20remedies%20are%20suggested.%22%2C%22date%22%3A%221979-11%22%2C%22proceedingsTitle%22%3A%22COMPSAC%2079.%20Proceedings.%20Computer%20Software%20and%20The%20IEEE%20Computer%20Society%27s%20Third%20International%20Applications%20Conference%2C%201979.%22%2C%22conferenceName%22%3A%22COMPSAC%2079.%20Proceedings.%20Computer%20Software%20and%20The%20IEEE%20Computer%20Society%27s%20Third%20International%20Applications%20Conference%2C%201979.%22%2C%22language%22%3A%22%22%2C%22DOI%22%3A%2210.1109%5C%2FCMPSAC.1979.762545%22%2C%22ISBN%22%3A%22%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fieeexplore.ieee.org%5C%2Fdocument%5C%2F762545%22%2C%22collections%22%3A%5B%22HQUL7BI3%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A09%3A16Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Automation%22%7D%2C%7B%22tag%22%3A%22Data%20Analysis%22%7D%5D%7D%7D%5D%7D</span>

				<div id="zp-ID-27-5384098-TWDLDCMT" data-zp-author-date='Tang-2025-12' data-zp-date-author='2025-12-Tang' data-zp-date='2025-12' data-zp-year='2025' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Tang, L. (2025) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s41592-025-02986-8'>Bridging smart microscopy with genomics</a>," <i>Nature Methods</i>, 22(12), pp. 2499–2499. Available at: <a href="https://doi.org/10.1038/s41592-025-02986-8" rel="nofollow">https://doi.org/10.1038/s41592-025-02986-8</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-BUIQZUCI" data-zp-author-date='Nature-Methods-2023-07' data-zp-date-author='2023-07-Nature-Methods' data-zp-date='2023-07' data-zp-year='2023' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Nature Methods (2023) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s41592-023-01950-8'>What's next for bioimage analysis?</a>," <i>Nature Methods</i>, 20(7), pp. 945–946. Available at: <a href="https://doi.org/10.1038/s41592-023-01950-8" rel="nofollow">https://doi.org/10.1038/s41592-023-01950-8</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-69VUKG2U" data-zp-author-date='Carpenter-et-al.-2023-07' data-zp-date-author='2023-07-Carpenter-et-al.' data-zp-date='2023-07' data-zp-year='2023' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Carpenter, A.E., Cimini, B.A. and Eliceiri, K.W. (2023) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s41592-023-01912-0'>Smart microscopes of the future</a>," <i>Nature Methods</i>, 20(7), pp. 962–964. Available at: <a href="https://doi.org/10.1038/s41592-023-01912-0" rel="nofollow">https://doi.org/10.1038/s41592-023-01912-0</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-GZPGA3QN" data-zp-author-date='Madhusoodanan-2023-02-07' data-zp-date-author='2023-02-07-Madhusoodanan' data-zp-date='2023-02-07' data-zp-year='2023' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Madhusoodanan, J. (2023) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/d41586-023-00336-2'>Smart microscopes spot fleeting biology</a>," <i>Nature</i>, 614(7947), pp. 378–380. Available at: <a href="https://doi.org/10.1038/d41586-023-00336-2" rel="nofollow">https://doi.org/10.1038/d41586-023-00336-2</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-YPXBQ6F2" data-zp-author-date='Ma-and-Wang-2023' data-zp-date-author='2023-Ma-and-Wang' data-zp-date='2023' data-zp-year='2023' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Ma, J. and Wang, B. (2023) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s41592-023-01885-0'>Towards foundation models of biological image segmentation</a>," <i>Nature Methods</i>, 20(7), pp. 953–955. Available at: <a href="https://doi.org/10.1038/s41592-023-01885-0" rel="nofollow">https://doi.org/10.1038/s41592-023-01885-0</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-E525TMMZ" data-zp-author-date='Pinkard-and-Waller-2022' data-zp-date-author='2022-Pinkard-and-Waller' data-zp-date='2022' data-zp-year='2022' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Pinkard, H. and Waller, L. (2022) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s41592-022-01566-4'>Microscopes are coming for your job</a>," <i>Nature Methods</i>, 19(10), pp. 1175–1176. Available at: <a href="https://doi.org/10.1038/s41592-022-01566-4" rel="nofollow">https://doi.org/10.1038/s41592-022-01566-4</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-48VNXAQ6" data-zp-author-date='Kugler-2021-04-24' data-zp-date-author='2021-04-24-Kugler' data-zp-date='2021-04-24' data-zp-year='2021' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Kugler, E. (2021) "<a class='zp-ItemURL' target='_blank' href='https://focalplane.biologists.com/2021/04/24/lsfm-series-partv-the-future/'>Part V The future: The hope of smart microscopes and phantoms</a>," <i>FocalPlane</i> [Preprint]. Available at: (Accessed: January 31, 2024).</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-YWWRYJ74" data-zp-author-date='Eisenstein-2020-11' data-zp-date-author='2020-11-Eisenstein' data-zp-date='2020-11' data-zp-year='2020' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Eisenstein, M. (2020) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s41592-020-00988-2'>Smart solutions for automated imaging</a>," <i>Nature Methods</i>, 17(11), pp. 1075–1079. Available at: <a href="https://doi.org/10.1038/s41592-020-00988-2" rel="nofollow">https://doi.org/10.1038/s41592-020-00988-2</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-CN3KALYL" data-zp-author-date='Strack-2020-01' data-zp-date-author='2020-01-Strack' data-zp-date='2020-01' data-zp-year='2020' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Strack, R. (2020) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s41592-019-0708-0'>Smarter microscopes</a>," <i>Nature Methods</i>, 17(1), pp. 23–23. Available at: <a href="https://doi.org/10.1038/s41592-019-0708-0" rel="nofollow">https://doi.org/10.1038/s41592-019-0708-0</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-RUG44S9D" data-zp-author-date='Huisken-2017-07-01' data-zp-date-author='2017-07-01-Huisken' data-zp-date='2017-07-01' data-zp-year='2017' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Huisken, J. (2017) "<a class='zp-ItemURL' target='_blank' href='https://doi.org/10.1017/S1431927617006493'>Smart Microscopy for Multi-Scale Developmental Biology in Real-Time</a>," <i>Microscopy and Microanalysis</i>, 23(S1), pp. 1166–1167. Available at: </div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-UF3MXTXS" data-zp-author-date='Scherf-and-Huisken-2015-08' data-zp-date-author='2015-08-Scherf-and-Huisken' data-zp-date='2015-08' data-zp-year='2015' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Scherf, N. and Huisken, J. (2015) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/nbt.3310'>The smart and gentle microscope</a>," <i>Nature Biotechnology</i>, 33(8), pp. 815–818. Available at: <a href="https://doi.org/10.1038/nbt.3310" rel="nofollow">https://doi.org/10.1038/nbt.3310</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-RN8R7ZQK" data-zp-author-date='Glaser-2012-12' data-zp-date-author='2012-12-Glaser' data-zp-date='2012-12' data-zp-year='2012' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Glaser, V. (2012) "<a class='zp-ItemURL' target='_blank' href='https://www.liebertpub.com/doi/full/10.1089/gen.32.21.13'>Close-Up View of Life Science Microscopy</a>," <i>Genetic Engineering & Biotechnology News</i>, 32(21), pp. 30–32. Available at: <a href="https://doi.org/10.1089/gen.32.21.13" rel="nofollow">https://doi.org/10.1089/gen.32.21.13</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-TYNYJA9G" data-zp-author-date='Prewitt-1979-11' data-zp-date-author='1979-11-Prewitt' data-zp-date='1979-11' data-zp-year='1979' data-zp-itemtype='conferencePaper' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Prewitt, J.M.S. (1979) "<a class='zp-ItemURL' target='_blank' href='https://ieeexplore.ieee.org/document/762545'>Intelligent microscopes: a scientific poem</a>," in <i>COMPSAC 79. Proceedings. Computer Software and The IEEE Computer Society's Third International Applications Conference, 1979.</i> <i>COMPSAC 79. Proceedings. Computer Software and The IEEE Computer Society's Third International Applications Conference, 1979.</i>, pp. 477–487. Available at: <a href="https://doi.org/10.1109/CMPSAC.1979.762545" rel="nofollow">https://doi.org/10.1109/CMPSAC.1979.762545</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->
			</div><!-- .zp-zp-SEO-Content -->
		</div><!-- .zp-List -->
	</div><!--.zp-Zotpress-->


    <div class="zotpress-table-container"></div>
  </div>
  <div class="zotpress-section" data-collection="UQ77P7I5">
    <div id="zotpress-662db4706e4a03b5c2e968421a31cb43" class="zp-Zotpress zp-Zotpress-Bib wp-block-group">

		<span class="ZP_API_USER_ID ZP_ATTR">5384098</span>
		<span class="ZP_ITEM_KEY ZP_ATTR"></span>
		<span class="ZP_COLLECTION_ID ZP_ATTR">UQ77P7I5</span>
		<span class="ZP_TAG_ID ZP_ATTR"></span>
		<span class="ZP_AUTHOR ZP_ATTR"></span>
		<span class="ZP_YEAR ZP_ATTR"></span>
        <span class="ZP_ITEMTYPE ZP_ATTR"></span>
		<span class="ZP_INCLUSIVE ZP_ATTR">1</span>
		<span class="ZP_STYLE ZP_ATTR">harvard1</span>
		<span class="ZP_LIMIT ZP_ATTR">50</span>
		<span class="ZP_SORTBY ZP_ATTR">date</span>
		<span class="ZP_ORDER ZP_ATTR">desc</span>
		<span class="ZP_TITLE ZP_ATTR"></span>
		<span class="ZP_SHOWIMAGE ZP_ATTR"></span>
		<span class="ZP_SHOWTAGS ZP_ATTR">1</span>
		<span class="ZP_DOWNLOADABLE ZP_ATTR">1</span>
		<span class="ZP_NOTES ZP_ATTR"></span>
		<span class="ZP_ABSTRACT ZP_ATTR"></span>
		<span class="ZP_CITEABLE ZP_ATTR"></span>
		<span class="ZP_TARGET ZP_ATTR">1</span>
		<span class="ZP_URLWRAP ZP_ATTR">title</span>
		<span class="ZP_FORCENUM ZP_ATTR"></span>
        <span class="ZP_HIGHLIGHT ZP_ATTR"></span>
        <span class="ZP_POSTID ZP_ATTR">27</span>
		<span class="ZOTPRESS_PLUGIN_URL ZP_ATTR"><a href="/wp-content/plugins/zotpress/">/wp-content/plugins/zotpress/</a></span>

		<div class="zp-List loading">
			<div class="zp-SEO-Content">
				<span class="ZP_JSON ZP_ATTR">%7B%22status%22%3A%22success%22%2C%22updateneeded%22%3Afalse%2C%22instance%22%3Afalse%2C%22meta%22%3A%7B%22request_last%22%3A0%2C%22request_next%22%3A0%2C%22used_cache%22%3Atrue%7D%2C%22data%22%3A%5B%7B%22key%22%3A%22AS5J555G%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Alvelid%20et%20al.%22%2C%22parsedDate%22%3A%222025-09-01%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BAlvelid%2C%20J.%2C%20Koerfer%2C%20A.%20and%20Eggeling%2C%20C.%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.biorxiv.org%5C%2Fcontent%5C%2F10.1101%5C%2F2025.08.27.672674v1%26%23039%3B%26gt%3BEvent-triggered%20MINFLUX%20microscopy%3A%20smart%20microscopy%20to%20catch%20and%20follow%20rare%20events%26lt%3B%5C%2Fa%26gt%3B.%26quot%3B%20bioRxiv.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1101%5C%2F2025.08.27.672674.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22preprint%22%2C%22title%22%3A%22Event-triggered%20MINFLUX%20microscopy%3A%20smart%20microscopy%20to%20catch%20and%20follow%20rare%20events%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jonatan%22%2C%22lastName%22%3A%22Alvelid%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Agnes%22%2C%22lastName%22%3A%22Koerfer%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Christian%22%2C%22lastName%22%3A%22Eggeling%22%7D%5D%2C%22abstractNote%22%3A%22MINFLUX%20microscopy%20is%20a%20powerful%20microscopy%20method%20allowing%20for%20the%20characterization%20of%20molecular%20organization%20and%20dynamics%20with%20single%20nanometer%20spatial%20resolution%20and%20sub-hundred%20microseconds%20temporal%20resolution.%20However%2C%20acquisition%20times%20often%20span%20minutes%20to%20hours%20as%20a%20single%20fluorophore%20is%20measured%20at%20a%20time.%20Applying%20it%20to%20study%20cellular%20processes%20in%20living%20cells%20therefore%20requires%20careful%20consideration%20of%20where%20and%20when%20to%20apply%20MINFLUX%20data%20acquisition%2C%20a%20consideration%20where%20manual%20control%20limits%20its%20potential%20applications.%20Here%2C%20to%20overcome%20the%20limitations%20of%20acquisition%20speed%2C%20acquisition%20initiation%2C%20and%20data%20throughput%2C%20we%20present%20a%20smart%20microscopy%20method%20that%20uses%20confocal%20imaging%20as%20a%20monitoring%20method%2C%20runs%20real-time%20image%20analysis%2C%20and%20only%20applies%20MINFLUX%20data%20acquisition%20exactly%20where%20and%20when%20deemed%20necessary%20based%20on%20the%20analysis%20outcome.%20The%20method%2C%20event-triggered%20MINFLUX%2C%20is%20controlled%20through%20a%20custom-written%20and%20open%20source%20Python%20widget%20that%20automatically%20controls%20a%20commercial%20MINFLUX%20microscope.%20We%20apply%20this%20method%20to%20investigate%20molecular%20membrane%20dynamics%20and%20organization%20during%20three%20different%20cellular%20events%3A%20two-dimensional%20lipid%20dynamics%20at%20caveolae%3B%20three-dimensional%20membrane%20topography%20during%20dynamin-mediated%20endocytosis%3B%20and%20three-dimensional%20membrane%20fluidity%20and%20topography%20during%20budding%20site%20formation%20of%20HIV-1%20proteins.%20Thanks%20to%20rapid%20event%20detection%20and%20minimal%20regions%20of%20interest%20the%20method%20provides%20data%20that%20would%20be%20unfeasible%20or%20impossible%20to%20acquire%20through%20manual%20control%20of%20the%20microscope.%22%2C%22genre%22%3A%22%22%2C%22repository%22%3A%22bioRxiv%22%2C%22archiveID%22%3A%22%22%2C%22date%22%3A%222025-09-01%22%2C%22DOI%22%3A%2210.1101%5C%2F2025.08.27.672674%22%2C%22citationKey%22%3A%22%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.biorxiv.org%5C%2Fcontent%5C%2F10.1101%5C%2F2025.08.27.672674v1%22%2C%22language%22%3A%22en%22%2C%22collections%22%3A%5B%22UQ77P7I5%22%5D%2C%22dateModified%22%3A%222025-10-02T12%3A19%3A18Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Application%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22HV29U83C%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Meirovitch%20et%20al.%22%2C%22parsedDate%22%3A%222025-05-15%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BMeirovitch%2C%20Y.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.biorxiv.org%5C%2Fcontent%5C%2F10.1101%5C%2F2023.10.05.561103v4%26%23039%3B%26gt%3BSmartEM%3A%20machine-learning%20guided%20electron%20microscopy%26lt%3B%5C%2Fa%26gt%3B.%26quot%3B%20bioRxiv.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1101%5C%2F2023.10.05.561103.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22preprint%22%2C%22title%22%3A%22SmartEM%3A%20machine-learning%20guided%20electron%20microscopy%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Yaron%22%2C%22lastName%22%3A%22Meirovitch%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ishaan%20Singh%22%2C%22lastName%22%3A%22Chandok%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Core%20Francisco%22%2C%22lastName%22%3A%22Park%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Pavel%22%2C%22lastName%22%3A%22Potocek%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Lu%22%2C%22lastName%22%3A%22Mi%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Shashata%22%2C%22lastName%22%3A%22Sawmya%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Yicong%22%2C%22lastName%22%3A%22Li%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Thomas%20L.%22%2C%22lastName%22%3A%22Athey%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Vladislav%22%2C%22lastName%22%3A%22Susoy%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Neha%22%2C%22lastName%22%3A%22Karlupia%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Yuelong%22%2C%22lastName%22%3A%22Wu%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Daniel%20R.%22%2C%22lastName%22%3A%22Berger%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Richard%22%2C%22lastName%22%3A%22Schalek%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Caitlyn%20A.%22%2C%22lastName%22%3A%22Bishop%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Daniel%22%2C%22lastName%22%3A%22Xenes%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Hannah%22%2C%22lastName%22%3A%22Martinez%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jordan%22%2C%22lastName%22%3A%22Matelsky%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Brock%20A.%22%2C%22lastName%22%3A%22Wester%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Hanspeter%22%2C%22lastName%22%3A%22Pfister%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Remco%22%2C%22lastName%22%3A%22Schoenmakers%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Maurice%22%2C%22lastName%22%3A%22Peemen%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jeff%20W.%22%2C%22lastName%22%3A%22Lichtman%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Aravinthan%20D.%20T.%22%2C%22lastName%22%3A%22Samuel%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Nir%22%2C%22lastName%22%3A%22Shavit%22%7D%5D%2C%22abstractNote%22%3A%22Connectomics%20provides%20nanometer-resolution%2C%20synapse-level%20maps%20of%20neural%20circuits%20to%20understand%20brain%20activity%20and%20behavior.%20However%2C%20few%20researchers%20have%20access%20to%20the%20high-throughput%20electron%20microscopes%20necessary%20to%20generate%20enough%20data%20for%20whole%20brain%20or%20even%20whole%20circuit%20reconstruction.%20To%20date%2C%20machine-learning%20methods%20have%20been%20used%20after%20the%20collection%20of%20images%20by%20electron%20microscopy%20%28EM%29%20to%20accelerate%20and%20improve%20neuronal%20segmentation%2C%20synapse%20reconstruction%20and%20other%20data%20analysis.%20With%20the%20computational%20improvements%20in%20processing%20EM%20images%2C%20acquiring%20EM%20images%20has%20now%20become%20the%20rate-limiting%20step%20in%20automated%20connectomics.%20Here%2C%20in%20order%20to%20speed%20up%20EM%20imaging%2C%20we%20integrate%20machine-learning%20into%20real-time%20image%20acquisition%20in%20a%20single-beam%20scanning%20electron%20microscope.%20This%20SmartEM%20approach%20allows%20an%20electron%20microscope%20to%20perform%20intelligent%2C%20data-aware%20imaging%20of%20specimens.%20SmartEM%20saves%20time%20by%20allocating%20the%20proper%20imaging%20time%20for%20each%20region%20of%20interest%20%5Cu2013%20first%20scanning%20all%20pixels%20rapidly%2C%20then%20rescanning%20more%20slowly%20only%20the%20small%20subareas%20where%20a%20higher%20quality%20signal%20is%20required.%20We%20demonstrate%20that%20SmartEM%20achieves%20up%20to%20a%20~7-fold%20acceleration%20of%20image%20acquisition%20time%20for%20connectomic%20samples%20using%20a%20commercial%20single-beam%20SEM%20in%20samples%20from%20nematodes%2C%20mice%20and%20human%20brain.%20We%20apply%20this%20fast%20imaging%20method%20to%20reconstruct%20a%20portion%20of%20mouse%20cerebral%20cortex%20with%20an%20accuracy%20comparable%20to%20traditional%20electron%20microscopy.%22%2C%22genre%22%3A%22%22%2C%22repository%22%3A%22bioRxiv%22%2C%22archiveID%22%3A%22%22%2C%22date%22%3A%222025-05-15%22%2C%22DOI%22%3A%2210.1101%5C%2F2023.10.05.561103%22%2C%22citationKey%22%3A%22%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.biorxiv.org%5C%2Fcontent%5C%2F10.1101%5C%2F2023.10.05.561103v4%22%2C%22language%22%3A%22en%22%2C%22collections%22%3A%5B%22UQ77P7I5%22%5D%2C%22dateModified%22%3A%222025-10-06T10%3A44%3A14Z%22%2C%22tags%22%3A%5B%5D%7D%7D%2C%7B%22key%22%3A%22QELVL8TC%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Yang%20et%20al.%22%2C%22parsedDate%22%3A%222025-04-14%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BYang%2C%20J.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttp%3A%5C%2F%5C%2Farxiv.org%5C%2Fabs%5C%2F2504.10281%26%23039%3B%26gt%3BZero-shot%20Autonomous%20Microscopy%20for%20Scalable%20and%20Intelligent%20Characterization%20of%202D%20Materials%26lt%3B%5C%2Fa%26gt%3B.%26quot%3B%20arXiv.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.48550%5C%2FarXiv.2504.10281.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22preprint%22%2C%22title%22%3A%22Zero-shot%20Autonomous%20Microscopy%20for%20Scalable%20and%20Intelligent%20Characterization%20of%202D%20Materials%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jingyun%22%2C%22lastName%22%3A%22Yang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ruoyan%20Avery%22%2C%22lastName%22%3A%22Yin%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Chi%22%2C%22lastName%22%3A%22Jiang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Yuepeng%22%2C%22lastName%22%3A%22Hu%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Xiaokai%22%2C%22lastName%22%3A%22Zhu%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Xingjian%22%2C%22lastName%22%3A%22Hu%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Sutharsika%22%2C%22lastName%22%3A%22Kumar%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Xiao%22%2C%22lastName%22%3A%22Wang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Xiaohua%22%2C%22lastName%22%3A%22Zhai%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Keran%22%2C%22lastName%22%3A%22Rong%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Yunyue%22%2C%22lastName%22%3A%22Zhu%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Tianyi%22%2C%22lastName%22%3A%22Zhang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Zongyou%22%2C%22lastName%22%3A%22Yin%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jing%22%2C%22lastName%22%3A%22Kong%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Neil%20Zhenqiang%22%2C%22lastName%22%3A%22Gong%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Zhichu%22%2C%22lastName%22%3A%22Ren%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Haozhe%22%2C%22lastName%22%3A%22Wang%22%7D%5D%2C%22abstractNote%22%3A%22Characterization%20of%20atomic-scale%20materials%20traditionally%20requires%20human%20experts%20with%20months%20to%20years%20of%20specialized%20training.%20Even%20for%20trained%20human%20operators%2C%20accurate%20and%20reliable%20characterization%20remains%20challenging%20when%20examining%20newly%20discovered%20materials%20such%20as%20two-dimensional%20%282D%29%20structures.%20This%20bottleneck%20drives%20demand%20for%20fully%20autonomous%20experimentation%20systems%20capable%20of%20comprehending%20research%20objectives%20without%20requiring%20large%20training%20datasets.%20In%20this%20work%2C%20we%20present%20ATOMIC%20%28Autonomous%20Technology%20for%20Optical%20Microscopy%20%26amp%3B%20Intelligent%20Characterization%29%2C%20an%20end-to-end%20framework%20that%20integrates%20foundation%20models%20to%20enable%20fully%20autonomous%2C%20zero-shot%20characterization%20of%202D%20materials.%20Our%20system%20integrates%20the%20vision%20foundation%20model%20%28i.e.%2C%20Segment%20Anything%20Model%29%2C%20large%20language%20models%20%28i.e.%2C%20ChatGPT%29%2C%20unsupervised%20clustering%2C%20and%20topological%20analysis%20to%20automate%20microscope%20control%2C%20sample%20scanning%2C%20image%20segmentation%2C%20and%20intelligent%20analysis%20through%20prompt%20engineering%2C%20eliminating%20the%20need%20for%20additional%20training.%20When%20analyzing%20typical%20MoS2%20samples%2C%20our%20approach%20achieves%2099.7%25%20segmentation%20accuracy%20for%20single%20layer%20identification%2C%20which%20is%20equivalent%20to%20that%20of%20human%20experts.%20In%20addition%2C%20the%20integrated%20model%20is%20able%20to%20detect%20grain%20boundary%20slits%20that%20are%20challenging%20to%20identify%20with%20human%20eyes.%20Furthermore%2C%20the%20system%20retains%20robust%20accuracy%20despite%20variable%20conditions%20including%20defocus%2C%20color%20temperature%20fluctuations%2C%20and%20exposure%20variations.%20It%20is%20applicable%20to%20a%20broad%20spectrum%20of%20common%202D%20materials-including%20graphene%2C%20MoS2%2C%20WSe2%2C%20SnSe-regardless%20of%20whether%20they%20were%20fabricated%20via%20chemical%20vapor%20deposition%20or%20mechanical%20exfoliation.%20This%20work%20represents%20the%20implementation%20of%20foundation%20models%20to%20achieve%20autonomous%20analysis%2C%20establishing%20a%20scalable%20and%20data-efficient%20characterization%20paradigm%20that%20fundamentally%20transforms%20the%20approach%20to%20nanoscale%20materials%20research.%22%2C%22genre%22%3A%22%22%2C%22repository%22%3A%22arXiv%22%2C%22archiveID%22%3A%22arXiv%3A2504.10281%22%2C%22date%22%3A%222025-04-14%22%2C%22DOI%22%3A%2210.48550%5C%2FarXiv.2504.10281%22%2C%22citationKey%22%3A%22%22%2C%22url%22%3A%22http%3A%5C%2F%5C%2Farxiv.org%5C%2Fabs%5C%2F2504.10281%22%2C%22language%22%3A%22%22%2C%22collections%22%3A%5B%22UQ77P7I5%22%5D%2C%22dateModified%22%3A%222025-04-29T14%3A01%3A27Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22CAZ4PPYG%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Friederich%20et%20al.%22%2C%22parsedDate%22%3A%222025-03-30%22%2C%22numChildren%22%3A1%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BFriederich%2C%20N.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttp%3A%5C%2F%5C%2Farxiv.org%5C%2Fabs%5C%2F2504.00047%26%23039%3B%26gt%3BEAP4EMSIG%20--%20Enhancing%20Event-Driven%20Microscopy%20for%20Microfluidic%20Single-Cell%20Analysis%26lt%3B%5C%2Fa%26gt%3B.%26quot%3B%20arXiv.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.48550%5C%2FarXiv.2504.00047.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22preprint%22%2C%22title%22%3A%22EAP4EMSIG%20--%20Enhancing%20Event-Driven%20Microscopy%20for%20Microfluidic%20Single-Cell%20Analysis%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Nils%22%2C%22lastName%22%3A%22Friederich%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Angelo%20Jovin%20Yamachui%22%2C%22lastName%22%3A%22Sitcheu%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Annika%22%2C%22lastName%22%3A%22Nassal%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Erenus%22%2C%22lastName%22%3A%22Yildiz%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Matthias%22%2C%22lastName%22%3A%22Pesch%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Maximilian%22%2C%22lastName%22%3A%22Beichter%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Lukas%22%2C%22lastName%22%3A%22Scholtes%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Bahar%22%2C%22lastName%22%3A%22Akbaba%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Thomas%22%2C%22lastName%22%3A%22Lautenschlager%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Oliver%22%2C%22lastName%22%3A%22Neumann%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Dietrich%22%2C%22lastName%22%3A%22Kohlheyer%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Hanno%22%2C%22lastName%22%3A%22Scharr%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Johannes%22%2C%22lastName%22%3A%22Seiffarth%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Katharina%22%2C%22lastName%22%3A%22N%5Cu00f6h%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ralf%22%2C%22lastName%22%3A%22Mikut%22%7D%5D%2C%22abstractNote%22%3A%22Microfluidic%20Live-Cell%20Imaging%20yields%20data%20on%20microbial%20cell%20factories.%20However%2C%20continuous%20acquisition%20is%20challenging%20as%20high-throughput%20experiments%20often%20lack%20realtime%20insights%2C%20delaying%20responses%20to%20stochastic%20events.%20We%20introduce%20three%20components%20in%20the%20Experiment%20Automation%20Pipeline%20for%20Event-Driven%20Microscopy%20to%20Smart%20Microfluidic%20Single-Cell%20Analysis%3A%20a%20fast%2C%20accurate%20Deep%20Learning%20autofocusing%20method%20predicting%20the%20focus%20offset%2C%20an%20evaluation%20of%20real-time%20segmentation%20methods%20and%20a%20realtime%20data%20analysis%20dashboard.%20Our%20autofocusing%20achieves%20a%20Mean%20Absolute%20Error%20of%200.0226%5C%5Ctextmu%20m%20with%20inference%20times%20below%2050~ms.%20Among%20eleven%20Deep%20Learning%20segmentation%20methods%2C%20Cellpose~3%20reached%20a%20Panoptic%20Quality%20of%2093.58%5C%5C%25%2C%20while%20a%20distance-based%20method%20is%20fastest%20%28121~ms%2C%20Panoptic%20Quality%2093.02%5C%5C%25%29.%20All%20six%20Deep%20Learning%20Foundation%20Models%20were%20unsuitable%20for%20real-time%20segmentation.%22%2C%22genre%22%3A%22%22%2C%22repository%22%3A%22arXiv%22%2C%22archiveID%22%3A%22arXiv%3A2504.00047%22%2C%22date%22%3A%222025-03-30%22%2C%22DOI%22%3A%2210.48550%5C%2FarXiv.2504.00047%22%2C%22citationKey%22%3A%22%22%2C%22url%22%3A%22http%3A%5C%2F%5C%2Farxiv.org%5C%2Fabs%5C%2F2504.00047%22%2C%22language%22%3A%22%22%2C%22collections%22%3A%5B%22UQ77P7I5%22%5D%2C%22dateModified%22%3A%222025-04-29T14%3A04%3A42Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Application%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22TCPZKI5S%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Kesavan%20and%20Bohra%22%2C%22parsedDate%22%3A%222025-03-04%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BKesavan%2C%20P.S.%20and%20Bohra%2C%20D.%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.biorxiv.org%5C%2Fcontent%5C%2F10.1101%5C%2F2025.02.25.639997v2%26%23039%3B%26gt%3Bdeepthought%3A%20domain%20driven%20design%20for%20microscopy%20with%20applications%20in%20DNA%20damage%20responses%26lt%3B%5C%2Fa%26gt%3B.%26quot%3B%20bioRxiv.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1101%5C%2F2025.02.25.639997.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22preprint%22%2C%22title%22%3A%22deepthought%3A%20domain%20driven%20design%20for%20microscopy%20with%20applications%20in%20DNA%20damage%20responses%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22P.%20S.%22%2C%22lastName%22%3A%22Kesavan%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Darshika%22%2C%22lastName%22%3A%22Bohra%22%7D%5D%2C%22abstractNote%22%3A%22Microscope%20systems%20have%20traditionally%20separated%20data%20acquisition%20and%20analysis%20into%20distinct%20phases%2C%20limiting%20real-time%20insights%20and%20experimental%20adaptability.%20While%20open-source%20tools%20exist%20for%20device%20control%2C%20there%20remains%20a%20need%20for%20integrated%20software%20frameworks%20that%20combine%20acquisition%20and%20analysis.%20Here%20we%20present%20deepthought%2C%20a%20framework%20built%20on%20domain-driven%20design%20principles%20that%20unifies%20microscope%20control%20and%20data%20analysis%20through%20a%20common%20domain%20language.%20This%20framework%20enables%20automated%20high-throughput%20imaging%20with%20real-time%20analysis%20capabilities%2C%20demonstrated%20through%20studies%20of%20DNA%20damage%20responses%20in%20live%20cells.%20By%20implementing%20deepthought%20on%20a%20standard%20wide-field%20microscope%2C%20we%20achieved%20sample%20sizes%20exceeding%2010%2C000%20cells%20per%20condition%20and%20identified%20rare%20cellular%20subpopulations%2C%20capabilities%20typically%20limited%20to%20specialized%20high-content%20systems.%20Our%20approach%20provides%20a%20foundation%20for%20developing%20context-aware%20microscopy%20software%20that%20can%20dynamically%20adjust%20experimental%20parameters%20based%20on%20real-time%20analysis%20of%20biological%20phenomena.%22%2C%22genre%22%3A%22%22%2C%22repository%22%3A%22bioRxiv%22%2C%22archiveID%22%3A%22%22%2C%22date%22%3A%222025-03-04%22%2C%22DOI%22%3A%2210.1101%5C%2F2025.02.25.639997%22%2C%22citationKey%22%3A%22%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.biorxiv.org%5C%2Fcontent%5C%2F10.1101%5C%2F2025.02.25.639997v2%22%2C%22language%22%3A%22en%22%2C%22collections%22%3A%5B%22UQ77P7I5%22%5D%2C%22dateModified%22%3A%222025-04-29T14%3A04%3A20Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Application%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22Y6KMZ8RU%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Berrevoets%20et%20al.%22%2C%22parsedDate%22%3A%222024-11-29%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BBerrevoets%2C%20E.S.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282024%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.biorxiv.org%5C%2Fcontent%5C%2F10.1101%5C%2F2024.11.28.625872v1%26%23039%3B%26gt%3BSmart%203D%20super-resolution%20microscopy%20reveals%20the%20architecture%20of%20the%20RNA%20scaffold%20in%20a%20nuclear%20body%26lt%3B%5C%2Fa%26gt%3B.%26quot%3B%20bioRxiv.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1101%5C%2F2024.11.28.625872.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22preprint%22%2C%22title%22%3A%22Smart%203D%20super-resolution%20microscopy%20reveals%20the%20architecture%20of%20the%20RNA%20scaffold%20in%20a%20nuclear%20body%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Enya%20S.%22%2C%22lastName%22%3A%22Berrevoets%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Laurell%20F.%22%2C%22lastName%22%3A%22Kessler%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ashwin%22%2C%22lastName%22%3A%22Balakrishnan%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Michaela%22%2C%22lastName%22%3A%22M%5Cu00fcller-McNicoll%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Bernd%22%2C%22lastName%22%3A%22Rieger%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Sjoerd%22%2C%22lastName%22%3A%22Stallinga%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Mike%22%2C%22lastName%22%3A%22Heilemann%22%7D%5D%2C%22abstractNote%22%3A%22Small%20subcellular%20organelles%20orchestrate%20key%20cellular%20functions.%20How%20biomolecules%20are%20spatially%20organized%20within%20these%20assemblies%20is%20poorly%20understood.%20Here%2C%20we%20report%20an%20automated%20super-%20resolution%20imaging%20and%20analysis%20workflow%20that%20integrates%20confocal%20microscopy%2C%20morphological%20object%20screening%2C%20targeted%203D%20super-resolution%20STED%20microscopy%20and%20quantitative%20image%20analysis.%20Using%20this%20smart%20microscopy%20workflow%2C%20we%20targeted%20the%203D%20organization%20of%20an%20architectural%20RNA%20that%20constitutes%20the%20structural%20backbone%20of%20paraspeckles%2C%20a%20membraneless%20nuclear%20organelle.%20Using%20site-specific%20labeling%2C%20morphological%20sorting%20and%20particle%20averaging%2C%20we%20reconstructed%20the%20morphological%20space%20of%20paraspeckles%20along%20their%20development%20cycle%20from%20over%2010%2C000%20individual%20particles.%20Applying%20spherical%20harmonics%20analysis%2C%20we%20report%20so-far%20unknown%20heterotypes%20of%20RNA%20organization.%20By%20integrating%20multi-%20positional%20labeling%2C%20we%20determined%20the%20coarse%20conformation%20of%20the%20RNA%20within%20the%20organelle%20and%20found%20the%203%5Cu2019%20end%20forming%20a%20loop-like%20structure%20at%20the%20surface%20of%20the%20paraspeckle.%20Our%20study%20reveals%20key%20structural%20features%20of%20nuclear%20paraspeckle%20structure%20and%20growth%2C%20as%20well%20as%20on%20the%20molecular%20organization%20of%20the%20scaffold%20RNA.%22%2C%22genre%22%3A%22%22%2C%22repository%22%3A%22bioRxiv%22%2C%22archiveID%22%3A%22%22%2C%22date%22%3A%222024-11-29%22%2C%22DOI%22%3A%2210.1101%5C%2F2024.11.28.625872%22%2C%22citationKey%22%3A%22%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.biorxiv.org%5C%2Fcontent%5C%2F10.1101%5C%2F2024.11.28.625872v1%22%2C%22language%22%3A%22en%22%2C%22collections%22%3A%5B%22UQ77P7I5%22%5D%2C%22dateModified%22%3A%222024-12-03T09%3A48%3A43Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Application%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22DFIINUT8%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Brooks%20et%20al.%22%2C%22parsedDate%22%3A%222024-10-19%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BBrooks%2C%20S.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282024%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.biorxiv.org%5C%2Fcontent%5C%2F10.1101%5C%2F2024.10.17.618897v1%26%23039%3B%26gt%3BCelFDrive%3A%20Artificial%20Intelligence%20assisted%20microscopy%20for%20automated%20detection%20of%20rare%20events%26lt%3B%5C%2Fa%26gt%3B.%26quot%3B%20bioRxiv.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1101%5C%2F2024.10.17.618897.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22preprint%22%2C%22title%22%3A%22CelFDrive%3A%20Artificial%20Intelligence%20assisted%20microscopy%20for%20automated%20detection%20of%20rare%20events%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Scott%22%2C%22lastName%22%3A%22Brooks%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Sara%22%2C%22lastName%22%3A%22Toral-P%5Cu00e9rez%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22David%20S.%22%2C%22lastName%22%3A%22Corcoran%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Karl%22%2C%22lastName%22%3A%22Kilborn%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Brian%22%2C%22lastName%22%3A%22Bodensteiner%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Hella%22%2C%22lastName%22%3A%22Baumann%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Nigel%20J.%22%2C%22lastName%22%3A%22Burroughs%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Andrew%20D.%22%2C%22lastName%22%3A%22McAinsh%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Till%22%2C%22lastName%22%3A%22Bretschneider%22%7D%5D%2C%22abstractNote%22%3A%221%20Abstract%5CnSummary%20CelFDrive%20automates%20high-resolution%203D%20imaging%20cells%20of%20interest%20across%20a%20variety%20of%20fluorescence%20microscopes%2C%20integrating%20deep%20learning%20cell%20classification%20from%20auxiliary%20low%20resolution%20widefield%20images.%20CelFDrive%20enables%20efficient%20detection%20of%20rare%20events%20in%20large%20cell%20populations%2C%20such%20as%20the%20onset%20of%20cell%20division%2C%20and%20subsequent%20rapid%20switching%20to%203D%20imaging%20modes%2C%20increasing%20the%20speed%20for%20finding%20cells%20of%20interest%20by%20an%20order%20of%20magnitude.%5CnAvailability%20and%20Implementation%20CelFDrive%20is%20available%20freely%20for%20academic%20purposes%20at%20the%20CelFDrive%20GitHub%20repository.%20and%20can%20be%20installed%20on%20Windows%2C%20macOS%20or%20Linux-based%20machines%20with%20relevant%20conda%20environments%20%5B1%5D.%20To%20interact%20with%20microscopy%20hardware%20requires%20additional%20software%3B%20we%20use%20SlideBook%20software%20from%20Intelligent%20Imaging%20Innovations%20%283i%29%2C%20but%20CelFDrive%20can%20be%20deployed%20with%20any%20microscope%20control%20software%20that%20can%20interact%20with%20a%20Python%20environment.%20Graphical%20Processing%20Units%20%28GPUs%29%20are%20recommended%20to%20increase%20the%20speed%20of%20application%20but%20are%20not%20required.%20On%203i%20systems%20the%20software%20can%20be%20deployed%20with%20a%20range%20of%20microscopes%20including%20their%20Lattice%20LightSheet%20microscope%20%28LLSM%29%20and%20spinning%20disk%20confocal%20%28SDC%29.%5CnContact%20s.brooks.2%7Bat%7Dwarwick.ac.uk%22%2C%22genre%22%3A%22%22%2C%22repository%22%3A%22bioRxiv%22%2C%22archiveID%22%3A%22%22%2C%22date%22%3A%222024-10-19%22%2C%22DOI%22%3A%2210.1101%5C%2F2024.10.17.618897%22%2C%22citationKey%22%3A%22%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.biorxiv.org%5C%2Fcontent%5C%2F10.1101%5C%2F2024.10.17.618897v1%22%2C%22language%22%3A%22en%22%2C%22collections%22%3A%5B%22UQ77P7I5%22%5D%2C%22dateModified%22%3A%222025-04-29T14%3A04%3A51Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22FQ7UJRZE%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Lei%20et%20al.%22%2C%22parsedDate%22%3A%222023-12-05%22%2C%22numChildren%22%3A1%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BLei%2C%20W.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282023%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttp%3A%5C%2F%5C%2Farxiv.org%5C%2Fabs%5C%2F2310.18351%26%23039%3B%26gt%3BBioImage.IO%20Chatbot%3A%20A%20Personalized%20Assistant%20for%20BioImage%20Analysis%20Augmented%20by%20Community%20Knowledge%20Base%26lt%3B%5C%2Fa%26gt%3B.%26quot%3B%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.5281%5C%2Fzenodo.10032227.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22BioImage.IO%20Chatbot%3A%20A%20Personalized%20Assistant%20for%20BioImage%20Analysis%20Augmented%20by%20Community%20Knowledge%20Base%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Wanlu%22%2C%22lastName%22%3A%22Lei%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Caterina%22%2C%22lastName%22%3A%22Fuster-Barcel%5Cu00f3%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Arrate%22%2C%22lastName%22%3A%22Mu%5Cu00f1oz-Barrutia%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Wei%22%2C%22lastName%22%3A%22Ouyang%22%7D%5D%2C%22abstractNote%22%3A%22The%20rapidly%20expanding%20landscape%20of%20bioimage%20analysis%20tools%20presents%20a%20navigational%20challenge%20for%20both%20experts%20and%20newcomers.%20Traditional%20search%20methods%20often%20fall%20short%20in%20assisting%20users%20in%20this%20complex%20environment.%20To%20address%20this%2C%20we%20introduce%20the%20BioImage%24.%24IO%20Chatbot%2C%20an%20AI-driven%20conversational%20assistant%20tailored%20for%20the%20bioimage%20community.%20Built%20upon%20large%20language%20models%2C%20this%20chatbot%20provides%20personalized%2C%20context-aware%20answers%20by%20aggregating%20and%20interpreting%20information%20from%20diverse%20databases%2C%20tool-specific%20documentation%2C%20and%20structured%20data%20sources.%20Enhanced%20by%20a%20community-contributed%20knowledge%20base%20and%20fine-tuned%20retrieval%20methods%2C%20the%20BioImage%24.%24IO%20Chatbot%20offers%20not%20just%20a%20personalized%20interaction%20but%20also%20a%20knowledge-enriched%2C%20context-aware%20experience.%20It%20fundamentally%20transforms%20the%20way%20biologists%2C%20bioimage%20analysts%2C%20and%20developers%20navigate%20and%20utilize%20advanced%20bioimage%20analysis%20tools%2C%20setting%20a%20new%20standard%20for%20community-driven%2C%20accessible%20scientific%20research.%22%2C%22date%22%3A%222023-12-05%22%2C%22language%22%3A%22%22%2C%22DOI%22%3A%2210.5281%5C%2Fzenodo.10032227%22%2C%22ISSN%22%3A%22%22%2C%22url%22%3A%22http%3A%5C%2F%5C%2Farxiv.org%5C%2Fabs%5C%2F2310.18351%22%2C%22collections%22%3A%5B%22UQ77P7I5%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A01%3A15Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Data%20Analysis%22%7D%2C%7B%22tag%22%3A%22Data%20handling%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22SD9WC6DX%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22G%5Cu00f3mez-de-Mariscal%20et%20al.%22%2C%22parsedDate%22%3A%222023-08-08%22%2C%22numChildren%22%3A1%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BG%5Cu00f3mez-de-Mariscal%2C%20E.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282023%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttp%3A%5C%2F%5C%2Farxiv.org%5C%2Fabs%5C%2F2308.04387%26%23039%3B%26gt%3BHarnessing%20Artificial%20Intelligence%20To%20Reduce%20Phototoxicity%20in%20Live%20Imaging%26lt%3B%5C%2Fa%26gt%3B.%26quot%3B%20arXiv.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.48550%5C%2FarXiv.2308.04387.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22preprint%22%2C%22title%22%3A%22Harnessing%20Artificial%20Intelligence%20To%20Reduce%20Phototoxicity%20in%20Live%20Imaging%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Estibaliz%22%2C%22lastName%22%3A%22G%5Cu00f3mez-de-Mariscal%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Mario%22%2C%22lastName%22%3A%22Del%20Rosario%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Joanna%20W.%22%2C%22lastName%22%3A%22Pylv%5Cu00e4n%5Cu00e4inen%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Guillaume%22%2C%22lastName%22%3A%22Jacquemet%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ricardo%22%2C%22lastName%22%3A%22Henriques%22%7D%5D%2C%22abstractNote%22%3A%22Fluorescence%20microscopy%2C%20widely%20used%20in%20the%20study%20of%20living%20cells%2C%20tissues%2C%20and%20organisms%2C%20often%20faces%20the%20challenge%20of%20photodamage.%20This%20is%20primarily%20caused%20by%20the%20interaction%20between%20light%20and%20biochemical%20components%20during%20the%20imaging%20process%2C%20leading%20to%20compromised%20accuracy%20and%20reliability%20of%20biological%20results.%20Methods%20necessitating%20extended%20high-intensity%20illumination%2C%20such%20as%20super-resolution%20microscopy%20or%20thick%20sample%20imaging%2C%20are%20particularly%20susceptible%20to%20this%20issue.%20As%20part%20of%20the%20solution%20to%20these%20problems%2C%20advanced%20imaging%20approaches%20involving%20artificial%20intelligence%20%28AI%29%20have%20been%20developed.%20Here%20we%20underscore%20the%20necessity%20of%20establishing%20constraints%20to%20maintain%20light-induced%20damage%20at%20levels%20that%20permit%20cells%20to%20sustain%20their%20live%20behaviour.%20From%20this%20perspective%2C%20data-driven%20live-cell%20imaging%20bears%20significant%20potential%20in%20aiding%20the%20development%20of%20AI-enhanced%20photodamage-aware%20microscopy.%20These%20technologies%20could%20streamline%20precise%20observations%20of%20natural%20biological%20dynamics%20while%20minimising%20phototoxicity%20risks.%22%2C%22genre%22%3A%22%22%2C%22repository%22%3A%22arXiv%22%2C%22archiveID%22%3A%22arXiv%3A2308.04387%22%2C%22date%22%3A%222023-08-08%22%2C%22DOI%22%3A%2210.48550%5C%2FarXiv.2308.04387%22%2C%22citationKey%22%3A%22%22%2C%22url%22%3A%22http%3A%5C%2F%5C%2Farxiv.org%5C%2Fabs%5C%2F2308.04387%22%2C%22language%22%3A%22%22%2C%22collections%22%3A%5B%22UQ77P7I5%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A02%3A29Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Data%20Analysis%22%7D%2C%7B%22tag%22%3A%22Data%20handling%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22Q2A9GDKT%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Chai%20et%20al.%22%2C%22parsedDate%22%3A%222023-07-19%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BChai%2C%20B.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282023%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttp%3A%5C%2F%5C%2Farxiv.org%5C%2Fabs%5C%2F2307.10500%26%23039%3B%26gt%3BOpportunities%20and%20challenges%20for%20deep%20learning%20in%20cell%20dynamics%20research%26lt%3B%5C%2Fa%26gt%3B.%26quot%3B%20arXiv.%20Available%20at%3A%20%28Accessed%3A%20February%2014%2C%202024%29.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22preprint%22%2C%22title%22%3A%22Opportunities%20and%20challenges%20for%20deep%20learning%20in%20cell%20dynamics%20research%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Binghao%22%2C%22lastName%22%3A%22Chai%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Christoforos%22%2C%22lastName%22%3A%22Efstathiou%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Haoran%22%2C%22lastName%22%3A%22Yue%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Viji%20M.%22%2C%22lastName%22%3A%22Draviam%22%7D%5D%2C%22abstractNote%22%3A%22With%20the%20growth%20of%20artificial%20intelligence%20%28AI%29%2C%20there%20has%20been%20an%20increase%20in%20the%20adoption%20of%20computer%20vision%20and%20deep%20learning%20%28DL%29%20techniques%20for%20the%20evaluation%20of%20microscopy%20images%20and%20movies.%20This%20adoption%20has%20not%20only%20addressed%20hurdles%20in%20quantitative%20analysis%20of%20dynamic%20cell%20biological%20processes%2C%20but%20it%20has%20also%20started%20supporting%20advances%20in%20drug%20development%2C%20precision%20medicine%20and%20genome-phenome%20mapping.%20Here%20we%20survey%20existing%20AI-based%20techniques%20and%20tools%2C%20and%20open-source%20datasets%2C%20with%20a%20specific%20focus%20on%20the%20computational%20tasks%20of%20segmentation%2C%20classification%2C%20and%20tracking%20of%20cellular%20and%20subcellular%20structures%20and%20dynamics.%20We%20summarise%20long-standing%20challenges%20in%20microscopy%20video%20analysis%20from%20the%20computational%20perspective%20and%20review%20emerging%20research%20frontiers%20and%20innovative%20applications%20for%20deep%20learning-guided%20automation%20for%20cell%20dynamics%20research.%22%2C%22genre%22%3A%22%22%2C%22repository%22%3A%22arXiv%22%2C%22archiveID%22%3A%22arXiv%3A2307.10500%22%2C%22date%22%3A%222023-07-19%22%2C%22DOI%22%3A%22%22%2C%22citationKey%22%3A%22%22%2C%22url%22%3A%22http%3A%5C%2F%5C%2Farxiv.org%5C%2Fabs%5C%2F2307.10500%22%2C%22language%22%3A%22%22%2C%22collections%22%3A%5B%22UQ77P7I5%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A02%3A23Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Data%20Analysis%22%7D%2C%7B%22tag%22%3A%22Data%20handling%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22KK2INVGX%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Mol%20and%20Vlijm%22%2C%22parsedDate%22%3A%222022-09-30%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BMol%2C%20F.N.%20and%20Vlijm%2C%20R.%20%282022%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.biorxiv.org%5C%2Fcontent%5C%2F10.1101%5C%2F2022.09.29.510126v1%26%23039%3B%26gt%3BAutomated%20STED%20nanoscopy%20for%20high-throughput%20imaging%20of%20cellular%20structures%26lt%3B%5C%2Fa%26gt%3B.%26quot%3B%20bioRxiv.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1101%5C%2F2022.09.29.510126.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22preprint%22%2C%22title%22%3A%22Automated%20STED%20nanoscopy%20for%20high-throughput%20imaging%20of%20cellular%20structures%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Frank%20N.%22%2C%22lastName%22%3A%22Mol%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Rifka%22%2C%22lastName%22%3A%22Vlijm%22%7D%5D%2C%22abstractNote%22%3A%22STimulated%20Emission%20Depletion%20%28STED%29%20nanoscopy%20uniquely%20combines%20a%20high%20spatial%20resolution%20%2820-50nm%20in%20cells%29%20with%20relatively%20fast%20imaging%20%28frame%20rate%20of%20%5Cu223c1-30Hz%29%2C%20straightforward%20sample%20preparation%20and%20direct%20image%20output%20%28no%20postprocessing%20required%29.%20Although%20these%20characteristics%20in%20principle%20make%20STED%20very%20suitable%20for%20high-throughput%20imaging%2C%20only%20few%20steps%20towards%20automation%20have%20been%20made.%20Here%2C%20we%20have%20developed%20fully%20automated%20STED%20imaging%2C%20eliminating%20all%20manual%20steps%20including%20the%20selection%20and%20characterisation%20of%20the%20relevant%20%28cellular%29%20regions%2C%20sample%20focusing%20and%20positioning%2C%20and%20microscope%20adjustments.%20This%20automatic%20STED%20image%20acquisition%20increases%20the%20data%20output%20by%20roughly%20two%20orders%20of%20magnitude%2C%20resulting%20in%20a%20more%20efficient%20use%20of%20the%20high-end%20microscope%2C%20and%20the%20ability%20to%20detect%20and%20characterise%20objects%20that%20are%20only%20present%20in%20a%20small%20subset%20of%20the%20sample.%22%2C%22genre%22%3A%22%22%2C%22repository%22%3A%22bioRxiv%22%2C%22archiveID%22%3A%22%22%2C%22date%22%3A%222022-09-30%22%2C%22DOI%22%3A%2210.1101%5C%2F2022.09.29.510126%22%2C%22citationKey%22%3A%22%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.biorxiv.org%5C%2Fcontent%5C%2F10.1101%5C%2F2022.09.29.510126v1%22%2C%22language%22%3A%22en%22%2C%22collections%22%3A%5B%22UQ77P7I5%22%5D%2C%22dateModified%22%3A%222024-11-15T14%3A17%3A37Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Application%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%5D%7D</span>

				<div id="zp-ID-27-5384098-AS5J555G" data-zp-author-date='Alvelid-et-al.-2025-09-01' data-zp-date-author='2025-09-01-Alvelid-et-al.' data-zp-date='2025-09-01' data-zp-year='2025' data-zp-itemtype='preprint' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Alvelid, J., Koerfer, A. and Eggeling, C. (2025) "<a class='zp-ItemURL' target='_blank' href='https://www.biorxiv.org/content/10.1101/2025.08.27.672674v1'>Event-triggered MINFLUX microscopy: smart microscopy to catch and follow rare events</a>." bioRxiv. Available at: <a href="https://doi.org/10.1101/2025.08.27.672674" rel="nofollow">https://doi.org/10.1101/2025.08.27.672674</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-HV29U83C" data-zp-author-date='Meirovitch-et-al.-2025-05-15' data-zp-date-author='2025-05-15-Meirovitch-et-al.' data-zp-date='2025-05-15' data-zp-year='2025' data-zp-itemtype='preprint' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Meirovitch, Y. <i>et al.</i> (2025) "<a class='zp-ItemURL' target='_blank' href='https://www.biorxiv.org/content/10.1101/2023.10.05.561103v4'>SmartEM: machine-learning guided electron microscopy</a>." bioRxiv. Available at: <a href="https://doi.org/10.1101/2023.10.05.561103" rel="nofollow">https://doi.org/10.1101/2023.10.05.561103</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-QELVL8TC" data-zp-author-date='Yang-et-al.-2025-04-14' data-zp-date-author='2025-04-14-Yang-et-al.' data-zp-date='2025-04-14' data-zp-year='2025' data-zp-itemtype='preprint' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Yang, J. <i>et al.</i> (2025) "<a class='zp-ItemURL' target='_blank' href='http://arxiv.org/abs/2504.10281'>Zero-shot Autonomous Microscopy for Scalable and Intelligent Characterization of 2D Materials</a>." arXiv. Available at: <a href="https://doi.org/10.48550/arXiv.2504.10281" rel="nofollow">https://doi.org/10.48550/arXiv.2504.10281</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-CAZ4PPYG" data-zp-author-date='Friederich-et-al.-2025-03-30' data-zp-date-author='2025-03-30-Friederich-et-al.' data-zp-date='2025-03-30' data-zp-year='2025' data-zp-itemtype='preprint' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Friederich, N. <i>et al.</i> (2025) "<a class='zp-ItemURL' target='_blank' href='http://arxiv.org/abs/2504.00047'>EAP4EMSIG -- Enhancing Event-Driven Microscopy for Microfluidic Single-Cell Analysis</a>." arXiv. Available at: <a href="https://doi.org/10.48550/arXiv.2504.00047" rel="nofollow">https://doi.org/10.48550/arXiv.2504.00047</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-TCPZKI5S" data-zp-author-date='Kesavan-and-Bohra-2025-03-04' data-zp-date-author='2025-03-04-Kesavan-and-Bohra' data-zp-date='2025-03-04' data-zp-year='2025' data-zp-itemtype='preprint' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Kesavan, P.S. and Bohra, D. (2025) "<a class='zp-ItemURL' target='_blank' href='https://www.biorxiv.org/content/10.1101/2025.02.25.639997v2'>deepthought: domain driven design for microscopy with applications in DNA damage responses</a>." bioRxiv. Available at: <a href="https://doi.org/10.1101/2025.02.25.639997" rel="nofollow">https://doi.org/10.1101/2025.02.25.639997</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-Y6KMZ8RU" data-zp-author-date='Berrevoets-et-al.-2024-11-29' data-zp-date-author='2024-11-29-Berrevoets-et-al.' data-zp-date='2024-11-29' data-zp-year='2024' data-zp-itemtype='preprint' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Berrevoets, E.S. <i>et al.</i> (2024) "<a class='zp-ItemURL' target='_blank' href='https://www.biorxiv.org/content/10.1101/2024.11.28.625872v1'>Smart 3D super-resolution microscopy reveals the architecture of the RNA scaffold in a nuclear body</a>." bioRxiv. Available at: <a href="https://doi.org/10.1101/2024.11.28.625872" rel="nofollow">https://doi.org/10.1101/2024.11.28.625872</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-DFIINUT8" data-zp-author-date='Brooks-et-al.-2024-10-19' data-zp-date-author='2024-10-19-Brooks-et-al.' data-zp-date='2024-10-19' data-zp-year='2024' data-zp-itemtype='preprint' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Brooks, S. <i>et al.</i> (2024) "<a class='zp-ItemURL' target='_blank' href='https://www.biorxiv.org/content/10.1101/2024.10.17.618897v1'>CelFDrive: Artificial Intelligence assisted microscopy for automated detection of rare events</a>." bioRxiv. Available at: <a href="https://doi.org/10.1101/2024.10.17.618897" rel="nofollow">https://doi.org/10.1101/2024.10.17.618897</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-FQ7UJRZE" data-zp-author-date='Lei-et-al.-2023-12-05' data-zp-date-author='2023-12-05-Lei-et-al.' data-zp-date='2023-12-05' data-zp-year='2023' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Lei, W. <i>et al.</i> (2023) "<a class='zp-ItemURL' target='_blank' href='http://arxiv.org/abs/2310.18351'>BioImage.IO Chatbot: A Personalized Assistant for BioImage Analysis Augmented by Community Knowledge Base</a>." Available at: <a href="https://doi.org/10.5281/zenodo.10032227" rel="nofollow">https://doi.org/10.5281/zenodo.10032227</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-SD9WC6DX" data-zp-author-date='Gómez-de-Mariscal-et-al.-2023-08-08' data-zp-date-author='2023-08-08-Gómez-de-Mariscal-et-al.' data-zp-date='2023-08-08' data-zp-year='2023' data-zp-itemtype='preprint' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Gómez-de-Mariscal, E. <i>et al.</i> (2023) "<a class='zp-ItemURL' target='_blank' href='http://arxiv.org/abs/2308.04387'>Harnessing Artificial Intelligence To Reduce Phototoxicity in Live Imaging</a>." arXiv. Available at: <a href="https://doi.org/10.48550/arXiv.2308.04387" rel="nofollow">https://doi.org/10.48550/arXiv.2308.04387</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-Q2A9GDKT" data-zp-author-date='Chai-et-al.-2023-07-19' data-zp-date-author='2023-07-19-Chai-et-al.' data-zp-date='2023-07-19' data-zp-year='2023' data-zp-itemtype='preprint' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Chai, B. <i>et al.</i> (2023) "<a class='zp-ItemURL' target='_blank' href='http://arxiv.org/abs/2307.10500'>Opportunities and challenges for deep learning in cell dynamics research</a>." arXiv. Available at: (Accessed: February 14, 2024).</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-KK2INVGX" data-zp-author-date='Mol-and-Vlijm-2022-09-30' data-zp-date-author='2022-09-30-Mol-and-Vlijm' data-zp-date='2022-09-30' data-zp-year='2022' data-zp-itemtype='preprint' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Mol, F.N. and Vlijm, R. (2022) "<a class='zp-ItemURL' target='_blank' href='https://www.biorxiv.org/content/10.1101/2022.09.29.510126v1'>Automated STED nanoscopy for high-throughput imaging of cellular structures</a>." bioRxiv. Available at: <a href="https://doi.org/10.1101/2022.09.29.510126" rel="nofollow">https://doi.org/10.1101/2022.09.29.510126</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->
			</div><!-- .zp-zp-SEO-Content -->
		</div><!-- .zp-List -->
	</div><!--.zp-Zotpress-->


    <div class="zotpress-table-container"></div>
  </div>
  <div class="zotpress-section" data-collection="4KIL6FA6">
    <div id="zotpress-d90354a61dc95d2c70e11ec45331d1f0" class="zp-Zotpress zp-Zotpress-Bib wp-block-group">

		<span class="ZP_API_USER_ID ZP_ATTR">5384098</span>
		<span class="ZP_ITEM_KEY ZP_ATTR"></span>
		<span class="ZP_COLLECTION_ID ZP_ATTR">4KIL6FA6</span>
		<span class="ZP_TAG_ID ZP_ATTR"></span>
		<span class="ZP_AUTHOR ZP_ATTR"></span>
		<span class="ZP_YEAR ZP_ATTR"></span>
        <span class="ZP_ITEMTYPE ZP_ATTR"></span>
		<span class="ZP_INCLUSIVE ZP_ATTR">1</span>
		<span class="ZP_STYLE ZP_ATTR">harvard1</span>
		<span class="ZP_LIMIT ZP_ATTR">50</span>
		<span class="ZP_SORTBY ZP_ATTR">date</span>
		<span class="ZP_ORDER ZP_ATTR">desc</span>
		<span class="ZP_TITLE ZP_ATTR"></span>
		<span class="ZP_SHOWIMAGE ZP_ATTR"></span>
		<span class="ZP_SHOWTAGS ZP_ATTR">1</span>
		<span class="ZP_DOWNLOADABLE ZP_ATTR">1</span>
		<span class="ZP_NOTES ZP_ATTR"></span>
		<span class="ZP_ABSTRACT ZP_ATTR"></span>
		<span class="ZP_CITEABLE ZP_ATTR"></span>
		<span class="ZP_TARGET ZP_ATTR">1</span>
		<span class="ZP_URLWRAP ZP_ATTR">title</span>
		<span class="ZP_FORCENUM ZP_ATTR"></span>
        <span class="ZP_HIGHLIGHT ZP_ATTR"></span>
        <span class="ZP_POSTID ZP_ATTR">27</span>
		<span class="ZOTPRESS_PLUGIN_URL ZP_ATTR"><a href="/wp-content/plugins/zotpress/">/wp-content/plugins/zotpress/</a></span>

		<div class="zp-List loading">
			<div class="zp-SEO-Content">
				<span class="ZP_JSON ZP_ATTR">%7B%22status%22%3A%22success%22%2C%22updateneeded%22%3Afalse%2C%22instance%22%3Afalse%2C%22meta%22%3A%7B%22request_last%22%3A0%2C%22request_next%22%3A0%2C%22used_cache%22%3Atrue%7D%2C%22data%22%3A%5B%7B%22key%22%3A%22QJYYQESZ%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Hsieh%20et%20al.%22%2C%22parsedDate%22%3A%222025-04-17%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BHsieh%2C%20H.-C.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282025%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-025-02647-w%26%23039%3B%26gt%3BImaging%203D%20cell%20cultures%20with%20optical%20microscopy%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BNature%20Methods%26lt%3B%5C%2Fi%26gt%3B%2C%20pp.%201%5Cu201324.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs41592-025-02647-w.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Imaging%203D%20cell%20cultures%20with%20optical%20microscopy%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Huai-Ching%22%2C%22lastName%22%3A%22Hsieh%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Qinghua%22%2C%22lastName%22%3A%22Han%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22David%22%2C%22lastName%22%3A%22Brenes%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Kevin%20W.%22%2C%22lastName%22%3A%22Bishop%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Rui%22%2C%22lastName%22%3A%22Wang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Yuli%22%2C%22lastName%22%3A%22Wang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Chetan%22%2C%22lastName%22%3A%22Poudel%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Adam%20K.%22%2C%22lastName%22%3A%22Glaser%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Benjamin%20S.%22%2C%22lastName%22%3A%22Freedman%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Joshua%20C.%22%2C%22lastName%22%3A%22Vaughan%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Nancy%20L.%22%2C%22lastName%22%3A%22Allbritton%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jonathan%20T.%20C.%22%2C%22lastName%22%3A%22Liu%22%7D%5D%2C%22abstractNote%22%3A%22Three-dimensional%20%283D%29%20cell%20cultures%20have%20gained%20popularity%20in%20recent%20years%20due%20to%20their%20ability%20to%20represent%20complex%20tissues%20or%20organs%20more%20faithfully%20than%20conventional%20two-dimensional%20%282D%29%20cell%20culture.%20This%20article%20reviews%20the%20application%20of%20both%202D%20and%203D%20microscopy%20approaches%20for%20monitoring%20and%20studying%203D%20cell%20cultures.%20We%20first%20summarize%20the%20most%20popular%20optical%20microscopy%20methods%20that%20have%20been%20used%20with%203D%20cell%20cultures.%20We%20then%20discuss%20the%20general%20advantages%20and%20disadvantages%20of%20various%20microscopy%20techniques%20for%20several%20broad%20categories%20of%20investigation%20involving%203D%20cell%20cultures.%20Finally%2C%20we%20provide%20perspectives%20on%20key%20areas%20of%20technical%20need%20in%20which%20there%20are%20clear%20opportunities%20for%20innovation.%20Our%20goal%20is%20to%20guide%20microscope%20engineers%20and%20biomedical%20end%20users%20toward%20optimal%20imaging%20methods%20for%20specific%20investigational%20scenarios%20and%20to%20identify%20use%20cases%20in%20which%20additional%20innovations%20in%20high-resolution%20imaging%20could%20be%20helpful.%22%2C%22date%22%3A%222025-04-17%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs41592-025-02647-w%22%2C%22ISSN%22%3A%221548-7105%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs41592-025-02647-w%22%2C%22collections%22%3A%5B%224KIL6FA6%22%5D%2C%22dateModified%22%3A%222025-04-29T14%3A05%3A14Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22QUM2DVR6%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Morgado%20et%20al.%22%2C%22parsedDate%22%3A%222024-03-06%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BMorgado%2C%20L.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282024%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fonlinelibrary.wiley.com%5C%2Fdoi%5C%2F10.1111%5C%2Fjmi.13282%26%23039%3B%26gt%3BThe%20rise%20of%20data%5Cu2010driven%20microscopy%20powered%20by%20machine%20learning%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BJournal%20of%20Microscopy%26lt%3B%5C%2Fi%26gt%3B%2C%20p.%20jmi.13282.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1111%5C%2Fjmi.13282.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22The%20rise%20of%20data%5Cu2010driven%20microscopy%20powered%20by%20machine%20learning%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Leonor%22%2C%22lastName%22%3A%22Morgado%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Estibaliz%22%2C%22lastName%22%3A%22G%5Cu00f3mez%5Cu2010de%5Cu2010Mariscal%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Hannah%20S.%22%2C%22lastName%22%3A%22Heil%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ricardo%22%2C%22lastName%22%3A%22Henriques%22%7D%5D%2C%22abstractNote%22%3A%22Abstract%5Cn%20%20%20%20%20%20%20%20%20%20%20%20Optical%20microscopy%20is%20an%20indispensable%20tool%20in%20life%20sciences%20research%2C%20but%20conventional%20techniques%20require%20compromises%20between%20imaging%20parameters%20like%20speed%2C%20resolution%2C%20field%20of%20view%20and%20phototoxicity.%20To%20overcome%20these%20limitations%2C%20data%5Cu2010driven%20microscopes%20incorporate%20feedback%20loops%20between%20data%20acquisition%20and%20analysis.%20This%20review%20overviews%20how%20machine%20learning%20enables%20automated%20image%20analysis%20to%20optimise%20microscopy%20in%20real%20time.%20We%20first%20introduce%20key%20data%5Cu2010driven%20microscopy%20concepts%20and%20machine%20learning%20methods%20relevant%20to%20microscopy%20image%20analysis.%20Subsequently%2C%20we%20highlight%20pioneering%20works%20and%20recent%20advances%20in%20integrating%20machine%20learning%20into%20microscopy%20acquisition%20workflows%2C%20including%20optimising%20illumination%2C%20switching%20modalities%20and%20acquisition%20rates%2C%20and%20triggering%20targeted%20experiments.%20We%20then%20discuss%20the%20remaining%20challenges%20and%20future%20outlook.%20Overall%2C%20intelligent%20microscopes%20that%20can%20sense%2C%20analyse%20and%20adapt%20promise%20to%20transform%20optical%20imaging%20by%20opening%20new%20experimental%20possibilities.%22%2C%22date%22%3A%222024-03-06%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1111%5C%2Fjmi.13282%22%2C%22ISSN%22%3A%220022-2720%2C%201365-2818%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fonlinelibrary.wiley.com%5C%2Fdoi%5C%2F10.1111%5C%2Fjmi.13282%22%2C%22collections%22%3A%5B%224KIL6FA6%22%5D%2C%22dateModified%22%3A%222024-04-23T10%3A06%3A06Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22FLSWKSGB%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Shroff%20et%20al.%22%2C%22parsedDate%22%3A%222024-01-02%22%2C%22numChildren%22%3A1%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BShroff%2C%20H.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282024%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttp%3A%5C%2F%5C%2Farxiv.org%5C%2Fabs%5C%2F2401.01438%26%23039%3B%26gt%3BLive-cell%20imaging%20powered%20by%20computation%26lt%3B%5C%2Fa%26gt%3B.%26quot%3B%20arXiv.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.48550%5C%2FarXiv.2401.01438.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22preprint%22%2C%22title%22%3A%22Live-cell%20imaging%20powered%20by%20computation%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Hari%22%2C%22lastName%22%3A%22Shroff%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ilaria%22%2C%22lastName%22%3A%22Testa%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Florian%22%2C%22lastName%22%3A%22Jug%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Suliana%22%2C%22lastName%22%3A%22Manley%22%7D%5D%2C%22abstractNote%22%3A%22The%20proliferation%20of%20microscopy%20methods%20for%20live-cell%20imaging%20offers%20many%20new%20possibilities%20for%20users%20but%20can%20also%20be%20challenging%20to%20navigate.%20We%20focus%20here%20on%20computational%20methods%20that%20promise%20to%20boost%20live-cell%20fluorescence%20microscopy%2C%20where%20intra-cellular%20dynamics%20and%20cell%20viability%20constrain%20measurements%20considerably.%20Considering%20the%20tradeoffs%20between%20signal-to-noise%20ratio%20%28SNR%29%2C%20spatial%20resolution%2C%20temporal%20resolution%2C%20and%20multi-color%20and%20multi-channel%20capacity%2C%20we%20review%20computational%20methods%20that%20can%20be%20layered%20on%20top%20of%20commonly%20used%20existing%20microscopies%2C%20as%20well%20as%20hybrid%20methods%20that%20integrate%20computation%20and%20microscope%20hardware.%22%2C%22genre%22%3A%22%22%2C%22repository%22%3A%22arXiv%22%2C%22archiveID%22%3A%22arXiv%3A2401.01438%22%2C%22date%22%3A%222024-01-02%22%2C%22DOI%22%3A%2210.48550%5C%2FarXiv.2401.01438%22%2C%22citationKey%22%3A%22%22%2C%22url%22%3A%22http%3A%5C%2F%5C%2Farxiv.org%5C%2Fabs%5C%2F2401.01438%22%2C%22language%22%3A%22%22%2C%22collections%22%3A%5B%224KIL6FA6%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A10%3A28Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Data%20Analysis%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22PAJYPWZV%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Pylv%5Cu00e4n%5Cu00e4inen%20et%20al.%22%2C%22parsedDate%22%3A%222023-12-01%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BPylv%5Cu00e4n%5Cu00e4inen%2C%20J.W.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282023%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.sciencedirect.com%5C%2Fscience%5C%2Farticle%5C%2Fpii%5C%2FS0955067423001205%26%23039%3B%26gt%3BLive-cell%20imaging%20in%20the%20deep%20learning%20era%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BCurrent%20Opinion%20in%20Cell%20Biology%26lt%3B%5C%2Fi%26gt%3B%2C%2085%2C%20p.%20102271.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1016%5C%2Fj.ceb.2023.102271.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Live-cell%20imaging%20in%20the%20deep%20learning%20era%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Joanna%20W.%22%2C%22lastName%22%3A%22Pylv%5Cu00e4n%5Cu00e4inen%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Estibaliz%22%2C%22lastName%22%3A%22G%5Cu00f3mez-de-Mariscal%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ricardo%22%2C%22lastName%22%3A%22Henriques%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Guillaume%22%2C%22lastName%22%3A%22Jacquemet%22%7D%5D%2C%22abstractNote%22%3A%22Live%20imaging%20is%20a%20powerful%20tool%2C%20enabling%20scientists%20to%20observe%20living%20organisms%20in%20real%20time.%20In%20particular%2C%20when%20combined%20with%20fluorescence%20microscopy%2C%20live%20imaging%20allows%20the%20monitoring%20of%20cellular%20components%20with%20high%20sensitivity%20and%20specificity.%20Yet%2C%20due%20to%20critical%20challenges%20%28i.e.%2C%20drift%2C%20phototoxicity%2C%20dataset%20size%29%2C%20implementing%20live%20imaging%20and%20analyzing%20the%20resulting%20datasets%20is%20rarely%20straightforward.%20Over%20the%20past%20years%2C%20the%20development%20of%20bioimage%20analysis%20tools%2C%20including%20deep%20learning%2C%20is%20changing%20how%20we%20perform%20live%20imaging.%20Here%20we%20briefly%20cover%20important%20computational%20methods%20aiding%20live%20imaging%20and%20carrying%20out%20key%20tasks%20such%20as%20drift%20correction%2C%20denoising%2C%20super-resolution%20imaging%2C%20artificial%20labeling%2C%20tracking%2C%20and%20time%20series%20analysis.%20We%20also%20cover%20recent%20advances%20in%20self-driving%20microscopy.%22%2C%22date%22%3A%222023-12-01%22%2C%22language%22%3A%22%22%2C%22DOI%22%3A%2210.1016%5C%2Fj.ceb.2023.102271%22%2C%22ISSN%22%3A%220955-0674%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.sciencedirect.com%5C%2Fscience%5C%2Farticle%5C%2Fpii%5C%2FS0955067423001205%22%2C%22collections%22%3A%5B%224KIL6FA6%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A12%3A06Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Data%20Analysis%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22CUSN9WPC%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Balasubramanian%20et%20al.%22%2C%22parsedDate%22%3A%222023-10-28%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BBalasubramanian%2C%20H.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282023%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs42003-023-05468-9%26%23039%3B%26gt%3BImagining%20the%20future%20of%20optical%20microscopy%3A%20everything%2C%20everywhere%2C%20all%20at%20once%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BCommunications%20Biology%26lt%3B%5C%2Fi%26gt%3B%2C%206%281%29%2C%20pp.%201%5Cu201312.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs42003-023-05468-9.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Imagining%20the%20future%20of%20optical%20microscopy%3A%20everything%2C%20everywhere%2C%20all%20at%20once%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Harikrushnan%22%2C%22lastName%22%3A%22Balasubramanian%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Chad%20M.%22%2C%22lastName%22%3A%22Hobson%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Teng-Leong%22%2C%22lastName%22%3A%22Chew%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jesse%20S.%22%2C%22lastName%22%3A%22Aaron%22%7D%5D%2C%22abstractNote%22%3A%22The%20optical%20microscope%20has%20revolutionized%20biology%20since%20at%20least%20the%2017th%20Century.%20Since%20then%2C%20it%20has%20progressed%20from%20a%20largely%20observational%20tool%20to%20a%20powerful%20bioanalytical%20platform.%20However%2C%20realizing%20its%20full%20potential%20to%20study%20live%20specimens%20is%20hindered%20by%20a%20daunting%20array%20of%20technical%20challenges.%20Here%2C%20we%20delve%20into%20the%20current%20state%20of%20live%20imaging%20to%20explore%20the%20barriers%20that%20must%20be%20overcome%20and%20the%20possibilities%20that%20lie%20ahead.%20We%20venture%20to%20envision%20a%20future%20where%20we%20can%20visualize%20and%20study%20everything%2C%20everywhere%2C%20all%20at%20once%20%5Cu2013%20from%20the%20intricate%20inner%20workings%20of%20a%20single%20cell%20to%20the%20dynamic%20interplay%20across%20entire%20organisms%2C%20and%20a%20world%20where%20scientists%20could%20access%20the%20necessary%20microscopy%20technologies%20anywhere.%22%2C%22date%22%3A%222023-10-28%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs42003-023-05468-9%22%2C%22ISSN%22%3A%222399-3642%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs42003-023-05468-9%22%2C%22collections%22%3A%5B%224KIL6FA6%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A12%3A30Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%2C%7B%22tag%22%3A%22Data%20Analysis%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22EAN93DZP%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Daetwyler%20and%20Fiolka%22%2C%22parsedDate%22%3A%222023-05-09%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BDaetwyler%2C%20S.%20and%20Fiolka%2C%20R.P.%20%282023%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs42003-023-04857-4%26%23039%3B%26gt%3BLight-sheets%20and%20smart%20microscopy%2C%20an%20exciting%20future%20is%20dawning%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BCommunications%20Biology%26lt%3B%5C%2Fi%26gt%3B%2C%206%281%29%2C%20pp.%201%5Cu201311.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1038%5C%2Fs42003-023-04857-4.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Light-sheets%20and%20smart%20microscopy%2C%20an%20exciting%20future%20is%20dawning%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Stephan%22%2C%22lastName%22%3A%22Daetwyler%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Reto%20Paul%22%2C%22lastName%22%3A%22Fiolka%22%7D%5D%2C%22abstractNote%22%3A%22Light-sheet%20fluorescence%20microscopy%20has%20transformed%20our%20ability%20to%20visualize%20and%20quantitatively%20measure%20biological%20processes%20rapidly%20and%20over%20long%20time%20periods.%20In%20this%20review%2C%20we%20discuss%20current%20and%20future%20developments%20in%20light-sheet%20fluorescence%20microscopy%20that%20we%20expect%20to%20further%20expand%20its%20capabilities.%20This%20includes%20smart%20and%20adaptive%20imaging%20schemes%20to%20overcome%20traditional%20imaging%20trade-offs%2C%20i.e.%2C%20spatiotemporal%20resolution%2C%20field%20of%20view%20and%20sample%20health.%20In%20smart%20microscopy%2C%20a%20microscope%20will%20autonomously%20decide%20where%2C%20when%2C%20what%20and%20how%20to%20image.%20We%20further%20assess%20how%20image%20restoration%20techniques%20provide%20avenues%20to%20overcome%20these%20tradeoffs%20and%20how%20%5Cu201copen%20top%5Cu201d%20light-sheet%20microscopes%20may%20enable%20multi-modal%20imaging%20with%20high%20throughput.%20As%20such%2C%20we%20predict%20that%20light-sheet%20microscopy%20will%20fulfill%20an%20important%20role%20in%20biomedical%20and%20clinical%20imaging%20in%20the%20future.%22%2C%22date%22%3A%222023-05-09%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1038%5C%2Fs42003-023-04857-4%22%2C%22ISSN%22%3A%222399-3642%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.nature.com%5C%2Farticles%5C%2Fs42003-023-04857-4%22%2C%22collections%22%3A%5B%224KIL6FA6%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A12%3A20Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%2C%7B%22tag%22%3A%22Data%20Analysis%22%7D%2C%7B%22tag%22%3A%22Data%20handling%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22P474WSZT%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Volpe%20et%20al.%22%2C%22parsedDate%22%3A%222023-03-07%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BVolpe%2C%20G.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282023%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttp%3A%5C%2F%5C%2Farxiv.org%5C%2Fabs%5C%2F2303.03793%26%23039%3B%26gt%3BRoadmap%20on%20Deep%20Learning%20for%20Microscopy%26lt%3B%5C%2Fa%26gt%3B.%26quot%3B%20arXiv.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.48550%5C%2FarXiv.2303.03793.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22preprint%22%2C%22title%22%3A%22Roadmap%20on%20Deep%20Learning%20for%20Microscopy%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Giovanni%22%2C%22lastName%22%3A%22Volpe%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Carolina%22%2C%22lastName%22%3A%22W%5Cu00e4hlby%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Lei%22%2C%22lastName%22%3A%22Tian%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Michael%22%2C%22lastName%22%3A%22Hecht%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Artur%22%2C%22lastName%22%3A%22Yakimovich%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Kristina%22%2C%22lastName%22%3A%22Monakhova%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Laura%22%2C%22lastName%22%3A%22Waller%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ivo%20F.%22%2C%22lastName%22%3A%22Sbalzarini%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Christopher%20A.%22%2C%22lastName%22%3A%22Metzler%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Mingyang%22%2C%22lastName%22%3A%22Xie%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Kevin%22%2C%22lastName%22%3A%22Zhang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Isaac%20C.%20D.%22%2C%22lastName%22%3A%22Lenton%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Halina%22%2C%22lastName%22%3A%22Rubinsztein-Dunlop%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Daniel%22%2C%22lastName%22%3A%22Brunner%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Bijie%22%2C%22lastName%22%3A%22Bai%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Aydogan%22%2C%22lastName%22%3A%22Ozcan%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Daniel%22%2C%22lastName%22%3A%22Midtvedt%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Hao%22%2C%22lastName%22%3A%22Wang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Nata%5Cu0161a%22%2C%22lastName%22%3A%22Sladoje%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Joakim%22%2C%22lastName%22%3A%22Lindblad%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jason%20T.%22%2C%22lastName%22%3A%22Smith%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Marien%22%2C%22lastName%22%3A%22Ochoa%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Margarida%22%2C%22lastName%22%3A%22Barroso%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Xavier%22%2C%22lastName%22%3A%22Intes%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Tong%22%2C%22lastName%22%3A%22Qiu%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Li-Yu%22%2C%22lastName%22%3A%22Yu%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Sixian%22%2C%22lastName%22%3A%22You%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Yongtao%22%2C%22lastName%22%3A%22Liu%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Maxim%20A.%22%2C%22lastName%22%3A%22Ziatdinov%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Sergei%20V.%22%2C%22lastName%22%3A%22Kalinin%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Arlo%22%2C%22lastName%22%3A%22Sheridan%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Uri%22%2C%22lastName%22%3A%22Manor%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Elias%22%2C%22lastName%22%3A%22Nehme%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ofri%22%2C%22lastName%22%3A%22Goldenberg%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Yoav%22%2C%22lastName%22%3A%22Shechtman%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Henrik%20K.%22%2C%22lastName%22%3A%22Moberg%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Christoph%22%2C%22lastName%22%3A%22Langhammer%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Barbora%22%2C%22lastName%22%3A%22%5Cu0160pa%5Cu010dkov%5Cu00e1%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Saga%22%2C%22lastName%22%3A%22Helgadottir%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Benjamin%22%2C%22lastName%22%3A%22Midtvedt%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Aykut%22%2C%22lastName%22%3A%22Argun%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Tobias%22%2C%22lastName%22%3A%22Thalheim%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Frank%22%2C%22lastName%22%3A%22Cichos%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Stefano%22%2C%22lastName%22%3A%22Bo%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Lars%22%2C%22lastName%22%3A%22Hubatsch%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Jesus%22%2C%22lastName%22%3A%22Pineda%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Carlo%22%2C%22lastName%22%3A%22Manzo%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Harshith%22%2C%22lastName%22%3A%22Bachimanchi%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Erik%22%2C%22lastName%22%3A%22Selander%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Antoni%22%2C%22lastName%22%3A%22Homs-Corbera%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Martin%22%2C%22lastName%22%3A%22Fr%5Cu00e4nzl%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Kevin%22%2C%22lastName%22%3A%22de%20Haan%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Yair%22%2C%22lastName%22%3A%22Rivenson%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Zofia%22%2C%22lastName%22%3A%22Korczak%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Caroline%20Beck%22%2C%22lastName%22%3A%22Adiels%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Mite%22%2C%22lastName%22%3A%22Mijalkov%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22D%5Cu00e1niel%22%2C%22lastName%22%3A%22Ver%5Cu00e9b%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Yu-Wei%22%2C%22lastName%22%3A%22Chang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Joana%20B.%22%2C%22lastName%22%3A%22Pereira%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Damian%22%2C%22lastName%22%3A%22Matuszewski%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Gustaf%22%2C%22lastName%22%3A%22Kylberg%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ida-Maria%22%2C%22lastName%22%3A%22Sintorn%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Juan%20C.%22%2C%22lastName%22%3A%22Caicedo%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Beth%20A.%22%2C%22lastName%22%3A%22Cimini%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Muyinatu%20A.%20Lediju%22%2C%22lastName%22%3A%22Bell%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Bruno%20M.%22%2C%22lastName%22%3A%22Saraiva%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Guillaume%22%2C%22lastName%22%3A%22Jacquemet%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ricardo%22%2C%22lastName%22%3A%22Henriques%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Wei%22%2C%22lastName%22%3A%22Ouyang%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Trang%22%2C%22lastName%22%3A%22Le%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Estibaliz%22%2C%22lastName%22%3A%22G%5Cu00f3mez-de-Mariscal%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Daniel%22%2C%22lastName%22%3A%22Sage%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Arrate%22%2C%22lastName%22%3A%22Mu%5Cu00f1oz-Barrutia%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ebba%20Josefson%22%2C%22lastName%22%3A%22Lindqvist%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Johanna%22%2C%22lastName%22%3A%22Bergman%22%7D%5D%2C%22abstractNote%22%3A%22Through%20digital%20imaging%2C%20microscopy%20has%20evolved%20from%20primarily%20being%20a%20means%20for%20visual%20observation%20of%20life%20at%20the%20micro-%20and%20nano-scale%2C%20to%20a%20quantitative%20tool%20with%20ever-increasing%20resolution%20and%20throughput.%20Artificial%20intelligence%2C%20deep%20neural%20networks%2C%20and%20machine%20learning%20are%20all%20niche%20terms%20describing%20computational%20methods%20that%20have%20gained%20a%20pivotal%20role%20in%20microscopy-based%20research%20over%20the%20past%20decade.%20This%20Roadmap%20is%20written%20collectively%20by%20prominent%20researchers%20and%20encompasses%20selected%20aspects%20of%20how%20machine%20learning%20is%20applied%20to%20microscopy%20image%20data%2C%20with%20the%20aim%20of%20gaining%20scientific%20knowledge%20by%20improved%20image%20quality%2C%20automated%20detection%2C%20segmentation%2C%20classification%20and%20tracking%20of%20objects%2C%20and%20efficient%20merging%20of%20information%20from%20multiple%20imaging%20modalities.%20We%20aim%20to%20give%20the%20reader%20an%20overview%20of%20the%20key%20developments%20and%20an%20understanding%20of%20possibilities%20and%20limitations%20of%20machine%20learning%20for%20microscopy.%20It%20will%20be%20of%20interest%20to%20a%20wide%20cross-disciplinary%20audience%20in%20the%20physical%20sciences%20and%20life%20sciences.%22%2C%22genre%22%3A%22%22%2C%22repository%22%3A%22arXiv%22%2C%22archiveID%22%3A%22arXiv%3A2303.03793%22%2C%22date%22%3A%222023-03-07%22%2C%22DOI%22%3A%2210.48550%5C%2FarXiv.2303.03793%22%2C%22citationKey%22%3A%22%22%2C%22url%22%3A%22http%3A%5C%2F%5C%2Farxiv.org%5C%2Fabs%5C%2F2303.03793%22%2C%22language%22%3A%22%22%2C%22collections%22%3A%5B%224KIL6FA6%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A10%3A18Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Data%20Analysis%22%7D%2C%7B%22tag%22%3A%22Data%20handling%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22MZQKQP7A%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Andr%5Cu00e9%20et%20al.%22%2C%22parsedDate%22%3A%222022-06-10%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BAndr%5Cu00e9%2C%20O.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282022%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.biorxiv.org%5C%2Fcontent%5C%2F10.1101%5C%2F2022.05.09.491153v2%26%23039%3B%26gt%3BData-driven%20microscopy%20allows%20for%20automated%20targeted%20acquisition%20of%20relevant%20data%20with%20higher%20fidelity%26lt%3B%5C%2Fa%26gt%3B.%26quot%3B%20bioRxiv.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1101%5C%2F2022.05.09.491153.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22preprint%22%2C%22title%22%3A%22Data-driven%20microscopy%20allows%20for%20automated%20targeted%20acquisition%20of%20relevant%20data%20with%20higher%20fidelity%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Oscar%22%2C%22lastName%22%3A%22Andr%5Cu00e9%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Johannes%20Kumra%22%2C%22lastName%22%3A%22Ahnlide%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Nils%22%2C%22lastName%22%3A%22Norlin%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Vinay%22%2C%22lastName%22%3A%22Swaminathan%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Pontus%22%2C%22lastName%22%3A%22Nordenfelt%22%7D%5D%2C%22abstractNote%22%3A%22Light%20microscopy%20is%20a%20powerful%20single-cell%20technique%20that%20allows%20for%20quantitative%20spatial%20information%20at%20subcellular%20resolution.%20However%2C%20unlike%20flow%20cytometry%20and%20single-cell%20sequencing%20techniques%2C%20microscopy%20has%20issues%20achieving%20high-quality%20population-wide%20sample%20characterization%20while%20maintaining%20high%20resolution.%20Here%2C%20we%20present%20a%20general%20framework%2C%20data-driven%20microscopy%20%28DDM%29%2C%20that%20uses%20population-wide%20cell%20characterization%20to%20enable%20data-driven%20high-fidelity%20imaging%20of%20relevant%20phenotypes.%20DDM%20combines%20data-independent%20and%20data-dependent%20steps%20to%20synergistically%20enhance%20data%20acquired%20using%20different%20imaging%20modalities.%20As%20proof-of-concept%2C%20we%20apply%20DDM%20with%20plugins%20for%20improved%20high-content%20screening%20and%20live%20adaptive%20microscopy.%20DDM%20also%20allows%20for%20easy%20correlative%20imaging%20in%20other%20systems%20with%20a%20plugin%20that%20uses%20the%20spatial%20relationship%20of%20the%20sample%20population%20for%20automated%20registration.%20We%20believe%20DDM%20will%20be%20a%20valuable%20approach%20for%20reducing%20human%20bias%2C%20increasing%20reproducibility%2C%20and%20placing%20singlecell%20characteristics%20in%20the%20context%20of%20the%20sample%20population%20when%20interpreting%20microscopy%20data%2C%20leading%20to%20an%20overall%20increase%20in%20data%20fidelity.%22%2C%22genre%22%3A%22%22%2C%22repository%22%3A%22bioRxiv%22%2C%22archiveID%22%3A%22%22%2C%22date%22%3A%222022-06-10%22%2C%22DOI%22%3A%2210.1101%5C%2F2022.05.09.491153%22%2C%22citationKey%22%3A%22%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.biorxiv.org%5C%2Fcontent%5C%2F10.1101%5C%2F2022.05.09.491153v2%22%2C%22language%22%3A%22en%22%2C%22collections%22%3A%5B%224KIL6FA6%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A10%3A37Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Acquisition%20frameworks%22%7D%2C%7B%22tag%22%3A%22Data%20Analysis%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22KRYJJPYL%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22von%20Chamier%20et%20al.%22%2C%22parsedDate%22%3A%222019-07-31%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3Bvon%20Chamier%2C%20L.%2C%20Laine%2C%20R.F.%20and%20Henriques%2C%20R.%20%282019%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1042%5C%2FBST20180391%26%23039%3B%26gt%3BArtificial%20intelligence%20for%20microscopy%3A%20what%20you%20should%20know%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BBiochemical%20Society%20Transactions%26lt%3B%5C%2Fi%26gt%3B%2C%2047%284%29%2C%20pp.%201029%5Cu20131040.%20Available%20at%3A%20%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Artificial%20intelligence%20for%20microscopy%3A%20what%20you%20should%20know%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Lucas%22%2C%22lastName%22%3A%22von%20Chamier%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Romain%20F.%22%2C%22lastName%22%3A%22Laine%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Ricardo%22%2C%22lastName%22%3A%22Henriques%22%7D%5D%2C%22abstractNote%22%3A%22Artificial%20Intelligence%20based%20on%20Deep%20Learning%20%28DL%29%20is%20opening%20new%20horizons%20in%20biomedical%20research%20and%20promises%20to%20revolutionize%20the%20microscopy%20field.%20It%20is%20now%20transitioning%20from%20the%20hands%20of%20experts%20in%20computer%20sciences%20to%20biomedical%20researchers.%20Here%2C%20we%20introduce%20recent%20developments%20in%20DL%20applied%20to%20microscopy%2C%20in%20a%20manner%20accessible%20to%20non-experts.%20We%20give%20an%20overview%20of%20its%20concepts%2C%20capabilities%20and%20limitations%2C%20presenting%20applications%20in%20image%20segmentation%2C%20classification%20and%20restoration.%20We%20discuss%20how%20DL%20shows%20an%20outstanding%20potential%20to%20push%20the%20limits%20of%20microscopy%2C%20enhancing%20resolution%2C%20signal%20and%20information%20content%20in%20acquired%20data.%20Its%20pitfalls%20are%20discussed%2C%20along%20with%20the%20future%20directions%20expected%20in%20this%20field.%22%2C%22date%22%3A%222019-07-31%22%2C%22language%22%3A%22%22%2C%22DOI%22%3A%2210.1042%5C%2FBST20180391%22%2C%22ISSN%22%3A%220300-5127%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1042%5C%2FBST20180391%22%2C%22collections%22%3A%5B%224KIL6FA6%22%5D%2C%22dateModified%22%3A%222024-04-23T10%3A06%3A32Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Data%20Analysis%22%7D%2C%7B%22tag%22%3A%22Data%20handling%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22FGGQKEF5%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Kalinin%20et%20al.%22%2C%22parsedDate%22%3A%222016-10-25%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BKalinin%2C%20S.V.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282016%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1021%5C%2Facsnano.6b04212%26%23039%3B%26gt%3BBig%2C%20Deep%2C%20and%20Smart%20Data%20in%20Scanning%20Probe%20Microscopy%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BACS%20Nano%26lt%3B%5C%2Fi%26gt%3B%2C%2010%2810%29%2C%20pp.%209068%5Cu20139086.%20Available%20at%3A%20%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Big%2C%20Deep%2C%20and%20Smart%20Data%20in%20Scanning%20Probe%20Microscopy%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Sergei%20V.%22%2C%22lastName%22%3A%22Kalinin%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Evgheni%22%2C%22lastName%22%3A%22Strelcov%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Alex%22%2C%22lastName%22%3A%22Belianinov%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Suhas%22%2C%22lastName%22%3A%22Somnath%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Rama%20K.%22%2C%22lastName%22%3A%22Vasudevan%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Eric%20J.%22%2C%22lastName%22%3A%22Lingerfelt%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Richard%20K.%22%2C%22lastName%22%3A%22Archibald%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Chaomei%22%2C%22lastName%22%3A%22Chen%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Roger%22%2C%22lastName%22%3A%22Proksch%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Nouamane%22%2C%22lastName%22%3A%22Laanait%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Stephen%22%2C%22lastName%22%3A%22Jesse%22%7D%5D%2C%22abstractNote%22%3A%22Scanning%20probe%20microscopy%20%28SPM%29%20techniques%20have%20opened%20the%20door%20to%20nanoscience%20and%20nanotechnology%20by%20enabling%20imaging%20and%20manipulation%20of%20the%20structure%20and%20functionality%20of%20matter%20at%20nanometer%20and%20atomic%20scales.%20Here%2C%20we%20analyze%20the%20scientific%20discovery%20process%20in%20SPM%20by%20following%20the%20information%20flow%20from%20the%20tip%5Cu2013surface%20junction%2C%20to%20knowledge%20adoption%20by%20the%20wider%20scientific%20community.%20We%20further%20discuss%20the%20challenges%20and%20opportunities%20offered%20by%20merging%20SPM%20with%20advanced%20data%20mining%2C%20visual%20analytics%2C%20and%20knowledge%20discovery%20technologies.%22%2C%22date%22%3A%222016-10-25%22%2C%22language%22%3A%22%22%2C%22DOI%22%3A%2210.1021%5C%2Facsnano.6b04212%22%2C%22ISSN%22%3A%221936-0851%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1021%5C%2Facsnano.6b04212%22%2C%22collections%22%3A%5B%224KIL6FA6%22%5D%2C%22dateModified%22%3A%222024-02-22T13%3A12%3A41Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Data%20Analysis%22%7D%2C%7B%22tag%22%3A%22Data%20handling%22%7D%5D%7D%7D%2C%7B%22key%22%3A%22ZJW7U833%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Tischer%20et%20al.%22%2C%22parsedDate%22%3A%222014-01-01%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BTischer%2C%20C.%20%26lt%3Bi%26gt%3Bet%20al.%26lt%3B%5C%2Fi%26gt%3B%20%282014%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fwww.sciencedirect.com%5C%2Fscience%5C%2Farticle%5C%2Fpii%5C%2FB9780124201385000264%26%23039%3B%26gt%3BChapter%2026%20-%20Adaptive%20fluorescence%20microscopy%20by%20online%20feedback%20image%20analysis%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20in%20J.C.%20Waters%20and%20T.%20Wittman%20%28eds.%29%20%26lt%3Bi%26gt%3BMethods%20in%20Cell%20Biology%26lt%3B%5C%2Fi%26gt%3B.%20Academic%20Press%20%28Quantitative%20Imaging%20in%20Cell%20Biology%29%2C%20pp.%20489%5Cu2013503.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1016%5C%2FB978-0-12-420138-5.00026-4.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22bookSection%22%2C%22title%22%3A%22Chapter%2026%20-%20Adaptive%20fluorescence%20microscopy%20by%20online%20feedback%20image%20analysis%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Christian%22%2C%22lastName%22%3A%22Tischer%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Volker%22%2C%22lastName%22%3A%22Hilsenstein%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Kirsten%22%2C%22lastName%22%3A%22Hanson%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Rainer%22%2C%22lastName%22%3A%22Pepperkok%22%7D%2C%7B%22creatorType%22%3A%22editor%22%2C%22firstName%22%3A%22Jennifer%20C.%22%2C%22lastName%22%3A%22Waters%22%7D%2C%7B%22creatorType%22%3A%22editor%22%2C%22firstName%22%3A%22Torsten%22%2C%22lastName%22%3A%22Wittman%22%7D%5D%2C%22abstractNote%22%3A%22Obtaining%20sufficient%20statistics%20in%20quantitative%20fluorescence%20microscopy%20is%20often%20hampered%20by%20the%20tedious%20and%20time-consuming%20task%20of%20manually%20locating%20comparable%20specimen%20and%20repeatedly%20launching%20the%20same%20acquisition%20protocol.%20Recent%20advances%20in%20combining%20fluorescence%20microscopy%20with%20online%20image%20analysis%20tackle%20this%20problem%20by%20fully%20integrating%20the%20task%20of%20identifying%20and%20locating%20the%20specimen%20of%20interest%20in%20an%20automated%20acquisition%20workflow.%20Here%2C%20we%20describe%20the%20general%20requirements%20and%20specific%20microscope%20control%20and%20image%20analysis%20software%20solutions%20for%20implementing%20such%20automated%20online%20feedback%20microscopy.%20We%20demonstrate%20the%20power%20of%20the%20method%20by%20two%20selected%20applications%20addressing%20high-throughput%203D%20imaging%20of%20sparsely%20parasite-infected%20tissue%20culture%20cells%20and%20automated%20fluorescence%20recovery%20after%20photobleaching%20experiments%20to%20quantify%20the%20turnover%20of%20vesicular%20coat%20proteins%20at%20ER%20exit%20sites.%22%2C%22bookTitle%22%3A%22Methods%20in%20Cell%20Biology%22%2C%22date%22%3A%222014-01-01%22%2C%22language%22%3A%22%22%2C%22ISBN%22%3A%22%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fwww.sciencedirect.com%5C%2Fscience%5C%2Farticle%5C%2Fpii%5C%2FB9780124201385000264%22%2C%22collections%22%3A%5B%224KIL6FA6%22%5D%2C%22dateModified%22%3A%222024-04-23T10%3A05%3A41Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Application%22%7D%2C%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%2C%7B%22key%22%3A%2266SBX2JH%22%2C%22library%22%3A%7B%22id%22%3A5384098%7D%2C%22meta%22%3A%7B%22creatorSummary%22%3A%22Tsukada%20and%20Hashimoto%22%2C%22parsedDate%22%3A%222013%22%2C%22numChildren%22%3A0%7D%2C%22bib%22%3A%22%26lt%3Bdiv%20class%3D%26quot%3Bcsl-bib-body%26quot%3B%20style%3D%26quot%3Bline-height%3A%201.35%3B%20%26quot%3B%26gt%3B%5Cn%20%26lt%3Bdiv%20class%3D%26quot%3Bcsl-entry%26quot%3B%26gt%3BTsukada%2C%20Y.%20and%20Hashimoto%2C%20K.%20%282013%29%20%26quot%3B%26lt%3Ba%20class%3D%26%23039%3Bzp-ItemURL%26%23039%3B%20target%3D%26%23039%3B_blank%26%23039%3B%20href%3D%26%23039%3Bhttps%3A%5C%2F%5C%2Fonlinelibrary.wiley.com%5C%2Fdoi%5C%2Fabs%5C%2F10.1111%5C%2Fdgd.12056%26%23039%3B%26gt%3BFeedback%20regulation%20of%20microscopes%20by%20image%20processing%26lt%3B%5C%2Fa%26gt%3B%2C%26quot%3B%20%26lt%3Bi%26gt%3BDevelopment%2C%20Growth%20%26amp%3B%20Differentiation%26lt%3B%5C%2Fi%26gt%3B%2C%2055%284%29%2C%20pp.%20550%5Cu2013562.%20Available%20at%3A%20https%3A%5C%2F%5C%2Fdoi.org%5C%2F10.1111%5C%2Fdgd.12056.%26lt%3B%5C%2Fdiv%26gt%3B%5Cn%26lt%3B%5C%2Fdiv%26gt%3B%22%2C%22data%22%3A%7B%22itemType%22%3A%22journalArticle%22%2C%22title%22%3A%22Feedback%20regulation%20of%20microscopes%20by%20image%20processing%22%2C%22creators%22%3A%5B%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Yuki%22%2C%22lastName%22%3A%22Tsukada%22%7D%2C%7B%22creatorType%22%3A%22author%22%2C%22firstName%22%3A%22Koichi%22%2C%22lastName%22%3A%22Hashimoto%22%7D%5D%2C%22abstractNote%22%3A%22Computational%20microscope%20systems%20are%20becoming%20a%20major%20part%20of%20imaging%20biological%20phenomena%2C%20and%20the%20development%20of%20such%20systems%20requires%20the%20design%20of%20automated%20regulation%20of%20microscopes.%20An%20important%20aspect%20of%20automated%20regulation%20is%20feedback%20regulation%2C%20which%20is%20the%20focus%20of%20this%20review.%20As%20modern%20microscope%20systems%20become%20more%20complex%2C%20often%20with%20many%20independent%20components%20that%20must%20work%20together%2C%20computer%20control%20is%20inevitable%20since%20the%20exact%20orchestration%20of%20parameters%20and%20timings%20for%20these%20multiple%20components%20is%20critical%20to%20acquire%20proper%20images.%20A%20number%20of%20techniques%20have%20been%20developed%20for%20biological%20imaging%20to%20accomplish%20this.%20Here%2C%20we%20summarize%20the%20basics%20of%20computational%20microscopy%20for%20the%20purpose%20of%20building%20automatically%20regulated%20microscopes%20focus%20on%20feedback%20regulation%20by%20image%20processing.%20These%20techniques%20allow%20high%20throughput%20data%20acquisition%20while%20monitoring%20both%20short-%20and%20long-term%20dynamic%20phenomena%2C%20which%20cannot%20be%20achieved%20without%20an%20automated%20system.%22%2C%22date%22%3A%222013%22%2C%22language%22%3A%22en%22%2C%22DOI%22%3A%2210.1111%5C%2Fdgd.12056%22%2C%22ISSN%22%3A%221440-169X%22%2C%22url%22%3A%22https%3A%5C%2F%5C%2Fonlinelibrary.wiley.com%5C%2Fdoi%5C%2Fabs%5C%2F10.1111%5C%2Fdgd.12056%22%2C%22collections%22%3A%5B%224KIL6FA6%22%5D%2C%22dateModified%22%3A%222024-04-23T10%3A06%3A19Z%22%2C%22tags%22%3A%5B%7B%22tag%22%3A%22Automation%22%7D%5D%7D%7D%5D%7D</span>

				<div id="zp-ID-27-5384098-QJYYQESZ" data-zp-author-date='Hsieh-et-al.-2025-04-17' data-zp-date-author='2025-04-17-Hsieh-et-al.' data-zp-date='2025-04-17' data-zp-year='2025' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Hsieh, H.-C. <i>et al.</i> (2025) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s41592-025-02647-w'>Imaging 3D cell cultures with optical microscopy</a>," <i>Nature Methods</i>, pp. 1–24. Available at: <a href="https://doi.org/10.1038/s41592-025-02647-w" rel="nofollow">https://doi.org/10.1038/s41592-025-02647-w</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-QUM2DVR6" data-zp-author-date='Morgado-et-al.-2024-03-06' data-zp-date-author='2024-03-06-Morgado-et-al.' data-zp-date='2024-03-06' data-zp-year='2024' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Morgado, L. <i>et al.</i> (2024) "<a class='zp-ItemURL' target='_blank' href='https://onlinelibrary.wiley.com/doi/10.1111/jmi.13282'>The rise of data‐driven microscopy powered by machine learning</a>," <i>Journal of Microscopy</i>, p. jmi.13282. Available at: <a href="https://doi.org/10.1111/jmi.13282" rel="nofollow">https://doi.org/10.1111/jmi.13282</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-FLSWKSGB" data-zp-author-date='Shroff-et-al.-2024-01-02' data-zp-date-author='2024-01-02-Shroff-et-al.' data-zp-date='2024-01-02' data-zp-year='2024' data-zp-itemtype='preprint' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Shroff, H. <i>et al.</i> (2024) "<a class='zp-ItemURL' target='_blank' href='http://arxiv.org/abs/2401.01438'>Live-cell imaging powered by computation</a>." arXiv. Available at: <a href="https://doi.org/10.48550/arXiv.2401.01438" rel="nofollow">https://doi.org/10.48550/arXiv.2401.01438</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-PAJYPWZV" data-zp-author-date='Pylvänäinen-et-al.-2023-12-01' data-zp-date-author='2023-12-01-Pylvänäinen-et-al.' data-zp-date='2023-12-01' data-zp-year='2023' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Pylvänäinen, J.W. <i>et al.</i> (2023) "<a class='zp-ItemURL' target='_blank' href='https://www.sciencedirect.com/science/article/pii/S0955067423001205'>Live-cell imaging in the deep learning era</a>," <i>Current Opinion in Cell Biology</i>, 85, p. 102271. Available at: <a href="https://doi.org/10.1016/j.ceb.2023.102271" rel="nofollow">https://doi.org/10.1016/j.ceb.2023.102271</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-CUSN9WPC" data-zp-author-date='Balasubramanian-et-al.-2023-10-28' data-zp-date-author='2023-10-28-Balasubramanian-et-al.' data-zp-date='2023-10-28' data-zp-year='2023' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Balasubramanian, H. <i>et al.</i> (2023) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s42003-023-05468-9'>Imagining the future of optical microscopy: everything, everywhere, all at once</a>," <i>Communications Biology</i>, 6(1), pp. 1–12. Available at: <a href="https://doi.org/10.1038/s42003-023-05468-9" rel="nofollow">https://doi.org/10.1038/s42003-023-05468-9</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-EAN93DZP" data-zp-author-date='Daetwyler-and-Fiolka-2023-05-09' data-zp-date-author='2023-05-09-Daetwyler-and-Fiolka' data-zp-date='2023-05-09' data-zp-year='2023' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Daetwyler, S. and Fiolka, R.P. (2023) "<a class='zp-ItemURL' target='_blank' href='https://www.nature.com/articles/s42003-023-04857-4'>Light-sheets and smart microscopy, an exciting future is dawning</a>," <i>Communications Biology</i>, 6(1), pp. 1–11. Available at: <a href="https://doi.org/10.1038/s42003-023-04857-4" rel="nofollow">https://doi.org/10.1038/s42003-023-04857-4</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-P474WSZT" data-zp-author-date='Volpe-et-al.-2023-03-07' data-zp-date-author='2023-03-07-Volpe-et-al.' data-zp-date='2023-03-07' data-zp-year='2023' data-zp-itemtype='preprint' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Volpe, G. <i>et al.</i> (2023) "<a class='zp-ItemURL' target='_blank' href='http://arxiv.org/abs/2303.03793'>Roadmap on Deep Learning for Microscopy</a>." arXiv. Available at: <a href="https://doi.org/10.48550/arXiv.2303.03793" rel="nofollow">https://doi.org/10.48550/arXiv.2303.03793</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-MZQKQP7A" data-zp-author-date='André-et-al.-2022-06-10' data-zp-date-author='2022-06-10-André-et-al.' data-zp-date='2022-06-10' data-zp-year='2022' data-zp-itemtype='preprint' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">André, O. <i>et al.</i> (2022) "<a class='zp-ItemURL' target='_blank' href='https://www.biorxiv.org/content/10.1101/2022.05.09.491153v2'>Data-driven microscopy allows for automated targeted acquisition of relevant data with higher fidelity</a>." bioRxiv. Available at: <a href="https://doi.org/10.1101/2022.05.09.491153" rel="nofollow">https://doi.org/10.1101/2022.05.09.491153</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-KRYJJPYL" data-zp-author-date='von-Chamier-et-al.-2019-07-31' data-zp-date-author='2019-07-31-von-Chamier-et-al.' data-zp-date='2019-07-31' data-zp-year='2019' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">von Chamier, L., Laine, R.F. and Henriques, R. (2019) "<a class='zp-ItemURL' target='_blank' href='https://doi.org/10.1042/BST20180391'>Artificial intelligence for microscopy: what you should know</a>," <i>Biochemical Society Transactions</i>, 47(4), pp. 1029–1040. Available at: </div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-FGGQKEF5" data-zp-author-date='Kalinin-et-al.-2016-10-25' data-zp-date-author='2016-10-25-Kalinin-et-al.' data-zp-date='2016-10-25' data-zp-year='2016' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Kalinin, S.V. <i>et al.</i> (2016) "<a class='zp-ItemURL' target='_blank' href='https://doi.org/10.1021/acsnano.6b04212'>Big, Deep, and Smart Data in Scanning Probe Microscopy</a>," <i>ACS Nano</i>, 10(10), pp. 9068–9086. Available at: </div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-ZJW7U833" data-zp-author-date='Tischer-et-al.-2014-01-01' data-zp-date-author='2014-01-01-Tischer-et-al.' data-zp-date='2014-01-01' data-zp-year='2014' data-zp-itemtype='bookSection' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Tischer, C. <i>et al.</i> (2014) "<a class='zp-ItemURL' target='_blank' href='https://www.sciencedirect.com/science/article/pii/B9780124201385000264'>Chapter 26 - Adaptive fluorescence microscopy by online feedback image analysis</a>," in J.C. Waters and T. Wittman (eds.) <i>Methods in Cell Biology</i>. Academic Press (Quantitative Imaging in Cell Biology), pp. 489–503. Available at: <a href="https://doi.org/10.1016/B978-0-12-420138-5.00026-4" rel="nofollow">https://doi.org/10.1016/B978-0-12-420138-5.00026-4</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->				<div id="zp-ID-27-5384098-66SBX2JH" data-zp-author-date='Tsukada-and-Hashimoto-2013' data-zp-date-author='2013-Tsukada-and-Hashimoto' data-zp-date='2013' data-zp-year='2013' data-zp-itemtype='journalArticle' class="zp-Entry zpSearchResultsItem">
<div class="csl-bib-body" style="line-height: 1.35; ">
 <div class="csl-entry">Tsukada, Y. and Hashimoto, K. (2013) "<a class='zp-ItemURL' target='_blank' href='https://onlinelibrary.wiley.com/doi/abs/10.1111/dgd.12056'>Feedback regulation of microscopes by image processing</a>," <i>Development, Growth & Differentiation</i>, 55(4), pp. 550–562. Available at: <a href="https://doi.org/10.1111/dgd.12056" rel="nofollow">https://doi.org/10.1111/dgd.12056</a>.</div>
</div>
				</div><!-- .zp-Entry .zpSearchResultsItem -->
			</div><!-- .zp-zp-SEO-Content -->
		</div><!-- .zp-List -->
	</div><!--.zp-Zotpress-->


    <div class="zotpress-table-container"></div>
  </div>

</div>

<!-- 🧠 STYLES & DEPENDENCIES -->
<link rel="stylesheet" href="https://cdn.datatables.net/1.13.6/css/jquery.dataTables.min.css">
<script src="https://code.jquery.com/jquery-3.7.0.min.js"></script>
<script src="https://cdn.datatables.net/1.13.6/js/jquery.dataTables.min.js"></script>

<style>
  .zp-Entry { display: none; }
  .zotpress-section { display: none; margin-top: 20px; }
  .zotpress-section.active { display: block; }
  .zotpress-table-container table { width: 100%; }
  
  /* Optional style enhancements */
  th:nth-child(3), td:nth-child(3) {
    width: 8%; /* Year column */
    text-align: center;
  }
  th:nth-child(2), td:nth-child(2) {
    width: 50%; /* Title column */
  }
  th:nth-child(4), td:nth-child(4) {
    width: 10%; /* Title column */
  }
  th:nth-child(1), td:nth-child(1) {
    width: 20%; /* Title column */
  }
</style>

<!-- ⚙️ JS LOGIC -->
<script>
jQuery(document).ready(function($) {

function parseEntry(text) {
  // Remove leading number and extract tags
  text = text.replace(/^\d+\.\s*/, '');
  const tagMatch = text.match(/Tags:\s*(.*)/i);
  const tags = tagMatch ? tagMatch[1].trim() : '';
  text = text.replace(/Tags:\s*.*/i, '').trim();

  // Extract author (up to first '(' for year)
  const authorMatch = text.match(/^(.+?)\s\(\d{4}\)/);
  const author = authorMatch ? authorMatch[1].trim() : '';

  // Extract year
  const yearMatch = text.match(/\((\d{4})\)/);
  const year = yearMatch ? yearMatch[1] : '';

  // Extract title in single quotes
  const titleMatch = text.match(/"([^']+)"/);
  const title = titleMatch ? titleMatch[1].slice(0, -1) : '';

  // Extract link after "Available at:"
  const linkMatch = text.match(/Available at:\s*(https?:\/\/\S+)/);
  const link = linkMatch ? `<a href="${linkMatch[1].slice(0, -1)}" target="_blank">Link</a>` : '';

  return { author, title, year, link, tags };
}


  function buildTable($section) {
    const $entries = $section.find('.zp-Entry');
    const $container = $section.find('.zotpress-table-container');
    const tableId = `table-${$section.data('collection')}`;
    const $table = $(`
      <div>
        <div class="collection" style="display: flex; gap: 2rem; margin-bottom: 1rem;">
        <label style="display:inline-block; margin-bottom:20px;">Filter by Category: <select class="tag-filter"><option value="">All Categories</option></select></label>
        </div>
        <table id="${tableId}" class="display">
          <thead><tr><th>Author</th><th>Title</th><th>Year</th><th>Link</th><th>Tags</th></tr></thead>
          <tbody></tbody>
        </table>
      </div>
    `);
    $container.append($table);

    const tagSet = new Set();
    const $tbody = $table.find('tbody');

    $entries.each(function() {
      const parsed = parseEntry($(this).text());
      parsed.tags.split(',').forEach(t => t.trim() && tagSet.add(t.trim()));
      $tbody.append(`
        <tr>
          <td>${parsed.author}</td>
          <td>${parsed.title}</td>
          <td>${parsed.year}</td>
          <td>${parsed.link}</td>
          <td>${parsed.tags}</td>
        </tr>
      `);
    });

    const dataTable = $(`#${tableId}`).DataTable({ pageLength: 10 });

    const $tagFilter = $table.find('.tag-filter');
    [...tagSet].sort().forEach(tag => {
      $tagFilter.append(`<option value="${tag}">${tag}</option>`);
    });

    $tagFilter.on('change', function() {
      const val = $(this).val();
      dataTable.column(4).search(val).draw();
    });
  }

  // After Zotpress renders, wait and then process all sections
  setTimeout(() => {
    $('.zotpress-section').each(function() {
      buildTable($(this));
    });
    // Show default
    $('[data-collection="all"]').addClass('active');
  }, 1500);

  // Collection switcher
  $('#collection-filter').on('change', function() {
    const val = $(this).val();
    $('.zotpress-section').removeClass('active');
    $(`[data-collection="${val}"]`).addClass('active');
  });

});
</script>
</div>
</div>
	</div>
</div>


</main>

<footer class="kind-footer">

			<div class="bottom-bar">
			<div class="kind-container">
				<div class="flex-row align-bottom">
					<div class="flex-cell flex-md-6">
						
			<a href="/" class="kind-footer-logo">
				<span class="kind-footer-logo__inner">
											<span class="kind-footer-logo__image" aria-hidden="true">
						<img width="630" height="630" src="https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_logo-1.png?fit=630%2C630&amp;ssl=1" class="attachment-full size-full" alt="Smart Microscopy" decoding="async" loading="lazy" srcset="https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_logo-1.png?w=630&amp;ssl=1 630w, https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_logo-1.png?resize=300%2C300&amp;ssl=1 300w, https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_logo-1.png?resize=150%2C150&amp;ssl=1 150w, https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_logo-1.png?resize=600%2C600&amp;ssl=1 600w, https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_logo-1.png?resize=400%2C400&amp;ssl=1 400w, https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_logo-1.png?resize=200%2C200&amp;ssl=1 200w" sizes="auto, (max-width: 630px) 100vw, 630px" data-attachment-id="193" data-permalink="/smartmicroscopy_logo-5/" data-orig-file="https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_logo-1.png?fit=630%2C630&amp;ssl=1" data-orig-size="630,630" data-comments-opened="1" data-image-meta="{&quot;aperture&quot;:&quot;0&quot;,&quot;credit&quot;:&quot;&quot;,&quot;camera&quot;:&quot;&quot;,&quot;caption&quot;:&quot;&quot;,&quot;created_timestamp&quot;:&quot;0&quot;,&quot;copyright&quot;:&quot;&quot;,&quot;focal_length&quot;:&quot;0&quot;,&quot;iso&quot;:&quot;0&quot;,&quot;shutter_speed&quot;:&quot;0&quot;,&quot;title&quot;:&quot;&quot;,&quot;orientation&quot;:&quot;0&quot;}" data-image-title="smartmicroscopy_logo" data-image-description="&lt;p&gt;©Alfredo Rates&lt;/p&gt;
" data-image-caption="" data-medium-file="https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_logo-1.png?fit=300%2C300&amp;ssl=1" data-large-file="https://i0.wp.com/smartmicroscopy.org/wp-content/uploads/2024/02/cropped-smartmicroscopy_logo-1.png?fit=630%2C630&amp;ssl=1" />						</span>
					
											<span class="kind-footer-logo__text">
															<span class="logo-name">Smart Microscopy</span>
																						<span class="logo-desc">All about imaging automation</span>
													</span>
					
				</span>
			</a>

								</div>

					<div class="flex-cell flex-md-6 links-right">
											</div>

				</div>
			</div>
		</div>
	
	<div class="kind-container">

					<div class="widget-area">

				
				<div class="widget-bottom widget-bottom-menu"><h2 class="widget-title">Language</h2><ul class="menu"><li id="menu-item-221-nl" class="lang-item lang-item-91016 lang-item-nl lang-item-first menu-item menu-item-type-custom menu-item-object-custom menu-item-221-nl"><a href="/nl/literatuur/" hreflang="nl-NL" lang="nl-NL">Nederlands</a></li>
<li id="menu-item-221-en" class="lang-item lang-item-91019 lang-item-en current-lang menu-item menu-item-type-custom menu-item-object-custom menu-item-221-en"><a href="/literature/" hreflang="en-US" lang="en-US">English</a></li>
</ul></div>
				
				
			</div>
		
					<div class="hr"></div>

			<div class="flex-row footer-credits align-center">

				<div class="flex-cell flex-sm-8 flex-md-6">
											<div class="copy">
							<p>
Maintained by <a href="mailto:a.ratessoriano@uu.nl" target="_blank">Alfredo Rates</a>. 
</p>
Hosted by the 
<b>
<a href="https://cellbiology.science.uu.nl" target="_blank">
Cell Biology department
</a>
</b>
 of 
<b>
<a href="https://uu.nl" target="_blank">
Utrecht University
</a>
</b>						</div>
									</div>

									<div class="flex-cell flex-sm-4 flex-md-6">
						<div class="kind-brand">
							<p>
<a href="https://smartmicroscopy.github.io" target="_blank">
Check out more at <b>&nbsp;smartmicroscopy.github.io</b>
</a>

smartmicroscopy.org is possible thanks to Euro-Bioimaging and 

<b>
<a href="http://eurobioimaging.nl" target="_blank">
NL-BioImaging
</a>
</b>
</p>						</div>
					</div>
				
			</div>
		
	</div>

</footer>


<!--  -->
<script type="speculationrules">
{"prefetch":[{"source":"document","where":{"and":[{"href_matches":"/*"},{"not":{"href_matches":["/wp-*.php","/wp-admin/*","/wp-content/uploads/*","/wp-content/*","/wp-content/plugins/*","/wp-content/themes/kind/*","/*\\?(.+)"]}},{"not":{"selector_matches":"a[rel~=\"nofollow\"]"}},{"not":{"selector_matches":".no-prefetch, .no-prefetch a"}}]},"eagerness":"conservative"}]}
</script>
<meta id="bilmur" property="bilmur:data" content="" data-customproperties="{&quot;woo_active&quot;:&quot;0&quot;,&quot;logged_in&quot;:&quot;0&quot;,&quot;wptheme&quot;:&quot;kind&quot;,&quot;wptheme_is_block&quot;:&quot;0&quot;}" data-provider="wordpress.com" data-service="atomic"  data-site-tz="Europe/Amsterdam" >
<script defer src="https://s0.wp.com/wp-content/js/bilmur.min.js?m=202551"></script>
<script type="text/template" id="static-snap-search-result-template">
  <article id="post-{{id}}" class="post-{{id}} page type-page status-publish hentry entry">
    <header class="entry-header">
      <h2 class="entry-title default-max-width">
        <a href="{{url}}">{{title}}</a>
      </h2>
    </header>


    <div class="entry-content">
      <p>{{excerpt}}</p>
    </div>


    <footer class="entry-footer default-max-width"></footer>

  </article>
</script>
<script type="importmap" id="wp-importmap">
{"imports":{"@wordpress/interactivity":"/wp-content/plugins/gutenberg/build/modules/interactivity/index.min.js?ver=1764773745501"}}
</script>
<script type="module" src="/wp-content/plugins/jetpack/jetpack_vendor/automattic/jetpack-forms/dist/modules/form/view.js?ver=15.4-a.3" id="jp-forms-view-js-module"></script>
<link rel="modulepreload" href="/wp-content/plugins/gutenberg/build/modules/interactivity/index.min.js?ver=1764773745501" id="@wordpress/interactivity-js-modulepreload" data-wp-fetchpriority="low">
<script type="application/json" id="wp-script-module-data-@wordpress/interactivity">
{"config":{"jetpack/form":{"error_types":{"is_required":"This field is required.","invalid_form_empty":"The form you are trying to submit is empty.","invalid_form":"Please fill out the form correctly.","network_error":"Connection issue while submitting the form. Check that you are connected to the Internet and try again."},"admin_ajax_url":"/wp-admin/admin-ajax.php"}}}
</script>
<script id="zotpress.dl.min.js-js-extra">
var zpDLAJAX = {"ajaxurl":"/wp-admin/admin-ajax.php","zpDL_nonce":"87a1d1a320","action":"zpDLViaAJAX"};
//# sourceURL=zotpress.dl.min.js-js-extra
</script>
<script id="zotpress.cite.min.js-js-extra">
var zpCiteAJAX = {"ajaxurl":"/wp-admin/admin-ajax.php","zpCite_nonce":"1f06ff7756","action":"zpCiteViaAJAX"};
//# sourceURL=zotpress.cite.min.js-js-extra
</script>
<script type='text/javascript'  src='/_static/??-eJwrL9BNzs8rSc0r0S/IKU3PzCvWr8ovKShKLS7Wz0Kw9VJy9HIz8/SyinXKidSRnFmSCtVjn2traG5mYGFmZmJmAgD+xipt'></script>
<script id="pll_cookie_script-js-after">
(function() {
				var expirationDate = new Date();
				expirationDate.setTime( expirationDate.getTime() + 31536000 * 1000 );
				document.cookie = "pll_language=en; expires=" + expirationDate.toUTCString() + "; path=/; secure; SameSite=Lax";
			}());

//# sourceURL=pll_cookie_script-js-after
</script>
<script src="/wp-content/plugins/gutenberg/build/scripts/hooks/index.min.js?ver=1764773745495" id="wp-hooks-js"></script>
<script src="/wp-content/plugins/gutenberg/build/scripts/i18n/index.min.js?ver=1764773747362" id="wp-i18n-js"></script>
<script id="wp-i18n-js-after">
wp.i18n.setLocaleData( { 'text direction\u0004ltr': [ 'ltr' ] } );
//# sourceURL=wp-i18n-js-after
</script>
<script id="static-snap-frontend-js-extra">
var StaticSnapFrontendConfig = {"has_translations":"false","is_static":"true","options":[],"locale":"en_US","is_admin_bar_showing":"false","captcha_type":null,"captcha_site_key":null};
//# sourceURL=static-snap-frontend-js-extra
</script>
<script src="/wp-content/plugins/static-snap/assets/js/frontend.js?ver=1c7741501d422b5e816b" id="static-snap-frontend-js"></script>
<script id="kind-js-extra">
var kind = {"i18n":{"a11y":{"expand":"Expand child menu","collapse":"Collapse child menu","offCanvasIsOpen":"Off-Canvas is open","offCanvasIsClosed":"Off-Canvas is closed"}}};
//# sourceURL=kind-js-extra
</script>
<script type='text/javascript'  src='/wp-content/themes/kind/assets/js/scripts.js?m=1706878058'></script>
<script id="jetpack-stats-js-before">
_stq = window._stq || [];
_stq.push([ "view", {"v":"ext","blog":"228298350","post":"27","tz":"1","srv":"smartmicroscopy.org","hp":"atomic","ac":"2","amp":"0","j":"1:15.4-a.3"} ]);
_stq.push([ "clickTrackerInit", "228298350", "27" ]);
//# sourceURL=jetpack-stats-js-before
</script>
<script src="https://stats.wp.com/e-202551.js" id="jetpack-stats-js" defer data-wp-strategy="defer"></script>
<script type='text/javascript'  src='/wp-includes/js/dist/vendor/wp-polyfill.min.js?m=1761030842'></script>
<script src="/wp-content/plugins/jetpack/jetpack_vendor/automattic/jetpack-forms/dist/blocks/view.js?minify=false&amp;ver=97115d02b739e14da240" id="jp-forms-blocks-js" defer data-wp-strategy="defer"></script>
<script id="zotpress.shortcode.bib.min.js-js-extra">
var zpShortcodeAJAX = {"ajaxurl":"/wp-admin/admin-ajax.php","zpShortcode_nonce":"bb3091abd6","action":"zpRetrieveViaShortcode","txt_removeimg":"Remove Image","txt_zperror":"There was a Zotpress error:","txt_noitemsfound":"No items found."};
//# sourceURL=zotpress.shortcode.bib.min.js-js-extra
</script>
<script type='text/javascript'  src='/_static/??-eJyNj8EKwjAQRH/IZhVK9CJ+i002zYY0iclui369gmg9SQ8D7zAPZpbSmZwYE0OJMlJq8MhcKrYGYWXVfK5sskU10KAmSiq03fJfDjfBeleRZnzTqlEyUSz+1EAI0Dk0vLHWeRp9fIU3zvl+seiuEj/aZTofjnp/0rrX/RPNrWap'></script>
<script id="wp-emoji-settings" type="application/json">
{"baseUrl":"https://s.w.org/images/core/emoji/17.0.2/72x72/","ext":".png","svgUrl":"https://s.w.org/images/core/emoji/17.0.2/svg/","svgExt":".svg","source":{"concatemoji":"/wp-includes/js/wp-emoji-release.min.js?ver=6.9"}}
</script>
<script type="module">
/*! This file is auto-generated */
const a=JSON.parse(document.getElementById("wp-emoji-settings").textContent),o=(window._wpemojiSettings=a,"wpEmojiSettingsSupports"),s=["flag","emoji"];function i(e){try{var t={supportTests:e,timestamp:(new Date).valueOf()};sessionStorage.setItem(o,JSON.stringify(t))}catch(e){}}function c(e,t,n){e.clearRect(0,0,e.canvas.width,e.canvas.height),e.fillText(t,0,0);t=new Uint32Array(e.getImageData(0,0,e.canvas.width,e.canvas.height).data);e.clearRect(0,0,e.canvas.width,e.canvas.height),e.fillText(n,0,0);const a=new Uint32Array(e.getImageData(0,0,e.canvas.width,e.canvas.height).data);return t.every((e,t)=>e===a[t])}function p(e,t){e.clearRect(0,0,e.canvas.width,e.canvas.height),e.fillText(t,0,0);var n=e.getImageData(16,16,1,1);for(let e=0;e<n.data.length;e++)if(0!==n.data[e])return!1;return!0}function u(e,t,n,a){switch(t){case"flag":return n(e,"\ud83c\udff3\ufe0f\u200d\u26a7\ufe0f","\ud83c\udff3\ufe0f\u200b\u26a7\ufe0f")?!1:!n(e,"\ud83c\udde8\ud83c\uddf6","\ud83c\udde8\u200b\ud83c\uddf6")&&!n(e,"\ud83c\udff4\udb40\udc67\udb40\udc62\udb40\udc65\udb40\udc6e\udb40\udc67\udb40\udc7f","\ud83c\udff4\u200b\udb40\udc67\u200b\udb40\udc62\u200b\udb40\udc65\u200b\udb40\udc6e\u200b\udb40\udc67\u200b\udb40\udc7f");case"emoji":return!a(e,"\ud83e\u1fac8")}return!1}function f(e,t,n,a){let r;const o=(r="undefined"!=typeof WorkerGlobalScope&&self instanceof WorkerGlobalScope?new OffscreenCanvas(300,150):document.createElement("canvas")).getContext("2d",{willReadFrequently:!0}),s=(o.textBaseline="top",o.font="600 32px Arial",{});return e.forEach(e=>{s[e]=t(o,e,n,a)}),s}function r(e){var t=document.createElement("script");t.src=e,t.defer=!0,document.head.appendChild(t)}a.supports={everything:!0,everythingExceptFlag:!0},new Promise(t=>{let n=function(){try{var e=JSON.parse(sessionStorage.getItem(o));if("object"==typeof e&&"number"==typeof e.timestamp&&(new Date).valueOf()<e.timestamp+604800&&"object"==typeof e.supportTests)return e.supportTests}catch(e){}return null}();if(!n){if("undefined"!=typeof Worker&&"undefined"!=typeof OffscreenCanvas&&"undefined"!=typeof URL&&URL.createObjectURL&&"undefined"!=typeof Blob)try{var e="postMessage("+f.toString()+"("+[JSON.stringify(s),u.toString(),c.toString(),p.toString()].join(",")+"));",a=new Blob([e],{type:"text/javascript"});const r=new Worker(URL.createObjectURL(a),{name:"wpTestEmojiSupports"});return void(r.onmessage=e=>{i(n=e.data),r.terminate(),t(n)})}catch(e){}i(n=f(s,u,c,p))}t(n)}).then(e=>{for(const n in e)a.supports[n]=e[n],a.supports.everything=a.supports.everything&&a.supports[n],"flag"!==n&&(a.supports.everythingExceptFlag=a.supports.everythingExceptFlag&&a.supports[n]);var t;a.supports.everythingExceptFlag=a.supports.everythingExceptFlag&&!a.supports.flag,a.supports.everything||((t=a.source||{}).concatemoji?r(t.concatemoji):t.wpemoji&&t.twemoji&&(r(t.twemoji),r(t.wpemoji)))});
//# sourceURL=/wp-includes/js/wp-emoji-loader.min.js
</script>

</body>
</html>
